<div class="list-entries"><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15869dfb941:14c3484:1e99f69f_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4368/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15869dfb941:14c3484:1e99f69f" data-navigation="inline" data-title="Nipype Mapnode to write to the same destination file after recon-all pipeline" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15869dfb941:14c3484:1e99f69f" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 64 --></div><!-- react-empty: 65 --><div class="content"><a href="http://neurostars.org/p/4368/" target="_blank" rel="noopener" class="title">Nipype Mapnode to write to the same destination file after recon-all pipeline</a><div class="summary">Hello, Nipype experts: I have a small question as the title, after run recon-all pipeline for 30 subjects, I want to read the last 1-4 lines from the recon-all.log file of every subject to one same destination, I know that I can use Node to write it in serialized, but for some reason, I want to use Mapnode, so this will be written in parallel, here is the code that I wrote: def recon_all_statistic</div></div><span class="ago" title="published:Tue, 15 Nov 2016 19:23:26 GMT -- Received: Tue, 15 Nov 2016 21:23:39 GMT">17d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1585da33ec1:1ef98fa:36ad7c89_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4361/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1585da33ec1:1ef98fa:36ad7c89" data-navigation="inline" data-title="Forum: Nipype workflow tutorial dmri_camino_dti.py" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1585da33ec1:1ef98fa:36ad7c89" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 80 --></div><!-- react-empty: 81 --><div class="content"><a href="http://neurostars.org/p/4361/" target="_blank" rel="noopener" class="title">Forum: Nipype workflow tutorial dmri_camino_dti.py</a><div class="summary">Hello: Recently, I am new to learn diffusion MRI, so I followed the tutorial from here: http://nipype.readthedocs.io/en/stable/users/examples/dmri_camino_dti.html Basically, my mac has pre-installed most of the software that i need( Freesurfer, afni, spm, matlab...) After I run it, I got some errors and stderr: I dont know if it is because of some other softwares for visualisation, i installed Tra</div></div><span class="ago" title="published:Sun, 13 Nov 2016 11:27:49 GMT -- Received: Sun, 13 Nov 2016 12:22:09 GMT">19d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_158549e83eb:117a90e:a36d59fd_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4357/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_158549e83eb:117a90e:a36d59fd" data-navigation="inline" data-title="News: Announcement: MRIQC 0.8.8 - release candidate 9 just published" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_158549e83eb:117a90e:a36d59fd" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 96 --></div><!-- react-empty: 97 --><div class="content"><a href="http://neurostars.org/p/4357/" target="_blank" rel="noopener" class="title">News: Announcement: MRIQC 0.8.8 - release candidate 9 just published</a><div class="summary"> A new version of MRIQC has just been released in PyPi , along with corresponding docker images . We will try to update the documentation ASAP. For reporting bugs, please use the github repository: https://github.com/poldracklab/mriqc/issues . For general questions, feel free to use neurostars, github, mailing lists, etc. Thanks very much, looking forward to hearing some feedback :) The MRIQC deve</div></div><span class="ago" title="published:Fri, 11 Nov 2016 17:52:50 GMT -- Received: Fri, 11 Nov 2016 18:20:24 GMT">21d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15853f9ae39:102b81f:a36d59fd_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4356/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15853f9ae39:102b81f:a36d59fd" data-navigation="inline" data-title="FSL design matrix outliers not identically 0 and 1?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15853f9ae39:102b81f:a36d59fd" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 112 --></div><!-- react-empty: 113 --><div class="content"><a href="http://neurostars.org/p/4356/" target="_blank" rel="noopener" class="title">FSL design matrix outliers not identically 0 and 1?</a><div class="summary"> I'm using this L1 processing script (variant of this: https://github.com/nipy/nipype/blob/master/examples/fmri_ants_openfmri.py), and I noticed that the outliers that appear in the FSL design matrix, and specifically are dumped to /model/task/subj/qa/model/run01_run0.mat and run01_run0.png are not columns of 0's with a single 1. Instead, all the elements except the outlier volume are "almost" zer</div></div><span class="ago" title="published:Fri, 11 Nov 2016 14:20:00 GMT -- Received: Fri, 11 Nov 2016 15:20:21 GMT">21d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15853f9ae39:102b81e:a36d59fd_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4355/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15853f9ae39:102b81e:a36d59fd" data-navigation="inline" data-title="Dynamic Causal Modeling (DCM) with outliers?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15853f9ae39:102b81e:a36d59fd" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 128 --></div><!-- react-empty: 129 --><div class="content"><a href="http://neurostars.org/p/4355/" target="_blank" rel="noopener" class="title">Dynamic Causal Modeling (DCM) with outliers?</a><div class="summary">Reposting from spm email list: I have preprocessed and extracted timeseries ROIs of my data with a non spm pipeline, and am unsure how to interface this data with spm's dcm with respect to outliers. (I took intermediate outputs from here: https://github.com/gablab/openfmri/blob/master/subject_level/fmri_ants_bids.py) I have identified all the needed criteria for specifying the DCM variables initia</div></div><span class="ago" title="published:Fri, 11 Nov 2016 14:09:48 GMT -- Received: Fri, 11 Nov 2016 15:20:21 GMT">21d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1584aebe7c6:b521a0e:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4346/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1584aebe7c6:b521a0e:18bc4566" data-navigation="inline" data-title="Forum: Nipype SPM realign error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1584aebe7c6:b521a0e:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 144 --></div><!-- react-empty: 145 --><div class="content"><a href="http://neurostars.org/p/4346/" target="_blank" rel="noopener" class="title">Forum: Nipype SPM realign error</a><div class="summary">Hello: I just use SPM to realign one fmri images, here is the code: # -*- coding: utf-8 -*- import nipype.interfaces.spm as spm # import spm import nipype.pipeline.engine as pe # import pypeline engine # What is the location of your experiment folder experiment_dir = '/Users/junhao.wen/Hao/tutorial/Nipype/Michael_Notter/nipype_tutorial' # # Create a realign node - Method 1: Specify inputs during n</div></div><span class="ago" title="published:Wed, 09 Nov 2016 21:00:53 GMT -- Received: Wed, 09 Nov 2016 21:08:44 GMT">23d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1584a471505:b3cabb0:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4345/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1584a471505:b3cabb0:18bc4566" data-navigation="inline" data-title="spm.SpecifySPMModel(): Are behavioral regressors tied to onsets?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1584a471505:b3cabb0:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 160 --></div><!-- react-empty: 161 --><div class="content"><a href="http://neurostars.org/p/4345/" target="_blank" rel="noopener" class="title">spm.SpecifySPMModel(): Are behavioral regressors tied to onsets?</a><div class="summary"> Hi all, I want to add regressors (e.g. right/wrong button presses) in an SPM design using spm.SpecifySPMModel(). In the bunched list where regressor_names and regressors are specified, to these have to be in the same format as the onsets and durations, i.e. do they have to be lists of lists (5 lists inside one list for 5 different conditions). I have one regressor value for each volume in the tim</div></div><span class="ago" title="published:Wed, 09 Nov 2016 15:29:21 GMT -- Received: Wed, 09 Nov 2016 18:08:42 GMT">23d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583ff84e5c:9edd929:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4337/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583ff84e5c:9edd929:18bc4566" data-navigation="inline" data-title="Forum: base_dir of Workflow in nipype does not work properly" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583ff84e5c:9edd929:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 176 --></div><!-- react-empty: 177 --><div class="content"><a href="http://neurostars.org/p/4337/" target="_blank" rel="noopener" class="title">Forum: base_dir of Workflow in nipype does not work properly</a><div class="summary">Hello, nipype experts: I have a question when i use Workflow's attribute base_dir, here is the code: if working_directory is None: working_directory = tempfile.mkdtemp() else : working_directory = working_directory # this is added later after None doesnt work surfstat_wf = pe.Workflow(name='surfstat_workflow', base_dir=working_directory) Basically, I know that for pe.Workflow , if we dont define b</div></div><span class="ago" title="published:Mon, 07 Nov 2016 17:01:19 GMT -- Received: Mon, 07 Nov 2016 18:06:27 GMT">25d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583ff84e5c:9edd928:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4336/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583ff84e5c:9edd928:18bc4566" data-navigation="inline" data-title="Forum: base_dir of Workflow in nipype does not work properly" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583ff84e5c:9edd928:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 192 --></div><!-- react-empty: 193 --><div class="content"><a href="http://neurostars.org/p/4336/" target="_blank" rel="noopener" class="title">Forum: base_dir of Workflow in nipype does not work properly</a><div class="summary">Hello, nipype experts: I have a question when i use Workflow's attribute base_dir, here is the code: if working_directory is None: working_directory = tempfile.mkdtemp() else : working_directory = working_directory # this is added later after None doesnt work surfstat_wf = pe.Workflow(name='surfstat_workflow', base_dir=working_directory) Basically, I know that for pe.Workflow , if we dont define b</div></div><span class="ago" title="published:Mon, 07 Nov 2016 17:01:17 GMT -- Received: Mon, 07 Nov 2016 18:06:27 GMT">25d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583127f7ec:86abf4b:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4321/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583127f7ec:86abf4b:18bc4566" data-navigation="inline" data-title="mriqc - download failed while running" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1583127f7ec:86abf4b:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 208 --></div><!-- react-empty: 209 --><div class="content"><a href="http://neurostars.org/p/4321/" target="_blank" rel="noopener" class="title">mriqc - download failed while running</a><div class="summary">Hi. I am attempting to run mriqc (v0.8.6a7) locally on Mac OSX 10.11.5. I receive an error when trying to download a MRI tar file. Any suggestions? This occurs on two different laptops and different wireless connections. Thanks. $ mriqc /Users/mhutchi1/Desktop/QP_test/ds001_cut /Users/mhutchi1/Desktop/QP_test/ds001_cut/OUT participant 161104-14:25:06,492 workflow INFO: Running MRIQC-0.8.6a7 (analy</div></div><span class="ago" title="published:Fri, 04 Nov 2016 18:32:12 GMT -- Received: Fri, 04 Nov 2016 21:04:12 GMT">28d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582fde3d59:8637252:30cfcb50_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4319/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582fde3d59:8637252:30cfcb50" data-navigation="inline" data-title="how to get individual gray matter mask by using freesurfer output" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582fde3d59:8637252:30cfcb50" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 224 --></div><!-- react-empty: 225 --><div class="content"><a href="http://neurostars.org/p/4319/" target="_blank" rel="noopener" class="title">how to get individual gray matter mask by using freesurfer output</a><div class="summary">Hi, I'm trying to get individual gray matter (GM) masks and to normalize these masks into MNI (3mm) template. I would like to describe my current approach, but would appreciate any comments or suggestions on it: 1) get GM mask with fs.Binarize() (inputs: aseg.mgz and GM labels) https://github.com/sheyma/stroke_preprop/blob/master/09_greyMatter.py#L26-L46 2) normalize GM mask to MNI1mm template wit</div></div><span class="ago" title="published:Fri, 04 Nov 2016 14:39:29 GMT -- Received: Fri, 04 Nov 2016 15:04:03 GMT">28d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582ab73f05:7ba712f:30cfcb50_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4314/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582ab73f05:7ba712f:30cfcb50" data-navigation="inline" data-title="Forum: Set different flag for different subjects for ReconAll interface in Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582ab73f05:7ba712f:30cfcb50" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 240 --></div><!-- react-empty: 241 --><div class="content"><a href="http://neurostars.org/p/4314/" target="_blank" rel="noopener" class="title">Forum: Set different flag for different subjects for ReconAll interface in Nipype</a><div class="summary">Hello, Nipype experts: I have a question for ReconAll interface( http://nipy.org/nipype/interfaces/generated/nipype.interfaces.freesurfer.preprocess.html#reconall ), here is the situation: Here is the code: def checkoutFOV (t1_list): import nibabel as nib output_flag = [] for i in t1_list: f = nib.load(i) voxel_size = f.header.get_zooms() data_shape = f.header.get_data_shape() if voxel_size[0] * d</div></div><span class="ago" title="published:Thu, 03 Nov 2016 15:01:43 GMT -- Received: Thu, 03 Nov 2016 15:03:22 GMT">29d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582ab73f05:7ba712e:30cfcb50_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4312/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582ab73f05:7ba712e:30cfcb50" data-navigation="inline" data-title="FSL (preprocessing BedPostX) Error: Error in logfile. What does this mean?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582ab73f05:7ba712e:30cfcb50" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 256 --></div><!-- react-empty: 257 --><div class="content"><a href="http://neurostars.org/p/4312/" target="_blank" rel="noopener" class="title">FSL (preprocessing BedPostX) Error: Error in logfile. What does this mean?</a><div class="summary">When I do BedpostX FSL preprocessing I get this message: ----- Bedpostx Monitor ----- 1 slices processed out of 62 2 slices processed out of 62 . . . 59 slices processed out of 62 60 slices processed out of 62 An error ocurred. Please check file /Users/...../bedpostx.bedpostX/logs/bpx_postproc.e68944 /Users/..../bedpostx.bedpostX/monitor: line 15: kill: (9187) - No such process logout [Process com</div></div><span class="ago" title="published:Thu, 03 Nov 2016 12:40:34 GMT -- Received: Thu, 03 Nov 2016 15:03:22 GMT">29d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582a1251b8:63d450c:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4311/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582a1251b8:63d450c:297c8405" data-navigation="inline" data-title="Error when I try to use fsl.dti via nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582a1251b8:63d450c:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 272 --></div><!-- react-empty: 273 --><div class="content"><a href="http://neurostars.org/p/4311/" target="_blank" rel="noopener" class="title">Error when I try to use fsl.dti via nipype</a><div class="summary">Dear all, I m new to python and nipype thus I really dont know what I should do with this error message! I want to do the fsl.dit command with nipype. I did it like it is explained on this site: http://nipy.org/nipype/interfaces/generated/nipype.interfaces.fsl.dti.html#dtifit so this is my code import numpy as nump import nipype as nip from nipype.interfaces import fsl dti = fsl.DTIFit() dti.input</div></div><span class="ago" title="published:Thu, 03 Nov 2016 11:57:00 GMT -- Received: Thu, 03 Nov 2016 12:03:13 GMT">29d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582634d3e3:7289c52:30cfcb50_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4310/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582634d3e3:7289c52:30cfcb50" data-navigation="inline" data-title="Displaying amplitude integrated electroencephalography" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1582634d3e3:7289c52:30cfcb50" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 288 --></div><!-- react-empty: 289 --><div class="content"><a href="http://neurostars.org/p/4310/" target="_blank" rel="noopener" class="title">Displaying amplitude integrated electroencephalography</a><div class="summary"> Hi, amplitdude integrated electroencephalography is an quite common dignostic method in neonatology. Does anyone have some experience using open source tools to viev aEEG Data (sigviewer, edfdisplay ...)? Using a Brainz BRM 3 monitor we just ran into the trap of proprietary file formats reslting in many .BRM-Files and a very little supported software even for windows systems. Before purchasing a </div></div><span class="ago" title="published:Wed, 02 Nov 2016 16:28:10 GMT -- Received: Wed, 02 Nov 2016 18:02:26 GMT">30d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15824eb131b:5af13ee:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4309/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15824eb131b:5af13ee:297c8405" data-navigation="inline" data-title="Wrapper for GLMDenoise?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15824eb131b:5af13ee:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 304 --></div><!-- react-empty: 305 --><div class="content"><a href="http://neurostars.org/p/4309/" target="_blank" rel="noopener" class="title">Wrapper for GLMDenoise?</a><div class="summary"> Hi all, Did somebody ever write a wrapper for GLMDenoise or started with it? http://kendrickkay.net/GLMdenoise/ Regards, Glad </div></div><span class="ago" title="published:Wed, 02 Nov 2016 10:32:49 GMT -- Received: Wed, 02 Nov 2016 12:02:15 GMT">30d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1581a9c75a4:5a6234b:30cfcb50_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4303/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1581a9c75a4:5a6234b:30cfcb50" data-navigation="inline" data-title="AttributeError on ignore_exception" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1581a9c75a4:5a6234b:30cfcb50" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 320 --></div><!-- react-empty: 321 --><div class="content"><a href="http://neurostars.org/p/4303/" target="_blank" rel="noopener" class="title">AttributeError on ignore_exception</a><div class="summary">Hi all, I am trying to work my way through a rather confusing bug. Firstly, I have defined my own class for performing n-ary math (like sum/product/etc.). It has an input spec that looks like this: class NaryOperationInputSpec(DynamicTraitedSpec, BaseInterfaceInputSpec): input_list = traits.List(File(exists=True)) out_file = File(desc='Output file') def create_n_inputs(self, n): add_traits(self, [</div></div><span class="ago" title="published:Mon, 31 Oct 2016 11:37:37 GMT -- Received: Mon, 31 Oct 2016 12:00:11 GMT">32d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1581a9c75a4:5a6234a:30cfcb50_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4302/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1581a9c75a4:5a6234a:30cfcb50" data-navigation="inline" data-title="AttributeError on ignore_exception" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1581a9c75a4:5a6234a:30cfcb50" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 336 --></div><!-- react-empty: 337 --><div class="content"><a href="http://neurostars.org/p/4302/" target="_blank" rel="noopener" class="title">AttributeError on ignore_exception</a><div class="summary">Hi all, I am trying to work my way through a rather confusing bug. Firstly, I have defined my own class for performing n-ary math (like sum/product/etc.). It has an input spec that looks like this: class NaryOperationInputSpec(DynamicTraitedSpec, BaseInterfaceInputSpec): input_list = traits.List(File(exists=True)) out_file = File(desc='Output file') def create_n_inputs(self, n): add_traits(self, [</div></div><span class="ago" title="published:Mon, 31 Oct 2016 11:37:16 GMT -- Received: Mon, 31 Oct 2016 12:00:11 GMT">32d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1580bb44e86:4168d3b:30cfcb50_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4294/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1580bb44e86:4168d3b:30cfcb50" data-navigation="inline" data-title="Tool: MRIQC --GRUP LEVEL ANALYSIS NOT GENERATED" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1580bb44e86:4168d3b:30cfcb50" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 352 --></div><!-- react-empty: 353 --><div class="content"><a href="http://neurostars.org/p/4294/" target="_blank" rel="noopener" class="title">Tool: MRIQC --GRUP LEVEL ANALYSIS NOT GENERATED</a><div class="summary"> Dear Oscar, I have been inspecting they outputs generated from MRIQC.&#8203; The command line I used was: docker run -v /home/rdacosta/docker_481_sujetos/bids/:/data -v /home/rdacosta/docker_481_sujetos/scratch:/scratch -w /scratch poldracklab /mriqc:latest /data /scratch/out participant -w /scratch/work I&#8203;t &#8203;seems that &#8203;with this command MRIQC only generated&#8203; individual runs but not group level analys</div></div><span class="ago" title="published:Fri, 28 Oct 2016 13:00:37 GMT -- Received: Fri, 28 Oct 2016 14:31:55 GMT">35d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fc3eae65:20d3837:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4288/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fc3eae65:20d3837:18bc4566" data-navigation="inline" data-title="spm.EstimateContrast(): Skip estimation of contrasts once already estimated?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fc3eae65:20d3837:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 368 --></div><!-- react-empty: 369 --><div class="content"><a href="http://neurostars.org/p/4288/" target="_blank" rel="noopener" class="title">spm.EstimateContrast(): Skip estimation of contrasts once already estimated?</a><div class="summary"> Hi, I noticed that EstimateContrast() runs every time I run the pipeline. Is there a way to not have it run every time? Or am I doing something wrong? Regards, Glad </div></div><span class="ago" title="published:Tue, 25 Oct 2016 12:56:26 GMT -- Received: Tue, 25 Oct 2016 14:29:08 GMT">38d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fb99b50d:198a3ea:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4287/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fb99b50d:198a3ea:297c8405" data-navigation="inline" data-title="dcm2nii, dcm2niix and MRIconvert and Sagittal Siemens data" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fb99b50d:198a3ea:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 384 --></div><!-- react-empty: 385 --><div class="content"><a href="http://neurostars.org/p/4287/" target="_blank" rel="noopener" class="title">dcm2nii, dcm2niix and MRIconvert and Sagittal Siemens data</a><div class="summary"> Hello everyone, Using dicom2nii, dcm2niix or MRIConvert to convert DICOM of Sagittal Siemens data to NIFTI, I get an error about DTI vectors: "please validate DTI vectors (matrix had a negative determinant, perhaps Siemens sagittal)". The .bvec file is not correct which makes wrong orientations of the computed DTI. Thanks for your help ! </div></div><span class="ago" title="published:Tue, 25 Oct 2016 10:53:12 GMT -- Received: Tue, 25 Oct 2016 11:28:57 GMT">38d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fb99b50d:198a3e9:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4286/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fb99b50d:198a3e9:297c8405" data-navigation="inline" data-title="Tool: Recommended metrics and methodology description with MRIQC" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157fb99b50d:198a3e9:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 400 --></div><!-- react-empty: 401 --><div class="content"><a href="http://neurostars.org/p/4286/" target="_blank" rel="noopener" class="title">Tool: Recommended metrics and methodology description with MRIQC</a><div class="summary"> Dear Oscar, I have two questions I would be more than grateful is you could answer me. The first one is about metrics used in MRIQC. I have the recommended papers about the metrics you use for assessing QC of the different images. If I interpreted you welll, the highest the number, the best. However, where are the range of values recommended?, Which is the minimum value for every metric we get th</div></div><span class="ago" title="published:Tue, 25 Oct 2016 09:29:31 GMT -- Received: Tue, 25 Oct 2016 11:28:57 GMT">38d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157f6726ddf:14c5b53:30cfcb50_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4284/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157f6726ddf:14c5b53:30cfcb50" data-navigation="inline" data-title="spm.SpecifySPMModel(): Do onsets have to be integers?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157f6726ddf:14c5b53:30cfcb50" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 416 --></div><!-- react-empty: 417 --><div class="content"><a href="http://neurostars.org/p/4284/" target="_blank" rel="noopener" class="title">spm.SpecifySPMModel(): Do onsets have to be integers?</a><div class="summary"> Hi, I am getting the following error: timelist = [np.max([0., _scalefactor * t]) for t in timelist] TypeError: can't multiply sequence by non-int of type 'float' Interface SpecifySPMModel failed to run. Does that mean that onsets have to be integers even though they are specified in seconds? If that is the case, then I suppose rounding them before bunching would be a solution. Regards, Glad </div></div><span class="ago" title="published:Mon, 24 Oct 2016 10:20:09 GMT -- Received: Mon, 24 Oct 2016 11:27:56 GMT">39d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157f3deeb79:100b036:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4283/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157f3deeb79:100b036:18bc4566" data-navigation="inline" data-title="&quot;AttributeError: 'Bunch' object has no attribute 'insert'&quot; when debugging pipeline" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157f3deeb79:100b036:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 432 --></div><!-- react-empty: 433 --><div class="content"><a href="http://neurostars.org/p/4283/" target="_blank" rel="noopener" class="title">"AttributeError: 'Bunch' object has no attribute 'insert'" when debugging pipeline</a><div class="summary">Hi all, I get the following error when I try to debug a pipeline. I thought it was because of a funky installation, but I tried installing nipype 0.12.1 from pip in virtualenvs both with --system-site-packages and without. Funny thing is with nipype 0.11 I do not get this problem, but I do get failures when I enable the debug mode (if I disable debug mode in nipype 0.11, it does pass that point bu</div></div><span class="ago" title="published:Sun, 23 Oct 2016 21:18:17 GMT -- Received: Sun, 23 Oct 2016 23:27:35 GMT">40d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157e7a21d80:619d4cd:844af92a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4281/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157e7a21d80:619d4cd:844af92a" data-navigation="inline" data-title="Visualize individual anatomical atlas region in nilearn" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157e7a21d80:619d4cd:844af92a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 448 --></div><!-- react-empty: 449 --><div class="content"><a href="http://neurostars.org/p/4281/" target="_blank" rel="noopener" class="title">Visualize individual anatomical atlas region in nilearn</a><div class="summary"> Hello, I am trying to plot AAL or Harvard Oxford (any 3d atlas) in nilearn. Using the standard example plotting.plot_roi (atlas_filename, title="Harvard Oxford atlas") plots all regions at once. However, there is also a labels-list provided and I'd like to display only one region - preferrably based on the label (e.g. labels['Frontal Pole']). I explored the region_extractor and the threshold_img,</div></div><span class="ago" title="published:Fri, 21 Oct 2016 13:29:59 GMT -- Received: Fri, 21 Oct 2016 14:25:43 GMT">42d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157e31ee5ef:485fd20:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4278/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157e31ee5ef:485fd20:297c8405" data-navigation="inline" data-title="ignore_exception doesn't prevent node from crashing if previous node failed for some input files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157e31ee5ef:485fd20:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 464 --></div><!-- react-empty: 465 --><div class="content"><a href="http://neurostars.org/p/4278/" target="_blank" rel="noopener" class="title">ignore_exception doesn't prevent node from crashing if previous node failed for some input files</a><div class="summary">I have a nipype workflow that processes MRI images. Part of this workflow consists of a MapNode that runs FSL BET and passes the output to another MapNode that runs FLIRT. Although I've set the ignore_exception trait for both nodes to True, the FLIRT node fails with the following error and causes the entire workflow to crash: Traceback (most recent call last): File "/home/lebedov/Work/projects/nip</div></div><span class="ago" title="published:Thu, 20 Oct 2016 17:18:30 GMT -- Received: Thu, 20 Oct 2016 17:23:55 GMT">43d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157dd52e38f:49c5e4b:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4276/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157dd52e38f:49c5e4b:18bc4566" data-navigation="inline" data-title="spm.SpecifySPMModle() pass .mat file with all conditions set?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157dd52e38f:49c5e4b:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 480 --></div><!-- react-empty: 481 --><div class="content"><a href="http://neurostars.org/p/4276/" target="_blank" rel="noopener" class="title">spm.SpecifySPMModle() pass .mat file with all conditions set?</a><div class="summary"> Dear Nipype Experts, I've noticed there is no way to pass a pre formated .mat file to be passed on to the models specification function of SPM. This would correspond to the multiple conditions entry in the SPM job. Was this option left out due to convenience? Or is the text file entry better? I suppose neither are BIDS compatible as the TSV files also include trial type names. Just wondering... R</div></div><span class="ago" title="published:Wed, 19 Oct 2016 13:38:14 GMT -- Received: Wed, 19 Oct 2016 14:22:59 GMT">44d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157dcae084b:4848279:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4275/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157dcae084b:4848279:18bc4566" data-navigation="inline" data-title="ICA loadings interpretation and processing" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157dcae084b:4848279:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 496 --></div><!-- react-empty: 497 --><div class="content"><a href="http://neurostars.org/p/4275/" target="_blank" rel="noopener" class="title">ICA loadings interpretation and processing</a><div class="summary"> Hi, I am interpreting the loading matrix output from a Group ICA experiment as the "level of expression" of one subject (each volume) to the corresponding independent component. In fMRI I would have one loading coefficient for each volume in each fMRI acquisition. I am looking for a method to sensibly calculate only one scalar loading coefficient for one subject. Do you think the average loading </div></div><span class="ago" title="published:Wed, 19 Oct 2016 09:23:15 GMT -- Received: Wed, 19 Oct 2016 11:22:55 GMT">44d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157d8d0aae3:4293bb8:844af92a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4272/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157d8d0aae3:4293bb8:844af92a" data-navigation="inline" data-title="Linking XNAT and dcm2nii()" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157d8d0aae3:4293bb8:844af92a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 512 --></div><!-- react-empty: 513 --><div class="content"><a href="http://neurostars.org/p/4272/" target="_blank" rel="noopener" class="title">Linking XNAT and dcm2nii()</a><div class="summary">Dear list, I'm looking to pull my dicom fmri data from my xnat instance and use it as the input to dcm2nii(). I've been looking at the XNATSource documentation but can't see how to generate the output such that I can connect to my dcm2nii() node? Maybe someone has a nice example of code or can point me in the right direction. I've included my code below, which I know is wrong, but is a starting po</div></div><span class="ago" title="published:Tue, 18 Oct 2016 15:49:58 GMT -- Received: Tue, 18 Oct 2016 17:22:16 GMT">45d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c573453d:20278ce:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4269/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c573453d:20278ce:18bc4566" data-navigation="inline" data-title="add .py file to nipype python path" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c573453d:20278ce:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 528 --></div><!-- react-empty: 529 --><div class="content"><a href="http://neurostars.org/p/4269/" target="_blank" rel="noopener" class="title">add .py file to nipype python path</a><div class="summary"> I have a function defined below. That function is wrapped as nipype node. It relies on another .py file subtract.py in which diffEntireFile is another defined function. How can I add "subtract" to the python path that gets executed by nipype? I need some way of specifying the full path to "subtract" because this runs on a slurm cluster. (Analogous to addpath requirement for matlab files when usin</div></div><span class="ago" title="published:Fri, 14 Oct 2016 21:20:45 GMT -- Received: Fri, 14 Oct 2016 23:07:28 GMT">49d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c4ce670e:1ed33f9:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4268/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c4ce670e:1ed33f9:18bc4566" data-navigation="inline" data-title="Change the default output structure for reconall in nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c4ce670e:1ed33f9:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 544 --></div><!-- react-empty: 545 --><div class="content"><a href="http://neurostars.org/p/4268/" target="_blank" rel="noopener" class="title">Change the default output structure for reconall in nipype</a><div class="summary">Hi, experts of nipype: I have a question for the interface of reconall in nipype, we know that in reconall command, you can define the output hierarchy structure like this: reconall.inputs.subject_dir = output_dir So, for exampe, I have 2 subjects( 001.nii, 002.nii), so after reconall command, it will create the output structure like: output_dir |---- 001 | |---- bem | |---- label ........... |---</div></div><span class="ago" title="published:Fri, 14 Oct 2016 19:44:10 GMT -- Received: Fri, 14 Oct 2016 20:07:23 GMT">49d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c4ce670e:1ed33f8:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4267/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c4ce670e:1ed33f8:18bc4566" data-navigation="inline" data-title="Change the default output structure for reconall in nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157c4ce670e:1ed33f8:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 560 --></div><!-- react-empty: 561 --><div class="content"><a href="http://neurostars.org/p/4267/" target="_blank" rel="noopener" class="title">Change the default output structure for reconall in nipype</a><div class="summary">Hi, experts of nipype: I have a question for the interface of reconall in nipype, we know that in reconall command, you can define the output hierarchy structure like this: reconall.inputs.subject_dir = output_dir So, for exampe, I have 2 subjects( 001.nii, 002.nii), so after reconall command, it will create the output structure like: output_dir |---- 001 | |---- bem | |---- label ........... |---</div></div><span class="ago" title="published:Fri, 14 Oct 2016 19:44:02 GMT -- Received: Fri, 14 Oct 2016 20:07:23 GMT">49d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157af8d22e3:2f00763:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4263/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157af8d22e3:2f00763:297c8405" data-navigation="inline" data-title="Confused about JoinNode()" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157af8d22e3:2f00763:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 576 --></div><!-- react-empty: 577 --><div class="content"><a href="http://neurostars.org/p/4263/" target="_blank" rel="noopener" class="title">Confused about JoinNode()</a><div class="summary">Hi, I want to use JoinNode() to pass all struct images to the NewSegment function, which in turn passes this output to the DARTEL template creation function. The Node that has iterables is infosource, as it iterates over subjects and runs. infosource = Node(interface=IdentityInterface(fields=['subject_id', 'run']), name="infosource") infosource.iterables = [('subject_id', subs), ('run', runs)] Her</div></div><span class="ago" title="published:Mon, 10 Oct 2016 14:49:34 GMT -- Received: Mon, 10 Oct 2016 17:04:04 GMT">53d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157970b698d:9027fd:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4261/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157970b698d:9027fd:297c8405" data-navigation="inline" data-title="Linear Regression - Neuro Images" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157970b698d:9027fd:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 592 --></div><!-- react-empty: 593 --><div class="content"><a href="http://neurostars.org/p/4261/" target="_blank" rel="noopener" class="title">Linear Regression - Neuro Images</a><div class="summary"> Hello, I am new to Neuro Imaging, and I got hold of a set of training and test images that I want to use to train an SVM to predict a subject's age based on an MRI of his/her brain. I have a set of images in .nii format whose voxels are aligned to ensure that the images have the same orientation. I have a basic Python script that traverses the images and reads the image data as a numpy array with</div></div><span class="ago" title="published:Wed, 05 Oct 2016 21:19:49 GMT -- Received: Wed, 05 Oct 2016 22:51:30 GMT">58d</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15786e6ef23:5665b0:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4253/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15786e6ef23:5665b0:297c8405" data-navigation="inline" data-title="Nipype DTI FSL example error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15786e6ef23:5665b0:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 608 --></div><!-- react-empty: 609 --><div class="content"><a href="http://neurostars.org/p/4253/" target="_blank" rel="noopener" class="title">Nipype DTI FSL example error</a><div class="summary">I am new to NiPype and I have been working through some examples. I have encountered a problem with running the dMRI example preprocessing workflow. http://nipy.org/nipype/0.10.0/users/examples/dmri_preprocessing.html I have downloaded the fsl example data set. The link on the NiPype example web page is broken. I am using the fdt1.tar.gz data set instead. This data set seemed to have all of the fi</div></div><span class="ago" title="published:Sun, 02 Oct 2016 17:17:48 GMT -- Received: Sun, 02 Oct 2016 19:37:43 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1577761a7d2:1ac53d5:e3157ec0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4248/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1577761a7d2:1ac53d5:e3157ec0" data-navigation="inline" data-title="Job: Job opening: AFNI group (NIMH, NIH, USA)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1577761a7d2:1ac53d5:e3157ec0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 624 --></div><!-- react-empty: 625 --><div class="content"><a href="http://neurostars.org/p/4248/" target="_blank" rel="noopener" class="title">Job: Job opening: AFNI group (NIMH, NIH, USA)</a><div class="summary"> From: Robert W Cox, PhD Director, Scientific and Statistical Computing Core NIMH &amp; NIH +1-301-594-9196 http://afni.nimh.nih.gov There is a job opening in the AFNI group at NIHM, NIH, USA. Please see this link if you or someone you know is a scientific programmer: https://kelly.secure.force.com/CandidateExperience/CandExpJobDetails?id=a7V80000000PChaEAG&amp;searchFlag=true You do not apply directly to</div></div><span class="ago" title="published:Thu, 29 Sep 2016 18:14:22 GMT -- Received: Thu, 29 Sep 2016 19:17:51 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15776bcadfa:1449730:e0992bbc_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4245/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15776bcadfa:1449730:e0992bbc" data-navigation="inline" data-title="best practices data storage" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15776bcadfa:1449730:e0992bbc" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 640 --></div><!-- react-empty: 641 --><div class="content"><a href="http://neurostars.org/p/4245/" target="_blank" rel="noopener" class="title">best practices data storage</a><div class="summary"> HI, does anyone have a reference for a best-practice document that describes the legal and technical issues around acquiring, storing, processing, and sharing (f)MRI data? I am particularly looking for e.g. an official guideline or list with checkboxes (e.g. raw-data archived -check-, data defaced, de-identified..-checked-, ...) that would help with IRB in order to be ethically (and legally) on t</div></div><span class="ago" title="published:Thu, 29 Sep 2016 13:34:34 GMT -- Received: Thu, 29 Sep 2016 16:17:39 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1577049c370:10ad1ed:18bc4566_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4241/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1577049c370:10ad1ed:18bc4566" data-navigation="inline" data-title="Transparency in Nilearn overlay plots" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1577049c370:10ad1ed:18bc4566" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 656 --></div><!-- react-empty: 657 --><div class="content"><a href="http://neurostars.org/p/4241/" target="_blank" rel="noopener" class="title">Transparency in Nilearn overlay plots</a><div class="summary"> Hello, I'd like to replicate the plots from this paper: http://www.sciencedirect.com/science/article/pii/S089662731200428X, where transparency is used to indicate T-stats. The Matlab code to do this is available here: http://mialab.mrn.org/datavis Is there a way to do this with nilearn's plotting tools? I guess not currently, and that I'd have to write a new add_overlay() function to OrthoSlicer?</div></div><span class="ago" title="published:Wed, 28 Sep 2016 08:39:55 GMT -- Received: Wed, 28 Sep 2016 10:14:24 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1576b1f26b0:76d0f2c:297c8405_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4239/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1576b1f26b0:76d0f2c:297c8405" data-navigation="inline" data-title="Nipype and Python logging" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1576b1f26b0:76d0f2c:297c8405" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 672 --></div><!-- react-empty: 673 --><div class="content"><a href="http://neurostars.org/p/4239/" target="_blank" rel="noopener" class="title">Nipype and Python logging</a><div class="summary">Hi everyone, I have the necessity to use Nipype inside a program that I'm writing, but I'm having a problem. The code actually is using Python logging for log information inside a file, but when I try to import nipype the program starts to print all the logging messages into the console instead that inside the file. I'll try to be clear with a little example. import logging if __name__ == '__main_</div></div><span class="ago" title="published:Tue, 27 Sep 2016 09:39:45 GMT -- Received: Tue, 27 Sep 2016 10:09:46 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15748eabdca:142c0c:e30dc255_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4236/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15748eabdca:142c0c:e30dc255" data-navigation="inline" data-title="Code review for wrapped ANTs function 'CreateJacobianDeterminantImage'?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15748eabdca:142c0c:e30dc255" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 688 --></div><!-- react-empty: 689 --><div class="content"><a href="http://neurostars.org/p/4236/" target="_blank" rel="noopener" class="title">Code review for wrapped ANTs function 'CreateJacobianDeterminantImage'?</a><div class="summary">Hi all, I noticed that the currently wrapped ANTs function ANTSJacobian is deprecated. 160919-16:10:48,212 interface INFO: stdout 2016-09-19T16:10:48.212057:Sorry! ANTSJacobian is deprecated 160919-16:10:48,212 interface INFO: stdout 2016-09-19T16:10:48.212057:Please use CreateJacobianDeterminantImage For the full log, see here . The CreateJacobianDeterminantImage ANTs function can be viewed here </div></div><span class="ago" title="published:Tue, 20 Sep 2016 18:12:27 GMT -- Received: Tue, 20 Sep 2016 18:45:26 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15748eabdca:142c0b:e30dc255_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4235/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15748eabdca:142c0b:e30dc255" data-navigation="inline" data-title="Strange effect in ICA spatial maps" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15748eabdca:142c0b:e30dc255" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 745 --></div><!-- react-empty: 746 --><div class="content"><a href="http://neurostars.org/p/4235/" target="_blank" rel="noopener" class="title">Strange effect in ICA spatial maps</a><div class="summary"> Hi, Have you ever seen this weird effect on ICA spatial maps from resting-state fMRI? I am seeing this "Gruyère" effect on the blobs of the ICA spatial maps. I can't figure out why. These images were preprocessed as: 6 sec trimmed - slicetime corrected - motion corrected - nuisance corrected (motion, CompCor CSF and WM) - bandpass filtered - smoothed (8mm). The warping was done using the avg EPI </div></div><span class="ago" title="published:Tue, 20 Sep 2016 16:51:31 GMT -- Received: Tue, 20 Sep 2016 18:45:26 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157325c6f52:31d0a9:3e511f58_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4230/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157325c6f52:31d0a9:3e511f58" data-navigation="inline" data-title="Normalize, split and concatenate time series" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157325c6f52:31d0a9:3e511f58" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 761 --></div><!-- react-empty: 762 --><div class="content"><a href="http://neurostars.org/p/4230/" target="_blank" rel="noopener" class="title">Normalize, split and concatenate time series</a><div class="summary"> Hello, I am currently working on creating connectivity matrices based on task data. The design is blocked and I'm interested in connectivity that is happening within specific blocks. Therefore, I am selecting only the volumes corresponding to specific task conditions. Afterwards I want to concatenate the data from all subjects and generate connectivity matrices from the concatenated time series. </div></div><span class="ago" title="published:Fri, 16 Sep 2016 07:46:14 GMT -- Received: Fri, 16 Sep 2016 09:38:21 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1573065a893:13a97b35:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4223/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1573065a893:13a97b35:e6b2a59c" data-navigation="inline" data-title="Can I merge/delete Neurostars accounts?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1573065a893:13a97b35:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 777 --></div><!-- react-empty: 778 --><div class="content"><a href="http://neurostars.org/p/4223/" target="_blank" rel="noopener" class="title">Can I merge/delete Neurostars accounts?</a><div class="summary"> Through inconsistent use of social login, I somehow created two separate Neurostars accounts. [This one](https://neurostars.org/u/264/) and another [one](http://neurostars.org/u/595/). I'd like to either merge the latter into the former, or just delete the latter to free up social logins. I've tried removing the social logins from the latter, but I can't wipe them all out (it complains I have no </div></div><span class="ago" title="published:Thu, 15 Sep 2016 21:56:40 GMT -- Received: Fri, 16 Sep 2016 00:29:11 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1572fc0a6d6:153ca7c1:e3157ec0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4220/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1572fc0a6d6:153ca7c1:e3157ec0" data-navigation="inline" data-title="Nipype distributed computing" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1572fc0a6d6:153ca7c1:e3157ec0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 793 --></div><!-- react-empty: 794 --><div class="content"><a href="http://neurostars.org/p/4220/" target="_blank" rel="noopener" class="title">Nipype distributed computing</a><div class="summary"> How does nipype actually schedules and distributes node computations over available cores/threads/compute nodes when using the multiproc or the slurm plugins? In the case of mriqc, where you can specify both the number of threads you want to use for the pipeline, and the number of threads to be used for antsRegistration, how would I go about allocating an optimal number of nodes? If I allocate on</div></div><span class="ago" title="published:Thu, 15 Sep 2016 19:22:47 GMT -- Received: Thu, 15 Sep 2016 21:28:57 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157294ec6af:11d32c57:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4218/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157294ec6af:11d32c57:e6b2a59c" data-navigation="inline" data-title="AFNI.RETROICOR() deletes output" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157294ec6af:11d32c57:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 809 --></div><!-- react-empty: 810 --><div class="content"><a href="http://neurostars.org/p/4218/" target="_blank" rel="noopener" class="title">AFNI.RETROICOR() deletes output</a><div class="summary">Hi there, I've found some peculiar behavior with afni.retroicor(). It runs just fine, but in the temporary directory the nifti file just disappears after the computation. I can see it fine, and I can see it grow in size during analysis, but after it's done it just goes away. I'm not even using the sinker and it still disappears. Here is my code: datasource = Node(interface=nio.DataGrabber(infields</div></div><span class="ago" title="published:Wed, 14 Sep 2016 13:34:43 GMT -- Received: Wed, 14 Sep 2016 15:26:51 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15723e29744:1191d342:b656daf7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4214/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15723e29744:1191d342:b656daf7" data-navigation="inline" data-title="scratch space needed for mriqc" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15723e29744:1191d342:b656daf7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 825 --></div><!-- react-empty: 826 --><div class="content"><a href="http://neurostars.org/p/4214/" target="_blank" rel="noopener" class="title">scratch space needed for mriqc</a><div class="summary"> Dear Oscar, We are under the first level process with mriqc with 481 participants. For now, SCRATCH folder occupies 65 GB and we only have 18 GB more...I would greatly appreciate if you could tell me something about the space needed for this kind of analysis. One thing I have thought is to stop the analysis once single subject level has finished...and I think this will once single pdf files for e</div></div><span class="ago" title="published:Tue, 13 Sep 2016 12:07:57 GMT -- Received: Tue, 13 Sep 2016 14:10:34 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1570a1a64a0:a512f77:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4197/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1570a1a64a0:a512f77:e6b2a59c" data-navigation="inline" data-title="mriqc running in a cluster: outputs generated" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1570a1a64a0:a512f77:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 841 --></div><!-- react-empty: 842 --><div class="content"><a href="http://neurostars.org/p/4197/" target="_blank" rel="noopener" class="title">mriqc running in a cluster: outputs generated</a><div class="summary">Dear Oscar, We have been testing again for two subjects in our cluster with mriqc. It seems that now it is working fine but there is nothing in logs folder and I wonder if this is what is expected. The run with two subjects has taken about 1 hour and here it is the output: In the scratch folder we have the following files: [rdacosta@node67 test_docker]$ ls scratch/work/ -lath total 3.4M drwxr-xr-x</div></div><span class="ago" title="published:Thu, 08 Sep 2016 12:13:33 GMT -- Received: Thu, 08 Sep 2016 14:01:24 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15705960706:9e08e68:e3157ec0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4188/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15705960706:9e08e68:e3157ec0" data-navigation="inline" data-title="SPM realign and unwarp implementation in nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15705960706:9e08e68:e3157ec0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 857 --></div><!-- react-empty: 858 --><div class="content"><a href="http://neurostars.org/p/4188/" target="_blank" rel="noopener" class="title">SPM realign and unwarp implementation in nipype</a><div class="summary"> I've searched the posts and there was talk about implementing the realign &amp; unwarp function of SPM. So far I haven't found it. Did anybody add it? If not I'll write it and send it to you and hopefully it gets added. Just to make sure I understand it correctly, the following must be added: class RealignUnwarpInputSpec(SPMCommandInputSpec) class RealignUnwarpOutputSpec(SPMCommandInputSpec) class Re</div></div><span class="ago" title="published:Wed, 07 Sep 2016 14:20:01 GMT -- Received: Wed, 07 Sep 2016 16:58:20 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15704f10b1f:9ac6e18:e3157ec0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4184/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15704f10b1f:9ac6e18:e3157ec0" data-navigation="inline" data-title="Problems running mirqc using HCP" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15704f10b1f:9ac6e18:e3157ec0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 873 --></div><!-- react-empty: 874 --><div class="content"><a href="http://neurostars.org/p/4184/" target="_blank" rel="noopener" class="title">Problems running mirqc using HCP</a><div class="summary">Dear mriqc users, I am having the following problem while executing mriqc in the cluster...but not sure about where it is the error...any help with this would be more than grateful: [rdacosta@node67 ~]$ docker run -v /home/rdacosta/test_docker/bids/:/data -v /home/rdacosta/test_docker/scratch:/scratch -w /scratch poldracklab/mriqc:latest /data /scratch/out participant -w /scratch/work 160907-14:25</div></div><span class="ago" title="published:Wed, 07 Sep 2016 11:49:44 GMT -- Received: Wed, 07 Sep 2016 13:58:08 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157044c133a:8ca92df:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4182/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157044c133a:8ca92df:e6b2a59c" data-navigation="inline" data-title="BIDS-like iterables for subjects AND runs" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157044c133a:8ca92df:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 889 --></div><!-- react-empty: 890 --><div class="content"><a href="http://neurostars.org/p/4182/" target="_blank" rel="noopener" class="title">BIDS-like iterables for subjects AND runs</a><div class="summary">I'm trying to set up my analysis pipeline and encountered the following problem. All my data are organized according to the BIDS standard, hence my functional runs are as follows: /sub-01/ /func/sub-01_task- _run-01_bold.nii /sub-01/ /func/sub-01_task- _run-02_bold.nii /sub-01/ /anat/sub-01_acq-T1w.nii /sub-02/ /func/sub-02_task- _run-01_bold.nii ... How can i set up an iterable when not only the </div></div><span class="ago" title="published:Wed, 07 Sep 2016 08:54:00 GMT -- Received: Wed, 07 Sep 2016 10:57:57 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157025c4065:8f7f083:e3157ec0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4179/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157025c4065:8f7f083:e3157ec0" data-navigation="inline" data-title="MNE Python .FDT File Error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_157025c4065:8f7f083:e3157ec0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 905 --></div><!-- react-empty: 906 --><div class="content"><a href="http://neurostars.org/p/4179/" target="_blank" rel="noopener" class="title">MNE Python .FDT File Error</a><div class="summary"> working in MNE python with .set file from EEGlab(matlab) have imported the data: x=mne.io.read_raw_eeglab('Subject1.set') trying to follow tutorials; always get this error: "IOError: [Errno 2] No such file or directory: u'Subject1.fdt'" One line generating the error is ln[8], here: https://plot.ly/ipython-notebooks/mne-tutorial/ The error arises basically any time the word "time" appears Help?!? </div></div><span class="ago" title="published:Wed, 07 Sep 2016 01:21:22 GMT -- Received: Wed, 07 Sep 2016 01:56:22 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156ffc86c86:7b04884:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4177/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156ffc86c86:7b04884:e6b2a59c" data-navigation="inline" data-title="Running mriqc in a HCP node (16GB and 16 cores) without access to internet" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156ffc86c86:7b04884:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 921 --></div><!-- react-empty: 922 --><div class="content"><a href="http://neurostars.org/p/4177/" target="_blank" rel="noopener" class="title">Running mriqc in a HCP node (16GB and 16 cores) without access to internet</a><div class="summary"> Dear mriqc members, I am trying to implement mriqc software in a node from a HCP. We have updated the server from Centos 5. to Centros 7. in order to get the minimum kernel requisits for installing Docker. This was ok. The problem now is when I try to run the follwing command which sets the following erros: docker run -it poldracklab/mriqc:latest -v Unable to find image 'poldracklab/mriqc:latest'</div></div><span class="ago" title="published:Tue, 06 Sep 2016 12:20:40 GMT -- Received: Tue, 06 Sep 2016 13:55:40 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156f955bdcd:6384d1a:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4176/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156f955bdcd:6384d1a:e6b2a59c" data-navigation="inline" data-title="Debugging Workflows wrapped in Function MapNode (Dynamic Iterables)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156f955bdcd:6384d1a:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 937 --></div><!-- react-empty: 938 --><div class="content"><a href="http://neurostars.org/p/4176/" target="_blank" rel="noopener" class="title">Debugging Workflows wrapped in Function MapNode (Dynamic Iterables)</a><div class="summary"> Hi all, I have a situation where I have defined an entire workflow which I am now including as a part of a new, larger workflow. In the larger workflow, I perform some clustering into an unknown (ahead of time) number of clusters, then run the old workflow on each of the results. So I need to define an iterable at runtime. As suggested here https://github.com/nipy/nipype/issues/819 I have wrapped</div></div><span class="ago" title="published:Mon, 05 Sep 2016 06:42:04 GMT -- Received: Mon, 05 Sep 2016 07:52:41 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156f61c46f7:59c5e85:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4175/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156f61c46f7:59c5e85:e6b2a59c" data-navigation="inline" data-title="nipype node finishes but doesn't close out" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156f61c46f7:59c5e85:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 953 --></div><!-- react-empty: 954 --><div class="content"><a href="http://neurostars.org/p/4175/" target="_blank" rel="noopener" class="title">nipype node finishes but doesn't close out</a><div class="summary">Any ideas on what would cause a previously working nipype workflow to suddenly enter a node-completes-but-then-hangs state? I say it comples but hangs because a look in the scratch directory shows the saved output file, yet the workflow never continues on. Furthermore, when I just kill all jobs, no crash pkl file is generated for that node despite it (I think) being interrupted by the kill all job</div></div><span class="ago" title="published:Sun, 04 Sep 2016 15:10:27 GMT -- Received: Sun, 04 Sep 2016 16:51:03 GMT">2mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156e890f18d:32879b5:e3157ec0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4174/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156e890f18d:32879b5:e3157ec0" data-navigation="inline" data-title="What digital resolution should be used for diffusion processing?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156e890f18d:32879b5:e3157ec0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 969 --></div><!-- react-empty: 970 --><div class="content"><a href="http://neurostars.org/p/4174/" target="_blank" rel="noopener" class="title">What digital resolution should be used for diffusion processing?</a><div class="summary"> On our system, digital resolution and physical resolution are often confused. Originally, our DTI images are 2.2x2.2x2.2mm, but the scanner exports 0.8x0.8x2.2mm (this is an old protocol used for a long term longitudinal study). The question that is often bounced around in our group is: "What digital resolution should be using for diffusion processing?". Should we upsample the slice dimension to </div></div><span class="ago" title="published:Fri, 02 Sep 2016 00:18:01 GMT -- Received: Fri, 02 Sep 2016 01:43:48 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156dafca2c0:8616ee:902735ce_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4160/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156dafca2c0:8616ee:902735ce" data-navigation="inline" data-title="HDF ERROR when importing nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156dafca2c0:8616ee:902735ce" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 985 --></div><!-- react-empty: 986 --><div class="content"><a href="http://neurostars.org/p/4160/" target="_blank" rel="noopener" class="title">HDF ERROR when importing nipype</a><div class="summary">Hi When I "import nipype" I get the error below, what might cause it? Thanks in advance Keren Lesinger &gt;&gt;&gt; import nipype Warning! ***HDF5 library version mismatched error*** The HDF5 header files used to compile this application do not match the version used by the HDF5 library to which this application is linked. Data corruption or segmentation faults may occur if the application continues. This </div></div><span class="ago" title="published:Tue, 30 Aug 2016 08:17:07 GMT -- Received: Tue, 30 Aug 2016 10:26:45 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156d068cc00:65fcaa9:45cbc242_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4153/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156d068cc00:65fcaa9:45cbc242" data-navigation="inline" data-title="Reference for Nipy" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156d068cc00:65fcaa9:45cbc242" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1001 --></div><!-- react-empty: 1002 --><div class="content"><a href="http://neurostars.org/p/4153/" target="_blank" rel="noopener" class="title">Reference for Nipy</a><div class="summary"> Hi all, another reference question: How do I cite Nipy (the software package, not the community) properly? I found this on researchgate: https://www.researchgate.net/publication/238805751_NIPY_an_open_library_and_development_framework_for_FMRI_data_analysis but cannot find such an article in the indicated Neuroimage issue. Thanks! Julia </div></div><span class="ago" title="published:Sun, 28 Aug 2016 08:05:42 GMT -- Received: Sun, 28 Aug 2016 09:09:04 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156c6beee1e:460b94b:db2061a7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4148/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156c6beee1e:460b94b:db2061a7" data-navigation="inline" data-title="nipy.SpaceTimeRealigner -->slice_times option description" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156c6beee1e:460b94b:db2061a7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1017 --></div><!-- react-empty: 1018 --><div class="content"><a href="http://neurostars.org/p/4148/" target="_blank" rel="noopener" class="title">nipy.SpaceTimeRealigner --&gt;slice_times option description</a><div class="summary"> Dear all, Is there a detailed documentation for nipy's SpaceTimeRealigner function? I'm especially searching for the meanings of optional strings for "slice_times" input. What type of slice time acquisitions do these strings refer to: 'asc_alt_2_1' 'asc_alt_half' 'asc_alt_siemens' 'ascending' 'desc_alt_2' 'desc_alt_half' 'descending' Thanks in advance! </div></div><span class="ago" title="published:Fri, 26 Aug 2016 09:30:19 GMT -- Received: Fri, 26 Aug 2016 12:06:56 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156befce8a6:2878d30:db2061a7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4146/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156befce8a6:2878d30:db2061a7" data-navigation="inline" data-title="New error following instructions for running mriqc" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156befce8a6:2878d30:db2061a7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1033 --></div><!-- react-empty: 1034 --><div class="content"><a href="http://neurostars.org/p/4146/" target="_blank" rel="noopener" class="title">New error following instructions for running mriqc</a><div class="summary">Dear mriqc users, I have tried again and now it appears the following error I do not know what is and how to cope with it: rosalia@rosalia-PORTEGE-Z930:/ media/rosalia/Rosalia_2016$ docker run -v /media/rosalia/Rosalia_2016/bids:/data -v /media/rosalia/Rosalia_2016/outputs:/mriqc_outputs -v /media/rosalia/Rosalia_2016/scratch:/scratch -w /scratch oesteban/mriqc -B /data -o /mriqc_outputs -w /scrat</div></div><span class="ago" title="published:Wed, 24 Aug 2016 21:23:43 GMT -- Received: Wed, 24 Aug 2016 23:57:40 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156befce8a6:2878d2f:db2061a7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4145/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156befce8a6:2878d2f:db2061a7" data-navigation="inline" data-title="Forum: Problems running mirqc using an external hard disk as a path" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156befce8a6:2878d2f:db2061a7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1049 --></div><!-- react-empty: 1050 --><div class="content"><a href="http://neurostars.org/p/4145/" target="_blank" rel="noopener" class="title">Forum: Problems running mirqc using an external hard disk as a path</a><div class="summary">Dear mriqc users, My name is Rosalia. This is the first time I use docker and mriqc. I have followed the instructions in the main page about running docker and about running mriqc via docker. The problem come when I try to run the following command line...I have tried two times and in the two times I had the following errors I do not know how to cope with. -First error: rosalia@rosalia-PORTEGE-Z93</div></div><span class="ago" title="published:Wed, 24 Aug 2016 21:11:28 GMT -- Received: Wed, 24 Aug 2016 23:57:40 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156bb1e6573:190645f:db2061a7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4144/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156bb1e6573:190645f:db2061a7" data-navigation="inline" data-title="[Nilearn] DIfference between nilearn MultiPCA and sklearn PCA" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156bb1e6573:190645f:db2061a7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1065 --></div><!-- react-empty: 1066 --><div class="content"><a href="http://neurostars.org/p/4144/" target="_blank" rel="noopener" class="title">[Nilearn] DIfference between nilearn MultiPCA and sklearn PCA</a><div class="summary"> Hi all, I have a set of 3D PET images of brain and I did PCA on them using both nilearn MultiPCA and sklearn PCA. For nilearn MultiPCA, I'm just needed to supply the brain mask and the list of images' file path to do the PCA. I wanted to duplicate nilearn results using sklearn. For sklearn, I have to manually extract the brain images (using the same mask) and then reshape them into 1D array, fina</div></div><span class="ago" title="published:Wed, 24 Aug 2016 05:29:55 GMT -- Received: Wed, 24 Aug 2016 05:55:46 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156b406c46c:862814b:3512bc0f_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4142/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156b406c46c:862814b:3512bc0f" data-navigation="inline" data-title="Any differences between various DICOM to NIfTI conversion tools?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156b406c46c:862814b:3512bc0f" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1081 --></div><!-- react-empty: 1082 --><div class="content"><a href="http://neurostars.org/p/4142/" target="_blank" rel="noopener" class="title">Any differences between various DICOM to NIfTI conversion tools?</a><div class="summary"> I am going to create an automatic pipeline for incoming DICOMs to be converted to NIfTI format, but I want to make sure I use the best possible software for this, i.e. the conversion tool does not make any problems or errors for me. (We'd heard that dcm2nii sometimes 'altered' the image data, or encoded it incorrectly, or some such) I am partial to using mri_convert, but as there are many possibi</div></div><span class="ago" title="published:Mon, 22 Aug 2016 18:20:32 GMT -- Received: Mon, 22 Aug 2016 20:52:37 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156aede5a58:6c263f4:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4140/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156aede5a58:6c263f4:e6b2a59c" data-navigation="inline" data-title="nipype and non-standard nested directory structures" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156aede5a58:6c263f4:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1097 --></div><!-- react-empty: 1098 --><div class="content"><a href="http://neurostars.org/p/4140/" target="_blank" rel="noopener" class="title">nipype and non-standard nested directory structures</a><div class="summary"> Hi all, I'm dealing with a dataset which has a non-standard directory structure, and I'm wondering how best to feed the data to nipype. Specifically, my data are laid out as follows at the top level (levels below this have a simple, homogeneous labeling scheme): sub001 session20160415at170000 session20160515at170000 sub 002 session20151225at170000 session20160125at170000 etc. That is, participant</div></div><span class="ago" title="published:Sun, 21 Aug 2016 20:43:15 GMT -- Received: Sun, 21 Aug 2016 20:50:22 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156a7c89892:5323377:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4138/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156a7c89892:5323377:e6b2a59c" data-navigation="inline" data-title="MGZ format is an incorrectly GZipped MGH ?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156a7c89892:5323377:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1113 --></div><!-- react-empty: 1114 --><div class="content"><a href="http://neurostars.org/p/4138/" target="_blank" rel="noopener" class="title">MGZ format is an incorrectly GZipped MGH ?</a><div class="summary"> Freesurfer's MGZ files are supposed to be the gzipped version of MGH format files. In all my old Freesurfer files I still can do: gunzip orig.mgz to get to the uncompressed MGH file. However, in more recent files gunzip fails systematically with the following error: invalid compressed data--crc error Are you aware of any change in the way in which MGZ files are produced from MGH files in the rece</div></div><span class="ago" title="published:Sat, 20 Aug 2016 10:16:07 GMT -- Received: Sat, 20 Aug 2016 11:49:16 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1569a4131b5:23b8a72:e6b2a59c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4137/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1569a4131b5:23b8a72:e6b2a59c" data-navigation="inline" data-title="Job: Hiring: Cloud and HPC Engineer at NIMH in Bethesda, MD" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1569a4131b5:23b8a72:e6b2a59c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1129 --></div><!-- react-empty: 1130 --><div class="content"><a href="http://neurostars.org/p/4137/" target="_blank" rel="noopener" class="title">Job: Hiring: Cloud and HPC Engineer at NIMH in Bethesda, MD</a><div class="summary">NIMH is hiring a Cloud and HPC Engineer. Please spread the word! https://github.com/nih-fmrif/hpc_job_ad HIRING: CLOUD AND HPC ENGINEER The National Institute of Mental Health (NIMH) is the lead federal agency for research on mental disorders. NIMH is one of the 27 Institutes and Centers that make up the National Institutes of Health (NIH), which is responsible for all federally funded biomedical </div></div><span class="ago" title="published:Wed, 17 Aug 2016 18:58:58 GMT -- Received: Wed, 17 Aug 2016 20:46:19 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15693cefa5a:c348cf:db2061a7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4135/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15693cefa5a:c348cf:db2061a7" data-navigation="inline" data-title="dcm2nii output data_type (INT16, FLOAT32)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15693cefa5a:c348cf:db2061a7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1145 --></div><!-- react-empty: 1146 --><div class="content"><a href="http://neurostars.org/p/4135/" target="_blank" rel="noopener" class="title">dcm2nii output data_type (INT16, FLOAT32)</a><div class="summary"> Dear all, I want to convert some raw dicom data into nifit. For that I apply dcm2nii and everything works fine. When I look at the data_type of the nifti file (fslinfo) I see that it ist INT16. I have however also other data, which were already converted to nifti, in FLOAT32. So, I would like to have both data in float. Is there a possibility to make dcm2nii to convert to float32 data? (Maybe the</div></div><span class="ago" title="published:Tue, 16 Aug 2016 12:35:01 GMT -- Received: Tue, 16 Aug 2016 14:43:50 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1567622e8fb:122fcf:99edc1d4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4123/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1567622e8fb:122fcf:99edc1d4" data-navigation="inline" data-title="Tutorial: Running mriqc using docker" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1567622e8fb:122fcf:99edc1d4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1161 --></div><!-- react-empty: 1162 --><div class="content"><a href="http://neurostars.org/p/4123/" target="_blank" rel="noopener" class="title">Tutorial: Running mriqc using docker</a><div class="summary">A quick tutorial on how to run mriqc with docker. Preliminaries Get the Docker Engine (https://docs.docker.com/engine/installation/) Have your data organized in BIDS ( ref1 , ref2 ) Running mriqc Test that the mriqc container works correctly: run -it oesteban/mriqc -v A successful run will show the mriqc version in the last line of the output. Run it: docker run -v </div></div><span class="ago" title="published:Wed, 10 Aug 2016 19:55:46 GMT -- Received: Wed, 10 Aug 2016 20:26:54 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1567622e8fb:122fce:99edc1d4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4122/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1567622e8fb:122fce:99edc1d4" data-navigation="inline" data-title="nipype workflow using ANTs interface and SLURM plug-in" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1567622e8fb:122fce:99edc1d4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1177 --></div><!-- react-empty: 1178 --><div class="content"><a href="http://neurostars.org/p/4122/" target="_blank" rel="noopener" class="title">nipype workflow using ANTs interface and SLURM plug-in</a><div class="summary">I've created a nipype workflow to run antsRegistration (currently, only rigid registration for testing purposes) over a list of subjects iterated using InfoSource. A SelectFiles node is grabbing three image modalities each for the fixed (lhtemplate[0,1,2].nii.gz) and moving images (T1, T2*, and label images), which gets passed into the antsRegistration node. We're running nipype 0.13 developer due</div></div><span class="ago" title="published:Wed, 10 Aug 2016 18:59:49 GMT -- Received: Wed, 10 Aug 2016 20:26:54 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1566fb18244:104e4db:5405e57d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4119/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1566fb18244:104e4db:5405e57d" data-navigation="inline" data-title="MapNode with iterfields of different lengths" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1566fb18244:104e4db:5405e57d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1193 --></div><!-- react-empty: 1194 --><div class="content"><a href="http://neurostars.org/p/4119/" target="_blank" rel="noopener" class="title">MapNode with iterfields of different lengths</a><div class="summary"> Hello, I am trying to run a workflow in parallel for multiple subjects by converting many of the nodes to mapnodes, but some nodes are already defined as MapNodes with some iterfields. I wonder if its possible to execute a MapNode of a workflow to have a mapnode with iterfields of different lengths? suppose, I have a mapnode and an IdentifyInterface inputnode that produces a list of files in its </div></div><span class="ago" title="published:Tue, 09 Aug 2016 13:58:35 GMT -- Received: Tue, 09 Aug 2016 14:25:19 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1565b0e6176:8dc63c1:d45364f0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4116/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1565b0e6176:8dc63c1:d45364f0" data-navigation="inline" data-title="Multi-subject maps of seed-based correlation" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1565b0e6176:8dc63c1:d45364f0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1209 --></div><!-- react-empty: 1210 --><div class="content"><a href="http://neurostars.org/p/4116/" target="_blank" rel="noopener" class="title">Multi-subject maps of seed-based correlation</a><div class="summary"> Hi, I have the following question: Is it possible to produce seed-based correlation maps for multiple subjects with Nilearn or to average already calculated (single subject) maps? I know how to perform resting-state analysis using seed time series as EV using FSL FEAT. I can average results across subjects there (for example with FLAME) using cope-s from first-level analysis. Such approach is des</div></div><span class="ago" title="published:Fri, 05 Aug 2016 13:17:05 GMT -- Received: Fri, 05 Aug 2016 14:14:44 GMT">3mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1564ce1b5c2:56c2741:d45364f0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4114/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1564ce1b5c2:56c2741:d45364f0" data-navigation="inline" data-title="Nipype import module crash" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1564ce1b5c2:56c2741:d45364f0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1225 --></div><!-- react-empty: 1226 --><div class="content"><a href="http://neurostars.org/p/4114/" target="_blank" rel="noopener" class="title">Nipype import module crash</a><div class="summary">Hi, I'm getting the following crash file when trying to run a simple script. I'm not sure why there is an error importing the module when all the files are in the same directory(which I've appended to the Python path). I need to create interfaces for scripts from my lab, so my idea was to create a bash script, wrap it, and execute it in a workflow to test Nipype itself without the analysis. I've a</div></div><span class="ago" title="published:Tue, 02 Aug 2016 19:26:40 GMT -- Received: Tue, 02 Aug 2016 20:11:15 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156350ae6db:2f305d:36858854_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4105/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156350ae6db:2f305d:36858854" data-navigation="inline" data-title="Structural T1 to MNI registration with ANTs: parameters recommendation?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156350ae6db:2f305d:36858854" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1241 --></div><!-- react-empty: 1242 --><div class="content"><a href="http://neurostars.org/p/4105/" target="_blank" rel="noopener" class="title">Structural T1 to MNI registration with ANTs: parameters recommendation?</a><div class="summary">Dear all, I'm struggling a bit with finding some robust settings for ANTs when mapping the T1-weighted image to the 1mm MNI T1 template. Trying the developers advice , the default execution of antsRegistrationSyN.sh -d 3 -f MNI.nii.gz -m T1.nii.gz -o XXX will result in a call like: /opt/ants/antsRegistration --dimensionality 3 --float 0 --output [XXX,XXXWarped.nii.gz] --interpolation Linear --wins</div></div><span class="ago" title="published:Fri, 29 Jul 2016 02:27:14 GMT -- Received: Fri, 29 Jul 2016 05:05:22 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1562e9a2f3a:1159304b:d45364f0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4102/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1562e9a2f3a:1159304b:d45364f0" data-navigation="inline" data-title="Question on food shipping" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1562e9a2f3a:1159304b:d45364f0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1257 --></div><!-- react-empty: 1258 --><div class="content"><a href="http://neurostars.org/p/4102/" target="_blank" rel="noopener" class="title">Question on food shipping</a><div class="summary"> How can I get a pizza delivered late at night? All the pizza stores near me close early. </div></div><span class="ago" title="published:Wed, 27 Jul 2016 20:42:17 GMT -- Received: Wed, 27 Jul 2016 23:04:31 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1562df4fdba:1174be51:97a41d4c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4101/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1562df4fdba:1174be51:97a41d4c" data-navigation="inline" data-title="Forum: Nilearn use case-MRI datatype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1562df4fdba:1174be51:97a41d4c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1273 --></div><!-- react-empty: 1274 --><div class="content"><a href="http://neurostars.org/p/4101/" target="_blank" rel="noopener" class="title">Forum: Nilearn use case-MRI datatype</a><div class="summary"> Bonjour: I have a stupid question for the MRI datatype that Nilearn uses, in the nilearn page, you have one sentence : Nilearn can readily be used on task fMRI , resting-state , or VBM data. So my question is that, can I use the sMRI(T1 image) to follow the tutorial that you provide, as I know that you just include one page for VBM data? I am new to this field:) So do you have some tutorial for M</div></div><span class="ago" title="published:Wed, 27 Jul 2016 18:59:53 GMT -- Received: Wed, 27 Jul 2016 20:04:05 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15628cdb051:1027d2e2:97a41d4c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4098/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15628cdb051:1027d2e2:97a41d4c" data-navigation="inline" data-title="mriqc image registration problem" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15628cdb051:1027d2e2:97a41d4c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1289 --></div><!-- react-empty: 1290 --><div class="content"><a href="http://neurostars.org/p/4098/" target="_blank" rel="noopener" class="title">mriqc image registration problem</a><div class="summary">I got an error when running mriqc regarding a missing file needed for image registration. A sample error trace from one of the work_log files says: Traceback (most recent call last): File "/Users/varadajc/anaconda/envs/qc/lib/python3.5/site-packages/nipype/interfaces/base.py", line 1122, in aggregate_outputs setattr(outputs, key, val) File "/Users/varadajc/anaconda/envs/qc/lib/python3.5/site-packa</div></div><span class="ago" title="published:Tue, 26 Jul 2016 19:31:32 GMT -- Received: Tue, 26 Jul 2016 20:03:03 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15624f0331c:efd4775:d45364f0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4092/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15624f0331c:efd4775:d45364f0" data-navigation="inline" data-title="multiple sessions in mriqc" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15624f0331c:efd4775:d45364f0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1305 --></div><!-- react-empty: 1306 --><div class="content"><a href="http://neurostars.org/p/4092/" target="_blank" rel="noopener" class="title">multiple sessions in mriqc</a><div class="summary">I have the following directory structure: top_bids sub-0001 ses-abc anat sub-0001_run-01_T2w.nii.gz sub-0001_run-02_T1w.nii.gz func … ses-def anat sub-0001_run-01_T2w.nii.gz sub-0001_run-02_T1w.nii.gz func … sub-0002 ses-abc … ses-def … … I am trying to launch mriqc as follows: mriqc -B </div></div><span class="ago" title="published:Mon, 25 Jul 2016 23:28:34 GMT -- Received: Tue, 26 Jul 2016 02:02:16 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1560938a2d0:8bc3927:85ca5f35_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4083/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1560938a2d0:8bc3927:85ca5f35" data-navigation="inline" data-title="Nipype: smri_fsreconall.py Error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1560938a2d0:8bc3927:85ca5f35" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1321 --></div><!-- react-empty: 1322 --><div class="content"><a href="http://neurostars.org/p/4083/" target="_blank" rel="noopener" class="title">Nipype: smri_fsreconall.py Error</a><div class="summary">Hi everyone, I'm using the smri_fsreconall.py, any idea why this might be happening? Traceback (most recent call last): File "/home/hpc/rpalmeida/conversion_workflow.py", line 23, in </div></div><span class="ago" title="published:Wed, 20 Jul 2016 15:07:39 GMT -- Received: Wed, 20 Jul 2016 16:52:01 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15607ee7822:863a7fd:85ca5f35_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4076/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15607ee7822:863a7fd:85ca5f35" data-navigation="inline" data-title="SpecifyModel: could not convert string to float" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15607ee7822:863a7fd:85ca5f35" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1337 --></div><!-- react-empty: 1338 --><div class="content"><a href="http://neurostars.org/p/4076/" target="_blank" rel="noopener" class="title">SpecifyModel: could not convert string to float</a><div class="summary"> Hi, Writing a new workflow of first level analysis I use SpecifyModel in the following way: ... specifymodel = pe.Node(SpecifyModel(), name='SpecifyModel') specifymodel.inputs.input_units = 'secs' apecifymodel.inputs.time_repetition = 2.5 specifymodel.inputs.high_pass_filter_cutoff = 128. / (2. * 2.) first_level_modelfit_workflow.connect(selector, 'bfsl_files', specifymodel, 'event_files') first_</div></div><span class="ago" title="published:Wed, 20 Jul 2016 10:47:51 GMT -- Received: Wed, 20 Jul 2016 10:51:24 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156055988bd:816bb12:97a41d4c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4073/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156055988bd:816bb12:97a41d4c" data-navigation="inline" data-title="NeuroStars as a support form rather than using a mailing list" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_156055988bd:816bb12:97a41d4c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1353 --></div><!-- react-empty: 1354 --><div class="content"><a href="http://neurostars.org/p/4073/" target="_blank" rel="noopener" class="title">NeuroStars as a support form rather than using a mailing list</a><div class="summary"> Would NeuroStars be a good place to post and respond to Mindboggle software (http://mindboggle.info) support questions after an upcoming launch, as opposed to using a mailing list? </div></div><span class="ago" title="published:Tue, 19 Jul 2016 22:19:59 GMT -- Received: Tue, 19 Jul 2016 22:49:29 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155f01116ee:3870b13:d45364f0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4067/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155f01116ee:3870b13:d45364f0" data-navigation="inline" data-title="Standard preprocessing pipeline" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155f01116ee:3870b13:d45364f0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1369 --></div><!-- react-empty: 1370 --><div class="content"><a href="http://neurostars.org/p/4067/" target="_blank" rel="noopener" class="title">Standard preprocessing pipeline</a><div class="summary"> Hi all, I need you to point me to some standard preprocessing pipeline of structural, functional and diffusion MRI scans. In particular, I'm interested in standard preprocessing pipelines or methods for structural and functional connectome analyses. It does not necessariliy have to be software, even just a document describing the common practises for these types of analyses is ok, I can implement</div></div><span class="ago" title="published:Fri, 15 Jul 2016 17:25:02 GMT -- Received: Fri, 15 Jul 2016 19:38:19 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155ef6c4152:3612699:97a41d4c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4063/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155ef6c4152:3612699:97a41d4c" data-navigation="inline" data-title="Choose convolution type per regressor in Nipype FSL interface (e.g. dgamma, gamma, none)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155ef6c4152:3612699:97a41d4c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1426 --></div><!-- react-empty: 1427 --><div class="content"><a href="http://neurostars.org/p/4063/" target="_blank" rel="noopener" class="title">Choose convolution type per regressor in Nipype FSL interface (e.g. dgamma, gamma, none)</a><div class="summary"> Hi all, I want to add a baseline (constant) to my model in FSL. It would be possible to do that in the FSL-gui by selecting no convolution for that specific regressor, but I doesn't seem to be possible to choose different convolutions per regressor using NiPype. Instead you can only provide convolution once for all regressors doing something like this: fsl.Level1Design().bases = {'dgamma': {'deri</div></div><span class="ago" title="published:Fri, 15 Jul 2016 14:23:19 GMT -- Received: Fri, 15 Jul 2016 16:38:17 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155ea43b203:2204b92:d45364f0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4057/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155ea43b203:2204b92:d45364f0" data-navigation="inline" data-title="Default value from input not being set with BaseInterface" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155ea43b203:2204b92:d45364f0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1442 --></div><!-- react-empty: 1443 --><div class="content"><a href="http://neurostars.org/p/4057/" target="_blank" rel="noopener" class="title">Default value from input not being set with BaseInterface</a><div class="summary"> With Nipype, I created an interface that inherits BaseInterface that runs some python code. I created an InputSpec class that has an input: threshold = traits.Float(default_value=0.9) I put my python code in the _run_interface method. If i set the threshold prior to running the interface, it works fine, but if I do not set the threshold input, my interface fails because the threshold input is sti</div></div><span class="ago" title="published:Thu, 14 Jul 2016 15:27:39 GMT -- Received: Thu, 14 Jul 2016 16:35:52 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155e326043c:5880fe:d45364f0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4052/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155e326043c:5880fe:d45364f0" data-navigation="inline" data-title="syntax error when trying to run mriqc" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155e326043c:5880fe:d45364f0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1458 --></div><!-- react-empty: 1459 --><div class="content"><a href="http://neurostars.org/p/4052/" target="_blank" rel="noopener" class="title">syntax error when trying to run mriqc</a><div class="summary"> Dear smart people, I am trying to run mriqc on a Neurodebian VM (v8.0.0). I have followed the documentation, but when I run mriqc, using: mriqc -B ~/Data/bids_dataset -o out/ -w work/ I get the following error: Traceback (most recent call last): File "/usr/local/bin/mriqc", line 7, in from mriqc.utils.mriqc_run import main File "/usr/local/lib/python3.4/dist-packages/mriqc-0.8.5a3-py3.4.egg/mriqc</div></div><span class="ago" title="published:Wed, 13 Jul 2016 04:38:59 GMT -- Received: Wed, 13 Jul 2016 07:26:07 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155dc1012df:15631c55:5405e57d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4051/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155dc1012df:15631c55:5405e57d" data-navigation="inline" data-title="Forum: regexp for parse string in rename input node" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155dc1012df:15631c55:5405e57d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1474 --></div><!-- react-empty: 1475 --><div class="content"><a href="http://neurostars.org/p/4051/" target="_blank" rel="noopener" class="title">Forum: regexp for parse string in rename input node</a><div class="summary">Dear all, I have used a merge node to convert all .mgz files to .nii files and reoriented them post ReconAll run. However when I do so the output file names are appended with *_out_reoriented.nii, for example brainmask. mgz with original name is outputted with brainmask_out_reoriented.nii post conversion and reorientation. I want to preserve original file names after removing suffix, in this case </div></div><span class="ago" title="published:Mon, 11 Jul 2016 20:52:54 GMT -- Received: Mon, 11 Jul 2016 22:24:48 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155c77037a8:1111afdc:5405e57d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4049/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155c77037a8:1111afdc:5405e57d" data-navigation="inline" data-title="use-plugin example for mriqc" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155c77037a8:1111afdc:5405e57d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1490 --></div><!-- react-empty: 1491 --><div class="content"><a href="http://neurostars.org/p/4049/" target="_blank" rel="noopener" class="title">use-plugin example for mriqc</a><div class="summary"> Hi, I'm running mriqc on a couple bids datasets, but can't get it to run through our server's resource manager, SLURM. Since nipype is being used, I'm sure this can be done. I see it's loaded as a yaml file, can someone point me to a case where this is used. Also, can you assign plugin-args? </div></div><span class="ago" title="published:Thu, 07 Jul 2016 21:19:25 GMT -- Received: Thu, 07 Jul 2016 22:17:48 GMT">4mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bbd7034f:e57d86b:36858854_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4043/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bbd7034f:e57d86b:36858854" data-navigation="inline" data-title="How to set &quot;IdentityInterface&quot; iterables from a not-hard-coded list" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bbd7034f:e57d86b:36858854" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1506 --></div><!-- react-empty: 1507 --><div class="content"><a href="http://neurostars.org/p/4043/" target="_blank" rel="noopener" class="title">How to set "IdentityInterface" iterables from a not-hard-coded list</a><div class="summary"> Hi, I would like to read-in subject_id's and their scanning sessions via "IdentityInterface" function of nipype. I have already a list for the subject_id's, so iterating over this list is no problem. But, scanning sessions, or sub-directories of subjects don't have the same naming: .../subject_01/ DWId00 FLAIRd00 FLAIRd01 rsd00 T1d00 .../subject_02/ DWId00 FLAIRd05 rsd05 T1d01 Here is the questio</div></div><span class="ago" title="published:Tue, 05 Jul 2016 15:01:02 GMT -- Received: Tue, 05 Jul 2016 16:14:38 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bb322313:e2b08a1:36858854_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4042/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bb322313:e2b08a1:36858854" data-navigation="inline" data-title="Assess significance for SearchLight Analysis" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bb322313:e2b08a1:36858854" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1522 --></div><!-- react-empty: 1523 --><div class="content"><a href="http://neurostars.org/p/4042/" target="_blank" rel="noopener" class="title">Assess significance for SearchLight Analysis</a><div class="summary"> Dear all, we used the Searchlight implementation of Nilearn to run our analysis. Now, we do not know how to prove significance. The first thing we tried is a one sided binomial test for each voxel and a fdr correction for multiple comparisons. However, the binomial test might not be the right approach and the fdr correction may be really severe due to the fact the pvalues are strongly correlated.</div></div><span class="ago" title="published:Tue, 05 Jul 2016 13:08:20 GMT -- Received: Tue, 05 Jul 2016 13:14:33 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bb322313:e2b08a0:36858854_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4041/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bb322313:e2b08a0:36858854" data-navigation="inline" data-title="Forum: MatlabCommand for the optional argument" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155bb322313:e2b08a0:36858854" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1538 --></div><!-- react-empty: 1539 --><div class="content"><a href="http://neurostars.org/p/4041/" target="_blank" rel="noopener" class="title">Forum: MatlabCommand for the optional argument</a><div class="summary"> Hello: I use MatlabCommand to connect the nipype with matlab, it works, but i have a small question, how do you deal with the optional args in matlab, how do you transfer the optional args from nipype interface to matlab, for example, my matlab script has 4 args, which are all numbers, so do I have to create an python array to contain these params, and use some if condition to deal with the matla</div></div><span class="ago" title="published:Tue, 05 Jul 2016 12:03:38 GMT -- Received: Tue, 05 Jul 2016 13:14:33 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155b6afd2b2:ca59137:85ca5f35_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4040/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155b6afd2b2:ca59137:85ca5f35" data-navigation="inline" data-title="Looking for Tailarach template/ Atlas of the Human brain by Mai template" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155b6afd2b2:ca59137:85ca5f35" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1554 --></div><!-- react-empty: 1555 --><div class="content"><a href="http://neurostars.org/p/4040/" target="_blank" rel="noopener" class="title">Looking for Tailarach template/ Atlas of the Human brain by Mai template</a><div class="summary"> Dear all, I want to coregister my anatomical images with a template corresponding to the space of the Atlas of the human brain by Mai (http://www.thehumanbrain.info/index.php). It seems to be quiet similar to the Tailarach space, but there is some difference concerning the alignment of anterior and posterior commissure. Could anyone please point me to such a template (as nii-file) or to a templat</div></div><span class="ago" title="published:Mon, 04 Jul 2016 15:46:26 GMT -- Received: Mon, 04 Jul 2016 16:13:43 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559d8d75a1:7a56144:36858854_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4035/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559d8d75a1:7a56144:36858854" data-navigation="inline" data-title="easythresh for multiple comparison correction?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559d8d75a1:7a56144:36858854" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1570 --></div><!-- react-empty: 1571 --><div class="content"><a href="http://neurostars.org/p/4035/" target="_blank" rel="noopener" class="title">easythresh for multiple comparison correction?</a><div class="summary"> Hey, just a short question: can I use easythresh on the flameo output for multiple comparison correction? If no, what are the best ways to do it? Thanks! Lisa </div></div><span class="ago" title="published:Wed, 29 Jun 2016 18:17:40 GMT -- Received: Wed, 29 Jun 2016 19:05:41 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559d8d75a1:7a56143:36858854_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4033/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559d8d75a1:7a56143:36858854" data-navigation="inline" data-title="Forum: wrap Matlab script using MatlabCommand in Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559d8d75a1:7a56143:36858854" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1586 --></div><!-- react-empty: 1587 --><div class="content"><a href="http://neurostars.org/p/4033/" target="_blank" rel="noopener" class="title">Forum: wrap Matlab script using MatlabCommand in Nipype</a><div class="summary">Bonjour: Hi everyone, I wrap a matlab script, in the matlab function, I didnt define the output, but I will save some output images to a specific folder, and I use the MatlabCommand in Nipype, so here is my NIpype code: from nipype.interfaces.utility import Function import nipype.pipeline.engine as pe def runmatlab(ContrastLinearModel, Format, CSVFilenames, PATH_TO_RECON_ALL_OUTPUTS, a_required_pa</div></div><span class="ago" title="published:Wed, 29 Jun 2016 17:38:20 GMT -- Received: Wed, 29 Jun 2016 19:05:41 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559b9e039b:7202825:5405e57d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4026/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559b9e039b:7202825:5405e57d" data-navigation="inline" data-title="nibabel header get_zooms" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1559b9e039b:7202825:5405e57d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1602 --></div><!-- react-empty: 1603 --><div class="content"><a href="http://neurostars.org/p/4026/" target="_blank" rel="noopener" class="title">nibabel header get_zooms</a><div class="summary"> Hi, I am using the nibabel function get_zooms to read the voxel dimensions from the header of my image. I got (3.0, 3.0, 7.5000005, 2.5) while both fslhd and spm display give me 7.5 for the third dimension. Is it a matter of rounding ? Thanks a lot ! </div></div><span class="ago" title="published:Wed, 29 Jun 2016 09:00:35 GMT -- Received: Wed, 29 Jun 2016 10:04:32 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15597bf2552:634c74b:5405e57d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4023/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15597bf2552:634c74b:5405e57d" data-navigation="inline" data-title="Tutorial: wrap a matlab script for Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15597bf2552:634c74b:5405e57d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1618 --></div><!-- react-empty: 1619 --><div class="content"><a href="http://neurostars.org/p/4023/" target="_blank" rel="noopener" class="title">Tutorial: wrap a matlab script for Nipype</a><div class="summary"> Hello: I am new to nipype, and recently, I want to wrap a matlab script for Nipype, but in your website, the example is not so specific,and I checked out all the posts that you have talked about, it is not so clear, whatever for the command line or to create the matlab interface by ourself, can you just make a good tutorial for us? in fact, the tutorial in the nipype website is not so well-struct</div></div><span class="ago" title="published:Tue, 28 Jun 2016 15:35:26 GMT -- Received: Tue, 28 Jun 2016 16:02:14 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155952ad6b0:581d17d:36858854_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4021/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155952ad6b0:581d17d:36858854" data-navigation="inline" data-title="using nibabel to produce similar results to fslmaths -add" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155952ad6b0:581d17d:36858854" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1634 --></div><!-- react-empty: 1635 --><div class="content"><a href="http://neurostars.org/p/4021/" target="_blank" rel="noopener" class="title">using nibabel to produce similar results to fslmaths -add</a><div class="summary"> If I have x number of masks that I'd like to combine into one mask, how would I do this in nibabel? using fslmaths, I'd do: `fslmaths img1.nii.gz -add img2.nii.gz outimg.nii.gz` in nibabel i've tried: nb.concat_images([img1.nii.gz, img2.nii.gz]) and def add(*images): to_add = [] for i in images: img = nb.load(i) data = img.get_data() affine = img.get_affine() to_add.append((data, affine)) data_su</div></div><span class="ago" title="published:Tue, 28 Jun 2016 01:12:04 GMT -- Received: Tue, 28 Jun 2016 04:01:00 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1558469548e:225dac2:36858854_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4019/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1558469548e:225dac2:36858854" data-navigation="inline" data-title="ReconAll error freesurfer" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1558469548e:225dac2:36858854" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1650 --></div><!-- react-empty: 1651 --><div class="content"><a href="http://neurostars.org/p/4019/" target="_blank" rel="noopener" class="title">ReconAll error freesurfer</a><div class="summary">Hi: I run the reconAlll command with two nii file, and I got some error here, can you help me to check it out, cuz I think I use the command correctly... here is the info for the error I think the problem is for the mri_convert command, I checked thr orig folder, there is no T1.mgz file! can you tell me why? File: crash-20160624-204118-guest-_recon_all0.pklz Node: _recon_all0 Working directory: /t</div></div><span class="ago" title="published:Fri, 24 Jun 2016 19:24:54 GMT -- Received: Fri, 24 Jun 2016 21:55:43 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155831f631a:1d5e84f:5405e57d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4018/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155831f631a:1d5e84f:5405e57d" data-navigation="inline" data-title="RETROICOR and nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155831f631a:1d5e84f:5405e57d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1666 --></div><!-- react-empty: 1667 --><div class="content"><a href="http://neurostars.org/p/4018/" target="_blank" rel="noopener" class="title">RETROICOR and nipype</a><div class="summary"> Hi all, is there a RETROICOR implementation/wrapper in nipype? There is an afni implementation (https://afni.nimh.nih.gov/pub/dist/doc/program_help/3dretroicor.html) and also an FSL implemenation (http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/PNM). Regards, Glad </div></div><span class="ago" title="published:Fri, 24 Jun 2016 13:10:53 GMT -- Received: Fri, 24 Jun 2016 15:55:20 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1557f3e998f:efe393:5405e57d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4017/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1557f3e998f:efe393:5405e57d" data-navigation="inline" data-title="Job: Postdoctoral Fellowship in Machine Learning" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1557f3e998f:efe393:5405e57d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1682 --></div><!-- react-empty: 1683 --><div class="content"><a href="http://neurostars.org/p/4017/" target="_blank" rel="noopener" class="title">Job: Postdoctoral Fellowship in Machine Learning</a><div class="summary">Description: A postdoctoral position in medical image analysis is available at the Department of Psychiatry &amp; Behavioral Sciences, Stanford University. The position will focus on creating machine learning technology for extracting image biomarkers from multi-modal, longitudinal MR images in combination with demographic and clinical data collected across multiple centers. The research projects are </div></div><span class="ago" title="published:Thu, 23 Jun 2016 21:27:34 GMT -- Received: Thu, 23 Jun 2016 21:50:56 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1557e99ac34:c809b3:5405e57d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4016/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1557e99ac34:c809b3:5405e57d" data-navigation="inline" data-title="Job: Biomedical Engineer Position at SRI - Stanford Lab" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1557e99ac34:c809b3:5405e57d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1698 --></div><!-- react-empty: 1699 --><div class="content"><a href="http://neurostars.org/p/4016/" target="_blank" rel="noopener" class="title">Job: Biomedical Engineer Position at SRI - Stanford Lab</a><div class="summary">SRI’s Center for Health Sciences Neuroscience Program is seeking a Biomedical Engineer who will provide on-site technical support for the processing and analysis of MR images, which includes quality control, analysis work flow management, and develop new MRI software tools under the supervision of research scientists in the lab. The applicant must have working experience in Linux and strong comput</div></div><span class="ago" title="published:Thu, 23 Jun 2016 18:48:21 GMT -- Received: Thu, 23 Jun 2016 18:50:48 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1556952fe70:406ebad:c3f7604e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4013/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1556952fe70:406ebad:c3f7604e" data-navigation="inline" data-title="mkdir permission denied in matlab via nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1556952fe70:406ebad:c3f7604e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1714 --></div><!-- react-empty: 1715 --><div class="content"><a href="http://neurostars.org/p/4013/" target="_blank" rel="noopener" class="title">mkdir permission denied in matlab via nipype</a><div class="summary"> Matlab script, in the course of running, uses mkdir. When run locally, no problem. When run remotely on an interactive matlab session on the cluster, it fails. When run via a nipype call locally, it works. When run via a nipype call on the cluster, it fails. When re-running the crashfile for that nipype node on the cluster, it works?! The fail error is within matlab. Matlab throws a "permission d</div></div><span class="ago" title="published:Sun, 19 Jun 2016 13:49:14 GMT -- Received: Sun, 19 Jun 2016 15:41:34 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1555a7f4e50:f561f2:c3f7604e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4011/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1555a7f4e50:f561f2:c3f7604e" data-navigation="inline" data-title="make a freesurfer .gca file" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1555a7f4e50:f561f2:c3f7604e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1730 --></div><!-- react-empty: 1731 --><div class="content"><a href="http://neurostars.org/p/4011/" target="_blank" rel="noopener" class="title">make a freesurfer .gca file</a><div class="summary"> Hello all, I'm using some routine from freesurfer calling a gca file. Now I created a .nii file and I'd like to saved it as .gca. Seems like mri_convert doesn't do that !? any idea how else I could convert to that format thanks Cyril </div></div><span class="ago" title="published:Thu, 16 Jun 2016 15:44:46 GMT -- Received: Thu, 16 Jun 2016 18:35:40 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15550298642:5686414:818adeed_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4004/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15550298642:5686414:818adeed" data-navigation="inline" data-title="fslmaths -Tstd - standard deviation in timeseries" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15550298642:5686414:818adeed" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1746 --></div><!-- react-empty: 1747 --><div class="content"><a href="http://neurostars.org/p/4004/" target="_blank" rel="noopener" class="title">fslmaths -Tstd - standard deviation in timeseries</a><div class="summary"> Hi, this is my bashscript. (it gives me the standard deviation in a voxel over a timeseries) fslmaths resting_timeseries.nii -Tstd StD_lowfreq.nii In interfaces.fsl.maths I can't find the right module/class to integrate this into a nipype pipeline. If this fslmaths-operation exists, is there a better way for me to search it? (other than searching for the flag?) If it doesn't exist, is there any d</div></div><span class="ago" title="published:Tue, 14 Jun 2016 16:13:18 GMT -- Received: Tue, 14 Jun 2016 18:25:46 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1554ba65443:44d5ef2:47e2180a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/4002/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1554ba65443:44d5ef2:47e2180a" data-navigation="inline" data-title="Dipy ValueError: streamlines' points are outside the target mask" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1554ba65443:44d5ef2:47e2180a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1762 --></div><!-- react-empty: 1763 --><div class="content"><a href="http://neurostars.org/p/4002/" target="_blank" rel="noopener" class="title">Dipy ValueError: streamlines' points are outside the target mask</a><div class="summary"> Hello, I am having issues with writing a Dipy script to filter streamlines that pass through a frontal lobe ROI. When I use utils from dipy.tracking on the streamlines I get an error stating that the streamlines are outside the target mask area. I know for a fact that the streamline file has normal tractography and the mask, though not completely white matter, is registered roughly to the area of</div></div><span class="ago" title="published:Mon, 13 Jun 2016 19:34:30 GMT -- Received: Mon, 13 Jun 2016 21:23:59 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155370649e9:3b38acb:818adeed_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3994/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155370649e9:3b38acb:818adeed" data-navigation="inline" data-title="Best way to publish an open PET/MR dataset" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155370649e9:3b38acb:818adeed" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1778 --></div><!-- react-empty: 1779 --><div class="content"><a href="http://neurostars.org/p/3994/" target="_blank" rel="noopener" class="title">Best way to publish an open PET/MR dataset</a><div class="summary"> Hi, We are preparing a paper for submission with a human PET/MR observational study. We'd like to publish the analysis and the data openly (probably under MIT and CC-BY licenses). For the code we will publish a repo in Github with a couple of Jupyter notebooks, but for the data I'm not sure what to do and would like to ask for your advice and common practices in the field. The data is ~50 nifti f</div></div><span class="ago" title="published:Thu, 09 Jun 2016 20:27:11 GMT -- Received: Thu, 09 Jun 2016 21:16:46 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15535bc118d:3487829:3821b62d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3993/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15535bc118d:3487829:3821b62d" data-navigation="inline" data-title="Forum: Need help with extractskeleton in nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15535bc118d:3487829:3821b62d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1794 --></div><!-- react-empty: 1795 --><div class="content"><a href="http://neurostars.org/p/3993/" target="_blank" rel="noopener" class="title">Forum: Need help with extractskeleton in nipype</a><div class="summary"> Hi, I would like to use nipype in my python program to skeletonize a simple binary image. I would like to use pruning such that I end up with a single, unbranched skeleton line. I am relatively new to python and I got stuck how to call the extractskeleton function. can anyone provide a simple example line of code how to import the correct function and call it on a dummy binary image? </div></div><span class="ago" title="published:Thu, 09 Jun 2016 15:14:25 GMT -- Received: Thu, 09 Jun 2016 15:16:05 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15527859dec:792c99c:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3988/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15527859dec:792c99c:9f72fccb" data-navigation="inline" data-title="FSL Nipype to FSL GUI group analysis" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15527859dec:792c99c:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1810 --></div><!-- react-empty: 1811 --><div class="content"><a href="http://neurostars.org/p/3988/" target="_blank" rel="noopener" class="title">FSL Nipype to FSL GUI group analysis</a><div class="summary"> I've successfully made an FSL preprocessing pipeline in Nipype. I've gotten the sense that there isn't that much support for group-level analyses using FSL in Nipype right now, so I wanted to at least initially get it working moving from Nipype to the FSL gui. When I try to run a 'Higher-level analysis' on the cope files output by Flame from my analysis across runs, I get the following error: "Re</div></div><span class="ago" title="published:Mon, 06 Jun 2016 19:10:28 GMT -- Received: Mon, 06 Jun 2016 21:01:56 GMT">5mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155176aef90:4584751:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3984/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155176aef90:4584751:9f72fccb" data-navigation="inline" data-title="processing stream in open fmri da0108" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_155176aef90:4584751:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1826 --></div><!-- react-empty: 1827 --><div class="content"><a href="http://neurostars.org/p/3984/" target="_blank" rel="noopener" class="title">processing stream in open fmri da0108</a><div class="summary"> Hi I'm working on fMRI data base named emotion regulation task and I have a question. are they the raw data? in other words you mentioned that some steps of preprocessing were implemented. now my question is should I preprocess the data from first stage or not? actually the number of volumes in 6runs of one subject are different, how can I enter them to FSL? </div></div><span class="ago" title="published:Fri, 03 Jun 2016 15:07:45 GMT -- Received: Fri, 03 Jun 2016 17:58:52 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1550c761a7a:1afa72b:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3979/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1550c761a7a:1afa72b:9f72fccb" data-navigation="inline" data-title="How do I get PMIDs associated with datasets in OpenfMRI?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1550c761a7a:1afa72b:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1842 --></div><!-- react-empty: 1843 --><div class="content"><a href="http://neurostars.org/p/3979/" target="_blank" rel="noopener" class="title">How do I get PMIDs associated with datasets in OpenfMRI?</a><div class="summary"> I have entered some of the scientific papers associated with OpenfMRI datasets in Wikidata [1]. However, I believe I am missing a few here and there. How do I most easily get papers associated with OpenfMRI datasets programmatically. Would I need to write a scraper starting at https://openfmri.org/dataset/ or is there a (secret) api or downloadable metadata dataset? [1] https://www.wikidata.org/w</div></div><span class="ago" title="published:Wed, 01 Jun 2016 14:26:54 GMT -- Received: Wed, 01 Jun 2016 14:55:14 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15506a92dce:35f22d:fa2e1b2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3975/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15506a92dce:35f22d:fa2e1b2" data-navigation="inline" data-title="Trying to understand nipype.RapidArt.outputs.statistics_files." data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15506a92dce:35f22d:fa2e1b2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1858 --></div><!-- react-empty: 1859 --><div class="content"><a href="http://neurostars.org/p/3975/" target="_blank" rel="noopener" class="title">Trying to understand nipype.RapidArt.outputs.statistics_files.</a><div class="summary"> Hi, My boss is used to the DPARSF motion statistics report, where it gives a boundary of movement for each fMRI acquisition series. For instance, it would say: 'this subject didn't move more than 3mm'. He uses that to exclude subjects with too much movement. I am now using RapidArt to detect outliers and calculate motion statistics and I am trying to understand the `statistic_files` output. I am </div></div><span class="ago" title="published:Tue, 31 May 2016 09:09:44 GMT -- Received: Tue, 31 May 2016 11:53:18 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15502cb0193:fe9bd56:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3965/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15502cb0193:fe9bd56:9f72fccb" data-navigation="inline" data-title="nipype OverflowError: size does not fit in an int Error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15502cb0193:fe9bd56:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1874 --></div><!-- react-empty: 1875 --><div class="content"><a href="http://neurostars.org/p/3965/" target="_blank" rel="noopener" class="title">nipype OverflowError: size does not fit in an int Error</a><div class="summary">I get the following error when trying to run a script. The error seems to be specific to nipype as when I remove the nipype functions and run the script to return the output everything works fine. I've tried adding `overwrite=True` to the nodes that connect to the failing node, as well to the failing node itself as well to the node that the failing node connects to. I've also tried deleting the wo</div></div><span class="ago" title="published:Mon, 30 May 2016 16:40:24 GMT -- Received: Mon, 30 May 2016 17:51:46 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15500db3691:f6d2fa5:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3963/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15500db3691:f6d2fa5:9f72fccb" data-navigation="inline" data-title="How do I make a gray matter mask?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15500db3691:f6d2fa5:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1890 --></div><!-- react-empty: 1891 --><div class="content"><a href="http://neurostars.org/p/3963/" target="_blank" rel="noopener" class="title">How do I make a gray matter mask?</a><div class="summary"> Is there any way to create a gray matter mask from anatomicals and apply it to functional data? I am running RSA on my data, and I'm curious what my results would look like if I only include voxels within gray matter. This function is the closest I could find in the Nipy documentation, but it would be helpful to see it in use. Thanks! </div></div><span class="ago" title="published:Mon, 30 May 2016 07:02:34 GMT -- Received: Mon, 30 May 2016 08:50:14 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ff912b63:f2560b7:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3961/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ff912b63:f2560b7:9f72fccb" data-navigation="inline" data-title="Perform affine-only registration but allowing for voxel anisotropy" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ff912b63:f2560b7:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1906 --></div><!-- react-empty: 1907 --><div class="content"><a href="http://neurostars.org/p/3961/" target="_blank" rel="noopener" class="title">Perform affine-only registration but allowing for voxel anisotropy</a><div class="summary"> I have a brain region atlas that I would want to align to my template. The brain region atlas has a very low rostrocaudal resolution, but save for that it fits the template quite nicely. I would like to avoid nonlinear transformations, as that could compromise symmetry, and lead to strangely formed blobs due to the low atlas resolution. I have tried editing the affines manually, but I guess the c</div></div><span class="ago" title="published:Mon, 30 May 2016 02:05:10 GMT -- Received: Mon, 30 May 2016 02:49:44 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154f3f76915:cfa0766:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3957/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154f3f76915:cfa0766:9f72fccb" data-navigation="inline" data-title="QC tools/packages for fMRI data" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154f3f76915:cfa0766:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1922 --></div><!-- react-empty: 1923 --><div class="content"><a href="http://neurostars.org/p/3957/" target="_blank" rel="noopener" class="title">QC tools/packages for fMRI data</a><div class="summary"> hello All, We have data organized in non- BIDS standard. Our data is generally organized in following way: sub-01 -Run1 -- img1.dcm -- img2.dcm -Run 2 -- img1.dcm -- img2.dcm As we are not following bids format for storing our data, we wanted to check if there are tools that can help us get quality check metrics as descriptive in mriqc . or are there any ways in which we can transform/curate our </div></div><span class="ago" title="published:Fri, 27 May 2016 18:05:32 GMT -- Received: Fri, 27 May 2016 20:45:58 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ef733ede:be40c15:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3955/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ef733ede:be40c15:9f72fccb" data-navigation="inline" data-title="Sharper or blurrier templates for registration?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ef733ede:be40c15:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1938 --></div><!-- react-empty: 1939 --><div class="content"><a href="http://neurostars.org/p/3955/" target="_blank" rel="noopener" class="title">Sharper or blurrier templates for registration?</a><div class="summary"> I have two templates at virtually the same resolution - one of which is blurry, and one of which is rather sharp. You can see them here . The difference arose from the way they were subsampled (the sharp one was done in one go with `flirt` and the blurry one with stepwise usage of `fslmaths -subsamp2` and `flirt` for the last step) - judgd purely on thsi info I would have said that the sharper re</div></div><span class="ago" title="published:Thu, 26 May 2016 23:32:06 GMT -- Received: Thu, 26 May 2016 23:43:07 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e66bda97:9b63c4b:fa2e1b2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3952/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e66bda97:9b63c4b:fa2e1b2" data-navigation="inline" data-title="Error with SPM model specification" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e66bda97:9b63c4b:fa2e1b2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1954 --></div><!-- react-empty: 1955 --><div class="content"><a href="http://neurostars.org/p/3952/" target="_blank" rel="noopener" class="title">Error with SPM model specification</a><div class="summary">New Nipype user here! I'm trying to define a 1st-level GLM on SPM, but my design matrix doesn't look right. In the task, a participant saw images from 10 different categories (houses, faces, bodies, etc.) over 10 runs. My design matrix should have 170 columns total: (10 conditions + 6 motion regressors) * 10 runs + 10 constants. Instead, my design matrix has 116 columns: 10 conditions * 10 runs + </div></div><span class="ago" title="published:Wed, 25 May 2016 04:25:13 GMT -- Received: Wed, 25 May 2016 05:38:28 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e333a83f:8ec3142:fa2e1b2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3946/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e333a83f:8ec3142:fa2e1b2" data-navigation="inline" data-title="FSL film_gls gives zeros for input, numTS=0" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e333a83f:8ec3142:fa2e1b2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1970 --></div><!-- react-empty: 1971 --><div class="content"><a href="http://neurostars.org/p/3946/" target="_blank" rel="noopener" class="title">FSL film_gls gives zeros for input, numTS=0</a><div class="summary">I'm setting up a nipype pipeline using FSL. The full pipeline will run, but all of the results are files with nothing but zeros. I've traced this back to the film_gls step - the input looks (to me) fine, it clearly has values. The output is all zeros. This is the terminal output when I run it: stdout 2016-05-23T13:47:56.583664:Log directory is: results stdout 2016-05-23T13:48:09.288887:numTS=0 std</div></div><span class="ago" title="published:Tue, 24 May 2016 14:12:43 GMT -- Received: Tue, 24 May 2016 14:38:14 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e333a83f:8ec3141:fa2e1b2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3944/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e333a83f:8ec3141:fa2e1b2" data-navigation="inline" data-title="dynamic nipype node memory" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154e333a83f:8ec3141:fa2e1b2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 1986 --></div><!-- react-empty: 1987 --><div class="content"><a href="http://neurostars.org/p/3944/" target="_blank" rel="noopener" class="title">dynamic nipype node memory</a><div class="summary"> In a parallel nipype stream, I have a single node being fed datasets of different sizes. E.g node_extract_feature sometimes operates on an input that is 1GB and sometimes 1 MB. However, I have to allocate memory for the node according to its max expected dataset size. Any way to do this dynamically? </div></div><span class="ago" title="published:Tue, 24 May 2016 12:04:21 GMT -- Received: Tue, 24 May 2016 14:38:14 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154df55be8a:7fd5e6f:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3942/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154df55be8a:7fd5e6f:9f72fccb" data-navigation="inline" data-title="[recommendation] note versions in Release history files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154df55be8a:7fd5e6f:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2002 --></div><!-- react-empty: 2003 --><div class="content"><a href="http://neurostars.org/p/3942/" target="_blank" rel="noopener" class="title">[recommendation] note versions in Release history files</a><div class="summary"> ATM Release history file on openfmri list entries based on date, while data files released with Revision numbers. So it would be nice if those followed the same convention as usually done for software, by grouping and listing changes (with dates ;) ) as associated with specific releases of the datasets, so no manual matching would be necessary. </div></div><span class="ago" title="published:Mon, 23 May 2016 20:35:28 GMT -- Received: Mon, 23 May 2016 20:36:59 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154d46232d1:5c351f1:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3939/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154d46232d1:5c351f1:9f72fccb" data-navigation="inline" data-title="What does &quot;task&quot; and &quot;condition&quot; mean in OpenfMRI ds052" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154d46232d1:5c351f1:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2018 --></div><!-- react-empty: 2019 --><div class="content"><a href="http://neurostars.org/p/3939/" target="_blank" rel="noopener" class="title">What does "task" and "condition" mean in OpenfMRI ds052</a><div class="summary"> In ds052 from OpenfMRI, the data is from the experiment 2 in the essay. There seems to be two kinds of condition, named positive and negative, in one run. Does positive mean experiment manipulation and negative mean control? Also there are weather prediction task and reversal weather prediction task. What does "reversal" mean? And I found that all the scan results extracted from one run is arrang</div></div><span class="ago" title="published:Sat, 21 May 2016 16:29:51 GMT -- Received: Sat, 21 May 2016 17:34:45 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154cf3acdc7:4c30c00:fa2e1b2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3929/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154cf3acdc7:4c30c00:fa2e1b2" data-navigation="inline" data-title="fMRI motion statistics" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154cf3acdc7:4c30c00:fa2e1b2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2034 --></div><!-- react-empty: 2035 --><div class="content"><a href="http://neurostars.org/p/3929/" target="_blank" rel="noopener" class="title">fMRI motion statistics</a><div class="summary"> Hi, I am looking for a Nipype workflow to calculate head motion statistics. I know Rapidart can do it, in fact I am using it for motion and nuisance correction like here: https://github.com/gablab/openfmri/blob/master/subject_level/fmri_ants_openfmri.py Otherwise I am also checking what C-PAC does: https://github.com/FCP-INDI/C-PAC/blob/master/CPAC/generate_motion_statistics/generate_motion_stati</div></div><span class="ago" title="published:Fri, 20 May 2016 14:53:15 GMT -- Received: Fri, 20 May 2016 17:33:38 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154cf3acdc7:4c30bff:fa2e1b2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3927/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154cf3acdc7:4c30bff:fa2e1b2" data-navigation="inline" data-title="Reference for nitime and rapidart" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154cf3acdc7:4c30bff:fa2e1b2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2050 --></div><!-- react-empty: 2051 --><div class="content"><a href="http://neurostars.org/p/3927/" target="_blank" rel="noopener" class="title">Reference for nitime and rapidart</a><div class="summary"> Hi! I was wondering how to properly cite: * The rapidart interface in Nipype (enough to cite Nipype?) * Nitime Thanks! Julia </div></div><span class="ago" title="published:Fri, 20 May 2016 14:38:00 GMT -- Received: Fri, 20 May 2016 17:33:38 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154c8c9b82d:332aa41:9f72fccb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3921/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154c8c9b82d:332aa41:9f72fccb" data-navigation="inline" data-title="Group ICA and spatial normalization" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154c8c9b82d:332aa41:9f72fccb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2107 --></div><!-- react-empty: 2108 --><div class="content"><a href="http://neurostars.org/p/3921/" target="_blank" rel="noopener" class="title">Group ICA and spatial normalization</a><div class="summary"> Hi, I am working on a pipeline for rs-fMRI preprocessing. To apply group ICA I need to spatially normalize all fMRI images to a common template. At what point in the preprocessing do you usually apply this warping? After temporal filtering? After smoothing? Thank you! Best, Alex </div></div><span class="ago" title="published:Thu, 19 May 2016 10:16:42 GMT -- Received: Thu, 19 May 2016 11:32:23 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154c3a0c09c:1eb5865:97af7a9a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3907/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154c3a0c09c:1eb5865:97af7a9a" data-navigation="inline" data-title="Blur (edges of) NIfTI template" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154c3a0c09c:1eb5865:97af7a9a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2123 --></div><!-- react-empty: 2124 --><div class="content"><a href="http://neurostars.org/p/3907/" target="_blank" rel="noopener" class="title">Blur (edges of) NIfTI template</a><div class="summary"> I am using a high-res NIfTI template for my plots. I prefer using white backgrounds in all of my plots. The original template (from the Australian Mouse Brain Mapping Consortium) - has a "black" background - with an actual value of 191919.8. I have thresholded the image at 191920, and obtained a template I can use with a white background, as seen below. The edges of the template in this figure lo</div></div><span class="ago" title="published:Wed, 18 May 2016 10:52:04 GMT -- Received: Wed, 18 May 2016 11:29:32 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154bbe41f41:27d48d:fa2e1b2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3905/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154bbe41f41:27d48d:fa2e1b2" data-navigation="inline" data-title="AttributeError after moving from nipype v 0.9 to 0.11" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154bbe41f41:27d48d:fa2e1b2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2139 --></div><!-- react-empty: 2140 --><div class="content"><a href="http://neurostars.org/p/3905/" target="_blank" rel="noopener" class="title">AttributeError after moving from nipype v 0.9 to 0.11</a><div class="summary"> I had been running my pipeline using nipype 0.9 and it ran without errors. I recently updated to 0.11.0.1, and now when I run I get the following error. Any idea what changed between the two versions that might cause this? 160516-18:23:14,943 workflow INFO: Executing node datasource.b0 in dir: /ncf/gershman/Lab/ObserveBet/Output/firstlevel/_subject_id_160505_OBSERVEBET_10/datasource Traceback (mo</div></div><span class="ago" title="published:Mon, 16 May 2016 22:29:37 GMT -- Received: Mon, 16 May 2016 23:26:10 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154bbe41f41:27d48c:fa2e1b2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3904/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154bbe41f41:27d48c:fa2e1b2" data-navigation="inline" data-title="Error specifying inputs for SPM realign?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154bbe41f41:27d48c:fa2e1b2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2155 --></div><!-- react-empty: 2156 --><div class="content"><a href="http://neurostars.org/p/3904/" target="_blank" rel="noopener" class="title">Error specifying inputs for SPM realign?</a><div class="summary">Hi everyone, This is my first time analyzing anything on Nipype, and I'm finding the syntax for workflows pretty counterintuitive. For now, I'm trying to run a really simple workflow with three steps: # Reorient to standard orientation reorient = pe.MapNode(interface = fsl.Reorient2Std(), name = 'reorient', iterfield = ['in_file']) reorient.inputs.in_file = bold_files # Clip first four frames fslr</div></div><span class="ago" title="published:Mon, 16 May 2016 20:50:11 GMT -- Received: Mon, 16 May 2016 23:26:10 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ad12d29f:12f5910:3821b62d_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3899/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ad12d29f:12f5910:3821b62d" data-navigation="inline" data-title="labels for fetch_atlas_basc_multiscale_2015" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154ad12d29f:12f5910:3821b62d" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2171 --></div><!-- react-empty: 2172 --><div class="content"><a href="http://neurostars.org/p/3899/" target="_blank" rel="noopener" class="title">labels for fetch_atlas_basc_multiscale_2015</a><div class="summary"> Hello, I was wondering if labels (anatomical description, possibly even network / large-scale system affiliations) exist for the newly added BASC multiscale functional brain parcellation atlas? Basically, I am looking for an equivalent of what is provided for AAL, HO, or Power2011. Assuming that is not the case, does anyone have suggestions on how to do that (in python)? I thought I may perhaps u</div></div><span class="ago" title="published:Sat, 14 May 2016 02:10:53 GMT -- Received: Sat, 14 May 2016 02:22:52 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1549c50e64a:527db28:377d4e46_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3878/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1549c50e64a:527db28:377d4e46" data-navigation="inline" data-title="Using event files to specify design for feat" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1549c50e64a:527db28:377d4e46" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2187 --></div><!-- react-empty: 2188 --><div class="content"><a href="http://neurostars.org/p/3878/" target="_blank" rel="noopener" class="title">Using event files to specify design for feat</a><div class="summary"> I am trying to build an FSL pipeline using nipype, and am stuck at specifying my level 1 analysis model. There is a mention of using event files to generate your glm model in the model gen help page here: http://nipy.org/nipype/interfaces/generated/nipype.algorithms.modelgen.html "Alternatively, you can provide information through event files. The event files have to be in 1, 2 or 3 column format</div></div><span class="ago" title="published:Tue, 10 May 2016 20:09:57 GMT -- Received: Tue, 10 May 2016 20:17:08 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1549b073477:500cda4:7461665c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3875/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1549b073477:500cda4:7461665c" data-navigation="inline" data-title="DataGraber, find .dcm in multiple folders" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1549b073477:500cda4:7461665c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2203 --></div><!-- react-empty: 2204 --><div class="content"><a href="http://neurostars.org/p/3875/" target="_blank" rel="noopener" class="title">DataGraber, find .dcm in multiple folders</a><div class="summary">Hi guys, I'm processing ADNI data using freesurfer, specifically recon-all. ADNI is organised in the following way: /Users/Raquel/Desktop/DATA/ADNI_DATA | -- 006_S_4150 | | -- MPRAGE | | | -- 2011-08-08_14_49_20.0 | | | | -- S117925 &#8203;| | | | | -- ADNI_006_S_4150_MR_MPRAGE_br_raw_20110809103032948_1_S117925_I249403.dcm &#8203;| | | | | -- ... | | | -- 2011-11-07_10_30_32.0 | | | | -- S128309 &#8203;| | | | | -</div></div><span class="ago" title="published:Tue, 10 May 2016 11:11:35 GMT -- Received: Tue, 10 May 2016 14:17:01 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1548f65412d:268b209:c3f7604e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3873/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1548f65412d:268b209:c3f7604e" data-navigation="inline" data-title="Changing GIFTI metadata in nibabel" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1548f65412d:268b209:c3f7604e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2219 --></div><!-- react-empty: 2220 --><div class="content"><a href="http://neurostars.org/p/3873/" target="_blank" rel="noopener" class="title">Changing GIFTI metadata in nibabel</a><div class="summary">Hello, I have been trying to put together a GIFTI 'metric' file with multiple overlays. The goal is to visualize some maps that were created in Python using Connectome Workbench. The approach is to get some example data (myelin maps from Conte atlas), clear the data array, add my own maps, change the metadata of each map (especially the name) and then save the files. I use the following code: # 1.</div></div><span class="ago" title="published:Sun, 08 May 2016 05:43:37 GMT -- Received: Sun, 08 May 2016 08:04:18 GMT">6mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15481377822:c716e5:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3870/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15481377822:c716e5:7bd7a919" data-navigation="inline" data-title="open fMRI dataset" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15481377822:c716e5:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2235 --></div><!-- react-empty: 2236 --><div class="content"><a href="http://neurostars.org/p/3870/" target="_blank" rel="noopener" class="title">open fMRI dataset</a><div class="summary"> Hi again I have a question about data set with ds000108 number, ' prefrontal subcortical pathways'. what's wrong with onsets? for example , 1 niftii file has 192 volumes with TR=2s ,whereas the onset vector is included 1329s ! I think there is "1" as an extra number in all the onsets vectors. can you chek them please??? I'm confused really </div></div><span class="ago" title="published:Thu, 05 May 2016 12:42:40 GMT -- Received: Thu, 05 May 2016 13:59:36 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15462ec89a2:3d77fc4:8d8172b5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3865/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15462ec89a2:3d77fc4:8d8172b5" data-navigation="inline" data-title="Nipype v.012.0 error in tutorial file smri_fsreconall.py &amp; stopped working for other files that were fine before the update" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15462ec89a2:3d77fc4:8d8172b5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2251 --></div><!-- react-empty: 2252 --><div class="content"><a href="http://neurostars.org/p/3865/" target="_blank" rel="noopener" class="title">Nipype v.012.0 error in tutorial file smri_fsreconall.py &amp; stopped working for other files that were fine before the update</a><div class="summary">Hey guys, I've been struggling with Nipype and was wondering if you could help me. I've created a script that gets my dcm files and processes each of them using Freesurfer 5.3, one image at a time, since I'm using ADNI data, I have an huge amount of dcm to process and it's taking a lot of time. So I've started reading about parallel processing and Nipype mapnodes and iterables and ended up decidin</div></div><span class="ago" title="published:Fri, 29 Apr 2016 16:46:36 GMT -- Received: Fri, 29 Apr 2016 16:49:09 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15462479d53:3acec47:8d8172b5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3864/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15462479d53:3acec47:8d8172b5" data-navigation="inline" data-title="Nipype JoinNode doesn't work as expected (minimal example included)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15462479d53:3acec47:8d8172b5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2267 --></div><!-- react-empty: 2268 --><div class="content"><a href="http://neurostars.org/p/3864/" target="_blank" rel="noopener" class="title">Nipype JoinNode doesn't work as expected (minimal example included)</a><div class="summary">Dear Nipype community, I gave the JoinNode a try and had a closer look at the tutorial . But it turns out that the JoinNode doesn't work for me as expected. Please have a look at this figure: According to the tutorial the JoinNode E joins all of the iterations caused by B and D, more precisely it just joins [D13,D14,D25,D26]. However, in my minimal implementation (see below) made from the tutorial</div></div><span class="ago" title="published:Fri, 29 Apr 2016 11:32:12 GMT -- Received: Fri, 29 Apr 2016 13:49:01 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15454b56f6f:6e0c28:8d8172b5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3861/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15454b56f6f:6e0c28:8d8172b5" data-navigation="inline" data-title="mapnode output to datasink, container specific files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15454b56f6f:6e0c28:8d8172b5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2283 --></div><!-- react-empty: 2284 --><div class="content"><a href="http://neurostars.org/p/3861/" target="_blank" rel="noopener" class="title">mapnode output to datasink, container specific files</a><div class="summary">Currently in my workflow everything runs, and the files I want to be created are created however, the list of outputs from mapnode are each saved into every container output directory (the participant output directory). Instead what I'd like to happen is each container output directory contains the 1 output file that belongs to it's container id. &#8203;from nipype.pipeline.engine import Node, MapNode, </div></div><span class="ago" title="published:Tue, 26 Apr 2016 20:18:54 GMT -- Received: Tue, 26 Apr 2016 22:34:17 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15453fa91ee:11b916:8d8172b5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3860/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15453fa91ee:11b916:8d8172b5" data-navigation="inline" data-title="How to automatize snapshots of different directions of DTI data ?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15453fa91ee:11b916:8d8172b5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2299 --></div><!-- react-empty: 2300 --><div class="content"><a href="http://neurostars.org/p/3860/" target="_blank" rel="noopener" class="title">How to automatize snapshots of different directions of DTI data ?</a><div class="summary"> Hi! I have a dataset of DTI data (10 B0 followed by 200 directions) and I would like to be able to take automatically snapshots (png or jpeg ...) of several orientations for all my subjects. Does anybody have a commandline to do that ? Thanks Céline </div></div><span class="ago" title="published:Tue, 26 Apr 2016 13:35:40 GMT -- Received: Tue, 26 Apr 2016 19:10:10 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154523c5a2d:4a59fc:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3859/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154523c5a2d:4a59fc:7bd7a919" data-navigation="inline" data-title="Where have I screwed up my resting state cleaning code?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154523c5a2d:4a59fc:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2315 --></div><!-- react-empty: 2316 --><div class="content"><a href="http://neurostars.org/p/3859/" target="_blank" rel="noopener" class="title">Where have I screwed up my resting state cleaning code?</a><div class="summary">Hi list, I'm having some problems with a resting state cleaning script that I've hacked together in python from my own code and code borrowed from CPAC (motion24 + regression). The script is part of a larger pipeline but in short it calculates the Friston 24 motion parameters (from 6 parameters), white matter and CSF masks + time series then the first 6 principle components using the method outlin</div></div><span class="ago" title="published:Tue, 26 Apr 2016 10:43:44 GMT -- Received: Tue, 26 Apr 2016 11:02:47 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154507bf481:14cd930:77051e1f_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3856/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154507bf481:14cd930:77051e1f" data-navigation="inline" data-title="RuntimeError: Workflow did not execute cleanly. Check log for details" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154507bf481:14cd930:77051e1f" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2331 --></div><!-- react-empty: 2332 --><div class="content"><a href="http://neurostars.org/p/3856/" target="_blank" rel="noopener" class="title">RuntimeError: Workflow did not execute cleanly. Check log for details</a><div class="summary">Hi all, I am very new to brain scan analysis and NiPy. I have just tried to run the following code and changed all paths and filenames to suit my data. However, I keep coming up with a runtime error at the end. When I try to check the log for more details I get a list of letters and numbers and I'm not quite sure what this means or what I'm doing wrong. I've tried to add as many details as I can h</div></div><span class="ago" title="published:Tue, 26 Apr 2016 02:32:53 GMT -- Received: Tue, 26 Apr 2016 02:53:01 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15444928ae3:357a739:80ff33e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3853/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15444928ae3:357a739:80ff33e3" data-navigation="inline" data-title="Smoothness estimation of cortical surface data" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15444928ae3:357a739:80ff33e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2347 --></div><!-- react-empty: 2348 --><div class="content"><a href="http://neurostars.org/p/3853/" target="_blank" rel="noopener" class="title">Smoothness estimation of cortical surface data</a><div class="summary"> Hi, I would like to estimate the spatial smoothness of functional data on a cortical surface, similar to the "RESEL" as implemented in SPM for volumetric data. The only tool I have come across to do so is the Matlab SurfStat toolbox (http://www.math.mcgill.ca/keith/surfstat/). Is anyone aware of alternatives, that don't depend on Matlab and Freesurfer File formats? Also, the SurfStat website stat</div></div><span class="ago" title="published:Sat, 23 Apr 2016 17:39:59 GMT -- Received: Sat, 23 Apr 2016 19:22:15 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543bc56328:13dc63e:80ff33e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3849/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543bc56328:13dc63e:80ff33e3" data-navigation="inline" data-title="Nipype dcm2nii interface issue" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543bc56328:13dc63e:80ff33e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2363 --></div><!-- react-empty: 2364 --><div class="content"><a href="http://neurostars.org/p/3849/" target="_blank" rel="noopener" class="title">Nipype dcm2nii interface issue</a><div class="summary">I'm having some issues running one of my nipype workflows on a new cluster. Specifically, the dcm2nii interface is running without errors, but is not storing the file for some reason. This workflow successfully runs on other systems and the file is correctly generated if I run the command.txt in the folder. Here is the command: dcm2nii -a y -c y -b config.ini -v y -d n -e y -g y -i n -n y -o . -p </div></div><span class="ago" title="published:Fri, 22 Apr 2016 00:46:25 GMT -- Received: Fri, 22 Apr 2016 02:21:12 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543b206c63:104ddaa:46c52675_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3848/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543b206c63:104ddaa:46c52675" data-navigation="inline" data-title="Dynamic nodes in Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543b206c63:104ddaa:46c52675" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2379 --></div><!-- react-empty: 2380 --><div class="content"><a href="http://neurostars.org/p/3848/" target="_blank" rel="noopener" class="title">Dynamic nodes in Nipype</a><div class="summary"> Scenario: Longitudinal image analysis with Nipype. Challenge: Differing number of timepoint for each subject Question: Is there any possibility that I can change number of inputs and outputs to nodes, depending on the available timepoints? Each subject might have different MRI sequences, and assume that at one node I need to collect all timepoints to do further analysis. I guess I can do the latt</div></div><span class="ago" title="published:Thu, 21 Apr 2016 21:30:30 GMT -- Received: Thu, 21 Apr 2016 23:21:01 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543494f488:cb9c4b:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3843/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543494f488:cb9c4b:7bd7a919" data-navigation="inline" data-title="Brainstem pulsation at 7T: literature?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543494f488:cb9c4b:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2395 --></div><!-- react-empty: 2396 --><div class="content"><a href="http://neurostars.org/p/3843/" target="_blank" rel="noopener" class="title">Brainstem pulsation at 7T: literature?</a><div class="summary"> Hi there, I'm debating how to design my experiment at the moment and the information I'm looking for is crucial in that decision. We're trying to do thalamic imaging but we're also interested in the inferior colliculus using 7T. As the latter is part of the midbrain it might experience pulsatile motion from the heart beat. At last year's OHBM2015 there were two oral presentations, one claiming th</div></div><span class="ago" title="published:Wed, 20 Apr 2016 16:04:54 GMT -- Received: Wed, 20 Apr 2016 16:50:57 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543494f488:cb9c4a:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3837/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543494f488:cb9c4a:7bd7a919" data-navigation="inline" data-title="edge_kwargs in plot_connectome" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1543494f488:cb9c4a:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2411 --></div><!-- react-empty: 2412 --><div class="content"><a href="http://neurostars.org/p/3837/" target="_blank" rel="noopener" class="title">edge_kwargs in plot_connectome</a><div class="summary"> Hi all, I am trying to plot a connectome plot using nilearns plot_connectome with different edges in various colors, depending on network membership (mostly). It seems edge_kwargs would allow me to do so as it says ("will be passed as kwargs for each edge matlotlib Line2D"). However, I could not figure out a way to lass a list of dicts, or a dict of dicts. Here is an example that gives all edges </div></div><span class="ago" title="published:Wed, 20 Apr 2016 14:06:10 GMT -- Received: Wed, 20 Apr 2016 16:50:57 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154242bc02b:690af02:1029b533_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3826/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154242bc02b:690af02:1029b533" data-navigation="inline" data-title="OpenfMRI dataset 115 [ utilizing preprocessing parameters ]" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154242bc02b:690af02:1029b533" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2427 --></div><!-- react-empty: 2428 --><div class="content"><a href="http://neurostars.org/p/3826/" target="_blank" rel="noopener" class="title">OpenfMRI dataset 115 [ utilizing preprocessing parameters ]</a><div class="summary"> Hi. Here I got some questions about ds-000115 on openfmri.org (https://openfmri.org/dataset/ds000115/) The data represent "Raw data (for subject XXX~)", but the attached files contain some preprocessing parameters. For example, files in directory '~/sub001/model/model001/task001_run001.feat/reg'. I hopefuly expect I can utilize the preprocessing parameters for this large dataset. However, when I </div></div><span class="ago" title="published:Sun, 17 Apr 2016 12:11:10 GMT -- Received: Sun, 17 Apr 2016 12:22:07 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154204d02f6:5cfb1ce:1029b533_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3825/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154204d02f6:5cfb1ce:1029b533" data-navigation="inline" data-title="MNI coordinates for AAL atlas" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_154204d02f6:5cfb1ce:1029b533" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2443 --></div><!-- react-empty: 2444 --><div class="content"><a href="http://neurostars.org/p/3825/" target="_blank" rel="noopener" class="title">MNI coordinates for AAL atlas</a><div class="summary"> Hi all, I am currently working with the ABIDE data (downloaded from http://fcon_1000.projects.nitrc.org/indi/abide/). I am using the AAL atlas parcellation and I was wondering where I could find the MNI coordinates corresponding to each region. Thanks in advance! </div></div><span class="ago" title="published:Sat, 16 Apr 2016 16:41:21 GMT -- Received: Sat, 16 Apr 2016 18:19:58 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15411f342a7:1a3c5c3:1029b533_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3823/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15411f342a7:1a3c5c3:1029b533" data-navigation="inline" data-title="Understanding Nipype IdentityInterface and SelectFiles with iterables" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15411f342a7:1a3c5c3:1029b533" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2459 --></div><!-- react-empty: 2460 --><div class="content"><a href="http://neurostars.org/p/3823/" target="_blank" rel="noopener" class="title">Understanding Nipype IdentityInterface and SelectFiles with iterables</a><div class="summary">I am having trouble understanding the behavior of Nipype's IdentityInterface and SelectFiles interfaces. I am creating a Workflow that iterates over a list of subject IDs, running the same operations for each subject ID. Eventually it will run my fMRI analysis; right now I just want the Workflow to print the file paths of each subject's anatomical images—to check that the paths are correct, and al</div></div><span class="ago" title="published:Wed, 13 Apr 2016 21:48:05 GMT -- Received: Wed, 13 Apr 2016 23:27:15 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540eb8da55:817418:1029b533_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3822/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540eb8da55:817418:1029b533" data-navigation="inline" data-title="Job: Post doc position in Machine Learning in Neurospin" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540eb8da55:817418:1029b533" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2475 --></div><!-- react-empty: 2476 --><div class="content"><a href="http://neurostars.org/p/3822/" target="_blank" rel="noopener" class="title">Job: Post doc position in Machine Learning in Neurospin</a><div class="summary">Segregation of a large population of patients with Bipolar Disorder using novel anatomical and diffusion weighted imaging processing tools and machine learning Postdoctoral position in Neurospin for 18 months with possible extensions, Neurospin, France Bipolar disorder (BD) is a chronic psychiatric illness affecting 1% of the population, characterized by the recurrences of major depressive and man</div></div><span class="ago" title="published:Wed, 13 Apr 2016 06:47:50 GMT -- Received: Wed, 13 Apr 2016 08:24:36 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540b7733cd:12b3e9f7:80ff33e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3821/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540b7733cd:12b3e9f7:80ff33e3" data-navigation="inline" data-title="Skull Stripping with ANTs" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540b7733cd:12b3e9f7:80ff33e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2491 --></div><!-- react-empty: 2492 --><div class="content"><a href="http://neurostars.org/p/3821/" target="_blank" rel="noopener" class="title">Skull Stripping with ANTs</a><div class="summary"> I was wondering whether ANTs could be used in order to do a skullstrip. Would it be possible to make ANTs position the template onto a NIfTI in such a way that there is a minimal mount of signal around the edge of the template (which would correspond to the signal gap gven by the skull). Does anybody know more about this? The reason I am asking is that I am working with mouse brains, for which I </div></div><span class="ago" title="published:Tue, 12 Apr 2016 16:13:39 GMT -- Received: Tue, 12 Apr 2016 17:14:02 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540a12f958:1237bc64:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3820/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540a12f958:1237bc64:7bd7a919" data-navigation="inline" data-title="Using Rename -> DataSink without symlinks?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540a12f958:1237bc64:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2507 --></div><!-- react-empty: 2508 --><div class="content"><a href="http://neurostars.org/p/3820/" target="_blank" rel="noopener" class="title">Using Rename -&gt; DataSink without symlinks?</a><div class="summary"> Hello! I have a pipeline using the Rename utility ( https://github.com/nipy/nipype/blob/4b392d55e469290139fc63d1a95928d665a2b08d/nipype/interfaces/utility.py#L155) connected to a DataSink. It seems that the DataSink's outputs are just symbolic links to the original workflow node that generated them (the input to the Rename node). I would prefer if the DataSink actually copied the files, rather th</div></div><span class="ago" title="published:Tue, 12 Apr 2016 08:24:40 GMT -- Received: Tue, 12 Apr 2016 10:44:56 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540a12f958:1237bc63:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3819/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540a12f958:1237bc63:7bd7a919" data-navigation="inline" data-title="all_fmb_pipeline kills ipython-notebook" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1540a12f958:1237bc63:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2523 --></div><!-- react-empty: 2524 --><div class="content"><a href="http://neurostars.org/p/3819/" target="_blank" rel="noopener" class="title">all_fmb_pipeline kills ipython-notebook</a><div class="summary"> To make it more userfriendly I am trying to move to ipython-notebook. I setup the all_fmb_pipeline and it's working fine, once. The combination of massive output and autosave caused a 2MB ipython-notebook file. This file seems to be too large to be load again. How can I redirect / disable the output? Interfaces have the terminal_output: ('stream' or 'allatonce' or 'file' or 'none') option but not</div></div><span class="ago" title="published:Tue, 12 Apr 2016 07:56:24 GMT -- Received: Tue, 12 Apr 2016 10:44:56 GMT">7mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e7a1d15e:72bed4d:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3815/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e7a1d15e:72bed4d:7bd7a919" data-navigation="inline" data-title="Logic error when using FSL's flameo" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e7a1d15e:72bed4d:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2539 --></div><!-- react-empty: 2540 --><div class="content"><a href="http://neurostars.org/p/3815/" target="_blank" rel="noopener" class="title">Logic error when using FSL's flameo</a><div class="summary"> I am trying to run flameo to fit a mixed-effects GLM. I am using the following command: flameo --copefile=cope_merged.nii.gz --covsplitfile=design.grp --designfile=design.mat --ld=stats --maskfile=ds_QBI_chr_bin.nii.gz --runmode=ols --tcontrastsfile=design.con --varcopefile=varcb_merged.nii.gz And I am getting this output: Log directory is: stats Setting up: An exception has been thrown Logic err</div></div><span class="ago" title="published:Tue, 05 Apr 2016 18:00:34 GMT -- Received: Tue, 05 Apr 2016 18:14:15 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e4568b1a:62e375b:80ff33e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3810/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e4568b1a:62e375b:80ff33e3" data-navigation="inline" data-title="BIDS-like structure for NiPype datasink" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e4568b1a:62e375b:80ff33e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2555 --></div><!-- react-empty: 2556 --><div class="content"><a href="http://neurostars.org/p/3810/" target="_blank" rel="noopener" class="title">BIDS-like structure for NiPype datasink</a><div class="summary">I have a workflow where I iterate over participants via infosource.iterables. I would like to store my data in a BIDS-like structure, meaning: /subject/ /scantype The closest I could get is currently: / /subject/scantype To visualize my directory structure, this currently looks like: $ ls DIAGNOSTIC/*/* DIAGNOSTIC/MELODIC_reports/_condition_ERC_ofM_subject_5502: _scan_type_7_EPI_CBV_alej _scan_typ</div></div><span class="ago" title="published:Tue, 05 Apr 2016 00:20:30 GMT -- Received: Tue, 05 Apr 2016 02:53:10 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e4568b1a:62e375a:80ff33e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3809/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e4568b1a:62e375a:80ff33e3" data-navigation="inline" data-title="BIDS-like structure for NiPype datasink" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153e4568b1a:62e375a:80ff33e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2571 --></div><!-- react-empty: 2572 --><div class="content"><a href="http://neurostars.org/p/3809/" target="_blank" rel="noopener" class="title">BIDS-like structure for NiPype datasink</a><div class="summary">I have a workflow where I iterate over participants via infosource.iterables. I would like to store my data in a BIDS-like structure, meaning: /subject/ /scantype The closest I could get is currently: / /subject/scantype To visualize my directory structure, this currently looks like: $ ls DIAGNOSTIC/*/* DIAGNOSTIC/MELODIC_reports/_condition_ERC_ofM_subject_5502: _scan_type_7_EPI_CBV_alej _scan_typ</div></div><span class="ago" title="published:Tue, 05 Apr 2016 00:20:26 GMT -- Received: Tue, 05 Apr 2016 02:53:10 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153d18862f2:b1ddab:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3808/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153d18862f2:b1ddab:7bd7a919" data-navigation="inline" data-title="Job: PhD student / Postdoc positions in systems neuroscience in Leuven Belgium" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153d18862f2:b1ddab:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2587 --></div><!-- react-empty: 2588 --><div class="content"><a href="http://neurostars.org/p/3808/" target="_blank" rel="noopener" class="title">Job: PhD student / Postdoc positions in systems neuroscience in Leuven Belgium</a><div class="summary"> PhD student / Postdoc positions in systems neuroscience available in the laboratory of Vincent Bonin at Neuro-Electronics Research Flanders in Leuven, Belgium. The successful candidate will apply optogenetic and cellular imaging techniques to study encoding of sensory information in the visual cortex of awake, behaving mice. Profile: – PhD in neuroscience, computer science, physics, mathematics o</div></div><span class="ago" title="published:Fri, 01 Apr 2016 11:14:25 GMT -- Received: Fri, 01 Apr 2016 11:14:49 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153d18862f2:b1ddaa:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3807/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153d18862f2:b1ddaa:7bd7a919" data-navigation="inline" data-title="Job: Neuro-engineer position in Leuven, Belgium" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153d18862f2:b1ddaa:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2603 --></div><!-- react-empty: 2604 --><div class="content"><a href="http://neurostars.org/p/3807/" target="_blank" rel="noopener" class="title">Job: Neuro-engineer position in Leuven, Belgium</a><div class="summary">We are looking for an enthusiastic and talented individual with broad practical technical skills to spur our laboratory’s software and hardware development. The ideal candidate is skilled at prototyping, systems integration, digital signal processing and software development. Responsibilities: Support researchers with development of software and hardware directed towards the study of brain functio</div></div><span class="ago" title="published:Fri, 01 Apr 2016 11:11:29 GMT -- Received: Fri, 01 Apr 2016 11:14:49 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153cf984e3d:1463d7:7bd7a919_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3806/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153cf984e3d:1463d7:7bd7a919" data-navigation="inline" data-title="Job: Python guru for the DataLad project at Dartmouth College, NH" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153cf984e3d:1463d7:7bd7a919" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2619 --></div><!-- react-empty: 2620 --><div class="content"><a href="http://neurostars.org/p/3806/" target="_blank" rel="noopener" class="title">Job: Python guru for the DataLad project at Dartmouth College, NH</a><div class="summary">Original full posting and application instructions could be found at http://searchjobs.dartmouth.edu/postings/36530 DataLad project ( http://datalad.org ) is a collaborative USA/Germany NSF/BMBF-funded project to establish data distribution for a variety of datasets in neuroscience and beyond. We are looking for a full-time (part-time might be considered) software developer to work on the project </div></div><span class="ago" title="published:Thu, 31 Mar 2016 23:40:35 GMT -- Received: Fri, 01 Apr 2016 02:12:58 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153c5e632e1:7bf08c:f050b7cf_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3804/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153c5e632e1:7bf08c:f050b7cf" data-navigation="inline" data-title="Coregistration and Subtraction" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153c5e632e1:7bf08c:f050b7cf" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2635 --></div><!-- react-empty: 2636 --><div class="content"><a href="http://neurostars.org/p/3804/" target="_blank" rel="noopener" class="title">Coregistration and Subtraction</a><div class="summary"> Hi all, Would it be possible to use nipy/nipype to coregister two sequences (e.g. non-contrast T1 and contrast-enhanced T1) and then subtract the two images to get a multidimensional representation of the enhancing areas only? Thank you, OA </div></div><span class="ago" title="published:Wed, 30 Mar 2016 02:06:52 GMT -- Received: Wed, 30 Mar 2016 05:01:51 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b3c69741:1582db66:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3801/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b3c69741:1582db66:bc27e115" data-navigation="inline" data-title="Iterables input double specified- what happens?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b3c69741:1582db66:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2651 --></div><!-- react-empty: 2652 --><div class="content"><a href="http://neurostars.org/p/3801/" target="_blank" rel="noopener" class="title">Iterables input double specified- what happens?</a><div class="summary"> If both lines are used in a node specification, what actually happens? The intent was to have one element of file_name_data_list be used per node, rather than passing the whole list into each iterable node copy. Program works and seems to be doing the "right" thing. Also works without the second, inputs.file_name_data line. node_get_all_data.iterables =[('file_name_data', file_name_data_list) ] n</div></div><span class="ago" title="published:Sat, 26 Mar 2016 15:27:55 GMT -- Received: Sat, 26 Mar 2016 16:34:09 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b3c69741:1582db65:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3800/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b3c69741:1582db65:bc27e115" data-navigation="inline" data-title="nipype debug slurm queue" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b3c69741:1582db65:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2667 --></div><!-- react-empty: 2668 --><div class="content"><a href="http://neurostars.org/p/3800/" target="_blank" rel="noopener" class="title">nipype debug slurm queue</a><div class="summary"> Why are jobs queued, or not queued in a nipype workflow. What are the computational bottlenecks to queuing jobs? How does the number of cores requested at each level cause things to be divided up (I thought initially, each node was independent)? Is a certain amount of memory needed as a critical mass per core to make things "go quick" as opposed to just "run without crashing"? How can I effective</div></div><span class="ago" title="published:Sat, 26 Mar 2016 15:24:27 GMT -- Received: Sat, 26 Mar 2016 16:34:09 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b1320a5e:14d8da9e:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3798/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b1320a5e:14d8da9e:bc27e115" data-navigation="inline" data-title="Job: Postdoctoral fellowship at UCLA" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153b1320a5e:14d8da9e:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2683 --></div><!-- react-empty: 2684 --><div class="content"><a href="http://neurostars.org/p/3798/" target="_blank" rel="noopener" class="title">Job: Postdoctoral fellowship at UCLA</a><div class="summary">Applications are invited for a postdoctoral fellowship under the mentorship of Dr. Carolyn Parkinson in the Department of Psychology at UCLA beginning Fall or Winter 2016 (start date is flexible). The successful candidate will have the opportunity to contribute to research projects that integrate neuroimaging, machine learning, social network analysis, and behavioral experimentation to investigate</div></div><span class="ago" title="published:Sat, 26 Mar 2016 01:53:57 GMT -- Received: Sat, 26 Mar 2016 04:32:40 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153adf863da:13f0c8f2:70e34146_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3796/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153adf863da:13f0c8f2:70e34146" data-navigation="inline" data-title="what drives workflow memory?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153adf863da:13f0c8f2:70e34146" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2699 --></div><!-- react-empty: 2700 --><div class="content"><a href="http://neurostars.org/p/3796/" target="_blank" rel="noopener" class="title">what drives workflow memory?</a><div class="summary"> What causes a running workflow to accumulate memory? I thought each node handled its own memory requirements, so the wf job itself should be quite light. https://neurostars.org/p/1873/#1874 alludes to the prov file consuming tons of memory, but the reply suggests otherwise. Is a 20GB prov file really reasonable? That would be a lot of text. I don't even see a single prov file because the workflow</div></div><span class="ago" title="published:Fri, 25 Mar 2016 13:04:20 GMT -- Received: Fri, 25 Mar 2016 13:30:50 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153ac07f07a:135462e2:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3795/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153ac07f07a:135462e2:bc27e115" data-navigation="inline" data-title="Nipy design matrix - setting amplitude of regressor" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153ac07f07a:135462e2:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2715 --></div><!-- react-empty: 2716 --><div class="content"><a href="http://neurostars.org/p/3795/" target="_blank" rel="noopener" class="title">Nipy design matrix - setting amplitude of regressor</a><div class="summary">Hi, I have a question about setting the amplitude of a parameter in a nipy design matrix. Basically, I have included the onsets of juice trials in the model, and a second regressor using the same juice trial onsets, with the amplitude modulated by a parameter from a VTA model. In the design matrix, which I have plotted below, showing the first 400 trials for the juice regressor (blue) and the VTA </div></div><span class="ago" title="published:Fri, 25 Mar 2016 02:16:19 GMT -- Received: Fri, 25 Mar 2016 04:28:34 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aabdf566:12f5699d:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3791/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aabdf566:12f5699d:bc27e115" data-navigation="inline" data-title="Simple affine transformation from 1mm MNI anatomical image to 2mm MNI anatomical image with ANTS and nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aabdf566:12f5699d:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2731 --></div><!-- react-empty: 2732 --><div class="content"><a href="http://neurostars.org/p/3791/" target="_blank" rel="noopener" class="title">Simple affine transformation from 1mm MNI anatomical image to 2mm MNI anatomical image with ANTS and nipype</a><div class="summary"> Currently I'm doing ants normalization by corregistering the mean functional to the subject structural using bbregister, followed by registering the subject structural to the 1mm MNI template. Since the functional image is with voxels 3x3x3, I'd like to re-sample the final image to be with the same dimensional (otherwise I get huge functional images (~8GB each run). Here it is suggested to do an </div></div><span class="ago" title="published:Thu, 24 Mar 2016 21:49:58 GMT -- Received: Thu, 24 Mar 2016 22:28:09 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aabdf566:12f5699c:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3790/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aabdf566:12f5699c:bc27e115" data-navigation="inline" data-title="News: OHBM Hackathon 2016 - Stipends Application" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aabdf566:12f5699c:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2788 --></div><!-- react-empty: 2789 --><div class="content"><a href="http://neurostars.org/p/3790/" target="_blank" rel="noopener" class="title">News: OHBM Hackathon 2016 - Stipends Application</a><div class="summary">The registration for the fourth OHBM Hackathon is now open! Thanks to the generous support of our sponsors, the OHBM special interest group in open science will award a limited number of stipends to students attending the OHBM Hackathon 2016, including free registration and 500 USD for travel and accommodations. If you would like to apply for one of the stipends, please email the following materia</div></div><span class="ago" title="published:Thu, 24 Mar 2016 20:46:21 GMT -- Received: Thu, 24 Mar 2016 22:28:09 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aa1904a8:12bdb93c:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3789/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aa1904a8:12bdb93c:bc27e115" data-navigation="inline" data-title="create provenance despite workflow crash?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153aa1904a8:12bdb93c:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2804 --></div><!-- react-empty: 2805 --><div class="content"><a href="http://neurostars.org/p/3789/" target="_blank" rel="noopener" class="title">create provenance despite workflow crash?</a><div class="summary"> Is there anyway to capture the provenance file despite a workflow crash? </div></div><span class="ago" title="published:Thu, 24 Mar 2016 18:27:38 GMT -- Received: Thu, 24 Mar 2016 19:27:59 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a39f0e72:1085535e:70e34146_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3784/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a39f0e72:1085535e:70e34146" data-navigation="inline" data-title="Signal extraction for nuisance correction" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a39f0e72:1085535e:70e34146" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2820 --></div><!-- react-empty: 2821 --><div class="content"><a href="http://neurostars.org/p/3784/" target="_blank" rel="noopener" class="title">Signal extraction for nuisance correction</a><div class="summary"> Hi, I am not very familiar with the signal regression/nuisance correction part of a resting-state pre-processing pipeline. I am looking for a node in nipype to perform signal extraction from different ROIS, such as white-matter or CSF from co-registered anatomical segmentation. This step would be to perform signal regression of motion corrected resting state images. I could try to implement an in</div></div><span class="ago" title="published:Wed, 23 Mar 2016 11:02:50 GMT -- Received: Wed, 23 Mar 2016 13:17:03 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a2f72846:104868f3:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3783/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a2f72846:104868f3:bc27e115" data-navigation="inline" data-title="openfmri dataset ds006A - condition onsets and fmri volumes" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a2f72846:104868f3:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2836 --></div><!-- react-empty: 2837 --><div class="content"><a href="http://neurostars.org/p/3783/" target="_blank" rel="noopener" class="title">openfmri dataset ds006A - condition onsets and fmri volumes</a><div class="summary"> Hello, I have a question regarding the ds006A dataset from the openfMRI project. I want to extract fMRI volumes for certain conditions from this task, but I'm not sure how the fMRI volumes correspond to the given onset times for each condition. The nifti-files for each run contain 205 fMRI volumes, the latest condition onset time is around 400 sec. As the TR is 2.0 sec I assume that the runs took</div></div><span class="ago" title="published:Wed, 23 Mar 2016 09:46:43 GMT -- Received: Wed, 23 Mar 2016 10:13:40 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a106bc05:fb2f2fe:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3781/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a106bc05:fb2f2fe:bc27e115" data-navigation="inline" data-title="Use a new FSL design.mat in" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153a106bc05:fb2f2fe:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2852 --></div><!-- react-empty: 2853 --><div class="content"><a href="http://neurostars.org/p/3781/" target="_blank" rel="noopener" class="title">Use a new FSL design.mat in</a><div class="summary"> I modified a design.mat file extracted from a regular feat analysis and I now want to rerun the analysis using the new design.mat file I created. How do I call the feat function so that it grabs all the old information like the contrasts, registragion, design.fsf, etc but that uses my new design.mat file? Thanks for all the help! </div></div><span class="ago" title="published:Tue, 22 Mar 2016 22:31:36 GMT -- Received: Wed, 23 Mar 2016 01:11:26 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539fb8d02d:f43de23:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3778/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539fb8d02d:f43de23:bc27e115" data-navigation="inline" data-title="nipype wf directory with too long name causes crash using iterable" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539fb8d02d:f43de23:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2868 --></div><!-- react-empty: 2869 --><div class="content"><a href="http://neurostars.org/p/3778/" target="_blank" rel="noopener" class="title">nipype wf directory with too long name causes crash using iterable</a><div class="summary">Using iterables seems to lead to nipype making sub working directories with exorbitantly long names that can't be made on the file system, leading to the crash (I think, considering running with a shorter 'test' set works just fine). The solution is probably to use datagrabber rather than the incredibly intuitive and easy .synchronize option. It's using the iterable inside test_list to name the no</div></div><span class="ago" title="published:Tue, 22 Mar 2016 19:07:10 GMT -- Received: Tue, 22 Mar 2016 19:06:42 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539c76393a:e2bcf34:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3772/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539c76393a:e2bcf34:bc27e115" data-navigation="inline" data-title="interactive debug nipype list/ where not working?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539c76393a:e2bcf34:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2884 --></div><!-- react-empty: 2885 --><div class="content"><a href="http://neurostars.org/p/3772/" target="_blank" rel="noopener" class="title">interactive debug nipype list/ where not working?</a><div class="summary"> Using a Jupyter notebook and nipype_display_crash -i crash*.pklz: Why does "next" and "where" at the debug command only show the node name and the line number? I actually get no insight into the code itself. ipdb&gt; n :138: VisibleDeprecationWarning: boolean index did not match indexed array along dimension 0; dimension is 6553 but corresponding boolean dimension is 6552 &gt; (139)extraTreeReg() l (lo</div></div><span class="ago" title="published:Tue, 22 Mar 2016 01:15:57 GMT -- Received: Tue, 22 Mar 2016 03:55:06 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539c76393a:e2bcf33:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3771/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539c76393a:e2bcf33:bc27e115" data-navigation="inline" data-title="interactive debug nipype list/ where not working?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1539c76393a:e2bcf33:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2900 --></div><!-- react-empty: 2901 --><div class="content"><a href="http://neurostars.org/p/3771/" target="_blank" rel="noopener" class="title">interactive debug nipype list/ where not working?</a><div class="summary"> Using a Jupyter notebook and nipype_display_crash -i crash*.pklz: Why does "next" and "where" at the debug command only show the node name and the line number? I actually get no insight into the code itself. ipdb&gt; n :138: VisibleDeprecationWarning: boolean index did not match indexed array along dimension 0; dimension is 6553 but corresponding boolean dimension is 6552 &gt; (139)extraTreeReg() l (lo</div></div><span class="ago" title="published:Tue, 22 Mar 2016 01:15:56 GMT -- Received: Tue, 22 Mar 2016 03:55:06 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15396643804:c692aae:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3770/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15396643804:c692aae:bc27e115" data-navigation="inline" data-title="Optimal ANTs configuration for incorporating lesion mask?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15396643804:c692aae:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2916 --></div><!-- react-empty: 2917 --><div class="content"><a href="http://neurostars.org/p/3770/" target="_blank" rel="noopener" class="title">Optimal ANTs configuration for incorporating lesion mask?</a><div class="summary">Hi list, I'm working with stroke lesioned brains and have decided to use ANTs to normalise them to MNI space in a nipype pipeline (using a combined affine / syn estimation?) The problems however are the bewildering range of options in ANT, the range of lesions sizes and my lack of knowledge when it comes to what most of the settings mean (I've gone through the ANTs manual). So, does anyone out the</div></div><span class="ago" title="published:Sun, 20 Mar 2016 23:00:19 GMT -- Received: Sun, 20 Mar 2016 23:37:43 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538ac010db:92ee801:70e34146_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3769/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538ac010db:92ee801:70e34146" data-navigation="inline" data-title="Nipype EstimateContrast specification" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538ac010db:92ee801:70e34146" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2932 --></div><!-- react-empty: 2933 --><div class="content"><a href="http://neurostars.org/p/3769/" target="_blank" rel="noopener" class="title">Nipype EstimateContrast specification</a><div class="summary">I'm having trouble specifying a contrast in nipype with EstimateContrast node. I have the following situation (simplified): 4 conditions 2 sessions the first 2 conditions are present only in sessions 1-2, while the second two conditions are present only in sessions 3-4 I am using time and dispersion derivatives Let's say I want to contrast conditions 1 and 2 vs 3 and 4. Given the above restriction</div></div><span class="ago" title="published:Fri, 18 Mar 2016 15:38:56 GMT -- Received: Fri, 18 Mar 2016 17:22:36 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538588ff3b:78f6e1a:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3765/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538588ff3b:78f6e1a:bc27e115" data-navigation="inline" data-title="Openfmri.org dataset 115 task order" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538588ff3b:78f6e1a:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2948 --></div><!-- react-empty: 2949 --><div class="content"><a href="http://neurostars.org/p/3765/" target="_blank" rel="noopener" class="title">Openfmri.org dataset 115 task order</a><div class="summary">Dear colleagues, I am trying to analyze OpenfMRI dataset 115: https://openfmri.org/dataset/ds000115/ . The functional runs in the dataset correspond to 0-back, 1-back and 2-back tasks. I have encountered two issues: 1. task001, task002 and task003 are all labeled as 'letter n-back task', so it's not clear what is the relationship between task and run, i.e. does task001 correspond to 0-back, task00</div></div><span class="ago" title="published:Thu, 17 Mar 2016 17:02:19 GMT -- Received: Thu, 17 Mar 2016 17:04:21 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538588ff3b:78f6e19:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3764/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538588ff3b:78f6e19:bc27e115" data-navigation="inline" data-title="Tools for super resolution ?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1538588ff3b:78f6e19:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2964 --></div><!-- react-empty: 2965 --><div class="content"><a href="http://neurostars.org/p/3764/" target="_blank" rel="noopener" class="title">Tools for super resolution ?</a><div class="summary"> Hello! Do you know of any super-resolution tool to combine 3 structural MRIs with anisotropic voxels into a single 1 with isotropic voxels ? Is there something like that in FSL, AFNI our MINC ? thank you, roberto </div></div><span class="ago" title="published:Thu, 17 Mar 2016 15:06:31 GMT -- Received: Thu, 17 Mar 2016 17:04:21 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153790e40c7:35c0c03:70e34146_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3760/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153790e40c7:35c0c03:70e34146" data-navigation="inline" data-title="Resting state residuals - high correlation between voxels when only motion removed?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153790e40c7:35c0c03:70e34146" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2980 --></div><!-- react-empty: 2981 --><div class="content"><a href="http://neurostars.org/p/3760/" target="_blank" rel="noopener" class="title">Resting state residuals - high correlation between voxels when only motion removed?</a><div class="summary"> Hi list, I'm playing around with noise regressors using a single participant's resting state fMRI and have an odd pattern of correlation after removing the motion regressors (output from SPM - 6 parameters). In the below link I've got several images, each calculated the same way (raw epi image with voxels to be correlated masked by a grey / white matter mask, selecting every 10th voxel (RAM const</div></div><span class="ago" title="published:Tue, 15 Mar 2016 05:08:18 GMT -- Received: Tue, 15 Mar 2016 06:54:50 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536eb45bf4:4ba046:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3759/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536eb45bf4:4ba046:bc27e115" data-navigation="inline" data-title="Make NiPype node fail quietly" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536eb45bf4:4ba046:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 2996 --></div><!-- react-empty: 2997 --><div class="content"><a href="http://neurostars.org/p/3759/" target="_blank" rel="noopener" class="title">Make NiPype node fail quietly</a><div class="summary"> I am trying to write a diagnostic workflow which does through all of my subjects, tries to find any measurement conditions and scans types from a given list, and run a few basic tests on them. Which conditions ans scan types have been measured differs from participant to participant - so it is difficult to iterate correctly (the IdentityInterface iterables creates a cartesian product matrix, and </div></div><span class="ago" title="published:Sun, 13 Mar 2016 05:33:09 GMT -- Received: Sun, 13 Mar 2016 06:40:27 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536c1fb246:85a797e:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3758/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536c1fb246:85a797e:ae8aca10" data-navigation="inline" data-title="Allen Brain Atlas API JS files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536c1fb246:85a797e:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3012 --></div><!-- react-empty: 3013 --><div class="content"><a href="http://neurostars.org/p/3758/" target="_blank" rel="noopener" class="title">Allen Brain Atlas API JS files</a><div class="summary"> Hello, This might be elementary, but I'm trying to understand how this works: http://www.brain-map.org/api/examples/examples/structures/index.html There is a downloadable tar.gz archive to gather data from the Allen Brain Atlas's API and construct the structural similarity network seen on the page. I've been through the download_data.py file and have a general understanding of its functions, but </div></div><span class="ago" title="published:Sat, 12 Mar 2016 16:17:25 GMT -- Received: Sat, 12 Mar 2016 18:38:49 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153664d33f0:696e729:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3753/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153664d33f0:696e729:fb0a47d9" data-navigation="inline" data-title="FSL - calculate COPE at population level" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153664d33f0:696e729:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3028 --></div><!-- react-empty: 3029 --><div class="content"><a href="http://neurostars.org/p/3753/" target="_blank" rel="noopener" class="title">FSL - calculate COPE at population level</a><div class="summary"> I am using FSL's glm function to compute the COPEs (contrast parameter estimtes) for a given contrast. I am doing this over multiple subjects. How can I integrate the COPEs from all the subjects to get a population contrast estimate? </div></div><span class="ago" title="published:Fri, 11 Mar 2016 15:01:07 GMT -- Received: Fri, 11 Mar 2016 15:30:48 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536311f59c:58aa580:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3751/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536311f59c:58aa580:fb0a47d9" data-navigation="inline" data-title="Creating a custom function interface in nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1536311f59c:58aa580:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3044 --></div><!-- react-empty: 3045 --><div class="content"><a href="http://neurostars.org/p/3751/" target="_blank" rel="noopener" class="title">Creating a custom function interface in nipype</a><div class="summary"> I understand the general idea of how to create a custom function interface for nipype from here: http://nipy.org/nipype/users/function_interface.html I have a specific question about outputs. Let's say that my inputs are realignment motion parameters. I have written a custom function that produces plots about movement with pyplot. What should the function return so that when I call this node and </div></div><span class="ago" title="published:Thu, 10 Mar 2016 23:07:55 GMT -- Received: Fri, 11 Mar 2016 00:27:14 GMT">8mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15345fb73a3:adc3258:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3748/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15345fb73a3:adc3258:ae8aca10" data-navigation="inline" data-title="Generating new FreeSurfer annot files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15345fb73a3:adc3258:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3060 --></div><!-- react-empty: 3061 --><div class="content"><a href="http://neurostars.org/p/3748/" target="_blank" rel="noopener" class="title">Generating new FreeSurfer annot files</a><div class="summary"> I'm trying to generate a new annotation for a cortical surface in FreeSurfer, from scratch, and saving it with `nibabel.freesurfer.write_annot`, but I can't seem to get the ctab or something right, as freeview doesn't show colors on the surface when I load the resulting annot file, and `mri_annotation2label` says max id -1. Has anyone had success with this? What are the constraints on generating </div></div><span class="ago" title="published:Sat, 05 Mar 2016 08:47:24 GMT -- Received: Sat, 05 Mar 2016 08:53:40 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153348989d5:515a49d:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3746/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153348989d5:515a49d:ae8aca10" data-navigation="inline" data-title="unique file name based on node name" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153348989d5:515a49d:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3076 --></div><!-- react-empty: 3077 --><div class="content"><a href="http://neurostars.org/p/3746/" target="_blank" rel="noopener" class="title">unique file name based on node name</a><div class="summary"> Within a node, how I can create a unique file name? Nipype is able to do this when creating the temp directory for the node, and I'd like to access that directory name within my function to create file names. e.g doSomething01 and doSomething02 are two nipype folders created to hold doSomething node's outputs for multiple copies of the node. How can I access the 01 or 02? </div></div><span class="ago" title="published:Tue, 01 Mar 2016 21:51:42 GMT -- Received: Tue, 01 Mar 2016 23:35:42 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153333e3a2c:4932a6a:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3744/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153333e3a2c:4932a6a:fb0a47d9" data-navigation="inline" data-title="Job: Seeking postdoctoral candidates for datascience positions" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153333e3a2c:4932a6a:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3092 --></div><!-- react-empty: 3093 --><div class="content"><a href="http://neurostars.org/p/3744/" target="_blank" rel="noopener" class="title">Job: Seeking postdoctoral candidates for datascience positions</a><div class="summary">We are seeking two postdoctoral computational/data scientists with a PhD in computer science, electrical or biomedical engineering, neuroscience, statistics, or related fields to apply for positions in the McGovern Institute for Brain Research at Massachusetts Institute of Technology. The projects cover a broad array of neuroinformatics. - Datamining of gene-behavior-anatomy relationships - Archit</div></div><span class="ago" title="published:Tue, 01 Mar 2016 15:14:25 GMT -- Received: Tue, 01 Mar 2016 17:33:49 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532ffb4c3b:37f18fd:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3742/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532ffb4c3b:37f18fd:ae8aca10" data-navigation="inline" data-title="Nipype - generate level2 design matrix" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532ffb4c3b:37f18fd:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3108 --></div><!-- react-empty: 3109 --><div class="content"><a href="http://neurostars.org/p/3742/" target="_blank" rel="noopener" class="title">Nipype - generate level2 design matrix</a><div class="summary"> I am working on a GLM workflow using nipype and FSL. The second-level tools provided by FSL require a .mat design file in a specific format. The format is quite simple, and I could write the file by hand, or write a new script, but I would rather use (and/or extend) existing nipype tools - if there are any. Nipype provides this interface http://www.mit.edu/~satra/nipype-nightly/interfaces/generat</div></div><span class="ago" title="published:Tue, 01 Mar 2016 00:58:46 GMT -- Received: Tue, 01 Mar 2016 02:21:51 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532eb18edb:3088e06:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3741/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532eb18edb:3088e06:fb0a47d9" data-navigation="inline" data-title="datasink a variable?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532eb18edb:3088e06:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3124 --></div><!-- react-empty: 3125 --><div class="content"><a href="http://neurostars.org/p/3741/" target="_blank" rel="noopener" class="title">datasink a variable?</a><div class="summary"> Is there anyway to save a variable that is returned from a node? It's not a file but a scikit cross validation structure that is run within the function. Below, I just kept the function very simple and I'd like to save the f_x value returned. pickling within the function fails because the cross val struct is an instancemethod according to the error message. e.g. from http://nbviewer.jupyter.org/g</div></div><span class="ago" title="published:Mon, 29 Feb 2016 20:20:15 GMT -- Received: Mon, 29 Feb 2016 20:21:41 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532eb18edb:3088e05:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3740/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532eb18edb:3088e05:fb0a47d9" data-navigation="inline" data-title="Job: Hiring Research Specialist: Multimodal Neuroimaging" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532eb18edb:3088e05:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3140 --></div><!-- react-empty: 3141 --><div class="content"><a href="http://neurostars.org/p/3740/" target="_blank" rel="noopener" class="title">Job: Hiring Research Specialist: Multimodal Neuroimaging</a><div class="summary">Dear colleagues, We are hiring one full-time research specialist with a preferred background in neuroimaging and coding. This position is ideal for someone who may have the aptitude and interest to pursue doctoral research, but would like to spend a couple years in a research laboratory gaining full time experience before beginning a doctoral degree. Please distribute to any qualified candidates; </div></div><span class="ago" title="published:Mon, 29 Feb 2016 17:52:06 GMT -- Received: Mon, 29 Feb 2016 20:21:41 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532d67052e:2948e4a:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3738/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532d67052e:2948e4a:ae8aca10" data-navigation="inline" data-title="Affine Template Creation with DTI-TK" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532d67052e:2948e4a:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3156 --></div><!-- react-empty: 3157 --><div class="content"><a href="http://neurostars.org/p/3738/" target="_blank" rel="noopener" class="title">Affine Template Creation with DTI-TK</a><div class="summary">Hi there, I want to create a DTI template with DTI-TK, but the affine-registration step does not work properly. I am following the tutotial on: http://dti-tk.sourceforge.net/pmwiki/pmwiki.php?n=Documentation.Registration However, their is no detailed documentation for the affine registration, as it seems to be straight forward. But I am missing somehow a step. So after extracting my DTI data from </div></div><span class="ago" title="published:Mon, 29 Feb 2016 14:10:42 GMT -- Received: Mon, 29 Feb 2016 14:20:39 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532ad17815:1bbd953:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3734/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532ad17815:1bbd953:fb0a47d9" data-navigation="inline" data-title="aggregate mapnode output?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1532ad17815:1bbd953:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3172 --></div><!-- react-empty: 3173 --><div class="content"><a href="http://neurostars.org/p/3734/" target="_blank" rel="noopener" class="title">aggregate mapnode output?</a><div class="summary">On this page, the second graph, I'd like to remove node A, and just have a bunch of copies of B whose outputs are aggregated to C. http://www.mit.edu/~satra/nipype-nightly/users/mapnode_and_iterables.html # works fine node_b = pe.MapNode(niu.Function(input_names=['f1','f2', 'f3', 'f4'], output_names=['o1'], function=b), name="b", iterfield=['f1', 'f2']) node_b.inputs.f1 = f1 node_b.inputs.f2 = f2 </div></div><span class="ago" title="published:Mon, 29 Feb 2016 02:12:39 GMT -- Received: Mon, 29 Feb 2016 02:18:04 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153264bead8:7eb983:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3733/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153264bead8:7eb983:fb0a47d9" data-navigation="inline" data-title="Forum: [nipype] Developers roundup" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153264bead8:7eb983:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3188 --></div><!-- react-empty: 3189 --><div class="content"><a href="http://neurostars.org/p/3733/" target="_blank" rel="noopener" class="title">Forum: [nipype] Developers roundup</a><div class="summary"> Hi all, I'm gauging the interest on having a meeting to catch up with the progress, development and future of nipype. If interested, please let us know your availability on the following doodle poll: http://doodle.com/poll/2ueny72ita7vpkem Thanks a lot! Cheers, Oscar </div></div><span class="ago" title="published:Sun, 28 Feb 2016 02:50:21 GMT -- Received: Sun, 28 Feb 2016 05:13:42 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531d4275f3:f75f53f:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3730/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531d4275f3:f75f53f:fb0a47d9" data-navigation="inline" data-title="Join Nipype workflow split with IdentityInterface" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531d4275f3:f75f53f:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3204 --></div><!-- react-empty: 3205 --><div class="content"><a href="http://neurostars.org/p/3730/" target="_blank" rel="noopener" class="title">Join Nipype workflow split with IdentityInterface</a><div class="summary"> I am initiating a multi-participant, 2-condition workflow with the following nipype.interfaces.utility.IdentityInterface node: infosource = pe.Node(interface=util.IdentityInterface(fields=["condition","subject"]), name="infosource") infosource.iterables = [('condition', set(list(data_selection["condition"]))), ('subject', set(list(data_selection["subject"])))] I have succesfully run the preproces</div></div><span class="ago" title="published:Fri, 26 Feb 2016 10:59:15 GMT -- Received: Fri, 26 Feb 2016 11:06:48 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531d4275f3:f75f53e:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3729/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531d4275f3:f75f53e:fb0a47d9" data-navigation="inline" data-title="Tool: Accurate People Counters" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531d4275f3:f75f53e:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3220 --></div><!-- react-empty: 3221 --><div class="content"><a href="http://neurostars.org/p/3729/" target="_blank" rel="noopener" class="title">Tool: Accurate People Counters</a><div class="summary"> Flonomics is a global leader in people counting and retail analytics technologies. We provide businesses with a powerful solution to turn real-world data into real-world actions. The Flonomics suite of products includes solutions for highly accurate customer counts, traffic flow analysis, sales performance and marketing return on investment. </div></div><span class="ago" title="published:Fri, 26 Feb 2016 08:19:24 GMT -- Received: Fri, 26 Feb 2016 11:06:48 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531bf860d2:ee59954:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3728/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531bf860d2:ee59954:ae8aca10" data-navigation="inline" data-title="Forum: Many early OpenfMRI datasets recently updated with corrected orientation information." data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1531bf860d2:ee59954:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3236 --></div><!-- react-empty: 3237 --><div class="content"><a href="http://neurostars.org/p/3728/" target="_blank" rel="noopener" class="title">Forum: Many early OpenfMRI datasets recently updated with corrected orientation information.</a><div class="summary"> Based on user reports, we've corrected the embedded orientation information many of the early OpenfMRI data sets: ds001, ds002, ds003, ds005, ds006, ds007, ds008, ds009, ds017, ds108, and ds105. More information about the issues and the corrective actions can be found at this page . </div></div><span class="ago" title="published:Fri, 26 Feb 2016 02:19:08 GMT -- Received: Fri, 26 Feb 2016 05:06:15 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15316cfbddd:d21835e:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3727/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15316cfbddd:d21835e:ae8aca10" data-navigation="inline" data-title="Datasink node freezes when workflow runs on MultiProc plugin" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15316cfbddd:d21835e:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3252 --></div><!-- react-empty: 3253 --><div class="content"><a href="http://neurostars.org/p/3727/" target="_blank" rel="noopener" class="title">Datasink node freezes when workflow runs on MultiProc plugin</a><div class="summary"> Hi, I am trying to run a Nipype worflow using the multiproc plugin, iterating over subjects and their experimental sessions (visits). When I run this on a single subject (using linear or MultiProc plugins), it runs fine. However, when I proceed to iterate it with over subjects, it stops running (the screen freezes) with some subjects (unpredictable to me) specifically on a datasink node. After I </div></div><span class="ago" title="published:Thu, 25 Feb 2016 03:08:51 GMT -- Received: Thu, 25 Feb 2016 05:03:46 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cbb:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3724/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cbb:ae8aca10" data-navigation="inline" data-title="Dipy: viz.fvtk has not attribute ren" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cbb:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3268 --></div><!-- react-empty: 3269 --><div class="content"><a href="http://neurostars.org/p/3724/" target="_blank" rel="noopener" class="title">Dipy: viz.fvtk has not attribute ren</a><div class="summary"> Using Dipy, I am trying to load ren from viz.fvtk, but get an error: `"AttributeError: 'module' object has no attribute 'ren'"` when running: `ren = fvtk.ren()` What's that all about? </div></div><span class="ago" title="published:Thu, 25 Feb 2016 00:03:30 GMT -- Received: Thu, 25 Feb 2016 02:03:42 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cba:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3723/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cba:ae8aca10" data-navigation="inline" data-title="Job: Postdoc position in neuroimaging and data science at the University of Washington" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cba:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3284 --></div><!-- react-empty: 3285 --><div class="content"><a href="http://neurostars.org/p/3723/" target="_blank" rel="noopener" class="title">Job: Postdoc position in neuroimaging and data science at the University of Washington</a><div class="summary">We are seeking scientists with a PhD in neuroscience, computer science, electrical engineering, statistics, psychology or related fields, and with an interest in human brain function and data science to apply for a position as a post-doc at the Institute for Learning &amp; Brain Science (I-LABS) ( http://depts.washington.edu/bdelab/ ) and the eScience Institute ( http://escience.washington.edu ) at th</div></div><span class="ago" title="published:Thu, 25 Feb 2016 00:00:48 GMT -- Received: Thu, 25 Feb 2016 02:03:42 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cb9:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3722/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cb9:ae8aca10" data-navigation="inline" data-title="Retrieve expression data from Allen Brain Atlas API" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153162ae0ab:cee3cb9:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3300 --></div><!-- react-empty: 3301 --><div class="content"><a href="http://neurostars.org/p/3722/" target="_blank" rel="noopener" class="title">Retrieve expression data from Allen Brain Atlas API</a><div class="summary"> Hello, The Allen Brain Atlas curates gene expression data for different regions of the brain. I am currently looking at the thalamus and have a list of genes I want to extract the expression values for. Here is an example: http://mouse.brain-map.org/gene/show/42802 On the graph below the picture it shows the expression levels for various brain regions. I'm looking to write a script in python that</div></div><span class="ago" title="published:Wed, 24 Feb 2016 23:06:58 GMT -- Received: Thu, 25 Feb 2016 02:03:42 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153143ae937:c3d22f3:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3720/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153143ae937:c3d22f3:ae8aca10" data-navigation="inline" data-title="`Graph has no nodes or edges` encountered when trying to use Function interface for GLM" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153143ae937:c3d22f3:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3316 --></div><!-- react-empty: 3317 --><div class="content"><a href="http://neurostars.org/p/3720/" target="_blank" rel="noopener" class="title">`Graph has no nodes or edges` encountered when trying to use Function interface for GLM</a><div class="summary">I am trying to write a 2-level GLM. For this I want to iterate over my subjects and my conditions. In order to get a measurement path for each combination of subjects and conditions I am using a function which evaluates a pandas dataframe (since DataGrabber apparently cannot). This function makes my workflow fail with the following error: /usr/lib64/python2.7/site-packages/nipy/labs/glm/glm.py:9: </div></div><span class="ago" title="published:Wed, 24 Feb 2016 15:58:45 GMT -- Received: Wed, 24 Feb 2016 17:01:58 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153119d2e4a:b7d9c20:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3715/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153119d2e4a:b7d9c20:fb0a47d9" data-navigation="inline" data-title="Plotting multiple maps in one image using nilearn's plotting.plot_stat_map?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_153119d2e4a:b7d9c20:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3332 --></div><!-- react-empty: 3333 --><div class="content"><a href="http://neurostars.org/p/3715/" target="_blank" rel="noopener" class="title">Plotting multiple maps in one image using nilearn's plotting.plot_stat_map?</a><div class="summary"> Hello, is it possible to plot 2 stat-maps (e.g. one in red, one in blue) onto the same image using nilearn's plotting.plot_stat_map? I couldn't find a way and am now almost willing to go with 2 separate plots. Perhaps advisable from an aesthetic standpoint anyway... Thanks and best, Ralf. </div></div><span class="ago" title="published:Wed, 24 Feb 2016 02:09:23 GMT -- Received: Wed, 24 Feb 2016 04:50:27 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530f093e53:a7f2554:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3713/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530f093e53:a7f2554:ae8aca10" data-navigation="inline" data-title="News: DIPY 0.11.0 is now available for download" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530f093e53:a7f2554:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3348 --></div><!-- react-empty: 3349 --><div class="content"><a href="http://neurostars.org/p/3713/" target="_blank" rel="noopener" class="title">News: DIPY 0.11.0 is now available for download</a><div class="summary">Dear all, We are excited to announce a new public release of Diffusion Imaging in Python (DIPY). The 0.11 release follows closely on the heels of the 0.10 release, resolving issues that existed in that release on the Windows 64 bit platform. New features of the 0.11 and 0.10 release cycles include many additional fixes and new frameworks. Here are some of the highlights: DIPY 0.11.0 (Monday, 21 Fe</div></div><span class="ago" title="published:Tue, 23 Feb 2016 16:15:31 GMT -- Received: Tue, 23 Feb 2016 16:49:37 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530e644ec2:a61e38d:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3712/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530e644ec2:a61e38d:fb0a47d9" data-navigation="inline" data-title="FSl easythresh error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530e644ec2:a61e38d:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3364 --></div><!-- react-empty: 3365 --><div class="content"><a href="http://neurostars.org/p/3712/" target="_blank" rel="noopener" class="title">FSl easythresh error</a><div class="summary">Hello, I have a probably really simple question. I am trying to use easythresh with the following syntax: easythresh /path/zstat3.nii.gz /path/mask.nii.gz 2.3 0.05 /path/MNI_3mm.nii /path/threshed/ Yet it yields an error: Image Exception : #22 :: ERROR: Could not open image thresh_/path/threshed/ terminate called after throwing an instance of 'RBD_COMMON::BaseException' Aborted I don't really unde</div></div><span class="ago" title="published:Tue, 23 Feb 2016 13:39:12 GMT -- Received: Tue, 23 Feb 2016 13:49:28 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530a7efd7d:91db4f1:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3710/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530a7efd7d:91db4f1:ae8aca10" data-navigation="inline" data-title="News: Nipype needs your feedback" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530a7efd7d:91db4f1:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3380 --></div><!-- react-empty: 3381 --><div class="content"><a href="http://neurostars.org/p/3710/" target="_blank" rel="noopener" class="title">News: Nipype needs your feedback</a><div class="summary"> Dear all, We are trying to figure out which features of Nipype we should focus on in the upcoming year. Please help us by filling this very short wiki survey: http://www.allourideas.org/nipype Best, Chris </div></div><span class="ago" title="published:Mon, 22 Feb 2016 16:35:24 GMT -- Received: Mon, 22 Feb 2016 19:40:07 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15309d1680f:900f903:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3709/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15309d1680f:900f903:fb0a47d9" data-navigation="inline" data-title="behavioral folder in dataset ds000108" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15309d1680f:900f903:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3396 --></div><!-- react-empty: 3397 --><div class="content"><a href="http://neurostars.org/p/3709/" target="_blank" rel="noopener" class="title">behavioral folder in dataset ds000108</a><div class="summary"> Hi I downloaded the Dataset with association number ds000108 and this title ' Prefrontal-Subcortical Pathways Mediating Successful Emotion Regulation " but after unzipping the data. I found that behavioral folder is empty and there is no behavioral data. I ask this question from Tor Wager (The investigator of this project) he said that they should be attached!!! is there anyone that use this data</div></div><span class="ago" title="published:Mon, 22 Feb 2016 14:28:37 GMT -- Received: Mon, 22 Feb 2016 16:30:31 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530023ff6c:63a8b0e:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3704/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530023ff6c:63a8b0e:ae8aca10" data-navigation="inline" data-title="ERROR: &quot;Not enough data in&quot; event file" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1530023ff6c:63a8b0e:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3412 --></div><!-- react-empty: 3413 --><div class="content"><a href="http://neurostars.org/p/3704/" target="_blank" rel="noopener" class="title">ERROR: "Not enough data in" event file</a><div class="summary">I'm getting the error below. But the event file it references looks totally fine: Three columns, no tabs or invisible characters. wc [filename] and cat -v [filename] both look perfectly normal. Everything in that folder is here: https://drive.google.com/folderview?id=0BzJVbUt2zihJTEZQQmJJWGdQV2c&amp;usp=sharing and here is the node from my pipeline that is failing: #generate a run specific mat file fo</div></div><span class="ago" title="published:Sat, 20 Feb 2016 17:35:21 GMT -- Received: Sat, 20 Feb 2016 19:24:32 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f866644e:3e4e431:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3702/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f866644e:3e4e431:ae8aca10" data-navigation="inline" data-title="What is the purpose of the multiple merge and mean fsamples nodes in the bedpostx_parallel workflow?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f866644e:3e4e431:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3469 --></div><!-- react-empty: 3470 --><div class="content"><a href="http://neurostars.org/p/3702/" target="_blank" rel="noopener" class="title">What is the purpose of the multiple merge and mean fsamples nodes in the bedpostx_parallel workflow?</a><div class="summary">I have been having issues feeding the output of the bedpostx_parallel workflow into probtrackx2 . The bedpostx_parallel documentation states that it wraps all XFibres output, which includes fsamples , phsamples and thsamples , however my script is failing, with the error message from nipype_display_crash given as: ValueError: ProbTrackX2 requires a value for input 'phsamples' Looking closely at th</div></div><span class="ago" title="published:Fri, 19 Feb 2016 04:45:00 GMT -- Received: Fri, 19 Feb 2016 07:20:05 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f71c5a6d:387e896:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3698/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f71c5a6d:387e896:fb0a47d9" data-navigation="inline" data-title="ERROR: &quot;Not enough data in&quot; event file" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f71c5a6d:387e896:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3485 --></div><!-- react-empty: 3486 --><div class="content"><a href="http://neurostars.org/p/3698/" target="_blank" rel="noopener" class="title">ERROR: "Not enough data in" event file</a><div class="summary">I'm getting the error below. But the event file it references looks totally fine: Three columns, no tabs or invisible characters. wc [filename] and cat -v [filename] both look perfectly normal. Any idea what might be causing the error?? RuntimeError: Command: feat_model run0 Standard output: Not enough data in /data/WorkingDir/FFX/MasterWorkfow/withinSubject/modelfit/_model_name_Model_001_Rank_sub</div></div><span class="ago" title="published:Fri, 19 Feb 2016 00:24:19 GMT -- Received: Fri, 19 Feb 2016 01:19:36 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f676cc7f:34f22a4:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3697/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f676cc7f:34f22a4:fb0a47d9" data-navigation="inline" data-title="Nipype/nipy SpaceTimeRealigner reduces MELODIC signal detection" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f676cc7f:34f22a4:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3501 --></div><!-- react-empty: 3502 --><div class="content"><a href="http://neurostars.org/p/3697/" target="_blank" rel="noopener" class="title">Nipype/nipy SpaceTimeRealigner reduces MELODIC signal detection</a><div class="summary"> All the files used to demonstrate the issue can be downloaded from here . Hello, For diagnostic purposes I use MELODIC to look at structure in my data. Recently I noticed that performing space-time realignment on my data (via nipype's SpaceTimeRealigner interface) has a strong detrimental effect on the signal that I measure in MELODIC (as opposed to a slight beneficial one - as I would have expec</div></div><span class="ago" title="published:Thu, 18 Feb 2016 20:10:42 GMT -- Received: Thu, 18 Feb 2016 22:18:46 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f52ce2fc:2d64929:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3695/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f52ce2fc:2d64929:ae8aca10" data-navigation="inline" data-title="Nipype: iterables over sets of parameters" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f52ce2fc:2d64929:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3517 --></div><!-- react-empty: 3518 --><div class="content"><a href="http://neurostars.org/p/3695/" target="_blank" rel="noopener" class="title">Nipype: iterables over sets of parameters</a><div class="summary">Hi, I am trying to use a Node `iterables` attribute to loop over sets of parameters (a list of list of 2-tuple of str). However I get a TypeError when I do `node.run()`. Here is a running example: ``` from nipype import Function, Node records = [[('sid', '001'), ('diag', 'ad1')], [('sid', '002'), ('diag', 'ad2')], [('sid', '003'), ('diag', 'ad3')], [('sid', '004'), ('diag', 'ad4')], ] out_fields =</div></div><span class="ago" title="published:Thu, 18 Feb 2016 14:57:54 GMT -- Received: Thu, 18 Feb 2016 16:18:25 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f4877dca:29e3371:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3694/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f4877dca:29e3371:fb0a47d9" data-navigation="inline" data-title="Job: Scientific Funding Officer" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f4877dca:29e3371:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3533 --></div><!-- react-empty: 3534 --><div class="content"><a href="http://neurostars.org/p/3694/" target="_blank" rel="noopener" class="title">Job: Scientific Funding Officer</a><div class="summary">Karolinska Institutet, INCF The INCF Secretariat is hiring a Scientific Funding Officer Join the INCF Secretariat at the Karolinska Institute in Stockholm, Sweden. The International Neuroinformatics Coordinating Facility is an OECD-launched international organization, which coordinates standards for neuroscience including an international collaborative research data infrastructure. The Secretariat</div></div><span class="ago" title="published:Thu, 18 Feb 2016 12:47:10 GMT -- Received: Thu, 18 Feb 2016 13:17:46 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f0a456d6:1588220:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3689/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f0a456d6:1588220:fb0a47d9" data-navigation="inline" data-title="Nipype: report.rst from Node to outputs?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152f0a456d6:1588220:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3549 --></div><!-- react-empty: 3550 --><div class="content"><a href="http://neurostars.org/p/3689/" target="_blank" rel="noopener" class="title">Nipype: report.rst from Node to outputs?</a><div class="summary"> Hi, I'm wondering if there is a relatively simple way to direct a Node's report.rst file to its own output (or if this makes any sense) during execution, so it's possible to save it to a specific output folder through a DataSink for QC purposes. These files are nicely formatted and make it easy (at least to me) to check the actual node inputs, execution inputs and outputs, and the standard output</div></div><span class="ago" title="published:Wed, 17 Feb 2016 18:44:33 GMT -- Received: Wed, 17 Feb 2016 19:10:48 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152e7998a3c:963be71:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3685/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152e7998a3c:963be71:bc27e115" data-navigation="inline" data-title="Does bedpostx_parallel workflow respect n_fibres parameter?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152e7998a3c:963be71:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3565 --></div><!-- react-empty: 3566 --><div class="content"><a href="http://neurostars.org/p/3685/" target="_blank" rel="noopener" class="title">Does bedpostx_parallel workflow respect n_fibres parameter?</a><div class="summary"> I have just run a larger workflow that includes the bedpostx_parallel workflow from nipype workflows modules. In particular I have taken the code straight from the source for the setup as in: params = dict(n_fibres = 3, fudge = 1, burn_in = 1000, n_jumps = 1250, sample_every = 25) bpwf = bedpostx_parallel('nipype_bedpostx_parallel', params) with the only change being the number of fibres set to 3</div></div><span class="ago" title="published:Mon, 15 Feb 2016 23:20:18 GMT -- Received: Tue, 16 Feb 2016 01:02:25 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152d2537442:3301828:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3684/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152d2537442:3301828:ae8aca10" data-navigation="inline" data-title="Nipype - strange AttributeError for SpecifyModel interface" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152d2537442:3301828:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3581 --></div><!-- react-empty: 3582 --><div class="content"><a href="http://neurostars.org/p/3684/" target="_blank" rel="noopener" class="title">Nipype - strange AttributeError for SpecifyModel interface</a><div class="summary">I am trying to run this pipeline , but I am getting the following crashes for all my SpecifyModel nodes: File: crash-20160211-193523-chymera-specify_model.a3.pklz Node: FSL_GLM_work.first_level.specify_model.a3 Working directory: /home/chymera/NIdata/ofM.dr/FSL_GLM_work/first_level/_measurement_id_20151110_210648_4001_1_$/specify_model Node inputs: event_files = functional_runs = ['/mnt/data9/NIda</div></div><span class="ago" title="published:Thu, 11 Feb 2016 19:07:36 GMT -- Received: Thu, 11 Feb 2016 21:53:50 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152ca8f036e:8496a2:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3680/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152ca8f036e:8496a2:ae8aca10" data-navigation="inline" data-title="Create nipype interface outputting a list of Bunch() elements" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152ca8f036e:8496a2:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3597 --></div><!-- react-empty: 3598 --><div class="content"><a href="http://neurostars.org/p/3680/" target="_blank" rel="noopener" class="title">Create nipype interface outputting a list of Bunch() elements</a><div class="summary">I want to create a node that can use a delay piped from another node to compute a list of Bunch elements that I can use as subject_information input for a SpecifyModel() node. The interface looks as follows: class MakeSubjectInfoInputSpec(BaseInterfaceInputSpec): conditions = traits.List(traits.Str(exists=True)) durations = traits.List(traits.List(traits.Float(exists=True))) measurement_delay = tr</div></div><span class="ago" title="published:Wed, 10 Feb 2016 07:45:47 GMT -- Received: Wed, 10 Feb 2016 09:41:56 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c9ea3255:51d31b:ae8aca10_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3676/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c9ea3255:51d31b:ae8aca10" data-navigation="inline" data-title="Using nipype in a virtual environment - symlink problem" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c9ea3255:51d31b:ae8aca10" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3613 --></div><!-- react-empty: 3614 --><div class="content"><a href="http://neurostars.org/p/3676/" target="_blank" rel="noopener" class="title">Using nipype in a virtual environment - symlink problem</a><div class="summary">Hi all, I'm trying to get nipype running on an lubuntu (14.04) virtual box (Oracle VM) and run into an error when collecting data and making symlinks for the first part of the pipeline (see error below). I've checked that I've got the right (write?) permissions and don't have a clue what could be causing it. Suggestions would be greatly appreciated, Thanks, Peter 160210-12:48:03,191 workflow ERROR</div></div><span class="ago" title="published:Wed, 10 Feb 2016 04:10:59 GMT -- Received: Wed, 10 Feb 2016 06:41:54 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c60baf81:1028967b:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3670/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c60baf81:1028967b:bc27e115" data-navigation="inline" data-title="NiPype delete all but datasink" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c60baf81:1028967b:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3629 --></div><!-- react-empty: 3630 --><div class="content"><a href="http://neurostars.org/p/3670/" target="_blank" rel="noopener" class="title">NiPype delete all but datasink</a><div class="summary"> For one particular pipeline, I would like to delete all of the files but the datasink. I can do this quite easily via: if not debug_mode: import shutil shutil.rmtree(workflow_base+"/"+workflow_denominator+"_work") But this seems a bit hackish, AND has the added weaknesses that I risk deleting everything in my workflow_base if I make an unfortunate typo when edditing my code, and that, should anyb</div></div><span class="ago" title="published:Tue, 09 Feb 2016 12:35:19 GMT -- Received: Tue, 09 Feb 2016 12:40:00 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c60baf81:1028967a:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3669/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c60baf81:1028967a:bc27e115" data-navigation="inline" data-title="Forum: Beginner in Nilearn, Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152c60baf81:1028967a:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3645 --></div><!-- react-empty: 3646 --><div class="content"><a href="http://neurostars.org/p/3669/" target="_blank" rel="noopener" class="title">Forum: Beginner in Nilearn, Nipype</a><div class="summary"> Hi All, I am a medical graduate. Recently finished the codeacademy course on Python and took some lessons in Scikit-learn (Numpy, SciPy, Matplotlib). I have past experience working in Neuro-imaging with CT scans and MRIs, but not fMRI. Before starting codeacademy, I knew I was doing it to get here(NiPy), but I still feel quite lost. I can't find a bundle of pre-requisites that will help me unders</div></div><span class="ago" title="published:Tue, 09 Feb 2016 11:04:19 GMT -- Received: Tue, 09 Feb 2016 12:40:00 GMT">9mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152ac379a52:878889c:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3665/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152ac379a52:878889c:bc27e115" data-navigation="inline" data-title="How (and where) to save nifti-output from custom node in Nipype workflow ?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152ac379a52:878889c:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3661 --></div><!-- react-empty: 3662 --><div class="content"><a href="http://neurostars.org/p/3665/" target="_blank" rel="noopener" class="title">How (and where) to save nifti-output from custom node in Nipype workflow ?</a><div class="summary">First of all, a big thank you to the developers of the Nipype package - I find it, so far, an incredibly useful piece of software! Right now, I'm creating my first fMRI preprocessing workflow with mainly FSL-functions. I'm trying to mix in some custom (Map)Nodes, but I struggle to find out how I can save the result of my custom node (i.e. a nifti-image) in the 'current' folder of my workflow's wor</div></div><span class="ago" title="published:Thu, 04 Feb 2016 09:38:02 GMT -- Received: Thu, 04 Feb 2016 12:17:51 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152a32f7f66:56999eb:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3664/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152a32f7f66:56999eb:bc27e115" data-navigation="inline" data-title="Forum: OpenfMRI Dataset ds030: UCLA Consortium for Neuropsychiatric Phenomics LA5c Study" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152a32f7f66:56999eb:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3677 --></div><!-- react-empty: 3678 --><div class="content"><a href="http://neurostars.org/p/3664/" target="_blank" rel="noopener" class="title">Forum: OpenfMRI Dataset ds030: UCLA Consortium for Neuropsychiatric Phenomics LA5c Study</a><div class="summary"> Dear all, OpenfMRI is excited to announce the availability of the raw data acquired in the UCLA Consortium for Neuropsychiatric Phenomics LA5c Study. The dataset is available the OpenfMRI with the accession number ds030 . We encourage those using the dataset to use this thread for comments, feedback, and discussions related to ds030. Please use the tags openfmri and ds030. Thanks! </div></div><span class="ago" title="published:Tue, 02 Feb 2016 17:37:05 GMT -- Received: Tue, 02 Feb 2016 18:12:25 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152a1e4d890:4eedf11:bc27e115_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3663/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152a1e4d890:4eedf11:bc27e115" data-navigation="inline" data-title="Nifty change datatype with Nibabel" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152a1e4d890:4eedf11:bc27e115" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3693 --></div><!-- react-empty: 3694 --><div class="content"><a href="http://neurostars.org/p/3663/" target="_blank" rel="noopener" class="title">Nifty change datatype with Nibabel</a><div class="summary"> Hi all, How do you change Nifty header values using Nibabel to store an nii image using a different greyscale encoding? particularly from FLOAT 32 to UCHAR 8, I could do it manually but seems like something that could already be in a library. Thanks. :D </div></div><span class="ago" title="published:Tue, 02 Feb 2016 11:27:17 GMT -- Received: Tue, 02 Feb 2016 12:11:15 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152897d55e1:e202ce:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3661/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152897d55e1:e202ce:fb0a47d9" data-navigation="inline" data-title="News: 2016 OHBM Hackathon Thursday, June 23rd-Saturday, June 25th, 2016" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152897d55e1:e202ce:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3709 --></div><!-- react-empty: 3710 --><div class="content"><a href="http://neurostars.org/p/3661/" target="_blank" rel="noopener" class="title">News: 2016 OHBM Hackathon Thursday, June 23rd-Saturday, June 25th, 2016</a><div class="summary"> Dear OHBM Community, Please mark your calendars for the 2016 OHBM Hackathon! We are pleased to announce that there will be a 2016 OHBM Hackathon Satellite meeting that will take place the weekend prior to the 2016 OHBM Annual Meeting . The Annual Meeting is being held on June 26-30 in Geneva, Switzerland, and the dates of the Hackathon meeting will be Thursday, June 23rd-Saturday, June 25th, 2016</div></div><span class="ago" title="published:Thu, 28 Jan 2016 17:31:39 GMT -- Received: Thu, 28 Jan 2016 18:27:18 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15288b0b343:126b1e5:70e34146_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3658/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15288b0b343:126b1e5:70e34146" data-navigation="inline" data-title="Discussion about XNAT" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15288b0b343:126b1e5:70e34146" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3725 --></div><!-- react-empty: 3726 --><div class="content"><a href="http://neurostars.org/p/3658/" target="_blank" rel="noopener" class="title">Discussion about XNAT</a><div class="summary"> Hi, I would like to hear your opinion on XNAT. How useful is XNAT for you? Is there any other similar product? How often do you use it? Do you think it is the right tool to create a clinical database for brain PET-MRI that medical students could use for their studies? Thanks. Cheers, Alex </div></div><span class="ago" title="published:Thu, 28 Jan 2016 11:49:32 GMT -- Received: Thu, 28 Jan 2016 14:43:47 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1527844b254:3626091:70e34146_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3652/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1527844b254:3626091:70e34146" data-navigation="inline" data-title="Mirroring nifti files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1527844b254:3626091:70e34146" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3741 --></div><!-- react-empty: 3742 --><div class="content"><a href="http://neurostars.org/p/3652/" target="_blank" rel="noopener" class="title">Mirroring nifti files</a><div class="summary"> Hi all, I am doing structural analysis of a few brain areas in MR images, which are in nifti format. My colleagues and I have a hypothesis of asymmetry, meaning region A in the left hemisphere is bigger/smaller than its counterpart in right hemisphere. I am doing manual tracing on this images, but I would like to be blind to the hemisphere. Do you know of any software or even some python script t</div></div><span class="ago" title="published:Mon, 25 Jan 2016 09:03:24 GMT -- Received: Mon, 25 Jan 2016 10:11:54 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1526c0611de:2b948d:fb0a47d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3650/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1526c0611de:2b948d:fb0a47d9" data-navigation="inline" data-title="nipype: Custom behavior for handling node or sub-workflow failures" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1526c0611de:2b948d:fb0a47d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3757 --></div><!-- react-empty: 3758 --><div class="content"><a href="http://neurostars.org/p/3650/" target="_blank" rel="noopener" class="title">nipype: Custom behavior for handling node or sub-workflow failures</a><div class="summary"> I'd like to add some custom behavior to workflow and node failures, and curious about the best way to go about this. I'm a proficient programmer, so I don't mind digging in--just curious if someone else has already solved this problem or has suggestions on where to start. My requirements: Rerun some nodes with different parameters if they crash. For other nodes, I'd like to send out a failure ema</div></div><span class="ago" title="published:Sat, 23 Jan 2016 00:59:57 GMT -- Received: Sat, 23 Jan 2016 01:08:03 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15269723758:c30950:ab4bea13_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3647/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15269723758:c30950:ab4bea13" data-navigation="inline" data-title="Tool: hansel: a Python module to import crumbs to find your file paths." data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15269723758:c30950:ab4bea13" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3773 --></div><!-- react-empty: 3774 --><div class="content"><a href="http://neurostars.org/p/3647/" target="_blank" rel="noopener" class="title">Tool: hansel: a Python module to import crumbs to find your file paths.</a><div class="summary"> Hi, This is a small tool that I have been cooking in my head for a long time. https://pypi.python.org/pypi/hansel I am actively using this to parse/read my data folders, where usually the file tree structure is well defined. For now I find it very convenient and it has given me unexpectedly good surprises. Although this feature is still not documented, you can also add `fnmatch` or `re.match` reg</div></div><span class="ago" title="published:Fri, 22 Jan 2016 11:37:25 GMT -- Received: Fri, 22 Jan 2016 13:07:19 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1525c55f38e:672421:f44c6242_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3640/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1525c55f38e:672421:f44c6242" data-navigation="inline" data-title="Correlation by hand vs. connectome_measure(kind=correlation)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1525c55f38e:672421:f44c6242" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3789 --></div><!-- react-empty: 3790 --><div class="content"><a href="http://neurostars.org/p/3640/" target="_blank" rel="noopener" class="title">Correlation by hand vs. connectome_measure(kind=correlation)</a><div class="summary"> Hi all, I have what seems a simple question : When I compute the correlation/connectivity of a timeseris y hand, I get values that are twice as high as when I use the connectome_measure.fit_transform. For instance, my code is: print(clean_ts.shape) cm1 = numpy.corrcoef(clean_ts.T) connectome_measure = ConnectivityMeasure(kind='correlation') correlation = connectome_measure.fit_transform([clean_ts</div></div><span class="ago" title="published:Tue, 19 Jan 2016 22:23:17 GMT -- Received: Wed, 20 Jan 2016 00:01:22 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15245c58ee2:230437e:84c006c5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3637/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15245c58ee2:230437e:84c006c5" data-navigation="inline" data-title="How to measure the difference of the spatial maps of two ICA studies?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15245c58ee2:230437e:84c006c5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3805 --></div><!-- react-empty: 3806 --><div class="content"><a href="http://neurostars.org/p/3637/" target="_blank" rel="noopener" class="title">How to measure the difference of the spatial maps of two ICA studies?</a><div class="summary"> Hi, I am trying to compare the spatial maps from two sets of independent components. They have been taken from the same subjects but from 2 different modalities (PET and rs-fMRI). My first approach was to calculate the Dice coefficient from the matching spatial maps. One of my problems is that fMRI is much more sensitive than PET. Hence I need to lower the threshold to create the spatial maps for</div></div><span class="ago" title="published:Fri, 15 Jan 2016 13:10:23 GMT -- Received: Fri, 15 Jan 2016 14:52:01 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1523d9e27bb:7d77c5a:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3636/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1523d9e27bb:7d77c5a:b18dfa55" data-navigation="inline" data-title="Job: Director of Machine Learning Core" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1523d9e27bb:7d77c5a:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3821 --></div><!-- react-empty: 3822 --><div class="content"><a href="http://neurostars.org/p/3636/" target="_blank" rel="noopener" class="title">Job: Director of Machine Learning Core</a><div class="summary">Title 42 Staff Scientist / Director Machine Learning Core National Institute of Mental Health, National Institutes of Health Department of Health and Human Services, Bethesda, MD The National Institute of Mental Health (NIMH), a major research component of the National Institutes of Health (NIH), and the Department of Health and Human Services (DHHS), is seeking exceptional candidates for the Dire</div></div><span class="ago" title="published:Wed, 13 Jan 2016 22:26:39 GMT -- Received: Thu, 14 Jan 2016 00:52:01 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1523c54685c:6dadfc0:d9bd156b_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3634/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1523c54685c:6dadfc0:d9bd156b" data-navigation="inline" data-title="Unusally high zstat values in seed connectivity anlysis" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1523c54685c:6dadfc0:d9bd156b" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3837 --></div><!-- react-empty: 3838 --><div class="content"><a href="http://neurostars.org/p/3634/" target="_blank" rel="noopener" class="title">Unusally high zstat values in seed connectivity anlysis</a><div class="summary"> Dear experts, I am doing resting-state functional connectivity study using seed-based connectivity methods. For sanity check, I placed the seed in PCC region for baseline conditions (n=15) to see whether the Default Mode Network appears. My pipeline uses first three Principal components from both WM and CSF region masks each (eroded), along with 6 movement regressors as nuisance regressors - With</div></div><span class="ago" title="published:Wed, 13 Jan 2016 16:58:03 GMT -- Received: Wed, 13 Jan 2016 18:51:50 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152391c0ec8:5c6528c:d9bd156b_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3633/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152391c0ec8:5c6528c:d9bd156b" data-navigation="inline" data-title="FSL Interaction event file" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152391c0ec8:5c6528c:d9bd156b" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3853 --></div><!-- react-empty: 3854 --><div class="content"><a href="http://neurostars.org/p/3633/" target="_blank" rel="noopener" class="title">FSL Interaction event file</a><div class="summary"> In FSL I can specify an "interaction EV." # Basic waveform shape (EV 5) # 0 : Square # 1 : Sinusoid # 2 : Custom (1 entry per volume) # 3 : Custom (3 column format) # 4 : Interaction # 10 : Empty (all zeros) I can see in nipype how to specify a EV using a 1col or 3col format, but I can't figure out how to specify an "interaction EV". Is it posible to specify an "interaction EV" using nipype? </div></div><span class="ago" title="published:Wed, 13 Jan 2016 03:01:37 GMT -- Received: Wed, 13 Jan 2016 03:51:26 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15235e39e9b:4acb1cb:d9bd156b_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3630/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15235e39e9b:4acb1cb:d9bd156b" data-navigation="inline" data-title="FSL randomise Image Exception : #22 :: ERROR: Could not open image, ERROR: Program failed" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15235e39e9b:4acb1cb:d9bd156b" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3869 --></div><!-- react-empty: 3870 --><div class="content"><a href="http://neurostars.org/p/3630/" target="_blank" rel="noopener" class="title">FSL randomise Image Exception : #22 :: ERROR: Could not open image, ERROR: Program failed</a><div class="summary"> Hey, I hope you can help me. I am trying to run randomise to find differences between two groups (patients vs controls) in their hippocampal connectivity (I ran individual analysis before and merged patients and controls together respectively). I used GLM same way as it says on the picture (except I did 1's and 2's in the first column to distinguish between two groups), merged all the controls an</div></div><span class="ago" title="published:Tue, 12 Jan 2016 11:12:08 GMT -- Received: Tue, 12 Jan 2016 12:50:55 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15232aa3078:3ffc90e:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3627/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15232aa3078:3ffc90e:b18dfa55" data-navigation="inline" data-title="FSL PPI pipeline" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15232aa3078:3ffc90e:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3885 --></div><!-- react-empty: 3886 --><div class="content"><a href="http://neurostars.org/p/3627/" target="_blank" rel="noopener" class="title">FSL PPI pipeline</a><div class="summary"> I’m trying to implement this [workflow]( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/PPIHowToRun ) as a nipype pipeline, but I can’t figure out how to specify a model where some regressors (the psychological regressor) are convolved with an HDR but not others (physiological regressor) or how to make interaction regressors. </div></div><span class="ago" title="published:Mon, 11 Jan 2016 19:54:44 GMT -- Received: Mon, 11 Jan 2016 21:49:20 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15232aa3078:3ffc90d:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3623/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15232aa3078:3ffc90d:b18dfa55" data-navigation="inline" data-title="Deconvolve a time series" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15232aa3078:3ffc90d:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3901 --></div><!-- react-empty: 3902 --><div class="content"><a href="http://neurostars.org/p/3623/" target="_blank" rel="noopener" class="title">Deconvolve a time series</a><div class="summary"> I would like to deconvolve the FSL HDR out of an extracted time series. Is there a tool in nipype to do that? Thanks, Dalton </div></div><span class="ago" title="published:Mon, 11 Jan 2016 19:03:17 GMT -- Received: Mon, 11 Jan 2016 21:49:20 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152320550e5:3bcf23f:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3622/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152320550e5:3bcf23f:b18dfa55" data-navigation="inline" data-title="Just do an FSL HDR convolution" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152320550e5:3bcf23f:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3917 --></div><!-- react-empty: 3918 --><div class="content"><a href="http://neurostars.org/p/3622/" target="_blank" rel="noopener" class="title">Just do an FSL HDR convolution</a><div class="summary"> I would like to do an HDR convolution on an event file and then be given just the regressor. I do not want that regressor to be immediately put in to a matrix file. For a PPI analysis, I need to do some post precessing on it before it goes in to the design matrix. How do I *just* do a HDR convolution and then see the resulting vector? Thanks, Dalton </div></div><span class="ago" title="published:Mon, 11 Jan 2016 18:45:22 GMT -- Received: Mon, 11 Jan 2016 18:49:15 GMT">10mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15210849858:80a326:11a027f8_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3617/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15210849858:80a326:11a027f8" data-navigation="inline" data-title="varcope = 0 in some places" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15210849858:80a326:11a027f8" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3933 --></div><!-- react-empty: 3934 --><div class="content"><a href="http://neurostars.org/p/3617/" target="_blank" rel="noopener" class="title">varcope = 0 in some places</a><div class="summary"> I've run a bunch of different models in the same pipeline be using different collections of event files. Most of them are fine, but for one of the models I get this error when combining across subjects : WARNING: The passed in varcope file, /data/WorkingDir/Model_004_S/MFX/MasterWorkfow/_con _17/varcopemerge/varcope1_merged.nii.gz, contains voxels inside the mask with zero (or negative) values. T</div></div><span class="ago" title="published:Tue, 05 Jan 2016 06:22:49 GMT -- Received: Tue, 05 Jan 2016 06:41:11 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1520e95ea54:367eb08:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3615/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1520e95ea54:367eb08:48f117c4" data-navigation="inline" data-title="Fooling a Nipype node into thinking it's complete?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1520e95ea54:367eb08:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3949 --></div><!-- react-empty: 3950 --><div class="content"><a href="http://neurostars.org/p/3615/" target="_blank" rel="noopener" class="title">Fooling a Nipype node into thinking it's complete?</a><div class="summary"> Hey Nipype Folks, Weird question: does anyone know of a way to fool a Nipype node in the working directory into thinking it has been completed upon a subsequent run? The situation is a nipype pipeline breaks on an interface. A manual fix is inserted into the broken interface's folder. I want the rest of the pipeline to run from there onward thinking that interace worked fine. Thoughts? Ideas? Tha</div></div><span class="ago" title="published:Mon, 04 Jan 2016 20:00:21 GMT -- Received: Mon, 04 Jan 2016 21:40:51 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1520df11450:3326cd4:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3614/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1520df11450:3326cd4:48f117c4" data-navigation="inline" data-title="Getting at searchlight values" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1520df11450:3326cd4:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3965 --></div><!-- react-empty: 3966 --><div class="content"><a href="http://neurostars.org/p/3614/" target="_blank" rel="noopener" class="title">Getting at searchlight values</a><div class="summary">Hi all and a happy, healthy new year! I am trying to get at the actual voxel values from a searchlight. Currently I am starting from the nilearn searchlight example because their code is so clean and well-documented :-) However, my goal is not to run a category decoding (using e.g. a Y-vector containing face/house labels), but more similar to representational similarity analysis. In short, I want </div></div><span class="ago" title="published:Mon, 04 Jan 2016 17:48:25 GMT -- Received: Mon, 04 Jan 2016 18:40:49 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152096e2abb:1f0d5a0:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3612/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152096e2abb:1f0d5a0:1ba91209" data-navigation="inline" data-title="FSL PPI eventfiels" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_152096e2abb:1f0d5a0:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3981 --></div><!-- react-empty: 3982 --><div class="content"><a href="http://neurostars.org/p/3612/" target="_blank" rel="noopener" class="title">FSL PPI eventfiels</a><div class="summary"> I have a good whole brain GLM pipeline, but I need to make event files for a PPI analysis. Are there any pipelines I could use as a template? </div></div><span class="ago" title="published:Sun, 03 Jan 2016 19:31:59 GMT -- Received: Sun, 03 Jan 2016 21:39:20 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151c522212b:74c076:33c701d4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3611/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151c522212b:74c076:33c701d4" data-navigation="inline" data-title="[$$-1080p-$$] Watch Alvin and the Chipmunks: The Road Chip/movie HD" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151c522212b:74c076:33c701d4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 3997 --></div><!-- react-empty: 3998 --><div class="content"><a href="http://neurostars.org/p/3611/" target="_blank" rel="noopener" class="title">[$$-1080p-$$] Watch Alvin and the Chipmunks: The Road Chip/movie HD</a><div class="summary">WATCH ~ Alvin and the Chipmunks: The Road Chip (2015) ~ : CLICK BELOW TO WATCH FULL MOVIE HD 1080P: ============================================= CLICK HERE &gt;&gt;&gt; http://tinyurl.com/qg2vgdt CLICK HERE &gt;&gt;&gt; http://tinyurl.com/qg2vgdt ============================================ Tag: Watch Alvin and the Chipmunks: The Road Chip (2015) Online Box Office Watch Alvin and the Chipmunks: The Road Chip (2015</div></div><span class="ago" title="published:Mon, 21 Dec 2015 13:29:39 GMT -- Received: Mon, 21 Dec 2015 15:22:07 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151c522212b:74c075:33c701d4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3608/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151c522212b:74c075:33c701d4" data-navigation="inline" data-title="Forum: [ VIOzz-o0o ] Star Wars: The Force Awakens full movie HD-2015" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151c522212b:74c075:33c701d4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4013 --></div><!-- react-empty: 4014 --><div class="content"><a href="http://neurostars.org/p/3608/" target="_blank" rel="noopener" class="title">Forum: [ VIOzz-o0o ] Star Wars: The Force Awakens full movie HD-2015</a><div class="summary"> [ uFo-12 ] Watch Star Wars The Force Awakens -- In a future where the population is divided by personality types, one woman branded "Watch Star Wars The Force Awakens " uncovers a conspiracy to eliminate all who bear her misfit . Watch Star Wars The Force Awakens Online Free on your computer in high definition quality without even having to spend a dime. ===============FREE SIGN UP or DOWNLOAD===</div></div><span class="ago" title="published:Mon, 21 Dec 2015 13:16:39 GMT -- Received: Mon, 21 Dec 2015 15:22:07 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151a81fae88:2399b04:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3605/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151a81fae88:2399b04:1ba91209" data-navigation="inline" data-title="Lock acquisition with portalocker.py" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151a81fae88:2399b04:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4029 --></div><!-- react-empty: 4030 --><div class="content"><a href="http://neurostars.org/p/3605/" target="_blank" rel="noopener" class="title">Lock acquisition with portalocker.py</a><div class="summary">Hi Nipype folks: I'm trying to use C-PAC for neuroimaging analysis, and it makes calls to portalocker.py. This routine calls fcntl.flock(...), which I gather means using f-lock processing. This doesn't work in our system, where we have NFS v3 and v4 in use (depending on server). When you run it, you get lock exception 37 ("no lock available"). Various web commentators suggest using fctnl locking i</div></div><span class="ago" title="published:Tue, 15 Dec 2015 22:02:26 GMT -- Received: Wed, 16 Dec 2015 00:10:27 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151a0fedd40:57a0ae8:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3601/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151a0fedd40:57a0ae8:3534b282" data-navigation="inline" data-title="Forum: Email sign up closed temporarily" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151a0fedd40:57a0ae8:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4045 --></div><!-- react-empty: 4046 --><div class="content"><a href="http://neurostars.org/p/3601/" target="_blank" rel="noopener" class="title">Forum: Email sign up closed temporarily</a><div class="summary"> Hello astrocytes, As some of you might have noticed, we had a brief Spam outbreak over the last weeks. Thanks to Istvan and others who raised the issue, we have now identified the issue and proceeded to close new Email Signups. If you are a new user and still want to login via your personal email (instead of via "social" accounts), please make sure you use Mozilla Persona. Best regards, Sentient </div></div><span class="ago" title="published:Mon, 14 Dec 2015 13:26:38 GMT -- Received: Mon, 14 Dec 2015 14:57:16 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151960b0017:268b472:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3600/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151960b0017:268b472:3534b282" data-navigation="inline" data-title="Forum: (Stefania LaVie Owen), preoccupied with boy issues" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151960b0017:268b472:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4061 --></div><!-- react-empty: 4062 --><div class="content"><a href="http://neurostars.org/p/3600/" target="_blank" rel="noopener" class="title">Forum: (Stefania LaVie Owen), preoccupied with boy issues</a><div class="summary"> The first scene of “Krampus” features the abominable hordes descending on the hapless employees of a mega-mart during Christmas season, pushing and shoving and biting their way toward flat-screen TVs and stuffed animals with feral mercilessness. I kind of hoped one of the stock boys would mutate into the holiday demon and trap the grunting, heaving shoppers inside the store like some sort of purg</div></div><span class="ago" title="published:Sat, 12 Dec 2015 11:31:42 GMT -- Received: Sat, 12 Dec 2015 11:54:42 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151960b0017:268b471:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3599/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151960b0017:268b471:3534b282" data-navigation="inline" data-title="Tutorial: The Big Short Movie Online" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151960b0017:268b471:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4077 --></div><!-- react-empty: 4078 --><div class="content"><a href="http://neurostars.org/p/3599/" target="_blank" rel="noopener" class="title">Tutorial: The Big Short Movie Online</a><div class="summary"> The long run will depend on how other competitors such as The Revenant perform with critics and award groups, and how much of the adult audience they peel away from The Big Short. Since this one is more about prestige than black ink, and since it’s $28 million budget and modest marketing mean the bar for acceptable box office performance isn’t set too high, there should be plenty of reason to exp</div></div><span class="ago" title="published:Sat, 12 Dec 2015 09:19:10 GMT -- Received: Sat, 12 Dec 2015 11:54:42 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151922d1bfb:14e513d:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3598/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151922d1bfb:14e513d:3534b282" data-navigation="inline" data-title="Forum: A ladder to cut free a tangled sail" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151922d1bfb:14e513d:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4093 --></div><!-- react-empty: 4094 --><div class="content"><a href="http://neurostars.org/p/3598/" target="_blank" rel="noopener" class="title">Forum: A ladder to cut free a tangled sail</a><div class="summary"> Hemsworth is best when he’s by himself — either barking orders or doing something physical. Of all the actors he shares scenes with, it’s Cillian Murphy as his second mate who brings out something resembling emotion. But we never really care about the lead, so there’s little hope that we’ll be interested in the rest of the men once it becomes solely about survival. http://ibt.mcu.edu.tw/in-the-he</div></div><span class="ago" title="published:Fri, 11 Dec 2015 15:22:40 GMT -- Received: Fri, 11 Dec 2015 17:53:28 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15190e2aaa1:d24123:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3596/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15190e2aaa1:d24123:48f117c4" data-navigation="inline" data-title="999IVEWRS In the Heart of the Sea Online Free 1080p HDRip" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15190e2aaa1:d24123:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4150 --></div><!-- react-empty: 4151 --><div class="content"><a href="http://neurostars.org/p/3596/" target="_blank" rel="noopener" class="title">999IVEWRS In the Heart of the Sea Online Free 1080p HDRip</a><div class="summary"> http://www.peoplememory.com/groups/vigaz-in-the-heart-of-the-sea-online-free-1080p Watch In the Heart of the Sea , Watch In the Heart of the Sea 2015 Full Movie Online, Watch In the Heart of the Sea Cop 2 2015 Full Movie , Watch In the Heart of the Sea Online ,Download , Watch In the Heart of the Sea Full Movie , Watch In the Heart of the Sea Online , Watch In the Heart of the Sea Film ,, Watch I</div></div><span class="ago" title="published:Fri, 11 Dec 2015 11:02:54 GMT -- Received: Fri, 11 Dec 2015 11:52:32 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15188753212:372c10d:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3590/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15188753212:372c10d:48f117c4" data-navigation="inline" data-title="iterables input from node" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15188753212:372c10d:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4166 --></div><!-- react-empty: 4167 --><div class="content"><a href="http://neurostars.org/p/3590/" target="_blank" rel="noopener" class="title">iterables input from node</a><div class="summary"> I would like to be able to split my workflow deending on a set of values or outputs produced by a node. A MapNode will not do - because I do not want the workflow to converge right after the node where I split it. I want it to stay split and give me different outputs which I can compare. It seems this request was quite aptly described here as well: https://groups.google.com/forum/#!msg/nipy-user/</div></div><span class="ago" title="published:Wed, 09 Dec 2015 19:41:22 GMT -- Received: Wed, 09 Dec 2015 20:36:00 GMT">11mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15183ed175d:1df9ad6:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3585/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15183ed175d:1df9ad6:48f117c4" data-navigation="inline" data-title="Accessing NIFTI header info at runtime in Nipype pipeline" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15183ed175d:1df9ad6:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4182 --></div><!-- react-empty: 4183 --><div class="content"><a href="http://neurostars.org/p/3585/" target="_blank" rel="noopener" class="title">Accessing NIFTI header info at runtime in Nipype pipeline</a><div class="summary"> Hello all, I have what I hope is an easy question w.r.t extracting header info from 4D NIFTI files being processed in a Nipype pipeline. I'm pulling a 4D NIFTI from XNAT (via custom source node), splitting it up into 3D files, and feeding it to SPM preprocessing nodes. I need to know how many 3D files are in the 4D (it can vary) but I haven't found an straight-forward 'nipype'-ey way of determini</div></div><span class="ago" title="published:Tue, 08 Dec 2015 21:34:37 GMT -- Received: Tue, 08 Dec 2015 23:28:52 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1517e1de450:5c8fede:a527185a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3584/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1517e1de450:5c8fede:a527185a" data-navigation="inline" data-title="nipype run dartel (existing template)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1517e1de450:5c8fede:a527185a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4198 --></div><!-- react-empty: 4199 --><div class="content"><a href="http://neurostars.org/p/3584/" target="_blank" rel="noopener" class="title">nipype run dartel (existing template)</a><div class="summary"> Dear Forum, I am trying to normalize my structural/functional data via Dartel (SPM12). I would like to 'dartel' the data to the ixi templates provided as part of the vbm toolbox. However, I could only find a way to create own templates from the segmented structurals, which I heard is prone to outliers, especially with sample sizes Best, Ralf. </div></div><span class="ago" title="published:Mon, 07 Dec 2015 18:19:24 GMT -- Received: Mon, 07 Dec 2015 20:24:26 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516e8e99c0:156a3f6:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3583/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516e8e99c0:156a3f6:3534b282" data-navigation="inline" data-title="Making a template using ANTS nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516e8e99c0:156a3f6:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4214 --></div><!-- react-empty: 4215 --><div class="content"><a href="http://neurostars.org/p/3583/" target="_blank" rel="noopener" class="title">Making a template using ANTS nipype</a><div class="summary">I have been trying to use the script buildtemplateparallel.sh from ANTS with nipype. There is a nipype interface / nipype / interfaces / ants /legacy.py , which contains a wrapper for buildtemplateparallel . The problem is that there is a mistake in the interface and not all of the input options from the original ANTS function are supported. In particular, in class buildtemplateparallelInputSpec ,</div></div><span class="ago" title="published:Fri, 04 Dec 2015 19:48:26 GMT -- Received: Fri, 04 Dec 2015 19:53:37 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516d44806b:d97111:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3582/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516d44806b:d97111:3534b282" data-navigation="inline" data-title="Convert antsIntroduction.sh call to antsRegistration and WarpTimeSeriesImageMultiTransform to antsApplytransforms" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516d44806b:d97111:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4230 --></div><!-- react-empty: 4231 --><div class="content"><a href="http://neurostars.org/p/3582/" target="_blank" rel="noopener" class="title">Convert antsIntroduction.sh call to antsRegistration and WarpTimeSeriesImageMultiTransform to antsApplytransforms</a><div class="summary"> I would like to run the following registration script via the antsRegistration comand: antsIntroduction.sh -d 3 -r atlas.nii.gz -i struct.nii.gz -o str2qbi -m 10x10x5 I guess this should go along the lines of: antsRegistration --float --collapse-output-transforms 1 --dimensionality 3 --initial-moving-transform [ atlas.nii.gz, struct.nii.gz, 1 ] --output [str2qbi] but antsRegistration needs more p</div></div><span class="ago" title="published:Fri, 04 Dec 2015 13:47:04 GMT -- Received: Fri, 04 Dec 2015 13:53:04 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516c9f8cf9:9b144b:a527185a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3580/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516c9f8cf9:9b144b:a527185a" data-navigation="inline" data-title="Job: Postdoc position: representation of gustatory stimuli as primary rewards" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1516c9f8cf9:9b144b:a527185a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4246 --></div><!-- react-empty: 4247 --><div class="content"><a href="http://neurostars.org/p/3580/" target="_blank" rel="noopener" class="title">Job: Postdoc position: representation of gustatory stimuli as primary rewards</a><div class="summary">Applications are invited for a postdoctoral researcher who will be in charge of a study on the representation of gustatory stimuli in the context of visual perception of food and other cues, as well as their hedonic and motivational potential. The study combines standard fMRI acquisition procedures with custom gustatory stimulation sequences. The project will be executed as a collaboration of the </div></div><span class="ago" title="published:Fri, 04 Dec 2015 08:59:08 GMT -- Received: Fri, 04 Dec 2015 10:52:54 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15164dd12de:cf3971b:a527185a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3578/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15164dd12de:cf3971b:a527185a" data-navigation="inline" data-title="VTK 5 to 6 migration issue" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15164dd12de:cf3971b:a527185a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4262 --></div><!-- react-empty: 4263 --><div class="content"><a href="http://neurostars.org/p/3578/" target="_blank" rel="noopener" class="title">VTK 5 to 6 migration issue</a><div class="summary"> I am trying to update mindboggle code (mindboggle.info) to migrate VTK from version 5 to version 6 according to http://www.vtk.org/Wiki/VTK/VTK_6_Migration/Removal_of_Update According to the example on that website: vtkDataObject* dobj = someAlgorithm-&gt;GetOutput(1); dobj-&gt;Update(); should become: someAlgorithm-&gt;Update(1); However, I don’t know how to deal with the following pattern: void PointAre</div></div><span class="ago" title="published:Wed, 02 Dec 2015 22:27:48 GMT -- Received: Wed, 02 Dec 2015 22:43:08 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151518667f7:6c4daf2:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3576/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151518667f7:6c4daf2:3534b282" data-navigation="inline" data-title="Fisher's exact test in a new Neuron paper" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151518667f7:6c4daf2:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4278 --></div><!-- react-empty: 4279 --><div class="content"><a href="http://neurostars.org/p/3576/" target="_blank" rel="noopener" class="title">Fisher's exact test in a new Neuron paper</a><div class="summary"> In a newly published Neuron paper titled "Correspondence between Resting-State Activity and Brain Gene Expression", the authors used Fisher's exact test to investigate the cell-type specificity. But there is not any detail how to calculate this p-value. According to my background, Fisher's exact test is used for test associations between two categorical variables. So what are the two variables in</div></div><span class="ago" title="published:Sun, 29 Nov 2015 03:11:13 GMT -- Received: Sun, 29 Nov 2015 04:35:41 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151473018ad:3ba6244:a527185a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3571/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151473018ad:3ba6244:a527185a" data-navigation="inline" data-title="what is the meaning of pre-pregistration ?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151473018ad:3ba6244:a527185a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4294 --></div><!-- react-empty: 4295 --><div class="content"><a href="http://neurostars.org/p/3571/" target="_blank" rel="noopener" class="title">what is the meaning of pre-pregistration ?</a><div class="summary"> When people talk about ways to avoid p-hacking, they talk about "pre-registration". What does that mean ? </div></div><span class="ago" title="published:Fri, 27 Nov 2015 02:30:14 GMT -- Received: Fri, 27 Nov 2015 04:25:13 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1513ce28323:4ea367:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3564/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1513ce28323:4ea367:3534b282" data-navigation="inline" data-title="Nipype: Deleting output after completion" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1513ce28323:4ea367:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4310 --></div><!-- react-empty: 4311 --><div class="content"><a href="http://neurostars.org/p/3564/" target="_blank" rel="noopener" class="title">Nipype: Deleting output after completion</a><div class="summary"> Hi, I'm trying to run a bit of a non-standard preprocessing pipeline ( https://www.dropbox.com/s/p7sd9vg5fckqbnt/graph.dot.png?dl=0 ) through SPM12 and have a problem when ever it finishes running the normalise to MNI node. In short, when I run the processing pipeline and normalisation finishes the output is deleted, except this only happens when the normalisation node is connected to another nod</div></div><span class="ago" title="published:Wed, 25 Nov 2015 01:36:51 GMT -- Received: Wed, 25 Nov 2015 04:24:16 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1513aecef9e:5b2515e:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3562/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1513aecef9e:5b2515e:1ba91209" data-navigation="inline" data-title="Running Freesurfer using Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1513aecef9e:5b2515e:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4326 --></div><!-- react-empty: 4327 --><div class="content"><a href="http://neurostars.org/p/3562/" target="_blank" rel="noopener" class="title">Running Freesurfer using Nipype</a><div class="summary"> Hello All, I have been using Freesurfer from a while with its normal software available. I am trying to implement it using Nipype in Python. I have the code which is as follows: http://nipy.org/nipype/users/examples/smri_freesurfer.html I am having difficulties probably setting it up as it is getting crashed as soon as I am running it. Can anyone help me setting it up! Thank you, Pratik </div></div><span class="ago" title="published:Tue, 24 Nov 2015 17:53:37 GMT -- Received: Tue, 24 Nov 2015 19:16:25 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151366ac3f0:425b017:a527185a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3560/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151366ac3f0:425b017:a527185a" data-navigation="inline" data-title="Multiple Comparisons Correction" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_151366ac3f0:425b017:a527185a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4342 --></div><!-- react-empty: 4343 --><div class="content"><a href="http://neurostars.org/p/3560/" target="_blank" rel="noopener" class="title">Multiple Comparisons Correction</a><div class="summary"> What are the best interface for doing multiple comparisons correction? I found this entry on how to do it in FSL, but I can't find the nipype interfaces to actually use it in my pipeline. Dalton </div></div><span class="ago" title="published:Mon, 23 Nov 2015 21:47:05 GMT -- Received: Mon, 23 Nov 2015 22:15:45 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15126f5c0c0:14d13ec:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3555/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15126f5c0c0:14d13ec:48f117c4" data-navigation="inline" data-title="Pre-whitening for Resting-State data" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15126f5c0c0:14d13ec:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4358 --></div><!-- react-empty: 4359 --><div class="content"><a href="http://neurostars.org/p/3555/" target="_blank" rel="noopener" class="title">Pre-whitening for Resting-State data</a><div class="summary"> Hi, In one of the forums (FSL-jiscmail), I've read that it is recommended NOT to pre-whiten the resting state fMRI data, as it can erase the low frequency component of the signal that is deemed important in the resting state studies. Here are two questions related to that 1. I am trying to understand (intuitively, but mathematical explanations very welcome) how removing auto-correlation in the er</div></div><span class="ago" title="published:Fri, 20 Nov 2015 22:12:24 GMT -- Received: Fri, 20 Nov 2015 22:13:38 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1511ca7ea80:ee2fc55:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3551/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1511ca7ea80:ee2fc55:1ba91209" data-navigation="inline" data-title="sorting images from a 4d nifti file" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1511ca7ea80:ee2fc55:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4374 --></div><!-- react-empty: 4375 --><div class="content"><a href="http://neurostars.org/p/3551/" target="_blank" rel="noopener" class="title">sorting images from a 4d nifti file</a><div class="summary"> Dear neurostars, I have a 4d nifti file and want to reorder the files so that the order is the same for all subjects. I have already the indices of the order I want on Python and was wondering if you could help me figure out the easiest way to accomplish this. Thanks </div></div><span class="ago" title="published:Wed, 18 Nov 2015 19:22:11 GMT -- Received: Wed, 18 Nov 2015 22:12:25 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1511108eb46:ad8151b:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3545/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1511108eb46:ad8151b:1ba91209" data-navigation="inline" data-title="ANTs registration command (and nipype binding)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1511108eb46:ad8151b:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4390 --></div><!-- react-empty: 4391 --><div class="content"><a href="http://neurostars.org/p/3545/" target="_blank" rel="noopener" class="title">ANTs registration command (and nipype binding)</a><div class="summary"> I would like to calculate a transformation matrix to register the mean of an EPI time series to a template brain. I understand that applying the transformation can be done via `antsApplyTransforms`. As for the registration, I am still not sure. Different sources recommend to use either `antsRegistration` (in nipype as `Registration`) or `antsintroduction.sh` (in nipype as `antsIntroduction`). Wha</div></div><span class="ago" title="published:Mon, 16 Nov 2015 15:15:32 GMT -- Received: Mon, 16 Nov 2015 16:02:55 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f68b7894:277fc3f:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3539/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f68b7894:277fc3f:1ba91209" data-navigation="inline" data-title="Could you add support for fslslice in Nipype?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f68b7894:277fc3f:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4406 --></div><!-- react-empty: 4407 --><div class="content"><a href="http://neurostars.org/p/3539/" target="_blank" rel="noopener" class="title">Could you add support for fslslice in Nipype?</a><div class="summary"> I want to split a 3D file into lots of 2D files (along z-axis). This can not be done using the interfaces.fsl.utils.Split(), which wraps fslsplit. Since FSL already have fslslice to do this, can you add support to this command in Nipype? </div></div><span class="ago" title="published:Wed, 11 Nov 2015 12:11:36 GMT -- Received: Wed, 11 Nov 2015 12:35:46 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f49c0910:1d581d6:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3534/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f49c0910:1d581d6:48f117c4" data-navigation="inline" data-title="Skullstrip or mask NIfTI time series" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f49c0910:1d581d6:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4422 --></div><!-- react-empty: 4423 --><div class="content"><a href="http://neurostars.org/p/3534/" target="_blank" rel="noopener" class="title">Skullstrip or mask NIfTI time series</a><div class="summary"> I have a 4d fMRI time series in NIfTI format. I would like to take the mean across the temporal dimension, and use that to skullstrip my entire time series. FSL's BET offers the -F option for this, however that simply takes the first image from the tme course and uses that as a template. I believe, however, that I would get better results if I took the mean image. Any idea how I can skullstrip ba</div></div><span class="ago" title="published:Wed, 11 Nov 2015 01:06:41 GMT -- Received: Wed, 11 Nov 2015 03:34:38 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f3f71ee1:19f0c8b:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3533/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f3f71ee1:19f0c8b:1ba91209" data-navigation="inline" data-title="Specify slice dimension for nipy's FmriRealign4d" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150f3f71ee1:19f0c8b:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4438 --></div><!-- react-empty: 4439 --><div class="content"><a href="http://neurostars.org/p/3533/" target="_blank" rel="noopener" class="title">Specify slice dimension for nipy's FmriRealign4d</a><div class="summary">I have a 4d NIfTI file which has the following header (as seen via fslhd): filename /mnt/data9/NIdata/ofM.dr/FSL_GLM_work/GLM/_measurement_id_20151103_144306_4008_1_1/BET/9_brain.nii.gz sizeof_hdr 348 data_type FLOAT32 dim0 4 dim1 64 dim2 32 dim3 14 dim4 1440 dim5 1 dim6 1 dim7 1 vox_units mm time_units s datatype 16 nbyper 4 bitpix 32 pixdim0 0.000000 pixdim1 3.125000 pixdim2 2.812500 pixdim3 5.0</div></div><span class="ago" title="published:Wed, 11 Nov 2015 00:34:20 GMT -- Received: Wed, 11 Nov 2015 00:34:30 GMT">12mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150eae9cef2:6a5181f:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3528/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150eae9cef2:6a5181f:3534b282" data-navigation="inline" data-title="FEAT error when using multiple models" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150eae9cef2:6a5181f:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4454 --></div><!-- react-empty: 4455 --><div class="content"><a href="http://neurostars.org/p/3528/" target="_blank" rel="noopener" class="title">FEAT error when using multiple models</a><div class="summary">I'm trying to modify a pipeline so that it can run multiple models on multiple subjects in parallel. I use a IdentityInterface to iterate over the models and subjects: modle_subj_source = pe.Node(interface=util.IdentityInterface(fields=['model_name', 'subject_id']), name="modle_subj_source") modle_subj_source.iterables = [('subject_id', PC.subject_list), ('model_name', PC.models_list)] And it runs</div></div><span class="ago" title="published:Mon, 09 Nov 2015 05:50:36 GMT -- Received: Mon, 09 Nov 2015 06:23:22 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150eae9cef2:6a5181e:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3527/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150eae9cef2:6a5181e:3534b282" data-navigation="inline" data-title="Tool: Razor Blades Delivered" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150eae9cef2:6a5181e:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4470 --></div><!-- react-empty: 4471 --><div class="content"><a href="http://neurostars.org/p/3527/" target="_blank" rel="noopener" class="title">Tool: Razor Blades Delivered</a><div class="summary"> Get best quality razor blades delivered to your door steps. Best quality products from American companies at Teeth Monster. </div></div><span class="ago" title="published:Mon, 09 Nov 2015 05:18:28 GMT -- Received: Mon, 09 Nov 2015 06:23:22 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150ea44eef3:677e745:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3526/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150ea44eef3:677e745:3534b282" data-navigation="inline" data-title="MRtrix MRConvert Dicom folder" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150ea44eef3:677e745:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4486 --></div><!-- react-empty: 4487 --><div class="content"><a href="http://neurostars.org/p/3526/" target="_blank" rel="noopener" class="title">MRtrix MRConvert Dicom folder</a><div class="summary"> Hello there, I am trying to figure out how to be able to pass a file folder as my input to the MRConvert Node that I have created to convert a Dicom folder into a T1.nii.gz. In the regular terminal all i have to do is mrconvert ''folder'' T1.nii.gz but for some reason I get the error Trait Error: The 'in_file' trait of a MRConvertInputSpec instance must be an existing file name, but a value of 'd</div></div><span class="ago" title="published:Mon, 09 Nov 2015 01:57:37 GMT -- Received: Mon, 09 Nov 2015 03:23:17 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150de08737c:30d3136:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3524/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150de08737c:30d3136:3534b282" data-navigation="inline" data-title="Problem with 3dfim+: Incompatible dimensions for vector multiplication" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150de08737c:30d3136:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4502 --></div><!-- react-empty: 4503 --><div class="content"><a href="http://neurostars.org/p/3524/" target="_blank" rel="noopener" class="title">Problem with 3dfim+: Incompatible dimensions for vector multiplication</a><div class="summary"> Hello, I am trying to run 3dfim+ to calculate a correlation map for a region of interest. I deobliqued the data (the functional and the ROI mask which was converted to the same functional space) with 3drefit -deoblique (because I was getting an error message about data being oblique). However the deobliqued functional image now doesn't have the time dimension and the 1D file I'm getting when extr</div></div><span class="ago" title="published:Fri, 06 Nov 2015 15:37:37 GMT -- Received: Fri, 06 Nov 2015 18:21:47 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150d83c6bf3:fc64c3:a527185a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3522/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150d83c6bf3:fc64c3:a527185a" data-navigation="inline" data-title="Job: OHSU Fair Neuroimaging Lab Research Associate" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150d83c6bf3:fc64c3:a527185a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4518 --></div><!-- react-empty: 4519 --><div class="content"><a href="http://neurostars.org/p/3522/" target="_blank" rel="noopener" class="title">Job: OHSU Fair Neuroimaging Lab Research Associate</a><div class="summary">Damien Fair’s Neuroimaging Lab is looking for a candidate with expertise in neuroimaging, particularly in the field of MRI. While the main focus of this position is in neuroimaging, candidates with experience in bioinformatics and software development are highly encouraged to apply. The position will support a large federally funded longitudinal study to assess the short- and long-term impact of s</div></div><span class="ago" title="published:Thu, 05 Nov 2015 14:45:58 GMT -- Received: Thu, 05 Nov 2015 15:20:49 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150d5a90195:218e54:a527185a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3520/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150d5a90195:218e54:a527185a" data-navigation="inline" data-title="Copy NIfTI header from one file to another" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150d5a90195:218e54:a527185a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4534 --></div><!-- react-empty: 4535 --><div class="content"><a href="http://neurostars.org/p/3520/" target="_blank" rel="noopener" class="title">Copy NIfTI header from one file to another</a><div class="summary"> I want to copy the header from one NIfTI file to another. As it stands I might want to do it a few times in the future as well. Is there any utility that can (help me) copy all the header and onyl the header from one NIfTI file to another? I know how to manually edit the header in nibabel, but copying everything field-by-field in a python script sounds like a rather roundabout way to go. Does nib</div></div><span class="ago" title="published:Thu, 05 Nov 2015 00:50:08 GMT -- Received: Thu, 05 Nov 2015 03:20:34 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150cdecfc0f:7acbab1:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3518/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150cdecfc0f:7acbab1:1ba91209" data-navigation="inline" data-title="Job: Las Vegas Payday Loans" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150cdecfc0f:7acbab1:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4550 --></div><!-- react-empty: 4551 --><div class="content"><a href="http://neurostars.org/p/3518/" target="_blank" rel="noopener" class="title">Job: Las Vegas Payday Loans</a><div class="summary"> Payday LV offers online payday loans in Nevada. Apply for Las Vegas payday loans and get fastest cash into your account. Hassle free payday loans without checking any account. </div></div><span class="ago" title="published:Tue, 03 Nov 2015 14:07:24 GMT -- Received: Tue, 03 Nov 2015 15:17:51 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150c96af56b:6264466:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3516/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150c96af56b:6264466:1ba91209" data-navigation="inline" data-title="Setting up Nipype's IdentityInterface and SelectFiles node for database with multiple subjects and scans" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150c96af56b:6264466:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4566 --></div><!-- react-empty: 4567 --><div class="content"><a href="http://neurostars.org/p/3516/" target="_blank" rel="noopener" class="title">Setting up Nipype's IdentityInterface and SelectFiles node for database with multiple subjects and scans</a><div class="summary"> Hi, I have been trying to configure nipype's identityinterface and the datagrabber/selectfiles node to pick up data from our database's specific structure. Specifically, I have to extract T1, T2 and DTI images for each of our subjects from the database. Roughly, the structure is as follows: base_directory/subj001/T1/scan_date1/*/*.dcm base_directory/subj001/T1/scan_date2/*/*.dcm base_directory/su</div></div><span class="ago" title="published:Mon, 02 Nov 2015 16:10:07 GMT -- Received: Mon, 02 Nov 2015 18:17:21 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150c4e8ca7b:47b3a2a:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3511/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150c4e8ca7b:47b3a2a:48f117c4" data-navigation="inline" data-title="Job: Software engineer for open-source development in Montreal" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150c4e8ca7b:47b3a2a:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4582 --></div><!-- react-empty: 4583 --><div class="content"><a href="http://neurostars.org/p/3511/" target="_blank" rel="noopener" class="title">Job: Software engineer for open-source development in Montreal</a><div class="summary">At the Neuroimaging Research Laboratory ( www.neuro.polymtl.ca ) based at Ecole Polytechnique, Université de Montréal, we work to better understand the brain and the spinal cord using state-of-the-art imaging techniques, with many potential medical applications. We are developing advanced analysis methods to improve the quality of MRI data, as well as to extract meaningful clinical information. We</div></div><span class="ago" title="published:Sun, 01 Nov 2015 19:03:17 GMT -- Received: Sun, 01 Nov 2015 21:16:41 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150be781230:2ce9302:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3506/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150be781230:2ce9302:48f117c4" data-navigation="inline" data-title="Resampling ROI from MNI space to subject's functional space problem" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150be781230:2ce9302:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4598 --></div><!-- react-empty: 4599 --><div class="content"><a href="http://neurostars.org/p/3506/" target="_blank" rel="noopener" class="title">Resampling ROI from MNI space to subject's functional space problem</a><div class="summary"> Hello, I am pretty new to fMRI data analysis and I have a problem with resampling my ROI masks to match subjects functional data. Functional data has a resolution of 3.4 x 3.4 x 3.4 voxels and additional has a fourth dimension. However, I created ROIs in 2mm MNI standard space. When I try to use AFNI’s maskave my script throws the error about not matching dimensions. i tried to use fsl 3dresample</div></div><span class="ago" title="published:Sat, 31 Oct 2015 13:41:17 GMT -- Received: Sat, 31 Oct 2015 15:15:51 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150b9f60b25:19e9a43:1ba91209_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3502/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150b9f60b25:19e9a43:1ba91209" data-navigation="inline" data-title="Surface smoothing in python" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150b9f60b25:19e9a43:1ba91209" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4614 --></div><!-- react-empty: 4615 --><div class="content"><a href="http://neurostars.org/p/3502/" target="_blank" rel="noopener" class="title">Surface smoothing in python</a><div class="summary"> Hi everyone, do you know of a good way to smooth fMRI data on a surface in pure python? I have the fMRI data (vertices x timepoints) as well as the vertex coordinates and faces as numpy arrays. With the gdist library from the virtual brain project (https://github.com/the-virtual-brain/tvb-geodesic) it is easy to find all vertices in a certain radius from a given vertex, along with their exact dis</div></div><span class="ago" title="published:Fri, 30 Oct 2015 15:56:03 GMT -- Received: Fri, 30 Oct 2015 18:15:21 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150b04cf9b7:d1a9604:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3498/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150b04cf9b7:d1a9604:48f117c4" data-navigation="inline" data-title="Job: Biomedical Data Engineer" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150b04cf9b7:d1a9604:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4630 --></div><!-- react-empty: 4631 --><div class="content"><a href="http://neurostars.org/p/3498/" target="_blank" rel="noopener" class="title">Job: Biomedical Data Engineer</a><div class="summary">The Center for Health Sciences at SRI International has an opening for a Biomedical Data Engineer. The successful candidate will work as a collaborative team member supporting SRI and Stanford scientists to push forward our vision of open and reproducible neuroscience research. This person will manage data and implement enhancements to a biomedical data repository and analysis platform that levera</div></div><span class="ago" title="published:Wed, 28 Oct 2015 19:44:12 GMT -- Received: Wed, 28 Oct 2015 21:14:06 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150ab2604f7:b620bae:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3492/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150ab2604f7:b620bae:3534b282" data-navigation="inline" data-title="Write stdout to file for FSL cluster" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150ab2604f7:b620bae:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4646 --></div><!-- react-empty: 4647 --><div class="content"><a href="http://neurostars.org/p/3492/" target="_blank" rel="noopener" class="title">Write stdout to file for FSL cluster</a><div class="summary"> After using the FSL cluster tool I wanted to write stdout to a file for easy access to GRF-corrected p-values and cluster sizes, and I couldn’t find anything in the documents that would do the job. I basically copied the code below from the FSL smoothest tool and added it to the cluster class definition in my local copy of /nipype/interfaces/fsl/model.py, and it seems to work. Maybe this (or some</div></div><span class="ago" title="published:Tue, 27 Oct 2015 19:38:26 GMT -- Received: Tue, 27 Oct 2015 21:13:27 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150aa807e2d:b22a134:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3491/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150aa807e2d:b22a134:3534b282" data-navigation="inline" data-title="Nipype: Problem with DataGrabber" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150aa807e2d:b22a134:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4662 --></div><!-- react-empty: 4663 --><div class="content"><a href="http://neurostars.org/p/3491/" target="_blank" rel="noopener" class="title">Nipype: Problem with DataGrabber</a><div class="summary">Hello, I am pretty new to Nipype and I hope you can help me. I have been fighting with DataGrabber for a while producing different variants of the script, yet none of them works properly. What I want to do is to grab just one subject file (possible that later I will run this script on a list of subjects, but for now it's just one) and run some analysis on them. Here's how I do it (first version): </div></div><span class="ago" title="published:Tue, 27 Oct 2015 16:13:00 GMT -- Received: Tue, 27 Oct 2015 18:12:39 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1509ee87958:74ef505:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3487/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1509ee87958:74ef505:3534b282" data-navigation="inline" data-title="Resample Voxels size in FSL" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1509ee87958:74ef505:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4678 --></div><!-- react-empty: 4679 --><div class="content"><a href="http://neurostars.org/p/3487/" target="_blank" rel="noopener" class="title">Resample Voxels size in FSL</a><div class="summary"> Hi all, I'm analysing a group of subjects and I noticed that for a couple of them the upsampling option of the scanner was enabled. Is there a way in fsl to downsample these volumes to the same voxel size of the rest of the group? </div></div><span class="ago" title="published:Sun, 25 Oct 2015 10:50:43 GMT -- Received: Sun, 25 Oct 2015 12:10:47 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1509baff0a4:684319f:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3482/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1509baff0a4:684319f:3534b282" data-navigation="inline" data-title="Getting ROIs in 2mm MNI standard space to functional space" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1509baff0a4:684319f:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4694 --></div><!-- react-empty: 4695 --><div class="content"><a href="http://neurostars.org/p/3482/" target="_blank" rel="noopener" class="title">Getting ROIs in 2mm MNI standard space to functional space</a><div class="summary"> Hi everyone, I have created a large number of ROIs that were made using fsl ImageMaths through Nipype, when I then tried using fsl ImageMeants I ran into the error of the images being in different spaces. the ROIs were made using fsl's MNI 2mm structural image. My bold images with the time series I want to extract are still in functional space. How do I go about getting these ROIs into their indi</div></div><span class="ago" title="published:Sat, 24 Oct 2015 18:51:29 GMT -- Received: Sat, 24 Oct 2015 21:10:10 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150991c7885:5c3a90f:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3481/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150991c7885:5c3a90f:48f117c4" data-navigation="inline" data-title="voir ce que sac desigual soldes je ai frappé l'endroit" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150991c7885:5c3a90f:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4710 --></div><!-- react-empty: 4711 --><div class="content"><a href="http://neurostars.org/p/3481/" target="_blank" rel="noopener" class="title">voir ce que sac desigual soldes je ai frappé l'endroit</a><div class="summary">Résistance sac Kolkata tout dépend de la façon dont le traitement technique. Il y avait de repenser notre stratégie sur desigual spain en faisant un concept international. Les surfaces supérieures par rapport à Internet par lequel vous pouvez acheter une véritable produit sont en utilisant les sites desigual manteaux communs. Cela signifie essentiellement que des point de vente desigual, alors qu'</div></div><span class="ago" title="published:Sat, 24 Oct 2015 08:01:50 GMT -- Received: Sat, 24 Oct 2015 09:09:51 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15092aba301:3bceff1:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3479/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15092aba301:3bceff1:48f117c4" data-navigation="inline" data-title="Tutorial: offrent Manteaux desigual avec les frais d'escompte" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15092aba301:3bceff1:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4726 --></div><!-- react-empty: 4727 --><div class="content"><a href="http://neurostars.org/p/3479/" target="_blank" rel="noopener" class="title">Tutorial: offrent Manteaux desigual avec les frais d'escompte</a><div class="summary">Étant donné que ces entreprises offrent d'importantes réductions dans leurs propres actifs, mais peuvent aussi se révéler une occasion de profit d'aubaine pour ceux qui sont prêts à obtenir et offrir. Vous allez acheter un faux, s'il vous plaît assurez desigual sac à main -vous que vous avez la même consistance et l'apparence. Ils étaient beaux, ils n'étaient pas juste moi. Habitudes il y a de nom</div></div><span class="ago" title="published:Fri, 23 Oct 2015 02:32:39 GMT -- Received: Fri, 23 Oct 2015 03:08:53 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508ece4a20:25b36a4:48f117c4_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3478/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508ece4a20:25b36a4:48f117c4" data-navigation="inline" data-title="vous devez desigual france tenir votre esprit de la fièvre" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508ece4a20:25b36a4:48f117c4" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4742 --></div><!-- react-empty: 4743 --><div class="content"><a href="http://neurostars.org/p/3478/" target="_blank" rel="noopener" class="title">vous devez desigual france tenir votre esprit de la fièvre</a><div class="summary">Les facteurs peuvent vraiment faire la place. Ni la volonté de solde leur meilleur vetement Desigual femme en solde ou la plaine peur d'encourir des dommages majeurs sur une piste de course avec congestionnés charge à l'autre vetement desigual femme . Duchesse infantile comparable à Cornwall, les boutiques en ligne les plus populaires dans le monde, Toutes les personnes qui ont utilisé le dit cett</div></div><span class="ago" title="published:Thu, 22 Oct 2015 07:00:09 GMT -- Received: Thu, 22 Oct 2015 09:08:15 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508c3af3a2:1947d12:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3472/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508c3af3a2:1947d12:3534b282" data-navigation="inline" data-title="Forum: FSL, average runs in functional native space" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508c3af3a2:1947d12:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4758 --></div><!-- react-empty: 4759 --><div class="content"><a href="http://neurostars.org/p/3472/" target="_blank" rel="noopener" class="title">Forum: FSL, average runs in functional native space</a><div class="summary"> I have a standard 3-level analysis in FSL but I want for an ROI analysis to use the average of all my runs in the native space (instead of in the standard space). I know that I have the standard2example_func.mat that I can use to revert the statistical map to functional space bbut I was wondering if there was a better way to do this. Alternatively, as I understand it, each run is registered to th</div></div><span class="ago" title="published:Wed, 21 Oct 2015 20:29:08 GMT -- Received: Wed, 21 Oct 2015 21:08:05 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15089a744d9:997cb5:3534b282_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3471/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15089a744d9:997cb5:3534b282" data-navigation="inline" data-title="où amarango soldes boutique desigual vous pouvez apprendre" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15089a744d9:997cb5:3534b282" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4774 --></div><!-- react-empty: 4775 --><div class="content"><a href="http://neurostars.org/p/3471/" target="_blank" rel="noopener" class="title">où amarango soldes boutique desigual vous pouvez apprendre</a><div class="summary">Créé à partir de rédaction à la retenue Lug doit en effet être d'or pommade ingestion traverse monde gamme la création de la pensée derrière la pensée puissante pour se cacher extérieure. Donc beaucoup plus âgé encore osent défier un desigual en solde rose, ne prenez pas la ligne dames, imprimé à la main à remplir avec Chic est le choix intelligent, et immédiatement battu le style doux qui ressemb</div></div><span class="ago" title="published:Wed, 21 Oct 2015 07:59:17 GMT -- Received: Wed, 21 Oct 2015 09:07:32 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508713df42:b80a436:5a7b0f5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3469/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508713df42:b80a436:5a7b0f5" data-navigation="inline" data-title="preprocessing pipeline FSL 5.0.6 --> FSL 5.0.8" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1508713df42:b80a436:5a7b0f5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4831 --></div><!-- react-empty: 4832 --><div class="content"><a href="http://neurostars.org/p/3469/" target="_blank" rel="noopener" class="title">preprocessing pipeline FSL 5.0.6 --&gt; FSL 5.0.8</a><div class="summary">As with an earlier post , I'm having trouble running my old pipeline on my new computer (that has a the newest version of FSL and the newest version nipype). This time, it's something to do with my preprocessing pipeline. When I pre-process the functional scan on the old computer, they run fine in FILM_GLS, but when I preprocess them on the new computer, the parameter estimate files produced by FI</div></div><span class="ago" title="published:Tue, 20 Oct 2015 20:58:48 GMT -- Received: Tue, 20 Oct 2015 21:07:17 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1507ffe094a:9015a67:5a7b0f5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3466/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1507ffe094a:9015a67:5a7b0f5" data-navigation="inline" data-title="Dipy example &quot;Tracking Quick Start&quot; throws an error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1507ffe094a:9015a67:5a7b0f5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4847 --></div><!-- react-empty: 4848 --><div class="content"><a href="http://neurostars.org/p/3466/" target="_blank" rel="noopener" class="title">Dipy example "Tracking Quick Start" throws an error</a><div class="summary"> I tried running the Tracking Quick Start example that goes with Dipy. However, the line csd_peaks = peaks_from_model(model=csd_model, data=data, sphere=sphere, mask=mask, relative_peak_threshold=.5, min_separation_angle=25, parallel=False) results in the following error message: --------------------------------------------------------------------------- LinAlgError Traceback (most recent call las</div></div><span class="ago" title="published:Mon, 19 Oct 2015 09:31:09 GMT -- Received: Mon, 19 Oct 2015 12:06:06 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1507654cf57:6526ad6:5a7b0f5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3463/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1507654cf57:6526ad6:5a7b0f5" data-navigation="inline" data-title="How do I connect tuples in Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1507654cf57:6526ad6:5a7b0f5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4863 --></div><!-- react-empty: 4864 --><div class="content"><a href="http://neurostars.org/p/3463/" target="_blank" rel="noopener" class="title">How do I connect tuples in Nipype</a><div class="summary"> Hi, I am working on a mrtrix3 pipeline, my problem is how do I connect my bvecs and bvals to the input grad_fsl: the input in MRTrix3BaseInputSpec is defined as: grad_fsl = traits.Tuple( File(exists=True), File(exists=True), argstr='-fslgrad %s %s', desc='(bvecs, bvals) dw gradient scheme (FSL format') I tried mapping.connect([(inputnode_within, brainmask,[(("bvecs","bvals")," grad_fsl ")])]) but</div></div><span class="ago" title="published:Sat, 17 Oct 2015 13:54:34 GMT -- Received: Sat, 17 Oct 2015 15:04:41 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1506df5a30d:3b52c53:5a7b0f5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3462/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1506df5a30d:3b52c53:5a7b0f5" data-navigation="inline" data-title="Convert CSV to NifTi" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1506df5a30d:3b52c53:5a7b0f5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4879 --></div><!-- react-empty: 4880 --><div class="content"><a href="http://neurostars.org/p/3462/" target="_blank" rel="noopener" class="title">Convert CSV to NifTi</a><div class="summary"> I would like to do some motion correction and statistic parametric mapping on an intrinsic optical imaging dataset. I was able to convert the dataset to CSV values, with a semicolon separator for the third dimension (the data is 2D with a third temporal dimension). I am guessing that I could pretend this is just a 1-slice MR dataset, and analize it with the software usually used for MRI data anal</div></div><span class="ago" title="published:Thu, 15 Oct 2015 23:19:04 GMT -- Received: Fri, 16 Oct 2015 00:03:46 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15066df289a:11b487a:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3460/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15066df289a:11b487a:b18dfa55" data-navigation="inline" data-title="Errors encountered running Label2Vol interface in nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15066df289a:11b487a:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4895 --></div><!-- react-empty: 4896 --><div class="content"><a href="http://neurostars.org/p/3460/" target="_blank" rel="noopener" class="title">Errors encountered running Label2Vol interface in nipype</a><div class="summary">Hi there, I'm a new user of nipype. My research group is currently using the software to create a neuroimaging pipeline which we plan on using to track the progression of Alzheimer's Disease. I'm trying to use the Label2Vol freesurfer interface to push labels from freesurfer space to our subjects' original T1 space but I'm not able to get it to run. The command itself works fine on my local machin</div></div><span class="ago" title="published:Wed, 14 Oct 2015 14:38:11 GMT -- Received: Wed, 14 Oct 2015 15:01:52 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15052427926:7a74c32:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3457/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15052427926:7a74c32:b18dfa55" data-navigation="inline" data-title="Nipype Beginner's Guide Normalize Example Error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15052427926:7a74c32:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4911 --></div><!-- react-empty: 4912 --><div class="content"><a href="http://neurostars.org/p/3457/" target="_blank" rel="noopener" class="title">Nipype Beginner's Guide Normalize Example Error</a><div class="summary">I'm very new to nipype and neuroimaging. Recently, I run the Normalize Example from Nipype Beginner's Guide(http://miykael.github.io/nipype-beginner-s-guide/normalize.html). There're 3 ways in this tutorial. 1. ANTs' complete normalization. 2. ANTs' partial normalization. 3. SPM's normalization. In my case, No.2 and No.3 methods are OK. But when I run ANTs' complete normalization, I got the follow</div></div><span class="ago" title="published:Sat, 10 Oct 2015 12:59:38 GMT -- Received: Sat, 10 Oct 2015 14:58:19 GMT">13mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1504efecb5b:6e2c484:4f18d3b0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3456/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1504efecb5b:6e2c484:4f18d3b0" data-navigation="inline" data-title="News: Brainhack events throughout North, Central, and South Americas during October 23, 24 &amp; 25, 2015" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1504efecb5b:6e2c484:4f18d3b0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4927 --></div><!-- react-empty: 4928 --><div class="content"><a href="http://neurostars.org/p/3456/" target="_blank" rel="noopener" class="title">News: Brainhack events throughout North, Central, and South Americas during October 23, 24 &amp; 25, 2015</a><div class="summary">Brainhack ( http://brainhack.org ) is a unique conference that convenes researchers from across the globe and a myriad of disciplines to work together on innovative projects related to neuroscience. Brainhack Americas ( http://brainhack.org/americas ) will unite several regional Brainhack events throughout North, Central, and South Americas during October 23, 24 &amp; 25, 2015. Having several simultan</div></div><span class="ago" title="published:Fri, 09 Oct 2015 22:02:17 GMT -- Received: Fri, 09 Oct 2015 23:45:32 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1504efecb5b:6e2c483:4f18d3b0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3455/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1504efecb5b:6e2c483:4f18d3b0" data-navigation="inline" data-title="Job: Scientist or Postdoctoral Fellow in Multimodal MRI Image Analysis" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1504efecb5b:6e2c483:4f18d3b0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4943 --></div><!-- react-empty: 4944 --><div class="content"><a href="http://neurostars.org/p/3455/" target="_blank" rel="noopener" class="title">Job: Scientist or Postdoctoral Fellow in Multimodal MRI Image Analysis</a><div class="summary">We are seeking a Scientist or Postdoctoral Fellow with a background in computer science, bioengineering or related fields. The successful candidate will contribute to research and development in the coherent and robust multi-modal analysis of longitudinal neuro or cardiac MR images collected across different sites. The research projects are funded in part by the NIH Big Data to Knowledge (BD2K) In</div></div><span class="ago" title="published:Fri, 09 Oct 2015 21:55:51 GMT -- Received: Fri, 09 Oct 2015 23:45:32 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15047e7dada:47b0f70:4f18d3b0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3454/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15047e7dada:47b0f70:4f18d3b0" data-navigation="inline" data-title="Datasink output - directory structure" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15047e7dada:47b0f70:4f18d3b0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4959 --></div><!-- react-empty: 4960 --><div class="content"><a href="http://neurostars.org/p/3454/" target="_blank" rel="noopener" class="title">Datasink output - directory structure</a><div class="summary">Dear all, I am currently undertaking my first steps in nipype and have produced a pipeline that iterates over subject_id and series_pattern in a dicom directory tree, converts specific dicom series to nifti and stores the nifti files in an output folder: https://gist.github.com/michimichi/0f4068712533de9bf2bb#file-dicomconversion-py I am using two iterables to loop over the dicom directory, and co</div></div><span class="ago" title="published:Thu, 08 Oct 2015 11:48:41 GMT -- Received: Thu, 08 Oct 2015 14:43:08 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15045546b06:39bba86:4f18d3b0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3453/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15045546b06:39bba86:4f18d3b0" data-navigation="inline" data-title="Resampling fmri image changing the voxel dimension using R." data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15045546b06:39bba86:4f18d3b0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4975 --></div><!-- react-empty: 4976 --><div class="content"><a href="http://neurostars.org/p/3453/" target="_blank" rel="noopener" class="title">Resampling fmri image changing the voxel dimension using R.</a><div class="summary"> I would like to resample a fmri image (nii or.img) with dimensions 53x63x46 to an new image with 256x256x256. In fact, I just need the new 3D array (matrix). If is not possible, do you know how SPM (matlab)does it? If possible, the code. Source image files, example: IMG Dimension (53x63x46) : https://www.dropbox.com/s/25oa0wr7prbcbl7/fica_3.img?dl=0 HDR: https://www.dropbox.com/s/4quyx60wraiw2o4/</div></div><span class="ago" title="published:Thu, 08 Oct 2015 00:39:00 GMT -- Received: Thu, 08 Oct 2015 02:42:51 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15037cc33b3:359b5d6:4f18d3b0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3451/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15037cc33b3:359b5d6:4f18d3b0" data-navigation="inline" data-title="News: Finance manager - Payday Loans Denver" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15037cc33b3:359b5d6:4f18d3b0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 4991 --></div><!-- react-empty: 4992 --><div class="content"><a href="http://neurostars.org/p/3451/" target="_blank" rel="noopener" class="title">News: Finance manager - Payday Loans Denver</a><div class="summary"> Looking for urgent solution to get rid of financial burden from you head . Furnish loans ready to fulfill your need in very less time frame. You just need to fill an Payday loans Denver application form online with in a minuted and get instant approval . </div></div><span class="ago" title="published:Mon, 05 Oct 2015 11:18:27 GMT -- Received: Mon, 05 Oct 2015 11:39:00 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15028f15c33:4841d84:4f18d3b0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3450/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15028f15c33:4841d84:4f18d3b0" data-navigation="inline" data-title="Nipype and SPM for Octave" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_15028f15c33:4841d84:4f18d3b0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5007 --></div><!-- react-empty: 5008 --><div class="content"><a href="http://neurostars.org/p/3450/" target="_blank" rel="noopener" class="title">Nipype and SPM for Octave</a><div class="summary"> I am trying to run the famous faces example script from nipype: http://nipy.org/nipype/users/examples/fmri_spm_face.html . It fails with the following error output: https://bpaste.net/show/b056e3601f66 I guess this is because nipype sends the commands directly to MATLAB and not to SPM.... Is there any way to get nipype to work with octave-SPM? </div></div><span class="ago" title="published:Fri, 02 Oct 2015 12:45:40 GMT -- Received: Fri, 02 Oct 2015 14:25:17 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150246dd0f2:3149d72:4f18d3b0_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3448/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150246dd0f2:3149d72:4f18d3b0" data-navigation="inline" data-title="yet another question about SpaceTimeRealigner and how to write files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_150246dd0f2:3149d72:4f18d3b0" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5023 --></div><!-- react-empty: 5024 --><div class="content"><a href="http://neurostars.org/p/3448/" target="_blank" rel="noopener" class="title">yet another question about SpaceTimeRealigner and how to write files</a><div class="summary"> Dear all, I'm still trying to figure out 4D realignment. I had SpaceTimeRealigner running, but I realized that I was only realigning within run, and not between them. Referring to the reference article, it says that it realigns multiple runs by computing a mean image within run after 4D realignment, realigning those mean images to each other, and then composing that transform with the within run </div></div><span class="ago" title="published:Thu, 01 Oct 2015 15:04:44 GMT -- Received: Thu, 01 Oct 2015 17:23:08 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501fea61a3:18000cd:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3443/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501fea61a3:18000cd:b18dfa55" data-navigation="inline" data-title="FSL, MRICRON &amp; Nipype IOError" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501fea61a3:18000cd:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5039 --></div><!-- react-empty: 5040 --><div class="content"><a href="http://neurostars.org/p/3443/" target="_blank" rel="noopener" class="title">FSL, MRICRON &amp; Nipype IOError</a><div class="summary">Hi. This might be a beginners mistake, since I am fairly new to Nipype and Python. I get these IOError (please see below) when I try to run the Nipype FSL functions (FLIRT &amp; BET) and Dcm2nii function from MRICRON in both Anaconda and IPyhon. With BET &amp; FLIRT i use more options but the Dcm2nii are the same as the example: http://nipy.org/nipype/interfaces/generated/nipype.interfaces.fsl.preprocess.</div></div><span class="ago" title="published:Wed, 30 Sep 2015 20:05:27 GMT -- Received: Wed, 30 Sep 2015 20:21:05 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501fea61a3:18000cc:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3442/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501fea61a3:18000cc:b18dfa55" data-navigation="inline" data-title="fmri_fsl.py example in 0.11.0" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501fea61a3:18000cc:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5055 --></div><!-- react-empty: 5056 --><div class="content"><a href="http://neurostars.org/p/3442/" target="_blank" rel="noopener" class="title">fmri_fsl.py example in 0.11.0</a><div class="summary"> I'm revisiting a pipeline that I build last year based on the fmri_fsl.py example . One of the connections (live 407 in the example) takes an output from FILMGLS called 'corrections'. That output doesn't exist in the 0.11 documentation. The pipeline ran fine in 0.10, but on my new machine with 0.11 the script throws an error when I try to get the 'corrections' from FILMGLS. What would be the easi</div></div><span class="ago" title="published:Wed, 30 Sep 2015 17:25:22 GMT -- Received: Wed, 30 Sep 2015 20:21:05 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501c0d0954:29ece3:b18dfa55_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3441/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501c0d0954:29ece3:b18dfa55" data-navigation="inline" data-title="Using AFNI's 3dDeconvolve on Sparse data, event related design" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1501c0d0954:29ece3:b18dfa55" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5071 --></div><!-- react-empty: 5072 --><div class="content"><a href="http://neurostars.org/p/3441/" target="_blank" rel="noopener" class="title">Using AFNI's 3dDeconvolve on Sparse data, event related design</a><div class="summary"> So I'm thinking of using AFNI's 3dDeconvolve on my seed time series for a PPI analysis in a nipype pipeline. The data are sparse , have a TR of 6.0 and a TA of 2.0. I'm a bit confused as to what would be the best way to set up the 3dDeconvolve command. What's the "optimal" way of doing this? </div></div><span class="ago" title="published:Tue, 29 Sep 2015 23:25:47 GMT -- Received: Wed, 30 Sep 2015 02:20:27 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1500580a632:17f37542:98c27739_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3440/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1500580a632:17f37542:98c27739" data-navigation="inline" data-title="using jackknife estimator to generate per-subject FDG-PET functional connectivity networks" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1500580a632:17f37542:98c27739" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5087 --></div><!-- react-empty: 5088 --><div class="content"><a href="http://neurostars.org/p/3440/" target="_blank" rel="noopener" class="title">using jackknife estimator to generate per-subject FDG-PET functional connectivity networks</a><div class="summary"> I encountered a paper with the following methods: "The 3D static PET image is obtained one per subject. Since we do not have multiple PET scans per subject, we couldn’t generate a single network per subject. So we used the Jackknife resampling technique to generate additional networks. For a group with n subjects, one subject is removed and the remaining n − 1 subjects are used to construct a net</div></div><span class="ago" title="published:Fri, 25 Sep 2015 15:35:11 GMT -- Received: Fri, 25 Sep 2015 17:15:28 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffdc61564:15dd1331:fccac72e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3438/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffdc61564:15dd1331:fccac72e" data-navigation="inline" data-title="How to do PPI analysis in FSL using Nipype?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffdc61564:15dd1331:fccac72e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5103 --></div><!-- react-empty: 5104 --><div class="content"><a href="http://neurostars.org/p/3438/" target="_blank" rel="noopener" class="title">How to do PPI analysis in FSL using Nipype?</a><div class="summary"> Hi, everyone, My question is about how to do PPI analysis using interfaces.fsl in nipype? Thank you very much! </div></div><span class="ago" title="published:Thu, 24 Sep 2015 04:16:13 GMT -- Received: Thu, 24 Sep 2015 05:14:21 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffdc61564:15dd1330:fccac72e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3437/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffdc61564:15dd1330:fccac72e" data-navigation="inline" data-title="How to do parameter estimate extraction and hemodynamic response extraction after finishing FSL GLM analysis using Nipype?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffdc61564:15dd1330:fccac72e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5119 --></div><!-- react-empty: 5120 --><div class="content"><a href="http://neurostars.org/p/3437/" target="_blank" rel="noopener" class="title">How to do parameter estimate extraction and hemodynamic response extraction after finishing FSL GLM analysis using Nipype?</a><div class="summary"> Hi, everyone, Just as the tile, does anyone know how to do parameter estimate extraction and hemodynamic response extraction after finishing FSL GLM analysis using Nipype? Thank you very much! </div></div><span class="ago" title="published:Thu, 24 Sep 2015 03:46:47 GMT -- Received: Thu, 24 Sep 2015 05:14:21 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffb328530:148c20ef:98c27739_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3433/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffb328530:148c20ef:98c27739" data-navigation="inline" data-title="what is the best way to extract/replicate a piece of pipeline (e.g. warpall from fmri_ants_openfmri.py)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ffb328530:148c20ef:98c27739" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5135 --></div><!-- react-empty: 5136 --><div class="content"><a href="http://neurostars.org/p/3433/" target="_blank" rel="noopener" class="title">what is the best way to extract/replicate a piece of pipeline (e.g. warpall from fmri_ants_openfmri.py)</a><div class="summary"> We would like to extend the pipeline with applying derived flirt+ants transformations to input files and copes obtained at the very first (within each run) level. Sure thing we could probably just copy/paste piece of the pipeline, and pick up transformations from the DataSink (or output of the registration pipeline) but I wondered if there is some more kosher way where we locate (programmatically</div></div><span class="ago" title="published:Wed, 23 Sep 2015 15:08:22 GMT -- Received: Wed, 23 Sep 2015 17:13:56 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ff6ae4df0:137e6919:fccac72e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3430/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ff6ae4df0:137e6919:fccac72e" data-navigation="inline" data-title="SpaceTimeRealigner: raise ValueError(&quot;slice times should be smaller than repetition time&quot;)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ff6ae4df0:137e6919:fccac72e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5151 --></div><!-- react-empty: 5152 --><div class="content"><a href="http://neurostars.org/p/3430/" target="_blank" rel="noopener" class="title">SpaceTimeRealigner: raise ValueError("slice times should be smaller than repetition time")</a><div class="summary">Dear all, I'm now trying to run SpaceTimeRealigner (I installed NiPy from the github master). This is the code snippet I am using to run it: for sub in subjects.keys(): for run in subjects[sub]: infile = os.path.join(root, sub, 'func' , run, '4D.nii' ) realigner = SpaceTimeRealigner() realigner.inputs.in_file = [infile] realigner.inputs.tr = 2.2 realigner.inputs.slice_times = range(0, 34, 2) + ran</div></div><span class="ago" title="published:Tue, 22 Sep 2015 18:00:17 GMT -- Received: Tue, 22 Sep 2015 20:11:02 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd9085dec:9fa7612:98c27739_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3426/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd9085dec:9fa7612:98c27739" data-navigation="inline" data-title="Forum: Plan: Nipype on Python 3" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd9085dec:9fa7612:98c27739" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5167 --></div><!-- react-empty: 5168 --><div class="content"><a href="http://neurostars.org/p/3426/" target="_blank" rel="noopener" class="title">Forum: Plan: Nipype on Python 3</a><div class="summary"> Hi, I want to revive this question: https://neurostars.org/p/2311/ I'm willing to work on nipype to make it compatible with Python 3 as well. Unless someone else is already on that, please let me know and I could help. There are two options: - six (pros: very lightweigt and supports 2.5+ and 3.x; cons: more work for me and a bit for the rest of developers afterwards) - python-future (pros: easy t</div></div><span class="ago" title="published:Wed, 16 Sep 2015 23:20:13 GMT -- Received: Thu, 17 Sep 2015 02:00:48 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd717df66:9505aa4:98c27739_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3425/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd717df66:9505aa4:98c27739" data-navigation="inline" data-title="Push files from nipype to XNAT (XNATsink)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd717df66:9505aa4:98c27739" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5183 --></div><!-- react-empty: 5184 --><div class="content"><a href="http://neurostars.org/p/3425/" target="_blank" rel="noopener" class="title">Push files from nipype to XNAT (XNATsink)</a><div class="summary">Hi There, After retrieving data form XNAT (via XNATsource) I run a nipype pipeline on it and want to push it then back to XNAT (via XNATsink). However the last interface fails: &gt;URI references data outside of the archive. ______________________________________________________________ dataSink1 = pe.Node(io.DataSink(), name='datasink') dataSink1.inputs.base_directory = os.path.join(expDir, outputDi</div></div><span class="ago" title="published:Wed, 16 Sep 2015 15:26:32 GMT -- Received: Wed, 16 Sep 2015 16:58:30 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd3df57dd:861092a:fccac72e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3424/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd3df57dd:861092a:fccac72e" data-navigation="inline" data-title="News: Nipype 0.11 was just released" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd3df57dd:861092a:fccac72e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5199 --></div><!-- react-empty: 5200 --><div class="content"><a href="http://neurostars.org/p/3424/" target="_blank" rel="noopener" class="title">News: Nipype 0.11 was just released</a><div class="summary"> Dear all, I'm very proud to announce the next release of Nipype - 0.11. This release brings many improvements in support of FSL, SPM (especially SPM12), AFNI, ANTs, and MRTrix (especially MRTrix3). Among new features we introduced support for JSON , more efficient execution on SLURM clusters , and a new tool - nipype_cmd - that allows you to run any Nipype interfaces on the command line. the last</div></div><span class="ago" title="published:Tue, 15 Sep 2015 23:21:28 GMT -- Received: Wed, 16 Sep 2015 01:57:53 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd2954dc4:7cbccde:98c27739_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3423/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd2954dc4:7cbccde:98c27739" data-navigation="inline" data-title="two questions about simultaneous realignment and slice timing in NiPype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fd2954dc4:7cbccde:98c27739" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5215 --></div><!-- react-empty: 5216 --><div class="content"><a href="http://neurostars.org/p/3423/" target="_blank" rel="noopener" class="title">two questions about simultaneous realignment and slice timing in NiPype</a><div class="summary">Dear Neurostars, I am trying to perform simultaneous slice time correction and realignment with NiPype using FmriRealign4d, documented here . My first question is what is the difference between FmriRealign4d and SpaceTimeRealigner? My second question is this: I have FmriRealign4d apparently running, in that it spits out a lot of console output, but I cannot discern that it is making any modificati</div></div><span class="ago" title="published:Tue, 15 Sep 2015 17:16:46 GMT -- Received: Tue, 15 Sep 2015 19:57:24 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fbe9dfe85:1cd8b98:98c27739_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3417/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fbe9dfe85:1cd8b98:98c27739" data-navigation="inline" data-title="Need help returning interface to original state" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fbe9dfe85:1cd8b98:98c27739" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5231 --></div><!-- react-empty: 5232 --><div class="content"><a href="http://neurostars.org/p/3417/" target="_blank" rel="noopener" class="title">Need help returning interface to original state</a><div class="summary">Hi Neurostars, I'm using the FSL cluster tool via lyman (http://web.stanford.edu/~mwaskom/software/lyman/index.html), and I tweaked its specification in nipype/interfaces/fsl/model.py so that I could capture stdout (I've listed the specific changes I made at the end of this message). This worked but it caused other problems and I'd like to revert the cluster tool back to its original state. I trie</div></div><span class="ago" title="published:Fri, 11 Sep 2015 22:18:58 GMT -- Received: Fri, 11 Sep 2015 22:54:29 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fb787997d:1a2bc3d:cef97aaf_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3413/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fb787997d:1a2bc3d:cef97aaf" data-navigation="inline" data-title="Question about transforming the BBRegister transformation matrix to ITK format with C3dAffineTool" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fb787997d:1a2bc3d:cef97aaf" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5247 --></div><!-- react-empty: 5248 --><div class="content"><a href="http://neurostars.org/p/3413/" target="_blank" rel="noopener" class="title">Question about transforming the BBRegister transformation matrix to ITK format with C3dAffineTool</a><div class="summary">Hi, I want to transform the BBRegister transformation matrix to ITK format with C3dAffineTool. I installed the C3D followed the instructions in http://miykael.github.io/nipype-beginner-s-guide/normalize.html, and ran code like: c3=c3.C3dAffineTool(source_file='/data2/sub1101/results/epi2struct_init.mat', itk_transform = 'affine.txt', fsl2ras = True) c3_resuts=c3.run() c3_results.outputs An error o</div></div><span class="ago" title="published:Thu, 10 Sep 2015 11:17:46 GMT -- Received: Thu, 10 Sep 2015 13:52:41 GMT">14mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fad7874fd:1bdf48cd:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3406/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fad7874fd:1bdf48cd:46f3bb90" data-navigation="inline" data-title="Pros and cons of single-subject designs in fMRI" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fad7874fd:1bdf48cd:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5263 --></div><!-- react-empty: 5264 --><div class="content"><a href="http://neurostars.org/p/3406/" target="_blank" rel="noopener" class="title">Pros and cons of single-subject designs in fMRI</a><div class="summary"> Can someone point out good (or standard) references that discuss the pros and cons of single-subject experimental designs in fMRI? </div></div><span class="ago" title="published:Tue, 08 Sep 2015 14:35:10 GMT -- Received: Tue, 08 Sep 2015 14:59:56 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14faae4131f:1b964de7:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3405/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14faae4131f:1b964de7:1cbd0d70" data-navigation="inline" data-title="about dipy's peaks function" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14faae4131f:1b964de7:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5279 --></div><!-- react-empty: 5280 --><div class="content"><a href="http://neurostars.org/p/3405/" target="_blank" rel="noopener" class="title">about dipy's peaks function</a><div class="summary">hello, I have puzzle with dipy, Could you help me? As for dipy Examples in Reconstruction like CSD: in dipy, we usually reconstruct data with a model to get odf like this: &gt;&gt;data_small = data[20:50, 55:85, 38:39] &gt;&gt;csd_fit = csd_model.fit(data_small) &gt;&gt;csd_odf = csd_fit.odf(sphere) and if we want to get peaks of odf, we can : &gt;&gt;from dipy.direction import peaks_from_model &gt;&gt;csd_peaks = peaks_from_m</div></div><span class="ago" title="published:Tue, 08 Sep 2015 02:02:59 GMT -- Received: Tue, 08 Sep 2015 02:58:37 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fa284cfcb:1906de9a:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3402/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fa284cfcb:1906de9a:1cbd0d70" data-navigation="inline" data-title="N4BiasFieldCorrection: error while loading shared libraries" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fa284cfcb:1906de9a:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5295 --></div><!-- react-empty: 5296 --><div class="content"><a href="http://neurostars.org/p/3402/" target="_blank" rel="noopener" class="title">N4BiasFieldCorrection: error while loading shared libraries</a><div class="summary">I used ants.N4BiasFieldCorrection to do Bias field correction of the T1 and there is an error called "N4BiasFieldCorrection: error while loading shared libraries". The code and errors are attached below. If anyone know how to fixed it, pleasure reply. Thank you! Code: bias_field_correcton = ants.N4BiasFieldCorrection(input_image = t1file, dimension = 3,save_bias = True) bias_field_correcton_result</div></div><span class="ago" title="published:Sun, 06 Sep 2015 10:31:37 GMT -- Received: Sun, 06 Sep 2015 11:57:37 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fa284cfcb:1906de99:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3401/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fa284cfcb:1906de99:1cbd0d70" data-navigation="inline" data-title="Problem in Normalization (T1 to MNI corregistration) using ANTs" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14fa284cfcb:1906de99:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5311 --></div><!-- react-empty: 5312 --><div class="content"><a href="http://neurostars.org/p/3401/" target="_blank" rel="noopener" class="title">Problem in Normalization (T1 to MNI corregistration) using ANTs</a><div class="summary">I used ants.Registration to do normalization, the code and errors are attached below: Looking forward for your answer. Thank you Code: ants_reg = ants.Registration(fixed_image = os.path.join(os.getenv('FSLDIR'),'data/standard/MNI152_T1_2mm_brain.nii.gz'), moving_image = bet_results.outputs.out_file, transforms = ['Translation', 'Rigid', 'Affine', 'SyN'], transform_parameters = [(0.1,), (0.1,), (0.</div></div><span class="ago" title="published:Sun, 06 Sep 2015 10:27:29 GMT -- Received: Sun, 06 Sep 2015 11:57:37 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f8e8aba80:126840ca:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3393/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f8e8aba80:126840ca:46f3bb90" data-navigation="inline" data-title="nipype &quot;standard&quot; workflow to run 2nd level (across subjects) mixed effect (flame1) analysis using FSL" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f8e8aba80:126840ca:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5327 --></div><!-- react-empty: 5328 --><div class="content"><a href="http://neurostars.org/p/3393/" target="_blank" rel="noopener" class="title">nipype "standard" workflow to run 2nd level (across subjects) mixed effect (flame1) analysis using FSL</a><div class="summary"> At some point in the past I have used a gablab/level2vol.py functionality to drive 2nd level analyses in volume, with the hope that eventually something like that would appear within stock nipype codebase. But it seems that there remains only the example on how to do the 1st level analysis over openfmri layout data using FSL, but there is no example/script/demo/anything for the 2nd level (only SP</div></div><span class="ago" title="published:Wed, 02 Sep 2015 13:19:58 GMT -- Received: Wed, 02 Sep 2015 14:51:40 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f8aad72bc:111ee377:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3388/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f8aad72bc:111ee377:46f3bb90" data-navigation="inline" data-title="Questions about NiftiMasker" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f8aad72bc:111ee377:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5343 --></div><!-- react-empty: 5344 --><div class="content"><a href="http://neurostars.org/p/3388/" target="_blank" rel="noopener" class="title">Questions about NiftiMasker</a><div class="summary">Hi, I have a question about the Nilearn NiftiMasker arising from the tutorial about between-subject VBM analyses located at https://nilearn.github.io/auto_examples/decoding/plot_oasis_vbm.html . In the following section of the code, ### Preprocess data ########################################################### nifti_masker = NiftiMasker ( standardize=False, smoothing_fwhm=2, memory='nilearn_cache</div></div><span class="ago" title="published:Tue, 01 Sep 2015 18:11:37 GMT -- Received: Tue, 01 Sep 2015 20:51:07 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f88188531:10249805:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3384/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f88188531:10249805:46f3bb90" data-navigation="inline" data-title="Nibabel: Proper way to modify (and subsequently save) data of a Nifti" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f88188531:10249805:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5359 --></div><!-- react-empty: 5360 --><div class="content"><a href="http://neurostars.org/p/3384/" target="_blank" rel="noopener" class="title">Nibabel: Proper way to modify (and subsequently save) data of a Nifti</a><div class="summary"> For instance, this is a little example in which I create a Nifti image 'abs_image.nii' that takes the absolute value of another image 'image.nii': import nibabel img = nibabel.load('/path/to/image.nii') img.get_data()[:] = abs(img.get_data()) nibabel.save(img, '/path/to/abs_image.nii') This seems to work, but a) the syntax get_data()[:] feels a bit unusual and b) i haven't actually been able to c</div></div><span class="ago" title="published:Tue, 01 Sep 2015 08:05:36 GMT -- Received: Tue, 01 Sep 2015 08:49:12 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f74201770:a8b2e72:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3377/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f74201770:a8b2e72:1cbd0d70" data-navigation="inline" data-title="Nipype: Create SPM batch file from workflow" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f74201770:a8b2e72:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5375 --></div><!-- react-empty: 5376 --><div class="content"><a href="http://neurostars.org/p/3377/" target="_blank" rel="noopener" class="title">Nipype: Create SPM batch file from workflow</a><div class="summary"> Is it possible to generate an SPM batchfile from an SPM-based nipype workflow? That would be neat in order to directly compare nipype workflows to existing SPM batchfile-based analyses. Cheers, Matthias </div></div><span class="ago" title="published:Fri, 28 Aug 2015 09:07:39 GMT -- Received: Fri, 28 Aug 2015 11:45:04 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f7042e2bd:8fe74f6:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3375/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f7042e2bd:8fe74f6:46f3bb90" data-navigation="inline" data-title="Location for files in openfmri/BIDS layout for associated derived data from other sources" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f7042e2bd:8fe74f6:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5391 --></div><!-- react-empty: 5392 --><div class="content"><a href="http://neurostars.org/p/3375/" target="_blank" rel="noopener" class="title">Location for files in openfmri/BIDS layout for associated derived data from other sources</a><div class="summary"> If I want to provide some "ad-hoc" masks per each subject (so not per each run/session) in BOLD space which were derived from some other experiment/models, not shared within the same "dataset", what should be the logical location to place them? subXXX/BOLD/masks ? or better to come up with a new task "descriptor" (e.g. task 010) and probably a model (although without detailing its "design", e.g. </div></div><span class="ago" title="published:Thu, 27 Aug 2015 15:08:38 GMT -- Received: Thu, 27 Aug 2015 17:44:35 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f6b1bfbeb:737a4d4:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3370/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f6b1bfbeb:737a4d4:46f3bb90" data-navigation="inline" data-title="Scale voxel size in NIfTI header" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f6b1bfbeb:737a4d4:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5407 --></div><!-- react-empty: 5408 --><div class="content"><a href="http://neurostars.org/p/3370/" target="_blank" rel="noopener" class="title">Scale voxel size in NIfTI header</a><div class="summary"> I am working with neuroimaging data from mice, and I need to scale the voxel size by a factor of 10 so that a number of tools (which were desined mainly for human neuroimaging) will work on them. Sadly, I don't seem to be able to scale the voxel size - or rather, I do, but the original voxel size pops up again after I save the file or further downstream in my processing. I am thining the NIfTI he</div></div><span class="ago" title="published:Wed, 26 Aug 2015 15:15:53 GMT -- Received: Wed, 26 Aug 2015 17:44:00 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f6699d8d8:5b186b0:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3367/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f6699d8d8:5b186b0:46f3bb90" data-navigation="inline" data-title="One node workflow with iterables?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f6699d8d8:5b186b0:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5423 --></div><!-- react-empty: 5424 --><div class="content"><a href="http://neurostars.org/p/3367/" target="_blank" rel="noopener" class="title">One node workflow with iterables?</a><div class="summary"> How can I run a single node using iterables by itself? node_sweep = pe.Node(niu.Function(input_names=['aCT','aTA'], output_names=['file_path_save'], function=wrapper_MAIN), name="chop" ) node_sweep.iterables = [('aCT', np.arange(0.1, 0.9, 0.1)), ('aTA', np.arange(0.1, 0.9, 0.1))] node_sweep.run() wrapper_MAIN() takes exactly 2 arguments (0 given) Interface Function failed to run. </div></div><span class="ago" title="published:Tue, 25 Aug 2015 20:30:38 GMT -- Received: Tue, 25 Aug 2015 20:43:22 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f655030b9:5376992:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3365/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f655030b9:5376992:46f3bb90" data-navigation="inline" data-title="Question in nipype.workflows.dmri.fsl.utils.rotate_bvecs" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f655030b9:5376992:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5439 --></div><!-- react-empty: 5440 --><div class="content"><a href="http://neurostars.org/p/3365/" target="_blank" rel="noopener" class="title">Question in nipype.workflows.dmri.fsl.utils.rotate_bvecs</a><div class="summary"> Hello, Due to the fact that the diffusion MRI are registered (rigid transformation) to the b0 volume using FLIRT (see hmc_pipeline), why is it necessary to take the inverse of the rotation matrix to correct the B matrix in the function rotate_bvecs ? See Leemans 2009. The B-Matrix must be rotated when correcting for subject motion in dti data. Best regards. Diego </div></div><span class="ago" title="published:Tue, 25 Aug 2015 14:25:16 GMT -- Received: Tue, 25 Aug 2015 14:43:18 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f600e7370:39fb4f1:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3362/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f600e7370:39fb4f1:1cbd0d70" data-navigation="inline" data-title="Cannot edit some NIfTI header fields with nibabel" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f600e7370:39fb4f1:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5455 --></div><!-- react-empty: 5456 --><div class="content"><a href="http://neurostars.org/p/3362/" target="_blank" rel="noopener" class="title">Cannot edit some NIfTI header fields with nibabel</a><div class="summary">I am trying to edit a file with the following header: sizeof_hdr : 348 data_type : db_name : extents : 0 session_error : 0 regular : dim_info : 57 dim : [ 4 60 30 20 600 1 1 1] intent_p1 : 0.0 intent_p2 : 0.0 intent_p3 : 0.0 intent_code : none datatype : float64 bitpix : 64 slice_start : 0 pixdim : [ -1.00000000e+00 3.03997457e-01 3.00000012e-01 5.00000000e-01 1.50000000e+03 1.00000000e+00 1.00000</div></div><span class="ago" title="published:Mon, 24 Aug 2015 12:14:16 GMT -- Received: Mon, 24 Aug 2015 14:13:23 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f5f698d8a:34ab7e6:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3361/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f5f698d8a:34ab7e6:46f3bb90" data-navigation="inline" data-title="Print NIfTI headers from the cli with nibabel" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f5f698d8a:34ab7e6:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5512 --></div><!-- react-empty: 5513 --><div class="content"><a href="http://neurostars.org/p/3361/" target="_blank" rel="noopener" class="title">Print NIfTI headers from the cli with nibabel</a><div class="summary">Hi, is there any cli binding (preferably in nibabel) which I can use to print NIfTI headers directly to the terminal? I am looking for something which shows the tg names as they are specified in the NIfTI documentation, e.g.: sizeof_hdr : 540 magic : n+2 eol_check : [13 10 26 10] datatype : int16 bitpix : 16 dim : [ 4 32 20 12 2 1 1 1] intent_p1 : 0.0 intent_p2 : 0.0 intent_p3 : 0.0 pixdim : [ -1.</div></div><span class="ago" title="published:Mon, 24 Aug 2015 09:12:29 GMT -- Received: Mon, 24 Aug 2015 11:13:17 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f58f871ec:18f0aa9:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3360/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f58f871ec:18f0aa9:1cbd0d70" data-navigation="inline" data-title="Python Crashing while using Nibabel" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f58f871ec:18f0aa9:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5528 --></div><!-- react-empty: 5529 --><div class="content"><a href="http://neurostars.org/p/3360/" target="_blank" rel="noopener" class="title">Python Crashing while using Nibabel</a><div class="summary"> I was trying to use the code snippets from the 'Getting Started' page from the Nibabel website in Spyder 2. I could go this far: import os import numpy as np from nibabel.testing import data_path example_file = os.path.join(data_path,'example4d.nii.gz') import nibabel as nib img = nib.load(example_file) However as I typed in img.shape , the editor caused Python to crash. I tried the same command </div></div><span class="ago" title="published:Sun, 23 Aug 2015 01:41:42 GMT -- Received: Sun, 23 Aug 2015 05:12:01 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f4645e8dc:128fd7c6:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3359/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f4645e8dc:128fd7c6:1cbd0d70" data-navigation="inline" data-title="Custom crashdump_dir or rename crash files based on iterable field?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f4645e8dc:128fd7c6:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5544 --></div><!-- react-empty: 5545 --><div class="content"><a href="http://neurostars.org/p/3359/" target="_blank" rel="noopener" class="title">Custom crashdump_dir or rename crash files based on iterable field?</a><div class="summary"> I have an iterable based workflow which gets called with a single wf.run() command. Crash files are placed in a single directory and labeled by the node and the iterable instance of the node as determined by nipype: crash-20150818-155112-xxx-chop.a09.pklz crash-20150818-155237-xxx-formant.a79.pklz crash-20150818-155237-xxx-pitch.a79.pklz crash-20150818-155238-xxx-formant.a78.pklz Is there a way t</div></div><span class="ago" title="published:Wed, 19 Aug 2015 12:48:53 GMT -- Received: Wed, 19 Aug 2015 14:03:50 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f411d9808:1048c50e:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3357/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f411d9808:1048c50e:46f3bb90" data-navigation="inline" data-title="iterable vs iterfield for robust parallel workflow" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f411d9808:1048c50e:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5560 --></div><!-- react-empty: 5561 --><div class="content"><a href="http://neurostars.org/p/3357/" target="_blank" rel="noopener" class="title">iterable vs iterfield for robust parallel workflow</a><div class="summary"> I have a workflow that should run independently across N subjects. My initial approach was to create each of the processing nodes as mapnodes where the 'copies' of the node were 1:1 with the subject. However, if one subject fails, the aggregrate mapnode fails, and all the downstream mapnodes for the subjects do not execute even though all but the one subject executed without problems. Would using</div></div><span class="ago" title="published:Tue, 18 Aug 2015 13:46:44 GMT -- Received: Tue, 18 Aug 2015 14:01:42 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f38be6e86:e00b881:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3356/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f38be6e86:e00b881:1cbd0d70" data-navigation="inline" data-title="Magnitude of influence of attention on higher-order visual system activity?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f38be6e86:e00b881:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5576 --></div><!-- react-empty: 5577 --><div class="content"><a href="http://neurostars.org/p/3356/" target="_blank" rel="noopener" class="title">Magnitude of influence of attention on higher-order visual system activity?</a><div class="summary"> I'm analyzing data of visual cortex activity recorded on different days. This data has been proven to show activity for certain stimuli in the higher and lower visual system areas for the stimuli (images and videos of people) in another analysis experiment. I'm taking a different approach for analysis now. I'm also analyzing the experiment in a single-subject fashion. While I can always explain t</div></div><span class="ago" title="published:Sun, 16 Aug 2015 20:34:52 GMT -- Received: Sun, 16 Aug 2015 23:00:48 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f275ae176:8bac6b4:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3351/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f275ae176:8bac6b4:46f3bb90" data-navigation="inline" data-title="How would you align brain meshes with landmarks?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f275ae176:8bac6b4:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5592 --></div><!-- react-empty: 5593 --><div class="content"><a href="http://neurostars.org/p/3351/" target="_blank" rel="noopener" class="title">How would you align brain meshes with landmarks?</a><div class="summary"> Hello, I have been using Thomas Yeo's Spherical Demons (https://sites.google.com/site/yeoyeo02/software/sphericaldemonsrelease) for some time to align meshes of cortical surface reconstructions. Spherical Demons requires the original mesh, an inflated version of it, a spherical version, and maps of curvature at different scale levels. I have code to produce all that out of a single mesh. However,</div></div><span class="ago" title="published:Thu, 13 Aug 2015 13:08:11 GMT -- Received: Thu, 13 Aug 2015 13:58:31 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f1db0c8dd:58d50b0:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3346/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f1db0c8dd:58d50b0:46f3bb90" data-navigation="inline" data-title="Error message presented by nipype_display_crash" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f1db0c8dd:58d50b0:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5608 --></div><!-- react-empty: 5609 --><div class="content"><a href="http://neurostars.org/p/3346/" target="_blank" rel="noopener" class="title">Error message presented by nipype_display_crash</a><div class="summary"> Greetings Neurostars, I sometimes receive crash files from users having difficulty using a pipeline created by my lab. Most of these crash files open without issue, but some of them behave oddly. When I try to open these files with nipype_display_crash, I am presented with the error message printed below: Traceback (most recent call last): File "/usr/local/bin/nipype_display_crash", line 86, in </div></div><span class="ago" title="published:Tue, 11 Aug 2015 14:44:33 GMT -- Received: Tue, 11 Aug 2015 16:56:09 GMT">15mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f0a57d132:156c48:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3339/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f0a57d132:156c48:1cbd0d70" data-navigation="inline" data-title="Resource with examples of Artifacts from Multi-Modal Human MRI Scan Sequences" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f0a57d132:156c48:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5624 --></div><!-- react-empty: 5625 --><div class="content"><a href="http://neurostars.org/p/3339/" target="_blank" rel="noopener" class="title">Resource with examples of Artifacts from Multi-Modal Human MRI Scan Sequences</a><div class="summary"> Hi, I'm looking for a, preferably online, resource that shows examples of artifacts that can be seen in human MRI scans from a variety of modalities. For example, motion, ghosting in fMRI, T1/T2, DTI, etc. that are lined up next to what a "good scan" looks like. It would also be great if any of these actually have the NIfTI files to look through. Can anyone point me to such a resource(s)? </div></div><span class="ago" title="published:Fri, 07 Aug 2015 21:02:04 GMT -- Received: Fri, 07 Aug 2015 22:46:11 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f0530b59e:957a2:1cbd0d70_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3335/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f0530b59e:957a2:1cbd0d70" data-navigation="inline" data-title="Timing of permutation testing using Nipype MapNodes" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f0530b59e:957a2:1cbd0d70" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5640 --></div><!-- react-empty: 5641 --><div class="content"><a href="http://neurostars.org/p/3335/" target="_blank" rel="noopener" class="title">Timing of permutation testing using Nipype MapNodes</a><div class="summary">Hi all, My basic question is, has anyone used Nipype to run permutation testing and get it to finish in a reasonable amount of time (about a month or, ideally, less)? I'm attempting to use Nipype with PyMVPA to run permutation testing on some leave-one-subject-out cross-validation searchlight MVPA I'm running on our SGE cluster. Since letting one job handle all the permutations for all the searchl</div></div><span class="ago" title="published:Thu, 06 Aug 2015 20:19:32 GMT -- Received: Thu, 06 Aug 2015 22:45:22 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f029d1aab:3af9b5db:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3334/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f029d1aab:3af9b5db:cd74fcc6" data-navigation="inline" data-title="interpretting the NIfTI dim_info key" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14f029d1aab:3af9b5db:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5656 --></div><!-- react-empty: 5657 --><div class="content"><a href="http://neurostars.org/p/3334/" target="_blank" rel="noopener" class="title">interpretting the NIfTI dim_info key</a><div class="summary">I have converted a set of DICOMS to a NIfTI file, whose header looks like ths: sizeof_hdr : 348 data_type : db_name : extents : 0 session_error : 0 regular : dim_info : 57 dim : [ 4 60 30 20 600 1 1 1] intent_p1 : 0.0 intent_p2 : 0.0 intent_p3 : 0.0 intent_code : none datatype : float64 bitpix : 64 slice_start : 0 pixdim : [ -1. 3.03997469 3. 3.49999595 1. 1000. 1. 1. ] vox_offset : 0.0 scl_slope </div></div><span class="ago" title="published:Thu, 06 Aug 2015 08:51:16 GMT -- Received: Thu, 06 Aug 2015 10:44:54 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14efc2c6af0:3b32bf87:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3329/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14efc2c6af0:3b32bf87:46f3bb90" data-navigation="inline" data-title="Nipype: Get subject_id from SelectFiles for Datasink" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14efc2c6af0:3b32bf87:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5672 --></div><!-- react-empty: 5673 --><div class="content"><a href="http://neurostars.org/p/3329/" target="_blank" rel="noopener" class="title">Nipype: Get subject_id from SelectFiles for Datasink</a><div class="summary">Hello all, I am new here (and to Nipype) and have a question. I want to use Datasink to make a folder structure in which there is a main folder for each subjectnumber (with therein subfolders/subsubfolders/etc). I tried to accomplish this by using the node that I iterate over (an instance of SelectFiles). However, the code below (a few snippets from the whole pipeline) does not work. I assume that</div></div><span class="ago" title="published:Wed, 05 Aug 2015 02:00:16 GMT -- Received: Wed, 05 Aug 2015 04:44:06 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14efa3dce57:3a97e1ef:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3323/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14efa3dce57:3a97e1ef:46f3bb90" data-navigation="inline" data-title="Nipy fails to install on conda virtualenv" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14efa3dce57:3a97e1ef:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5688 --></div><!-- react-empty: 5689 --><div class="content"><a href="http://neurostars.org/p/3323/" target="_blank" rel="noopener" class="title">Nipy fails to install on conda virtualenv</a><div class="summary">I'm trying to install nipy on a miniconda virtualenv using: pip install https://github.com/nipy/nipy/archive/master.zip but I'm getting the following error: ****************ERROR BELOW****************** compiling C sources C compiler: gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC creating build/temp.linux-x86_64-2.7 creating build/temp.linux-x86_6</div></div><span class="ago" title="published:Tue, 04 Aug 2015 18:29:38 GMT -- Received: Tue, 04 Aug 2015 19:43:51 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ef3281c57:360a4f36:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3317/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ef3281c57:360a4f36:cd74fcc6" data-navigation="inline" data-title="Longitudinal MRI data from young and middle-aged healthy controls" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ef3281c57:360a4f36:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5704 --></div><!-- react-empty: 5705 --><div class="content"><a href="http://neurostars.org/p/3317/" target="_blank" rel="noopener" class="title">Longitudinal MRI data from young and middle-aged healthy controls</a><div class="summary"> I was wondering whether any one can point me to a dataset of longitudinal brain MRI (3D T1) from young and middle aged (more than 20 less than 50) individuals? HCP seems not to have the longitudinal data I am looking for. Thanks </div></div><span class="ago" title="published:Mon, 03 Aug 2015 09:54:57 GMT -- Received: Mon, 03 Aug 2015 10:42:49 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ee3b36971:340685ba:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3315/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ee3b36971:340685ba:46f3bb90" data-navigation="inline" data-title="Make background brain in pysurfer more transparent" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ee3b36971:340685ba:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5720 --></div><!-- react-empty: 5721 --><div class="content"><a href="http://neurostars.org/p/3315/" target="_blank" rel="noopener" class="title">Make background brain in pysurfer more transparent</a><div class="summary"> I'd like to make the background brain in pysurfer a little more transparent, so I'm looking for an "alpha" flag or something similar. I don't think this yet exists according to the documentation on changing the curvature color scheme but I wonder if anyone has a hack that I could implement? An alternative to the alpha parameter would be to create a fifth (personalised) color scheme perhaps? Thank</div></div><span class="ago" title="published:Fri, 31 Jul 2015 09:57:54 GMT -- Received: Fri, 31 Jul 2015 10:41:02 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14edc9d318d:31c1dd9c:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3310/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14edc9d318d:31c1dd9c:46f3bb90" data-navigation="inline" data-title="Obtain detailed acyclic graph from nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14edc9d318d:31c1dd9c:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5736 --></div><!-- react-empty: 5737 --><div class="content"><a href="http://neurostars.org/p/3310/" target="_blank" rel="noopener" class="title">Obtain detailed acyclic graph from nipype</a><div class="summary"> The nipype documentation tells me that with workflow.write_graph() , I can obtain acyclic graphs like this: However, my workflows, (all containing workflow.write_graph() ), output acyclic graphs like this: (which do not specify inputs and outputs) How can I force my graphs to display this additional information? </div></div><span class="ago" title="published:Wed, 29 Jul 2015 22:44:42 GMT -- Received: Thu, 30 Jul 2015 01:39:26 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed4e249c0:2d502f0a:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3298/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed4e249c0:2d502f0a:cd74fcc6" data-navigation="inline" data-title="Nipy script interface is not saving any files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed4e249c0:2d502f0a:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5752 --></div><!-- react-empty: 5753 --><div class="content"><a href="http://neurostars.org/p/3298/" target="_blank" rel="noopener" class="title">Nipy script interface is not saving any files</a><div class="summary"> I wrote an interface to increase the voxel size in a series of NIfTI files: class VoxelResizeInputSpec(BaseInterfaceInputSpec): nii_files = traits.List(File(exists=True, mandatory=True)) resize_factor = traits.Int(10, usedefault=True, desc="Factor by which to multiply the voxel size in the header") class VoxelResizeOutputSpec(TraitedSpec): resized_files = traits.List(File(exists=True)) class Voxe</div></div><span class="ago" title="published:Tue, 28 Jul 2015 11:19:28 GMT -- Received: Tue, 28 Jul 2015 13:37:56 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed24e9f1d:2e587412:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3294/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed24e9f1d:2e587412:46f3bb90" data-navigation="inline" data-title="Pipe outputs from different nodes to one node in nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed24e9f1d:2e587412:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5768 --></div><!-- react-empty: 5769 --><div class="content"><a href="http://neurostars.org/p/3294/" target="_blank" rel="noopener" class="title">Pipe outputs from different nodes to one node in nipype</a><div class="summary"> I have a node called `stacker` which stacks multi-echo DICOM files to a series of NIfTI files, and also outputs a list of the echo times. I have another node that resizes the voxels in the NIfTI files. I would like to pass both the resized-voxel NIfTI files and the echo times to another node. I was thinking of doing something like this: workflow.connect([ (infosource, datasource, [('subject_id', </div></div><span class="ago" title="published:Tue, 28 Jul 2015 01:08:44 GMT -- Received: Tue, 28 Jul 2015 01:37:24 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed1a9c843:2c5539c1:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3293/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed1a9c843:2c5539c1:cd74fcc6" data-navigation="inline" data-title="Resources for teaching a half-day lab on machine learning in neuroimaging (nilearn)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed1a9c843:2c5539c1:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5784 --></div><!-- react-empty: 5785 --><div class="content"><a href="http://neurostars.org/p/3293/" target="_blank" rel="noopener" class="title">Resources for teaching a half-day lab on machine learning in neuroimaging (nilearn)</a><div class="summary"> Hi all, As part of the new-student "boot camp" for the Temporal Dynamics of Learning Center , I am planning to give a 4 hour "lab" on machine learning in neuroimaging (fMRI). The students will be familiar with very basics of fMRI and Python programming, and not familiar with machine learning. I'm wondering if the nilearn team, or others, have slides / standard workshop suggestions for something l</div></div><span class="ago" title="published:Mon, 27 Jul 2015 20:14:53 GMT -- Received: Mon, 27 Jul 2015 22:37:21 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed104f2f9:2c2077af:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3291/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed104f2f9:2c2077af:cd74fcc6" data-navigation="inline" data-title="Editing NIfTI headers in Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ed104f2f9:2c2077af:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5800 --></div><!-- react-empty: 5801 --><div class="content"><a href="http://neurostars.org/p/3291/" target="_blank" rel="noopener" class="title">Editing NIfTI headers in Nipype</a><div class="summary"> I would like to edit the headers of a few NIfTI files. More specifically, I would like to change the voxel size - this is commonly done with small animal fMRI so as to be able to use functons designed primarily for human fMRI analysis (which assume larger voxels). I have two questions: How can I edit the headers (peferably via an interface that I can wrap into my workflow) Which parts of the head</div></div><span class="ago" title="published:Mon, 27 Jul 2015 17:24:53 GMT -- Received: Mon, 27 Jul 2015 19:37:19 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ecc82c58d:2c7ff0bd:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3286/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ecc82c58d:2c7ff0bd:46f3bb90" data-navigation="inline" data-title="How to use wavelet decomposition for feature extraction (for fMRI images)?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ecc82c58d:2c7ff0bd:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5816 --></div><!-- react-empty: 5817 --><div class="content"><a href="http://neurostars.org/p/3286/" target="_blank" rel="noopener" class="title">How to use wavelet decomposition for feature extraction (for fMRI images)?</a><div class="summary"> Hello I have a dataset consisting of fMRI images (from mice) which are divided into 4 groups (different drug dose levels applied). Each fMRI image is 4D, that means each voxel is a time series. For each fMRI image I want to extract one feature vector. Now I want to use wavelet decomposition for feature extraction. In Matlab there exist no 4D wavelet decomposition, so I turn the 4D images into 3D </div></div><span class="ago" title="published:Sun, 26 Jul 2015 19:49:32 GMT -- Received: Sun, 26 Jul 2015 22:36:39 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ebc68a9ee:2683aa4a:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3279/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ebc68a9ee:2683aa4a:cd74fcc6" data-navigation="inline" data-title="Importing Nilearn in IPython" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ebc68a9ee:2683aa4a:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5832 --></div><!-- react-empty: 5833 --><div class="content"><a href="http://neurostars.org/p/3279/" target="_blank" rel="noopener" class="title">Importing Nilearn in IPython</a><div class="summary"> I am a new user of Nilearn and I am trying to use it with Anaconda. When I open shell and execute "pip install --user nilearn", it says: "Requirement already satisfied (use --upgrade to upgrade): nilearn in /Users/username/.local/lib/python2.7/site-packages" so it is already installed. However when I try to run 'import nilearn' on ipython console (or in Spyder Editor), it says: "Traceback (most r</div></div><span class="ago" title="published:Thu, 23 Jul 2015 17:58:09 GMT -- Received: Thu, 23 Jul 2015 19:34:12 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ea915c8a5:21b4547c:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3276/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ea915c8a5:21b4547c:46f3bb90" data-navigation="inline" data-title="Nipy interface with multiple outputs" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ea915c8a5:21b4547c:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5848 --></div><!-- react-empty: 5849 --><div class="content"><a href="http://neurostars.org/p/3276/" target="_blank" rel="noopener" class="title">Nipy interface with multiple outputs</a><div class="summary">I am trying to wrap a function I wrote, which can stack multi-echo .dicom files. I would like it to output the list of files as well as a list of the echo times. This function works up until I go and try to add a second output: class DcmToNiiInputSpec(BaseInterfaceInputSpec): dcm_dir = Directory(exists=True, mandatory=True) group_by = traits.Str(desc='everything below this value will be set to zer</div></div><span class="ago" title="published:Mon, 20 Jul 2015 01:00:46 GMT -- Received: Mon, 20 Jul 2015 01:30:53 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e9b8f913a:1d2a445b:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3274/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e9b8f913a:1d2a445b:cd74fcc6" data-navigation="inline" data-title="best automated way for making a cerebellum segmentation" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e9b8f913a:1d2a445b:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5864 --></div><!-- react-empty: 5865 --><div class="content"><a href="http://neurostars.org/p/3274/" target="_blank" rel="noopener" class="title">best automated way for making a cerebellum segmentation</a><div class="summary"> Hello everyone, I am new to segmentation and I am looking for the best automated way to segment the cerebellum. I saw FreeSurfer can segment whole cerebellum but I would also want to segment cerebellar lobules, or vermian lobules. I found two toolkits for that (SUIT and ACCLAIM), but I'm not sure of there fiabilities. I also found another algorithm (CATK) that claims to do much better than the pr</div></div><span class="ago" title="published:Fri, 17 Jul 2015 09:55:21 GMT -- Received: Fri, 17 Jul 2015 10:29:14 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e97b22889:1c06f13d:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3273/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e97b22889:1c06f13d:cd74fcc6" data-navigation="inline" data-title="realigment parameters output missing?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e97b22889:1c06f13d:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5880 --></div><!-- react-empty: 5881 --><div class="content"><a href="http://neurostars.org/p/3273/" target="_blank" rel="noopener" class="title">realigment parameters output missing?</a><div class="summary">Hello, Just a quick question, I am running SPM realignment in a pre-processing workflow in ipython. Everything is running well except that I can't seem to find the realignment parameters for my model (rp.txt file). This is probably a very stupid question, but it doesn't appear to be output with the realigned files. I have tried running realignment as both estimate and est/write. This doesn't appea</div></div><span class="ago" title="published:Thu, 16 Jul 2015 15:00:42 GMT -- Received: Thu, 16 Jul 2015 16:28:32 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e93d4e90b:1bacda7f:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3272/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e93d4e90b:1bacda7f:46f3bb90" data-navigation="inline" data-title="python 3 support" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e93d4e90b:1bacda7f:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5896 --></div><!-- react-empty: 5897 --><div class="content"><a href="http://neurostars.org/p/3272/" target="_blank" rel="noopener" class="title">python 3 support</a><div class="summary"> Hi there, I'm just getting started with using nipype, and I have been having a bit of a time with the miniconda installation. I suspect that the whole thing would be easier if I knew that this project is only supported by python 2 or 3. Otherwise, I think I will have to find a way to make a direct connection between some of the replaced modules (traits as py3traits) and I would love some assistan</div></div><span class="ago" title="published:Wed, 15 Jul 2015 21:40:49 GMT -- Received: Wed, 15 Jul 2015 22:28:00 GMT">16mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e7f387d01:15826476:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3265/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e7f387d01:15826476:46f3bb90" data-navigation="inline" data-title="How can I extract features from fMRI network connectivity analysis (FSL nets)?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e7f387d01:15826476:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5912 --></div><!-- react-empty: 5913 --><div class="content"><a href="http://neurostars.org/p/3265/" target="_blank" rel="noopener" class="title">How can I extract features from fMRI network connectivity analysis (FSL nets)?</a><div class="summary"> Hello I have a set of 37 fMRI images from mice which are divided into 4 classes (different drug doses applied). My task is to train classifiers (SVM etc.) on this dataset. Of course feature extraction is a very important step. I'm now investigating a feature extraction method based on FLS nets (network connectivity analysis). I did not myself perform this step, I just got the FSL net results. The</div></div><span class="ago" title="published:Sat, 11 Jul 2015 20:40:20 GMT -- Received: Sat, 11 Jul 2015 22:24:45 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e59ddf7b6:a244a09:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3259/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e59ddf7b6:a244a09:cd74fcc6" data-navigation="inline" data-title="How to check output of datagrabber" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e59ddf7b6:a244a09:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5928 --></div><!-- react-empty: 5929 --><div class="content"><a href="http://neurostars.org/p/3259/" target="_blank" rel="noopener" class="title">How to check output of datagrabber</a><div class="summary"> This is a very simple question, but is there a simple way to print the files returned by datagrabber? Here is my code, I just want to check that it worked (which it probably didn't :) print(x) doesn't seem to work. datasource1 = pe.Node(interface=nio.DataGrabber(infields=["subject_id"], outfields=["func", "struc"]), name = "datasource1") datasource1.inputs.base_directory = experiment_dir datasour</div></div><span class="ago" title="published:Sat, 04 Jul 2015 14:26:15 GMT -- Received: Sat, 04 Jul 2015 16:19:55 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e50d9cb66:7a57889:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3255/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e50d9cb66:7a57889:cd74fcc6" data-navigation="inline" data-title="slice timing syntax error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e50d9cb66:7a57889:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5944 --></div><!-- react-empty: 5945 --><div class="content"><a href="http://neurostars.org/p/3255/" target="_blank" rel="noopener" class="title">slice timing syntax error</a><div class="summary">I am trying to do a simple work flow to processing fMRI scans in SPM (using ipython). I have gone over and over my code and I can't figure why I am getting a "syntax error" on the following code. This isn't a very useful error message. Does anyone have any idea what the problem might be here?? I have bolded the code where I am getting a syntax error. I am new to ipython and know that I am missing </div></div><span class="ago" title="published:Thu, 02 Jul 2015 20:22:29 GMT -- Received: Thu, 02 Jul 2015 22:18:47 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e4e46547d:6dcd6a6:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3254/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e4e46547d:6dcd6a6:46f3bb90" data-navigation="inline" data-title="Drawing a deformation grid and overlaying over an image" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e4e46547d:6dcd6a6:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5960 --></div><!-- react-empty: 5961 --><div class="content"><a href="http://neurostars.org/p/3254/" target="_blank" rel="noopener" class="title">Drawing a deformation grid and overlaying over an image</a><div class="summary"> Hello, 1. Is there an interface in nipype or a tool elsewhere to display a dense deformation field saved as .nii or .mha (3D vector at each voxel) by deforming a rectangular grid and overlaying it over a selected slice of a 3D image ? I'd like the display to be something like the image shown at: https://wiki.mouseimaging.ca/display/MICePub/minc-stuffs+%28formerly+mice-minc-tools%29#minc-stuffsfor</div></div><span class="ago" title="published:Thu, 02 Jul 2015 08:25:38 GMT -- Received: Thu, 02 Jul 2015 10:18:28 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e4541e818:3e048d6:46f3bb90_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3250/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e4541e818:3e048d6:46f3bb90" data-navigation="inline" data-title="Differences between Neuro-informatics, NeuroBioinformatics, Computational Neuroscience" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e4541e818:3e048d6:46f3bb90" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5976 --></div><!-- react-empty: 5977 --><div class="content"><a href="http://neurostars.org/p/3250/" target="_blank" rel="noopener" class="title">Differences between Neuro-informatics, NeuroBioinformatics, Computational Neuroscience</a><div class="summary"> Pretty much like the title. There are actually differences or they are pretty much the same? Also there is a particular neuroscientific field dedicated to Molecular Dynamics Simulations? Thanks in advance for the replies </div></div><span class="ago" title="published:Tue, 30 Jun 2015 15:22:14 GMT -- Received: Tue, 30 Jun 2015 16:17:04 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e43f82ec6:35b82ff:cd74fcc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3249/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e43f82ec6:35b82ff:cd74fcc6" data-navigation="inline" data-title="Job: MNE software research engineer in Paris France" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e43f82ec6:35b82ff:cd74fcc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 5992 --></div><!-- react-empty: 5993 --><div class="content"><a href="http://neurostars.org/p/3249/" target="_blank" rel="noopener" class="title">Job: MNE software research engineer in Paris France</a><div class="summary">hi, I have an open position as engineer in my research team in Telecom ParisTech / CNRS LTCI in downtown Paris. The position is for two years to work on MNE-Python and should start before the end of the year 2015. The goal is to help us to develop, maintain, document and promote the MNE open source project [1,2]. [1] http://martinos.org/mne/ [2] https://github.com/mne-tools/mne-python In terms of </div></div><span class="ago" title="published:Tue, 30 Jun 2015 08:03:29 GMT -- Received: Tue, 30 Jun 2015 10:16:55 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e212fbb62:8c1130b:6956f5d6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3135/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e212fbb62:8c1130b:6956f5d6" data-navigation="inline" data-title="Threshold and threshold statistics" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e212fbb62:8c1130b:6956f5d6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6008 --></div><!-- react-empty: 6009 --><div class="content"><a href="http://neurostars.org/p/3135/" target="_blank" rel="noopener" class="title">Threshold and threshold statistics</a><div class="summary"> I would like to threshold my data, and used threshold statistics node to get an estimation of the results. But I can't find the correlation between the output of ThresholdStatistics and Threshold , and the proper way to use the first into the second. </div></div><span class="ago" title="published:Tue, 23 Jun 2015 14:56:27 GMT -- Received: Tue, 23 Jun 2015 16:10:53 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1f40d0cc:813cbdd:6956f5d6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3133/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1f40d0cc:813cbdd:6956f5d6" data-navigation="inline" data-title="Real-time fMRI open datasets" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1f40d0cc:813cbdd:6956f5d6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6024 --></div><!-- react-empty: 6025 --><div class="content"><a href="http://neurostars.org/p/3133/" target="_blank" rel="noopener" class="title">Real-time fMRI open datasets</a><div class="summary"> Does anyone know of any freely available real-time fMRI datasets? I am looking to study some techniques for studying real-time fMRI and would like to validate these approaches using an open dataset </div></div><span class="ago" title="published:Tue, 23 Jun 2015 06:25:15 GMT -- Received: Tue, 23 Jun 2015 07:10:18 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1f40d0cc:813cbdc:6956f5d6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3132/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1f40d0cc:813cbdc:6956f5d6" data-navigation="inline" data-title="Real-time fMRI open datasets" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1f40d0cc:813cbdc:6956f5d6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6040 --></div><!-- react-empty: 6041 --><div class="content"><a href="http://neurostars.org/p/3132/" target="_blank" rel="noopener" class="title">Real-time fMRI open datasets</a><div class="summary"> Does anyone know of any freely available real-time fMRI datasets? I am looking to study some techniques for studying real-time fMRI and would like to validate these approaches using an open dataset </div></div><span class="ago" title="published:Tue, 23 Jun 2015 06:25:11 GMT -- Received: Tue, 23 Jun 2015 07:10:18 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1df7167f:7b558de:6956f5d6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3131/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1df7167f:7b558de:6956f5d6" data-navigation="inline" data-title="Smoothing value different in command than script" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1df7167f:7b558de:6956f5d6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6056 --></div><!-- react-empty: 6057 --><div class="content"><a href="http://neurostars.org/p/3131/" target="_blank" rel="noopener" class="title">Smoothing value different in command than script</a><div class="summary"> Hi all, I'm using fsl.Smooth() in a workflow and noticed that the fwhm value in the command being used is different than the one that I specified in my script. In my script I have the smoothing node set up as such: smooth = pe.Node(interface = fsl.Smooth(), name = 'smooth') smooth.inputs.fwhm = 7 However, the command that appears in terminal (and recorded in command.txt) is the following: fslmath</div></div><span class="ago" title="published:Mon, 22 Jun 2015 23:26:20 GMT -- Received: Tue, 23 Jun 2015 01:10:09 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1d52393f:77bedfb:f56d963b_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3130/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1d52393f:77bedfb:f56d963b" data-navigation="inline" data-title="Job: Postdoctoral Fellow - Clinical Neuroscience and Neuroimaging Job" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e1d52393f:77bedfb:f56d963b" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6072 --></div><!-- react-empty: 6073 --><div class="content"><a href="http://neurostars.org/p/3130/" target="_blank" rel="noopener" class="title">Job: Postdoctoral Fellow - Clinical Neuroscience and Neuroimaging Job</a><div class="summary"> Postdoctoral Fellow - Clinical Neuroscience and Neuroimaging Job ID # 550BR Center for Health Sciences Menlo Park, CA A NIH-funded postdoctoral research fellowship position is currently available in the Neuroscience Program at SRI International, Menlo Park, California. The research project will investigate the effects of aging on the functional and structural subcortical-cortical brain circuits t</div></div><span class="ago" title="published:Mon, 22 Jun 2015 21:25:24 GMT -- Received: Mon, 22 Jun 2015 22:10:04 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e02e8f609:c956ac4:843e64d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3123/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e02e8f609:c956ac4:843e64d9" data-navigation="inline" data-title="SPM Workflow Regression" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14e02e8f609:c956ac4:843e64d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6088 --></div><!-- react-empty: 6089 --><div class="content"><a href="http://neurostars.org/p/3123/" target="_blank" rel="noopener" class="title">SPM Workflow Regression</a><div class="summary">Hi all, I'm troubleshooting a transformation matrix problem using SPM preprocessing in nipype. This is a basic toy example, but holds for more complex workflows. I'm still looking into errors I found while asking about 3d/4d SPM file handling, but I think this is unrelated. I'm using SPM for preprocessing, and in this case using the realign and coregister interfaces. I'm trying to follow best-prac</div></div><span class="ago" title="published:Wed, 17 Jun 2015 18:41:55 GMT -- Received: Wed, 17 Jun 2015 19:04:58 GMT">17mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14de349f3fa:2de5427:843e64d9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3121/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14de349f3fa:2de5427:843e64d9" data-navigation="inline" data-title="Problem with SPM model generation and Datasink" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14de349f3fa:2de5427:843e64d9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6104 --></div><!-- react-empty: 6105 --><div class="content"><a href="http://neurostars.org/p/3121/" target="_blank" rel="noopener" class="title">Problem with SPM model generation and Datasink</a><div class="summary">Hello All, I'm trying to debug some code that uses SpecifySPMModel as it is not generating the right output. Here's the code I have so far: info = pickle.load(open('subinf.pkl')) datadir = '/sdata/images/projects/STRADL/CHECK/VOLUNTEER_199/OrigData/output' import nipype.pipeline.engine as pe from nipype.interfaces.base import Bunch from nipype.algorithms.modelgen import SpecifySPMModel import glob</div></div><span class="ago" title="published:Thu, 11 Jun 2015 15:20:19 GMT -- Received: Thu, 11 Jun 2015 15:43:04 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dc797e050:1fb781c:e4f689e2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3116/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dc797e050:1fb781c:e4f689e2" data-navigation="inline" data-title="suppress nipype.interfaces.dcm2nii.Dcm2nii() from creating config.ini when run" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dc797e050:1fb781c:e4f689e2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6120 --></div><!-- react-empty: 6121 --><div class="content"><a href="http://neurostars.org/p/3116/" target="_blank" rel="noopener" class="title">suppress nipype.interfaces.dcm2nii.Dcm2nii() from creating config.ini when run</a><div class="summary"> When nipype.interfaces.dcm2nii.Dcm2nii() is run, it creates a config.ini, because of the call to _gen_filename() in nipype/interfaces/dcm2nii.py. What is the purpose of that file? Is there any way to suppress that file from being created? It's annoying that it gets left around. </div></div><span class="ago" title="published:Sat, 06 Jun 2015 05:02:27 GMT -- Received: Sat, 06 Jun 2015 06:38:48 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dc504433d:1441bf7:e4f689e2_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3115/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dc504433d:1441bf7:e4f689e2" data-navigation="inline" data-title="detail about significance testing in nilearn" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dc504433d:1441bf7:e4f689e2" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6136 --></div><!-- react-empty: 6137 --><div class="content"><a href="http://neurostars.org/p/3115/" target="_blank" rel="noopener" class="title">detail about significance testing in nilearn</a><div class="summary">At the upcoming OHBM 2015, Jo Etzel and I will host a brief morning (Wed) session on significance testing in MVPA studies. In my short (15 min) talk I would like to provide audience a pragmatic overview of methodologies available in publicly available toolkits for MVPA (in Python, Matlab, etc all alike). I am not intending to provide comparison of all of the toolkits' functionality, but rather wan</div></div><span class="ago" title="published:Fri, 05 Jun 2015 17:35:21 GMT -- Received: Fri, 05 Jun 2015 18:38:20 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dbf37cd86:2d5af2c:6956f5d6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3112/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dbf37cd86:2d5af2c:6956f5d6" data-navigation="inline" data-title="Feeding in (too) large argument list in Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dbf37cd86:2d5af2c:6956f5d6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6193 --></div><!-- react-empty: 6194 --><div class="content"><a href="http://neurostars.org/p/3112/" target="_blank" rel="noopener" class="title">Feeding in (too) large argument list in Nipype</a><div class="summary"> Hello, I'm really new to Nipype. For analysing a resting state fMRI dataset I used preprocessing steps provided here: https://github.com/NeuroanatomyAndConnectivity/pipelines/tree/master/src/lsd_lemon During the preprocessing, at one step the resting state images are split into single images and later on merged again in time. As we have quite a few images (1505), here is where the problem was: fe</div></div><span class="ago" title="published:Thu, 04 Jun 2015 14:51:11 GMT -- Received: Thu, 04 Jun 2015 15:36:54 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dbab57efa:158b2c1:6956f5d6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3111/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dbab57efa:158b2c1:6956f5d6" data-navigation="inline" data-title="Problem raising Exceptions within Function node" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dbab57efa:158b2c1:6956f5d6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6209 --></div><!-- react-empty: 6210 --><div class="content"><a href="http://neurostars.org/p/3111/" target="_blank" rel="noopener" class="title">Problem raising Exceptions within Function node</a><div class="summary"> I have a function node wrapping a custom function that goes through an atlas nifti file, zeroing out specific ROIs set by the user. I want to raise an exception whenever the function is told to remove an ROI that it either can't find or that it finds conflicting definitions for. I initially tried using `assert`, which led returned a TypeError: exceptions must be old-style classes or derived from </div></div><span class="ago" title="published:Wed, 03 Jun 2015 16:38:56 GMT -- Received: Wed, 03 Jun 2015 18:36:06 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dba109758:1116fdc:18a325ad_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3109/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dba109758:1116fdc:18a325ad" data-navigation="inline" data-title="Realignment Nipype Options" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14dba109758:1116fdc:18a325ad" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6225 --></div><!-- react-empty: 6226 --><div class="content"><a href="http://neurostars.org/p/3109/" target="_blank" rel="noopener" class="title">Realignment Nipype Options</a><div class="summary"> Hello, Relatively new to Nipype, sorry. 2 questions about the realignment options for SPM... Looking at the documentation for nipype.interfaces.spm.Realign 1) Can I assume that when you select... realign.inputs.register_to_mean = False the images are then aligned to the first image? If so, how do you get it to register to another image? 2) Can I also assume that when you make no selection for wri</div></div><span class="ago" title="published:Wed, 03 Jun 2015 15:29:24 GMT -- Received: Wed, 03 Jun 2015 15:35:59 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14da0f27f58:1a9c37e0:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3105/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14da0f27f58:1a9c37e0:7af40b3e" data-navigation="inline" data-title="Raise custom error from Workflow" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14da0f27f58:1a9c37e0:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6241 --></div><!-- react-empty: 6242 --><div class="content"><a href="http://neurostars.org/p/3105/" target="_blank" rel="noopener" class="title">Raise custom error from Workflow</a><div class="summary"> I am writing a workflow and I would like to wrap it in a try block. I have a custom error raised for a very specific error in an interface, but whenever I run the workflow it is a RuntimeError('Workflow... at the end. Is there any way to raise a custom error through the workflow interface? </div></div><span class="ago" title="published:Fri, 29 May 2015 16:11:24 GMT -- Received: Fri, 29 May 2015 18:32:36 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d9b268d90:187270b3:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3102/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d9b268d90:187270b3:de01cf22" data-navigation="inline" data-title="SPM Dicom Convert in python (Ipython/ Nipype)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d9b268d90:187270b3:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6257 --></div><!-- react-empty: 6258 --><div class="content"><a href="http://neurostars.org/p/3102/" target="_blank" rel="noopener" class="title">SPM Dicom Convert in python (Ipython/ Nipype)</a><div class="summary"> I am new to python or more specifically ipython. I have been running through the steps to run what should be a very simple Dicom Conversion in a statistical package called SPM for an MRI image file as described by NiPype. I can't get it to run and was wondering what I was doing wrong. I am not getting an error message, instead, there is no file change or output. It just hangs. Does anyone have an</div></div><span class="ago" title="published:Thu, 28 May 2015 13:50:07 GMT -- Received: Thu, 28 May 2015 15:31:44 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d97ee26c5:17677a56:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3098/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d97ee26c5:17677a56:de01cf22" data-navigation="inline" data-title="Job: Data curator job at Stanford" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d97ee26c5:17677a56:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6273 --></div><!-- react-empty: 6274 --><div class="content"><a href="http://neurostars.org/p/3098/" target="_blank" rel="noopener" class="title">Job: Data curator job at Stanford</a><div class="summary">Poldracklab at Department of Psychology, Stanford University is looking for a full time Data Curator (DC) to help grow a repository for sharing and analysing scientific data at OpenfMRI.org. OpenfMRI.org is a project dedicated to the free and open sharing of functional magnetic resonance imaging (fMRI) datasets, including raw datas. To learn more about the OpenfMRI project, please go to https://op</div></div><span class="ago" title="published:Wed, 27 May 2015 23:15:05 GMT -- Received: Thu, 28 May 2015 00:31:16 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d941102ad:160c44c0:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3094/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d941102ad:160c44c0:de01cf22" data-navigation="inline" data-title="How to get a register.dat file like the mni152.register.dat?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d941102ad:160c44c0:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6289 --></div><!-- react-empty: 6290 --><div class="content"><a href="http://neurostars.org/p/3094/" target="_blank" rel="noopener" class="title">How to get a register.dat file like the mni152.register.dat?</a><div class="summary">Hi, all. I want to assign volume values to freesurfer surface. I use the command line, mri_vol2surf. If we assign volume in MNI152 space, we can use the mni152.register.dat file in the fsaverage folder by using --mni152reg. However, if I want to assign volume values (in MNI152 space) to individual's surface (not fsaverage), how can I get the register.dat file? I thought tkregister might be helpful</div></div><span class="ago" title="published:Wed, 27 May 2015 05:00:23 GMT -- Received: Wed, 27 May 2015 06:30:52 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d936c23a2:1605a62f:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3092/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d936c23a2:1605a62f:7af40b3e" data-navigation="inline" data-title="Example script: dmri_tbss_nki.py error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d936c23a2:1605a62f:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6305 --></div><!-- react-empty: 6306 --><div class="content"><a href="http://neurostars.org/p/3092/" target="_blank" rel="noopener" class="title">Example script: dmri_tbss_nki.py error</a><div class="summary">Hello When running the example script dmri_tbss_nki.py and tried on the nki sample data, as well as fsl tutorial data, and my own data. Everything runs smoothly in first part of workflow, with FA and MD, regardless of data. Then, when trying to run tbss_all node to create distance map etc, /Users/macbook/anaconda/lib/python2.7/site-packages/nipype/interfaces/base.py:359: UserWarning: Input all_axi</div></div><span class="ago" title="published:Wed, 27 May 2015 02:52:13 GMT -- Received: Wed, 27 May 2015 03:30:47 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d90d8a9a5:1527a7b0:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3090/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d90d8a9a5:1527a7b0:7af40b3e" data-navigation="inline" data-title="Downsampling of fMRI image with FSL" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d90d8a9a5:1527a7b0:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6321 --></div><!-- react-empty: 6322 --><div class="content"><a href="http://neurostars.org/p/3090/" target="_blank" rel="noopener" class="title">Downsampling of fMRI image with FSL</a><div class="summary"> Hello I have a set of fMRI images. One group has a dimensionality of 90 x 60 x 12 x 350 with voxel dimension 1 x 1 x 1 mm (350 volumes). The other group has a dimensionality of 80 x 35 x 12 x 350 with voxel dimension 0.2 x 0.2 x 0.5 mm. I'm using one of the images as reference image for registration. Due to the difference in resolution the registration fails (with flirt). So I have to first downs</div></div><span class="ago" title="published:Tue, 26 May 2015 13:28:09 GMT -- Received: Tue, 26 May 2015 15:30:27 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d82ad4575:11425d08:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3088/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d82ad4575:11425d08:7af40b3e" data-navigation="inline" data-title="Feature Extraction and Cross-Validation of an fmri image dataset" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d82ad4575:11425d08:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6337 --></div><!-- react-empty: 6338 --><div class="content"><a href="http://neurostars.org/p/3088/" target="_blank" rel="noopener" class="title">Feature Extraction and Cross-Validation of an fmri image dataset</a><div class="summary">Hi I have a dataset consisting of fMRI images. Each image belongs to one class. The dataset is as follows: Class 1: 9 images Class 2: 10 images Class 3: 6 images Class 4: 12 images Each image is 4D (time series), i.e. 90x60x10x350 where 350 is the time dimension (i.e. 350 3D volumes). I want to train a classifier on this data. Now I want to first extract features and then apply feature selection b</div></div><span class="ago" title="published:Sat, 23 May 2015 21:22:07 GMT -- Received: Sat, 23 May 2015 21:28:23 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7e2a5590:101676f1:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3087/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7e2a5590:101676f1:7af40b3e" data-navigation="inline" data-title="Realign Structural Images" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7e2a5590:101676f1:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6353 --></div><!-- react-empty: 6354 --><div class="content"><a href="http://neurostars.org/p/3087/" target="_blank" rel="noopener" class="title">Realign Structural Images</a><div class="summary">I am trying to set up a fairly basic pipeline, and I am noticing that both nipy's SpaceTimeRealigner and FmriRealign4d fail to run on the structural stack, with crash reports such as: /usr/lib64/python2.7/site-packages/nipy/labs/glm/glm.py:7: FutureWarning: Module nipy.labs.utils.routines deprecated, will be removed from ..utils import mahalanobis /usr/lib64/python2.7/site-packages/nipype/interfac</div></div><span class="ago" title="published:Fri, 22 May 2015 22:59:16 GMT -- Received: Sat, 23 May 2015 00:26:53 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7d85787d:fb8914c:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3082/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7d85787d:fb8914c:de01cf22" data-navigation="inline" data-title="Manage both structural and functional image sets with nipype's DataGrabber" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7d85787d:fb8914c:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6369 --></div><!-- react-empty: 6370 --><div class="content"><a href="http://neurostars.org/p/3082/" target="_blank" rel="noopener" class="title">Manage both structural and functional image sets with nipype's DataGrabber</a><div class="summary">I am currently trying to write a very basic nipype workflow. I am using IdentityInterface and DataGrabber to manage multiple participants. Of course, for each of those I have both structural and functional images in need of processing . This guide I am following indicates that it is possible to use DataGrabber in the same workflow for both structural and functional images, but does not provide a n</div></div><span class="ago" title="published:Fri, 22 May 2015 19:14:42 GMT -- Received: Fri, 22 May 2015 21:26:48 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7ce092d1:fb26f3b:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3081/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7ce092d1:fb26f3b:7af40b3e" data-navigation="inline" data-title="Multisession (Longitudinal) recon_all in NiPype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d7ce092d1:fb26f3b:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6385 --></div><!-- react-empty: 6386 --><div class="content"><a href="http://neurostars.org/p/3081/" target="_blank" rel="noopener" class="title">Multisession (Longitudinal) recon_all in NiPype</a><div class="summary">Hi, I'm (a) new to nipype and (b) having difficulty using recon_all on multiple structural scans per participant. Superficially, the problem seems similar to that described in https://neurostars.org/p/803/, and I've tried my best to adapt the solution there to my case, but something seems to be amiss. Background: I have Time-1 and Time-2 structural (and functional, but ignore that for now) scans t</div></div><span class="ago" title="published:Fri, 22 May 2015 17:26:04 GMT -- Received: Fri, 22 May 2015 18:26:42 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d6842d298:8ca4257:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3078/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d6842d298:8ca4257:de01cf22" data-navigation="inline" data-title="How do diffusion models compare?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d6842d298:8ca4257:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6401 --></div><!-- react-empty: 6402 --><div class="content"><a href="http://neurostars.org/p/3078/" target="_blank" rel="noopener" class="title">How do diffusion models compare?</a><div class="summary"> This paper nicely covers the sparse fascicle model and the comparison with the diffusion tensor model. http://arxiv.org/abs/1411.0721 how does it compare with models like csd, and more generally with propagator based methods? how would one dissociate between applications: 1) tractography and 2) structural difference detection. </div></div><span class="ago" title="published:Mon, 18 May 2015 17:43:16 GMT -- Received: Mon, 18 May 2015 18:21:59 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d679df054:88ccf12:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3077/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d679df054:88ccf12:de01cf22" data-navigation="inline" data-title="Are there dipy or other tools to visualize whole head ODFs from dipy?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d679df054:88ccf12:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6417 --></div><!-- react-empty: 6418 --><div class="content"><a href="http://neurostars.org/p/3077/" target="_blank" rel="noopener" class="title">Are there dipy or other tools to visualize whole head ODFs from dipy?</a><div class="summary"> Are there dipy tools to visualize whole head ODFs? I'm currently visualizing CSD odf's a slice at a time, but it would be great to be able to fly through the entire brain. my data volumes are of shape (140, 140, 78) - 1.8mm resolution. </div></div><span class="ago" title="published:Mon, 18 May 2015 14:26:00 GMT -- Received: Mon, 18 May 2015 15:21:53 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4c91231f:952388:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3067/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4c91231f:952388:de01cf22" data-navigation="inline" data-title="One smaller, one larger data set: which one for exploration, which one for confirmation?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4c91231f:952388:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6433 --></div><!-- react-empty: 6434 --><div class="content"><a href="http://neurostars.org/p/3067/" target="_blank" rel="noopener" class="title">One smaller, one larger data set: which one for exploration, which one for confirmation?</a><div class="summary"> Dear data analysts, We have two data sets: A) n = 150 with more exclusion criteria (i.e., potentially less representative of the general population) and B) n = ~1500 with less exclusion criteria. We would like to use one set for exploration to create hypotheses that will then be tested in the other. Which data set would be better used as the former, which one for the latter? What are your recomme</div></div><span class="ago" title="published:Wed, 13 May 2015 08:50:36 GMT -- Received: Wed, 13 May 2015 09:18:09 GMT">18mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4aa1b019:138c065e:603b4255_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3065/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4aa1b019:138c065e:603b4255" data-navigation="inline" data-title="Dipy tractography with Eudx: how to place seeds at the center of the voxels?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4aa1b019:138c065e:603b4255" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6449 --></div><!-- react-empty: 6450 --><div class="content"><a href="http://neurostars.org/p/3065/" target="_blank" rel="noopener" class="title">Dipy tractography with Eudx: how to place seeds at the center of the voxels?</a><div class="summary"> I am trying to perform tractography from a ROI and create seeds at the center of each voxel of this ROI. The ROI includes 8 voxels: 9 voxels forming a square, less one corner. To create seeds at the center of each voxel, do the tractography and display the results, I run the following commands: tenmodel = dti.TensorModel(my_gradient_table) tenfit = tenmodel.fit(my_dmri_data) seeds = seeds_from_ma</div></div><span class="ago" title="published:Tue, 12 May 2015 22:25:40 GMT -- Received: Wed, 13 May 2015 00:16:59 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4aa1b019:138c065d:603b4255_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3063/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4aa1b019:138c065d:603b4255" data-navigation="inline" data-title="Dipy visualization: is it possible to position tensor ellipsoid slice in fvtk?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d4aa1b019:138c065d:603b4255" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6465 --></div><!-- react-empty: 6466 --><div class="content"><a href="http://neurostars.org/p/3063/" target="_blank" rel="noopener" class="title">Dipy visualization: is it possible to position tensor ellipsoid slice in fvtk?</a><div class="summary"> I am trying to display tensor information with fvtk.tensor and add other information such as streamlines or FA slices but they do not align. It seems like fvtk.tensor does not have a parameter to set the 3D position of a slice. For example I do not see any option to change the z-position of the axial slice below in the visualization. Plotting another axial slice ( iAxial = 25 ) will simply be ove</div></div><span class="ago" title="published:Tue, 12 May 2015 22:04:48 GMT -- Received: Wed, 13 May 2015 00:16:59 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d461f4336:11f61a2b:603b4255_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3060/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d461f4336:11f61a2b:603b4255" data-navigation="inline" data-title="Dipy visualization with missing (?) affine parameter" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d461f4336:11f61a2b:603b4255" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6481 --></div><!-- react-empty: 6482 --><div class="content"><a href="http://neurostars.org/p/3060/" target="_blank" rel="noopener" class="title">Dipy visualization with missing (?) affine parameter</a><div class="summary"> I would like to add slices and other elements in dipy fvtk visualization but no affine parameter is available for element such as dipy. viz .fvtk.slicer . The voxsz parameter helps rescaling but what about rotation/translation/etc.? Any help would be greatly appreciated. </div></div><span class="ago" title="published:Tue, 12 May 2015 01:03:30 GMT -- Received: Tue, 12 May 2015 03:16:03 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d2f941fba:b55e0a6:69c6e04e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3059/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d2f941fba:b55e0a6:69c6e04e" data-navigation="inline" data-title="Dipy visualization (fvtk) crash when saving series of images" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d2f941fba:b55e0a6:69c6e04e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6497 --></div><!-- react-empty: 6498 --><div class="content"><a href="http://neurostars.org/p/3059/" target="_blank" rel="noopener" class="title">Dipy visualization (fvtk) crash when saving series of images</a><div class="summary"> I wrote a script to save a series of slice visualization (ellipsoids) with dipy vtk module (fvtk). However after a few slices (around 6) python crashes with only error message "Segmentation fault (core dumped)". The script goes: tenmodel = dti.TensorModel(my_gradient_table) tenfit = tenmodel.fit(my_masked_dwi_data) RGB = color_fa(my_FA_image, tenfit.evecs) sphere = get_sphere('symmetric724') ren </div></div><span class="ago" title="published:Thu, 07 May 2015 18:07:20 GMT -- Received: Thu, 07 May 2015 18:12:26 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d29233197:9534926:69c6e04e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3058/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d29233197:9534926:69c6e04e" data-navigation="inline" data-title="DARTEL, class images and DARTELNorm2MNI" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14d29233197:9534926:69c6e04e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6513 --></div><!-- react-empty: 6514 --><div class="content"><a href="http://neurostars.org/p/3058/" target="_blank" rel="noopener" class="title">DARTEL, class images and DARTELNorm2MNI</a><div class="summary">Hello, this is probably a very simple problem, but still it puzzles me. I would like to use the DARTELNorm2MNI on the output of a DARTEL_Template not on one but several class images. When using the "getclass1images" function found in spm.fmri.preprocess it works fine, for any class of images that was generated previously; when using the flattened output (one list) of the "get2classes" function fou</div></div><span class="ago" title="published:Wed, 06 May 2015 09:37:59 GMT -- Received: Wed, 06 May 2015 12:11:21 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cbf7a1a3e:d99dbbb:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3047/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cbf7a1a3e:d99dbbb:7af40b3e" data-navigation="inline" data-title="nipype spm realign/smooth file question" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cbf7a1a3e:d99dbbb:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6529 --></div><!-- react-empty: 6530 --><div class="content"><a href="http://neurostars.org/p/3047/" target="_blank" rel="noopener" class="title">nipype spm realign/smooth file question</a><div class="summary">Hi all, I have been playing with making a nipype workflow using nodes from spm to mimic pre-processing that I usually do in spm. I have encountered a systematic bug that occurs in between realignment and smoothing. Typically, when realigned files are passed into smoothing using spm, the realigned files (e.g. ra*.nii files) all become smoothed &amp; realigned files (sra*.nii files). When I setup these </div></div><span class="ago" title="published:Wed, 15 Apr 2015 23:09:55 GMT -- Received: Wed, 15 Apr 2015 23:46:32 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb9097bf6:b5c6960:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3041/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb9097bf6:b5c6960:7af40b3e" data-navigation="inline" data-title="Nipype SPM Preproc 3d/4d Handling" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb9097bf6:b5c6960:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6545 --></div><!-- react-empty: 6546 --><div class="content"><a href="http://neurostars.org/p/3041/" target="_blank" rel="noopener" class="title">Nipype SPM Preproc 3d/4d Handling</a><div class="summary"> Hi @satra and all, In nipype 0.10.0 it looks like two similar nipype interfaces - slice timing and realign in particular - are handling the filename conversion of input files differently. They both pass the inputs through scans_for_fnames, but slicetiming uses `keep4d = False` and realign uses `keep4d = True`. I'm trying to replicate a manual run through the SPM GUI (v5236), and slicetiming is re</div></div><span class="ago" title="published:Tue, 14 Apr 2015 15:08:00 GMT -- Received: Tue, 14 Apr 2015 17:45:48 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb864a6be:b024bc3:be345dbb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3040/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb864a6be:b024bc3:be345dbb" data-navigation="inline" data-title="skipping nodes (wf branch without smoothing)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb864a6be:b024bc3:be345dbb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6561 --></div><!-- react-empty: 6562 --><div class="content"><a href="http://neurostars.org/p/3040/" target="_blank" rel="noopener" class="title">skipping nodes (wf branch without smoothing)</a><div class="summary"> Dear all, I am doing smoothing with fsl and would like to have fwhm as iterable including no smoothing (fwhm=0). However, fslmaths requires a value &gt; 0 for the fwhm to work. There is a 4 year old post with a similar question (https://neurostars.org/p/2639/). Chris and Satra suggested some hacks but also implied that there might be more elegant ways to do this in the future. Since the future is no</div></div><span class="ago" title="published:Tue, 14 Apr 2015 14:23:48 GMT -- Received: Tue, 14 Apr 2015 14:45:46 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb3e2859a:9a34662:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3038/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb3e2859a:9a34662:7af40b3e" data-navigation="inline" data-title="spm.threshold counts from 1, other nodes from 0" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb3e2859a:9a34662:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6577 --></div><!-- react-empty: 6578 --><div class="content"><a href="http://neurostars.org/p/3038/" target="_blank" rel="noopener" class="title">spm.threshold counts from 1, other nodes from 0</a><div class="summary"> When iterating with a mapnode over Threshold I noticed that the first generated directories which names were labelled starting from 0: + while the latter output files were named starting from 1: '_spmT_' + + '_thr.hdr' which results in a structure where numbers are not coherent, and therefore need extra processing: dir0 -&gt; file1 dir1 -&gt; file2 etc is there a reason for that? </div></div><span class="ago" title="published:Mon, 13 Apr 2015 16:05:33 GMT -- Received: Mon, 13 Apr 2015 17:45:09 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb33d9f1a:944035a:be345dbb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3036/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb33d9f1a:944035a:be345dbb" data-navigation="inline" data-title="Plugins options in config file" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14cb33d9f1a:944035a:be345dbb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6593 --></div><!-- react-empty: 6594 --><div class="content"><a href="http://neurostars.org/p/3036/" target="_blank" rel="noopener" class="title">Plugins options in config file</a><div class="summary"> Dear Experts, Is it not possible to have plugin=LSF in configuration file while i setup the execution because i saw in documentation that "possible values: Linear, MultiProc, SGE, IPython; default value: Linear)" . If I use Linear for working in LSF system does it effects the execution ? Thank you. Regards, Nabin </div></div><span class="ago" title="published:Mon, 13 Apr 2015 14:42:41 GMT -- Received: Mon, 13 Apr 2015 14:45:02 GMT">19mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c98d5f859:12b3c04:be345dbb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3034/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c98d5f859:12b3c04:be345dbb" data-navigation="inline" data-title="store all results from a datasink in one directory without subdirectories" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c98d5f859:12b3c04:be345dbb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6609 --></div><!-- react-empty: 6610 --><div class="content"><a href="http://neurostars.org/p/3034/" target="_blank" rel="noopener" class="title">store all results from a datasink in one directory without subdirectories</a><div class="summary"> Hi all, following my previous question , I would like to store all result files in a common directory without any subdirectory, so I can run batch post-treatment. The thing is: as I used a Mapnode as one of my inputs to datasink, it seems it cannot get over the subdirectories organization; even when setting 'remove_dest_dir' to 'True'. I cannot either filter the subdirectory names with 'strip_dir</div></div><span class="ago" title="published:Wed, 08 Apr 2015 10:16:06 GMT -- Received: Wed, 08 Apr 2015 11:41:41 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c9453ce57:2bf1db1:7af40b3e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3032/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c9453ce57:2bf1db1:7af40b3e" data-navigation="inline" data-title="using spm thresholding on several contrast results" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c9453ce57:2bf1db1:7af40b3e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6625 --></div><!-- react-empty: 6626 --><div class="content"><a href="http://neurostars.org/p/3032/" target="_blank" rel="noopener" class="title">using spm thresholding on several contrast results</a><div class="summary"> Hi all, I designed a basic two-sample T test pipeline to analyze MRI files from patients and controls. It follows the Nipype Beginner's guide here , excepted that it is a two sample T test (not one sample T test) and that I use four contrasts (see here ). I wanted to threshold the resulting contrast data and I naively thought that the 'stat_image' and 'contrast_index' inputs could contain several</div></div><span class="ago" title="published:Tue, 07 Apr 2015 12:57:53 GMT -- Received: Tue, 07 Apr 2015 14:41:02 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c76b2fa3f:94555ec:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3031/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c76b2fa3f:94555ec:de01cf22" data-navigation="inline" data-title="Get Estimated Params from SPM Corregister w/ Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c76b2fa3f:94555ec:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6641 --></div><!-- react-empty: 6642 --><div class="content"><a href="http://neurostars.org/p/3031/" target="_blank" rel="noopener" class="title">Get Estimated Params from SPM Corregister w/ Nipype</a><div class="summary"> I'm running nmi correg between a CT and T1 with nipype's spm interface and I want to grab the resultant matrix with estimated parameters for later use. Is there an easy way to do this with the nipype wrapper? Alternatively, I see in the SPM manual that the results are stored in the header of the source image, but I'm not sure what field it's stored under. If anyone happens to know that I can just</div></div><span class="ago" title="published:Wed, 01 Apr 2015 18:58:43 GMT -- Received: Wed, 01 Apr 2015 20:36:23 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c571d1184:2ae4260:be345dbb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3025/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c571d1184:2ae4260:be345dbb" data-navigation="inline" data-title="Non-linear registration/normalization in Nipy (other than FSL, SPM or ANTs)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c571d1184:2ae4260:be345dbb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6657 --></div><!-- react-empty: 6658 --><div class="content"><a href="http://neurostars.org/p/3025/" target="_blank" rel="noopener" class="title">Non-linear registration/normalization in Nipy (other than FSL, SPM or ANTs)</a><div class="summary"> Hello all, I'm very new to neuroimaging, so bear with me as I explain what I am interested in doing. I'm building a pipeline for resting-state fMRI data; part of that includes normalization to MNI space. The pipeline is in Python and we would like to avoid using FSL and SPM as much as possible. Is there a built-in tool within Nipy that does nonlinear spatial registration? I know using FNIRT, SPM </div></div><span class="ago" title="published:Thu, 26 Mar 2015 16:20:12 GMT -- Received: Thu, 26 Mar 2015 17:24:25 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c345494d7:1bc9790:24a98465_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3023/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c345494d7:1bc9790:24a98465" data-navigation="inline" data-title="News: Dipy 0.9.2 is now available for download!" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c345494d7:1bc9790:24a98465" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6673 --></div><!-- react-empty: 6674 --><div class="content"><a href="http://neurostars.org/p/3023/" target="_blank" rel="noopener" class="title">News: Dipy 0.9.2 is now available for download!</a><div class="summary">Dear all, We are very happy to announce a new release of Diffusion Imaging in Python (DIPY) Here is a quick summary of our new features. DIPY 0.9.2 (Wednesday, 18 March 2015) * Anatomically Constrained Tissue Classifiers for Tracking * Massive speedup of Constrained Spherical Deconvolution (CSD) * Recursive calibration of response function for CSD * New experimental framework for clustering * Impr</div></div><span class="ago" title="published:Thu, 19 Mar 2015 21:57:01 GMT -- Received: Thu, 19 Mar 2015 23:18:20 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c3265e876:1035e1d:24a98465_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3021/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c3265e876:1035e1d:24a98465" data-navigation="inline" data-title="How to use HCP data with Amazon Elastic Compute Cloud (EC2)?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c3265e876:1035e1d:24a98465" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6689 --></div><!-- react-empty: 6690 --><div class="content"><a href="http://neurostars.org/p/3021/" target="_blank" rel="noopener" class="title">How to use HCP data with Amazon Elastic Compute Cloud (EC2)?</a><div class="summary"> As we know, HCP data is now mirrored on Amazon Web Services. Do you known how we can use these data directly with Amazon EC2? </div></div><span class="ago" title="published:Thu, 19 Mar 2015 14:07:21 GMT -- Received: Thu, 19 Mar 2015 14:18:01 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2e8878f4:984294f:24a98465_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3014/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2e8878f4:984294f:24a98465" data-navigation="inline" data-title="create_bedpostx_pipeline 0 based files do not exist" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2e8878f4:984294f:24a98465" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6705 --></div><!-- react-empty: 6706 --><div class="content"><a href="http://neurostars.org/p/3014/" target="_blank" rel="noopener" class="title">create_bedpostx_pipeline 0 based files do not exist</a><div class="summary"> Hello, I'm creating a dwi pipeline that incorporates the existing create_bedpostx_pipeline but am getting an error on a Merge Node. The crash file gives me the following message: TraitError: Each element of the 'in_files' trait of a DynamicTraitedSpec instance must be an existing file name, but a value of '/scratch/madlab/cmind/probtrack/dwiproc/bedpostx/_subject_id_IRC04H_17M003/_n_fibres_1/xfib</div></div><span class="ago" title="published:Wed, 18 Mar 2015 18:02:43 GMT -- Received: Wed, 18 Mar 2015 20:17:18 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2d3ea46a:8f8f4c1:e0550d54_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3013/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2d3ea46a:8f8f4c1:e0550d54" data-navigation="inline" data-title="InterfaceResult is not iterable" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2d3ea46a:8f8f4c1:e0550d54" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6721 --></div><!-- react-empty: 6722 --><div class="content"><a href="http://neurostars.org/p/3013/" target="_blank" rel="noopener" class="title">InterfaceResult is not iterable</a><div class="summary">Dear Experts, I am trying to run the example code 'dmri_fsl_dti.py' with the sample data in LSF cluster with the bsub options. It runs well at the beginning and generates some output as well but at the end it gives the error as 150318-13:00:11,46 workflow INFO: Saving crash info to /gpfs/fs1/home/nkoirala/nipype-0.10.0/examples/crash-20150318-130011-nkoirala-_xfibres16.pklz 150318-13:00:11,47 work</div></div><span class="ago" title="published:Wed, 18 Mar 2015 13:28:32 GMT -- Received: Wed, 18 Mar 2015 14:17:02 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2a060f4e:7f0d487:e0550d54_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3008/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2a060f4e:7f0d487:e0550d54" data-navigation="inline" data-title="What QC tools are available for ADNI phantoms?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2a060f4e:7f0d487:e0550d54" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6737 --></div><!-- react-empty: 6738 --><div class="content"><a href="http://neurostars.org/p/3008/" target="_blank" rel="noopener" class="title">What QC tools are available for ADNI phantoms?</a><div class="summary"> We collect an ADNI phantom from our scanner every day we run a participant, and I'm curious what type of QC routines folks run on these phantoms. So far, the only package I've found is Aqual2 ( http://www.loni.usc.edu/Software/moreinfo.php?package=Aqual2). Does anyone know if there are other software packages available for monitoring ADNI phantoms? </div></div><span class="ago" title="published:Tue, 17 Mar 2015 20:55:01 GMT -- Received: Tue, 17 Mar 2015 23:16:22 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2a060f4e:7f0d486:e0550d54_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3007/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2a060f4e:7f0d486:e0550d54" data-navigation="inline" data-title="What QC tools are available for ADNI phantoms?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c2a060f4e:7f0d486:e0550d54" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6753 --></div><!-- react-empty: 6754 --><div class="content"><a href="http://neurostars.org/p/3007/" target="_blank" rel="noopener" class="title">What QC tools are available for ADNI phantoms?</a><div class="summary"> We collect an ADNI phantom from our scanner every day we run a participant, and I'm curious what type of QC routines folks run on these phantoms. So far, the only package I've found is Aqual2 ( http://www.loni.usc.edu/Software/moreinfo.php?package=Aqual2). Does anyone know if there are other software packages available for monitoring ADNI phantoms? </div></div><span class="ago" title="published:Tue, 17 Mar 2015 20:54:59 GMT -- Received: Tue, 17 Mar 2015 23:16:22 GMT">20mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c0f9d5b75:94ab1cb:e0550d54_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3005/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c0f9d5b75:94ab1cb:e0550d54" data-navigation="inline" data-title="News: Nipype coding sprint" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c0f9d5b75:94ab1cb:e0550d54" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6769 --></div><!-- react-empty: 6770 --><div class="content"><a href="http://neurostars.org/p/3005/" target="_blank" rel="noopener" class="title">News: Nipype coding sprint</a><div class="summary"> We would like to organise a worldwide Internet facilitated coding Nipype coding sprint. The goal is to convince as many developers as possible to set aside one day to focus on reviewing PRs, fixing bugs, adding new features and preparing for the new release. I have create a Doodle pool to help us choose the day: http://doodle.com/zcdz2hnmiwigyqx3 I'm looking forward to this (even though it will b</div></div><span class="ago" title="published:Thu, 12 Mar 2015 17:23:12 GMT -- Received: Thu, 12 Mar 2015 20:11:53 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c0ef882b8:906946e:cc12bc64_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3002/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c0ef882b8:906946e:cc12bc64" data-navigation="inline" data-title="[nipy] Spatial interpolation in SpaceTimeRealign algorithm" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c0ef882b8:906946e:cc12bc64" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6785 --></div><!-- react-empty: 6786 --><div class="content"><a href="http://neurostars.org/p/3002/" target="_blank" rel="noopener" class="title">[nipy] Spatial interpolation in SpaceTimeRealign algorithm</a><div class="summary"> Hi, I am agonising about whether or not to use SpaceTimeRealign for my resting state data, that I want to sample on a surface. The advantage of doing motion correction separately is that it gives me transformations for projecting the surface onto each volume separately, thus completely avoiding to interpolate the functional data. Now I tried to understand SpaceTimeRealign better to decide whether</div></div><span class="ago" title="published:Thu, 12 Mar 2015 15:37:52 GMT -- Received: Thu, 12 Mar 2015 17:11:49 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c09d1545e:74fe427:e0550d54_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/3000/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c09d1545e:74fe427:e0550d54" data-navigation="inline" data-title="Job: Research Assistant Position in Neuroimaging at NIH" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c09d1545e:74fe427:e0550d54" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6801 --></div><!-- react-empty: 6802 --><div class="content"><a href="http://neurostars.org/p/3000/" target="_blank" rel="noopener" class="title">Job: Research Assistant Position in Neuroimaging at NIH</a><div class="summary"> Research Assistant in Neuroimaging General Description: The Translational Neuroradiology Unit (TNU) at the National Institute of Neurological Disorders and Stroke (NINDS) is a world-class laboratory focusing on the use of advanced magnetic resonance imaging (MRI) techniques to understand the sources of disability in multiple sclerosis (MS) and other inflammatory diseases of the central nervous sy</div></div><span class="ago" title="published:Wed, 11 Mar 2015 15:42:03 GMT -- Received: Wed, 11 Mar 2015 17:10:55 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c05f4076f:5ff190f:e0550d54_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2999/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c05f4076f:5ff190f:e0550d54" data-navigation="inline" data-title="MapNode Repeats Computations Serially After Completion?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14c05f4076f:5ff190f:e0550d54" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6817 --></div><!-- react-empty: 6818 --><div class="content"><a href="http://neurostars.org/p/2999/" target="_blank" rel="noopener" class="title">MapNode Repeats Computations Serially After Completion?</a><div class="summary"> This has been asked a few times before, but I have run into it again. I run a MapNode through parallel plugin (MultiProc or IPython) and it runs as it should, but after all of the children have completed, each one is run again in a serial fashion. I know this probably has to do with how the hash files are being used, but I cannot seem to figure it out. I am on the most current master from github.</div></div><span class="ago" title="published:Tue, 10 Mar 2015 21:05:50 GMT -- Received: Tue, 10 Mar 2015 23:10:21 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bf0b276e5:3b6f7fe:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2998/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bf0b276e5:3b6f7fe:de01cf22" data-navigation="inline" data-title="News: Registration open for HBM Hackathon 2015" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bf0b276e5:3b6f7fe:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6874 --></div><!-- react-empty: 6875 --><div class="content"><a href="http://neurostars.org/p/2998/" target="_blank" rel="noopener" class="title">News: Registration open for HBM Hackathon 2015</a><div class="summary"> 2015 OHBM Hackathon Registration Now Open! 2015 OHBM Hackathon Registration is now open! Aloha! Registration is now open for the 3rd Annual OHBM Hackathon taking place at the Hawaii Convention Center in Hawaii on the beautiful island of Oahu from June 12-14, 2015. During this three day event, the first two days will focus on collaborative and open neuroscience projects in data analysis and method</div></div><span class="ago" title="published:Fri, 06 Mar 2015 18:50:53 GMT -- Received: Fri, 06 Mar 2015 20:06:42 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bed79be22:19c9a69:603b4255_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2997/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bed79be22:19c9a69:603b4255" data-navigation="inline" data-title="News: Call for participation: Fiber Tractography Challenge at ISMRM 2015" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bed79be22:19c9a69:603b4255" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6890 --></div><!-- react-empty: 6891 --><div class="content"><a href="http://neurostars.org/p/2997/" target="_blank" rel="noopener" class="title">News: Call for participation: Fiber Tractography Challenge at ISMRM 2015</a><div class="summary">[SORRY IF THIS IS A DUPLICATE] Dear Colleagues: The Diffusion Study Group is sponsoring a novel Fiber Tractography Challenge to stimulate an interactive discussion on diffusion reconstruction and tractography methods at this year’s study group meeting at ISMRM 2015 in Toronto. The Challenge will use a simulated but highly realistic diffusion-weighted dataset in which 26 specific white matter bundl</div></div><span class="ago" title="published:Fri, 06 Mar 2015 02:24:29 GMT -- Received: Fri, 06 Mar 2015 05:05:53 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14beaa96b6c:165e2e2:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2996/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14beaa96b6c:165e2e2:de01cf22" data-navigation="inline" data-title="Neo python for .nev and .ns3 files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14beaa96b6c:165e2e2:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6906 --></div><!-- react-empty: 6907 --><div class="content"><a href="http://neurostars.org/p/2996/" target="_blank" rel="noopener" class="title">Neo python for .nev and .ns3 files</a><div class="summary">I'm trying to figure out a way to use Python's neo module to deal with .nev and .ns3 but can't find an appropriate IO function. I'm able to read the files with BlackrockIO but it says the headers are empty which doesn't make sense. The documentation exclusively mentions .ns5 files which are closely related to yet some how distinct from .ns3 and .nev files. Can anyone direct me to a more appropriat</div></div><span class="ago" title="published:Thu, 05 Mar 2015 14:46:55 GMT -- Received: Thu, 05 Mar 2015 15:59:06 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14be391bd3d:746e068:be345dbb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2992/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14be391bd3d:746e068:be345dbb" data-navigation="inline" data-title="Mrtrix_group_connectivity tutorial, cannot set undefined parsellation name" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14be391bd3d:746e068:be345dbb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6922 --></div><!-- react-empty: 6923 --><div class="content"><a href="http://neurostars.org/p/2992/" target="_blank" rel="noopener" class="title">Mrtrix_group_connectivity tutorial, cannot set undefined parsellation name</a><div class="summary">Hello, &#8203;I am running tutorial pipeline dmri_group_connectivity.py I get this error: import nipype.interfaces.fsl as fsl import nipype.interfaces.freesurfer as fs # freesurfer import os.path as op # system functions import cmp from nipype.workflows.dmri.mrtrix.group_connectivity import create_group_connectivity_pipeline from nipype.workflows.dmri.connectivity.group_connectivity import (create_merge</div></div><span class="ago" title="published:Wed, 04 Mar 2015 04:34:39 GMT -- Received: Wed, 04 Mar 2015 06:55:54 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bdbd5dbe1:411ac30:be345dbb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2990/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bdbd5dbe1:411ac30:be345dbb" data-navigation="inline" data-title="[nipype] copy directory to node directory" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bdbd5dbe1:411ac30:be345dbb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6938 --></div><!-- react-empty: 6939 --><div class="content"><a href="http://neurostars.org/p/2990/" target="_blank" rel="noopener" class="title">[nipype] copy directory to node directory</a><div class="summary"> I am trying to write an interface for Freesurfer's mri_glmfit-sim to be able to do cluster correction with precomputed permutation data. The plan is to first run a node using the GLMFit interface and passing its results to a node using this GLMFitSim interface to do cluster correction. When I just pass glm_dir from the GLMFit node to the GLMFitSim node, the results of the cluster corrections are </div></div><span class="ago" title="published:Mon, 02 Mar 2015 17:04:14 GMT -- Received: Mon, 02 Mar 2015 18:53:20 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bc8836364:5a4c43e:de01cf22_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2986/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bc8836364:5a4c43e:de01cf22" data-navigation="inline" data-title="[nipype] copy directory to node directory" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bc8836364:5a4c43e:de01cf22" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6954 --></div><!-- react-empty: 6955 --><div class="content"><a href="http://neurostars.org/p/2986/" target="_blank" rel="noopener" class="title">[nipype] copy directory to node directory</a><div class="summary"> I am trying to write an interface for Freesurfer's mri_glmfit-sim to be able to do cluster correction with precomputed permutation data. The plan is to first run a node using the GLMFit interface and passing its results to a node using this GLMFitSim interface to do cluster correction. When I just pass glm_dir from the GLMFit node to the GLMFitSim node, the results of the cluster corrections are </div></div><span class="ago" title="published:Thu, 26 Feb 2015 23:56:09 GMT -- Received: Fri, 27 Feb 2015 00:50:28 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bc35c3367:3e3be65:e0550d54_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2984/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bc35c3367:3e3be65:e0550d54" data-navigation="inline" data-title="Looking for a PET Modeling Toolbox" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bc35c3367:3e3be65:e0550d54" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6970 --></div><!-- react-empty: 6971 --><div class="content"><a href="http://neurostars.org/p/2984/" target="_blank" rel="noopener" class="title">Looking for a PET Modeling Toolbox</a><div class="summary"> Does anyone know of a good toolbox (python/open preferred, but I'll take what I can get) for modelling PET Time Activity Curves? I've been using PMOD with a simple reference tissue model (SRTM) but it seems to be buggy and I'd like to cross-check results, if anything is available. I know NiftyRec is available for reconstruction, but I couldn't find anything that would do the modeling. Anyone work</div></div><span class="ago" title="published:Wed, 25 Feb 2015 23:02:11 GMT -- Received: Thu, 26 Feb 2015 00:49:34 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bad721ca3:4282690:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2979/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bad721ca3:4282690:64bd10e3" data-navigation="inline" data-title="second level analysis Nipype - model estimation" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14bad721ca3:4282690:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 6986 --></div><!-- react-empty: 6987 --><div class="content"><a href="http://neurostars.org/p/2979/" target="_blank" rel="noopener" class="title">second level analysis Nipype - model estimation</a><div class="summary">Dear all, I ran this code following the Nipype tutorial I have the level 1 results in '/Users/emiliesalvia/Documents/post-doc_Marseille/expe_fMRI/NiPy/NiPyPe/NiPyPe_test/test/test_2/resultats/s*/level1/contrastestimate/con_%04d.img' with s1 and s2 and 8 contrasts, e.g. con_0001.img ########## CODE ##### import nipype.interfaces.matlab as mlab mlab.MatlabCommand.set_default_matlab_cmd('/Application</div></div><span class="ago" title="published:Sat, 21 Feb 2015 17:20:24 GMT -- Received: Sat, 21 Feb 2015 18:41:51 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ba46de328:192d13d:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2976/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ba46de328:192d13d:64bd10e3" data-navigation="inline" data-title="[nipype] Matlab version used by NiPype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ba46de328:192d13d:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7002 --></div><!-- react-empty: 7003 --><div class="content"><a href="http://neurostars.org/p/2976/" target="_blank" rel="noopener" class="title">[nipype] Matlab version used by NiPype</a><div class="summary"> Our admin just installed the 2014b version of Matlab. But the default one is still 2011b. In my shell config file I alias 'matlab' to the newer version. I also set the environment variable $MATLABCMD to the 2014er version. Still, matlab nodes run in nipype use the 2011er version. How can I change this?! </div></div><span class="ago" title="published:Fri, 20 Feb 2015 00:38:32 GMT -- Received: Fri, 20 Feb 2015 00:40:39 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b978bac80:1eadad3a:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2972/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b978bac80:1eadad3a:64bd10e3" data-navigation="inline" data-title="Forum: Node _prepfa0 failed to run on host" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b978bac80:1eadad3a:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7018 --></div><!-- react-empty: 7019 --><div class="content"><a href="http://neurostars.org/p/2972/" target="_blank" rel="noopener" class="title">Forum: Node _prepfa0 failed to run on host</a><div class="summary">Dear Experts, I am trying to run TBSS first step create tbss_1_preproc() with just two subjects to make sure about the things. But i am getting an error message saying Node _prepfa0 failed to run on host. The crash file displays the following error. (PYTHON2.7)[nkoirala@login01 PYTHON2.7]$ nipype_display_crash crash-20150217-104448-nkoirala-_prepfa0.pklz File: crash-20150217-104448-nkoirala-_prepf</div></div><span class="ago" title="published:Tue, 17 Feb 2015 10:08:59 GMT -- Received: Tue, 17 Feb 2015 12:38:08 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b92649216:1d971757:69c6e04e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2971/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b92649216:1d971757:69c6e04e" data-navigation="inline" data-title="Forum: unexpected keyword argument 'plugin'" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b92649216:1d971757:69c6e04e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7034 --></div><!-- react-empty: 7035 --><div class="content"><a href="http://neurostars.org/p/2971/" target="_blank" rel="noopener" class="title">Forum: unexpected keyword argument 'plugin'</a><div class="summary"> Dear Experts, What could be the reason that i am getting the error " run() got an unexpected keyword argument 'plugin' " when trying to run it with plugin 'LSF'. The command i used is as folllows: mybet.run(plugin='LSF', plugin_args=dict(bsub_args='-J testrosen -app 2G -W 10')) Traceback (most recent call last): File " ", line 1, in TypeError: run() got an unexpected keyword argument 'plugin' Tha</div></div><span class="ago" title="published:Mon, 16 Feb 2015 10:35:56 GMT -- Received: Mon, 16 Feb 2015 12:37:19 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b7e43792c:173ed5d2:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2960/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b7e43792c:173ed5d2:64bd10e3" data-navigation="inline" data-title="Forum: cmp not installed" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b7e43792c:173ed5d2:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7050 --></div><!-- react-empty: 7051 --><div class="content"><a href="http://neurostars.org/p/2960/" target="_blank" rel="noopener" class="title">Forum: cmp not installed</a><div class="summary">Dear Experts, With your kind support I was able to install nipype but to make it running i am having lot of difficulties, may be because i could not get the hold of exact procedure to follow to perform an analysis. Just as a trial i wanted to use tbss.py but while importing it says an error as , nipype/__init__.py:57: UserWarning: Running the tests from the install directory may trigger some failu</div></div><span class="ago" title="published:Thu, 12 Feb 2015 14:34:46 GMT -- Received: Thu, 12 Feb 2015 14:48:46 GMT">21mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b7a66269e:15fef389:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2957/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b7a66269e:15fef389:64bd10e3" data-navigation="inline" data-title="[nipype] Using interfaces.ants.registration" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b7a66269e:15fef389:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7066 --></div><!-- react-empty: 7067 --><div class="content"><a href="http://neurostars.org/p/2957/" target="_blank" rel="noopener" class="title">[nipype] Using interfaces.ants.registration</a><div class="summary"> Hi, I have installed ants from neurodebian (version 1.9.2) in order to try to run the fmri_ants_openfmri example. However, the interface interfaces.ants.registration fails because it can find the antsRegeistration command. It seems that this command is not installed. Do I need a specific version of ants? </div></div><span class="ago" title="published:Wed, 11 Feb 2015 18:22:35 GMT -- Received: Wed, 11 Feb 2015 20:48:09 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b73501013:138d0674:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2949/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b73501013:138d0674:64bd10e3" data-navigation="inline" data-title="Job: Research position (PhD) available in Dresden" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b73501013:138d0674:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7082 --></div><!-- react-empty: 7083 --><div class="content"><a href="http://neurostars.org/p/2949/" target="_blank" rel="noopener" class="title">Job: Research position (PhD) available in Dresden</a><div class="summary">The Translational Developmental Neuroscience Section of the Department of Child and Adolescent Psychiatry, Faculty of Medicine, Technische Universität Dresden, Germany invites applications for a Research position (PhD) (with a focus on autism spectrum disorder and eating disorders) within the Translational Developmental Neuroscience Group (Prof. Stefan Ehrlich) for projects with a focus on social </div></div><span class="ago" title="published:Tue, 10 Feb 2015 10:40:17 GMT -- Received: Tue, 10 Feb 2015 11:46:41 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b70179128:12ef145b:69c6e04e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2947/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b70179128:12ef145b:69c6e04e" data-navigation="inline" data-title="[nipype] MultiProc different number of processors for different nodes" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b70179128:12ef145b:69c6e04e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7098 --></div><!-- react-empty: 7099 --><div class="content"><a href="http://neurostars.org/p/2947/" target="_blank" rel="noopener" class="title">[nipype] MultiProc different number of processors for different nodes</a><div class="summary"> I am wrapping a Matlab script in a node that uses a toolbox that our lab only has 3 licenses for. The rest of the nodes of my workflow uses free software. Is it possible to run all other nodes in parallel (e.g. 'n_procs' : 36), but not the Matlab node? </div></div><span class="ago" title="published:Mon, 09 Feb 2015 20:43:23 GMT -- Received: Mon, 09 Feb 2015 20:46:07 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b6f729533:124c7a32:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2944/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b6f729533:124c7a32:64bd10e3" data-navigation="inline" data-title="Calculation of norm_motion (rapidart)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b6f729533:124c7a32:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7114 --></div><!-- react-empty: 7115 --><div class="content"><a href="http://neurostars.org/p/2944/" target="_blank" rel="noopener" class="title">Calculation of norm_motion (rapidart)</a><div class="summary"> Dear all, I want to know how rapidart exactly calculates the parameter "norm_motion". The six rigid body parameters are used for calculation and each value refers to the previous frame. But what is the exact formula or respectively the meaning of the formula? Thank you! Juliane </div></div><span class="ago" title="published:Mon, 09 Feb 2015 16:13:32 GMT -- Received: Mon, 09 Feb 2015 17:45:55 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b5ffce044:e0bb86d:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2936/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b5ffce044:e0bb86d:64bd10e3" data-navigation="inline" data-title="[nipype] Doc formatting issues in rsfmri_vol_surface_preprocessing_nipy" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b5ffce044:e0bb86d:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7130 --></div><!-- react-empty: 7131 --><div class="content"><a href="http://neurostars.org/p/2936/" target="_blank" rel="noopener" class="title">[nipype] Doc formatting issues in rsfmri_vol_surface_preprocessing_nipy</a><div class="summary"> Hi, I have noticed some formatting issues in the rsfmri_vol_surface_preprocessing_nipy example (see below the definition of create_reg_workflow here ). make html prints several ERROR: Unexpected indentation . The first offending line (378) looks like this: register = Workflow(name=name) inputnode = Node(interface=IdentityInterface(fields=['source_files', 'mean_image', 'subject_id', 'subjects_dir'</div></div><span class="ago" title="published:Fri, 06 Feb 2015 14:54:16 GMT -- Received: Fri, 06 Feb 2015 17:43:02 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b5275d54a:986583b:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2931/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b5275d54a:986583b:64bd10e3" data-navigation="inline" data-title="Intensity normalization via 4D Median vs. 4D Mean" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b5275d54a:986583b:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7146 --></div><!-- react-empty: 7147 --><div class="content"><a href="http://neurostars.org/p/2931/" target="_blank" rel="noopener" class="title">Intensity normalization via 4D Median vs. 4D Mean</a><div class="summary"> Hey All, I'm working with some pre-processed (EPI) data and realized that the voxel intensities across scans were scaled using the 4D median (using the fslmaths -ing N), as opposed to scaling to a 4D mean of N. One of the only places I could find that talked about the reasoning behind using the median is in a preprocessing how-to by Mark Jenkinson and Steve Smith (http://afni.nimh.nih.gov/sscc/st</div></div><span class="ago" title="published:Wed, 04 Feb 2015 02:29:54 GMT -- Received: Wed, 04 Feb 2015 02:40:29 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4df336aa:824c523:69c6e04e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2919/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4df336aa:824c523:69c6e04e" data-navigation="inline" data-title="DICOMConvertInputSpec error help (from DICOM import documentation)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4df336aa:824c523:69c6e04e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7162 --></div><!-- react-empty: 7163 --><div class="content"><a href="http://neurostars.org/p/2919/" target="_blank" rel="noopener" class="title">DICOMConvertInputSpec error help (from DICOM import documentation)</a><div class="summary"> Does anybody know why I might be getting the error, when using line-for-line copies (except for path changes) of the nipype documentation at https://github.com/miykael/nipype-beginner-s-guide/blob/master/dicom2nifti.py: traits.trait_errors.TraitError: The 'dicom_dir' trait of a DICOMConvertInputSpec instance must be an existing directory name, but a value of '~/experiment1/dicom/subject1' was spe</div></div><span class="ago" title="published:Tue, 03 Feb 2015 04:22:09 GMT -- Received: Tue, 03 Feb 2015 05:39:19 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4ca94a2f:797fe45:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2916/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4ca94a2f:797fe45:64bd10e3" data-navigation="inline" data-title="QA Metrics for Whole Brain Statistical Maps" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4ca94a2f:797fe45:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7178 --></div><!-- react-empty: 7179 --><div class="content"><a href="http://neurostars.org/p/2916/" target="_blank" rel="noopener" class="title">QA Metrics for Whole Brain Statistical Maps</a><div class="summary"> Hi Neurostars, There are most definitely quality analysis (QA) stuffs out there for raw functional data, but less so for statistical maps. We are putting together a tool, and I thought it would be fun to get others feedback about the kind of features / plots that would be useful. Here is our tiny list so far: - standard measures of central tendency and variance - normality of the distribution acr</div></div><span class="ago" title="published:Mon, 02 Feb 2015 21:22:57 GMT -- Received: Mon, 02 Feb 2015 23:38:57 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4aba733c:6e66547:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2906/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4aba733c:6e66547:64bd10e3" data-navigation="inline" data-title="[nipype] Documentation formatting issue in nipype.interfaces.fsl.preprocess" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4aba733c:6e66547:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7194 --></div><!-- react-empty: 7195 --><div class="content"><a href="http://neurostars.org/p/2906/" target="_blank" rel="noopener" class="title">[nipype] Documentation formatting issue in nipype.interfaces.fsl.preprocess</a><div class="summary"> While updating the doc, I noticed that documentation for FUGUE (in nipype.interfaces.fsl.preprocess ) fails in docstrings. More specifically, line 1267: &gt;&gt;&gt; fugue.cmdline #doctest: +ELLIPSIS 'fugue --in=epi.nii --mask=epi_mask.nii --loadshift=vsm.nii --unwarpdir=y --unwarp=epi_unwarped.nii.gz' produces the following error: /home/md238665/Code/nipype/doc/interfaces/generated/nipype.interfaces.fsl.</div></div><span class="ago" title="published:Mon, 02 Feb 2015 14:20:29 GMT -- Received: Mon, 02 Feb 2015 14:38:28 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4aba733c:6e66546:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2905/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4aba733c:6e66546:64bd10e3" data-navigation="inline" data-title="Installation error: Sphinx" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b4aba733c:6e66546:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7210 --></div><!-- react-empty: 7211 --><div class="content"><a href="http://neurostars.org/p/2905/" target="_blank" rel="noopener" class="title">Installation error: Sphinx</a><div class="summary"> Dear experts, I am trying to install nipype but it give me the error saying somthing like 'Sphinx is not installed , cannot build docs ' . I installed sphinx and also build sphinx. But still the same error. Could not figure out what i am missing . I am installing all the packages required using pip or easy_install commnads and used the same for installing nipype. It would be really great if someo</div></div><span class="ago" title="published:Mon, 02 Feb 2015 13:24:24 GMT -- Received: Mon, 02 Feb 2015 14:38:28 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b3fc702df:3ed76f9:69c6e04e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2899/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b3fc702df:3ed76f9:69c6e04e" data-navigation="inline" data-title="NiPyPe 1st pipeline" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b3fc702df:3ed76f9:69c6e04e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7226 --></div><!-- react-empty: 7227 --><div class="content"><a href="http://neurostars.org/p/2899/" target="_blank" rel="noopener" class="title">NiPyPe 1st pipeline</a><div class="summary">Dear all, I tried to run a pipeline and I got the following error. Last login: Sat Jan 31 07:39:58 on console MacBook-Air-de-Emilie:~ emiliesalvia$ cd Documents/post-doc_Marseille/expe_fMRI/NiPy/NiPyPe/NiPyPe_test/test/ MacBook-Air-de-Emilie:test emiliesalvia$ MacBook-Air-de-Emilie:test emiliesalvia$ ls nipype_test.py test.nii MacBook-Air-de-Emilie:test emiliesalvia$ MacBook-Air-de-Emilie:test emi</div></div><span class="ago" title="published:Sat, 31 Jan 2015 09:19:47 GMT -- Received: Sat, 31 Jan 2015 11:36:21 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b361d4402:e6c5dd:64bd10e3_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2883/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b361d4402:e6c5dd:64bd10e3" data-navigation="inline" data-title="[nipype] Using octave instead of matlab" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b361d4402:e6c5dd:64bd10e3" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7242 --></div><!-- react-empty: 7243 --><div class="content"><a href="http://neurostars.org/p/2883/" target="_blank" rel="noopener" class="title">[nipype] Using octave instead of matlab</a><div class="summary"> Hi, This may seem a bit strange but I would like to launch some interface with (the main reason is that I don't have matlab at home and my scripts are mostly compatible with octave). The main problem is that octave don't accept -r as an option (this is used to pass the expression to evaluate which include the generated script). The corresponding option in octave is --eval (note that by default oc</div></div><span class="ago" title="published:Thu, 29 Jan 2015 13:08:50 GMT -- Received: Thu, 29 Jan 2015 14:34:22 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b3578574d:a6ffde:69c6e04e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2882/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b3578574d:a6ffde:69c6e04e" data-navigation="inline" data-title="[nipype] Overloading MatlabCommand" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b3578574d:a6ffde:69c6e04e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7258 --></div><!-- react-empty: 7259 --><div class="content"><a href="http://neurostars.org/p/2882/" target="_blank" rel="noopener" class="title">[nipype] Overloading MatlabCommand</a><div class="summary">Hi, I have tried to wrap some matlab commands as explained in http://nipy.org/nipype/devel/matlab_interface_devel.html (example 2). The interface is a subclass of MatlabCommand (overriding run ) and the input spec inherits from MatlabInputSpec (which has a mandatory script input). In this example, the mandatory script input is set in run by a call to _my_script and then MatlabCommand.run is called</div></div><span class="ago" title="published:Thu, 29 Jan 2015 09:20:07 GMT -- Received: Thu, 29 Jan 2015 11:34:14 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b319adaa6:8f05946:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2879/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b319adaa6:8f05946:6b730af7" data-navigation="inline" data-title="Philips scanner and FSL: correct diffusion gradient tables?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b319adaa6:8f05946:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7274 --></div><!-- react-empty: 7275 --><div class="content"><a href="http://neurostars.org/p/2879/" target="_blank" rel="noopener" class="title">Philips scanner and FSL: correct diffusion gradient tables?</a><div class="summary">Dear experts, I am trying to analyse DTI data (60 directions, b=1300) acquired on a Philips Achieva system, and stored as PAR/REC files. I want to analyze the data using TRACULA ( http://surfer.nmr.mgh.harvard.edu/fswiki/Tracula ), an analysis tool which combines FSL (dtifit, bedpost) and FreeSurfer scripts. However, based on visual inspection of the dtifit_FA.nii.gz and dtifit_V1.nii.gz images (p</div></div><span class="ago" title="published:Wed, 28 Jan 2015 15:37:43 GMT -- Received: Wed, 28 Jan 2015 17:33:27 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b26a74441:54017cd:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2874/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b26a74441:54017cd:d38cab4a" data-navigation="inline" data-title="[nipype] split_filename differs from standard library" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b26a74441:54017cd:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7290 --></div><!-- react-empty: 7291 --><div class="content"><a href="http://neurostars.org/p/2874/" target="_blank" rel="noopener" class="title">[nipype] split_filename differs from standard library</a><div class="summary"> Hello, I have noticed that for some cornercases, the first component of split_filename differs from standard dirname (in os.path). &gt;&gt;&gt; from nipype.utils.filemanip import split_filename &gt;&gt;&gt; import os &gt;&gt;&gt; path="/abs/path/" &gt;&gt;&gt; split_filename(path) ('/abs', 'path', '') &gt;&gt;&gt; os.path.dirname(path) '/abs/path' The behaviour of dirname seems more coherent. Looking at the code of split_filename , I think </div></div><span class="ago" title="published:Mon, 26 Jan 2015 12:50:50 GMT -- Received: Mon, 26 Jan 2015 14:31:11 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b0d821895:13787a5e:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2856/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b0d821895:13787a5e:d38cab4a" data-navigation="inline" data-title="[nipype] Checking inputs in _run_interface()" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b0d821895:13787a5e:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7306 --></div><!-- react-empty: 7307 --><div class="content"><a href="http://neurostars.org/p/2856/" target="_blank" rel="noopener" class="title">[nipype] Checking inputs in _run_interface()</a><div class="summary"> Hi, While complexifying my interface (derived from BaseInterface ), I have noticed that some input configuration could be impossible. I don't think that it could be detected with traits options. Therefore, if I'm correct, I should check that in _run_interface() . What is the convention? Is there nipype specific exceptions (something like InputError )? To give you some context, I have declared an </div></div><span class="ago" title="published:Wed, 21 Jan 2015 15:15:38 GMT -- Received: Wed, 21 Jan 2015 17:20:04 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b085afdd3:11d58ead:113fbbc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2851/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b085afdd3:11d58ead:113fbbc6" data-navigation="inline" data-title="News: Dipy 0.8.0 is released" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14b085afdd3:11d58ead:113fbbc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7322 --></div><!-- react-empty: 7323 --><div class="content"><a href="http://neurostars.org/p/2851/" target="_blank" rel="noopener" class="title">News: Dipy 0.8.0 is released</a><div class="summary">We are very happy to announce a new release of Diffusion Imaging in Python ( DIPY ). Here is a summary of the most important new features and developments. DIPY 0.8.0 (Released on Tuesday, 6 Jan 2015) Nonlinear Image-based Registration (SyN) An implementation of the Symmetric Normalization method for nonlinear diffeomorphic registration. This implementation is lightweight, and does not depend on I</div></div><span class="ago" title="published:Tue, 20 Jan 2015 14:36:17 GMT -- Received: Tue, 20 Jan 2015 17:19:15 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14af463b2b8:c382704:113fbbc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2843/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14af463b2b8:c382704:113fbbc6" data-navigation="inline" data-title="Nipype and SGE clusters" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14af463b2b8:c382704:113fbbc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7338 --></div><!-- react-empty: 7339 --><div class="content"><a href="http://neurostars.org/p/2843/" target="_blank" rel="noopener" class="title">Nipype and SGE clusters</a><div class="summary"> Hi all, When submitting large pipelines to the SGE cluster, I'll occasionally get really strange problems that seem to be a result of how Nipype and the cluster are interacting. For instance, sometimes a node will finish running but not produce an output (though I gather I should set the job_finished_timeout value to 65 in order to fix that), sometimes a node will just stall and wait until it tim</div></div><span class="ago" title="published:Fri, 16 Jan 2015 19:06:25 GMT -- Received: Fri, 16 Jan 2015 20:16:22 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14af319eef7:bc1a492:113fbbc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2837/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14af319eef7:bc1a492:113fbbc6" data-navigation="inline" data-title="Selectfiles/ too many subfolders" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14af319eef7:bc1a492:113fbbc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7354 --></div><!-- react-empty: 7355 --><div class="content"><a href="http://neurostars.org/p/2837/" target="_blank" rel="noopener" class="title">Selectfiles/ too many subfolders</a><div class="summary">Hello nipypers, I am trying to run a pipeline with the set up like so (example 1). When I run it, it does work but it in the workflow-directory it creates a folder for each subject. And in each subject folder there are subfolders for each subject in the list. All of them contain the results after computing the process. So in the case of the example below I will have 9 folders full of the data I wa</div></div><span class="ago" title="published:Fri, 16 Jan 2015 12:40:12 GMT -- Received: Fri, 16 Jan 2015 14:16:10 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14aef3c9e69:a95b25e:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2834/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14aef3c9e69:a95b25e:d38cab4a" data-navigation="inline" data-title="[nipype] When is _list_output() called?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14aef3c9e69:a95b25e:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7370 --></div><!-- react-empty: 7371 --><div class="content"><a href="http://neurostars.org/p/2834/" target="_blank" rel="noopener" class="title">[nipype] When is _list_output() called?</a><div class="summary"> Hi, I have started to write my own interfaces. Starting from examples, I have overloaded the _run_interface() method and the _list_output() method. This method is called by nipype machinery (probably in a generic run() method) because it's absence cause an error. Also, as I learn from the examples, it's common to use it in _run_interface() (but I think I can avoid that). I wanted to know in which</div></div><span class="ago" title="published:Thu, 15 Jan 2015 17:21:18 GMT -- Received: Thu, 15 Jan 2015 20:15:35 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ae4ee99fd:7332d9b:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2831/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ae4ee99fd:7332d9b:d38cab4a" data-navigation="inline" data-title="[nipype] Specifying onsets in a sparse model" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ae4ee99fd:7332d9b:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7386 --></div><!-- react-empty: 7387 --><div class="content"><a href="http://neurostars.org/p/2831/" target="_blank" rel="noopener" class="title">[nipype] Specifying onsets in a sparse model</a><div class="summary"> Hi all, Can someone help clarify the sparse model specification for me? 1. Is time = 0 the onset of the first volume acquisition or the onset of the first event? 2. How does the scan_onset input affect this? And if my scan comes AFTER the first event, should this number be positive or negative? So, for instance, in my paradigm with TR = 10s: the first event occurs, then 5s later the first volume </div></div><span class="ago" title="published:Tue, 13 Jan 2015 17:16:02 GMT -- Received: Tue, 13 Jan 2015 20:14:09 GMT">22mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ade7dccf1:4fa951f:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2821/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ade7dccf1:4fa951f:d38cab4a" data-navigation="inline" data-title="[nipype] Collecting outputs from a node" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ade7dccf1:4fa951f:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7402 --></div><!-- react-empty: 7403 --><div class="content"><a href="http://neurostars.org/p/2821/" target="_blank" rel="noopener" class="title">[nipype] Collecting outputs from a node</a><div class="summary"> Hi, I am learning nipype and I would need some help on a topic. I have read the nipype tutorials so I have a basic understanding of pipelines and data access. My project is to read the cortical thickness from feesurfer 'aseg.stats' files corresponding to a list of subjects ID and collect them in a matrix (with the same order than in the ID list). I have written a simple python function to read 'a</div></div><span class="ago" title="published:Mon, 12 Jan 2015 12:51:14 GMT -- Received: Mon, 12 Jan 2015 14:13:14 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ace6371a5:9990bc:113fbbc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2817/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ace6371a5:9990bc:113fbbc6" data-navigation="inline" data-title="nipype pipelines for HCP data" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ace6371a5:9990bc:113fbbc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7418 --></div><!-- react-empty: 7419 --><div class="content"><a href="http://neurostars.org/p/2817/" target="_blank" rel="noopener" class="title">nipype pipelines for HCP data</a><div class="summary"> Hi. I'm starting to take a look at some HCP (human connectome project) MEG data and would like to proceed using nipype rather than the matlab and bash scripts that are provided from the connectomeDB website. I'm aware of a few pipelines here and there for fMRI and DWI that can be easily adapted to HCP data, but not for MEG. I would be aiming for at least partial overlap with the analyses of fMRI </div></div><span class="ago" title="published:Fri, 09 Jan 2015 09:19:56 GMT -- Received: Fri, 09 Jan 2015 11:10:31 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14abc5a8c29:162fc92e:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2800/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14abc5a8c29:162fc92e:d38cab4a" data-navigation="inline" data-title="What is a good repository to publicly host a collection of rat DTI scans for free?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14abc5a8c29:162fc92e:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7434 --></div><!-- react-empty: 7435 --><div class="content"><a href="http://neurostars.org/p/2800/" target="_blank" rel="noopener" class="title">What is a good repository to publicly host a collection of rat DTI scans for free?</a><div class="summary"> I'm looking for a database or data repository where I can store a collection of DTI reconstructions for a rat study. Does anyone have recommendations? This is for a paper being published in Plos One. Thanks! </div></div><span class="ago" title="published:Mon, 05 Jan 2015 21:54:08 GMT -- Received: Mon, 05 Jan 2015 23:07:38 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14abb10bbfd:15bd0242:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2797/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14abb10bbfd:15bd0242:d38cab4a" data-navigation="inline" data-title="NiPype - Multiple subjects &amp; functional runs?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14abb10bbfd:15bd0242:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7450 --></div><!-- react-empty: 7451 --><div class="content"><a href="http://neurostars.org/p/2797/" target="_blank" rel="noopener" class="title">NiPype - Multiple subjects &amp; functional runs?</a><div class="summary"> For the majority of cognitive science fMRI projects, there are multiple functional runs for each subject. In some cases, for example with motion correction, all volumes across all runs are registered to the first volume of the first run etc. Can someone point me to a simple resource on how to setup a workflow so that you can iterate over subjects &amp; over runs? </div></div><span class="ago" title="published:Mon, 05 Jan 2015 15:47:53 GMT -- Received: Mon, 05 Jan 2015 17:07:23 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ab87d40f6:151621d0:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2796/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ab87d40f6:151621d0:8340a896" data-navigation="inline" data-title="Tutorial: Output was blank on tracks, DMRI_Camino_DTI.py" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14ab87d40f6:151621d0:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7466 --></div><!-- react-empty: 7467 --><div class="content"><a href="http://neurostars.org/p/2796/" target="_blank" rel="noopener" class="title">Tutorial: Output was blank on tracks, DMRI_Camino_DTI.py</a><div class="summary"> Hello, So I have managed to run dmri_camino_dti.py smoothly on my mac, and returned no errors. The output gave files including data_DT_tracked.trk, which is 198.6 MB in size, but appears blank when I open in TrackVis. All trk outputs were blank here. Let me know, it is appreciated Thanks </div></div><span class="ago" title="published:Mon, 05 Jan 2015 02:38:42 GMT -- Received: Mon, 05 Jan 2015 05:07:04 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14aa002fdcd:f137be0:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2793/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14aa002fdcd:f137be0:d38cab4a" data-navigation="inline" data-title="Which covariates to use in twin analyses of the HCP data?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14aa002fdcd:f137be0:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7482 --></div><!-- react-empty: 7483 --><div class="content"><a href="http://neurostars.org/p/2793/" target="_blank" rel="noopener" class="title">Which covariates to use in twin analyses of the HCP data?</a><div class="summary"> Hello, I'm starting to look at the heritability of different measurements using the Human Connectome Project (HCP). At the moment there are ~200 subjects: ~50 monozygotic (MZ) twin pairs, and ~50 dyzygotic (DZ) twin pairs. I'm using OpenMx, with the twin analysis script described in their manual. Given the number of subjects, I should be well powered to reproduce the results in the literature. Ho</div></div><span class="ago" title="published:Wed, 31 Dec 2014 10:51:37 GMT -- Received: Wed, 31 Dec 2014 11:02:38 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a9c25b48c:e09753d:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2792/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a9c25b48c:e09753d:d38cab4a" data-navigation="inline" data-title="[nitime] Filtering fMRI time series: ValueError: filtfilt only accepts 1-d arrays." data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a9c25b48c:e09753d:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7498 --></div><!-- react-empty: 7499 --><div class="content"><a href="http://neurostars.org/p/2792/" target="_blank" rel="noopener" class="title">[nitime] Filtering fMRI time series: ValueError: filtfilt only accepts 1-d arrays.</a><div class="summary"> Hi, I am trying to normalize and temporally filter an fMRI time series data using the following command: T= io.time_series_from_file(in_file, TR=tr, normalize='zscore', filter={'lb':highpass, 'ub':lowpass, 'method':'fir'} and keep getting this error: ValueError: filtfilt only accepts 1-d arrays. Normalizing the data without the filtering part works fine. I am using python 2.7, scipy 0.14.0, numpy</div></div><span class="ago" title="published:Tue, 30 Dec 2014 16:26:33 GMT -- Received: Tue, 30 Dec 2014 17:02:04 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a757f900a:40db271:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2789/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a757f900a:40db271:8340a896" data-navigation="inline" data-title="Tutorial: pickled problem - how do you unpickle output files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a757f900a:40db271:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7555 --></div><!-- react-empty: 7556 --><div class="content"><a href="http://neurostars.org/p/2789/" target="_blank" rel="noopener" class="title">Tutorial: pickled problem - how do you unpickle output files</a><div class="summary"> How do you unpickle the output files (.pklz)? Will the result be in NIFTI format? Any help will be appreciated. Thanks a lot. </div></div><span class="ago" title="published:Tue, 23 Dec 2014 03:40:33 GMT -- Received: Tue, 23 Dec 2014 04:55:01 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a6376bd99:a249858:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2787/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a6376bd99:a249858:8340a896" data-navigation="inline" data-title="Nipype custom function node: Interface Function failed to run error" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a6376bd99:a249858:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7571 --></div><!-- react-empty: 7572 --><div class="content"><a href="http://neurostars.org/p/2787/" target="_blank" rel="noopener" class="title">Nipype custom function node: Interface Function failed to run error</a><div class="summary">I'm putting together an MVPA pipeline using Nipype and so am using a bunch of custom function nodes. (Unrelated: are there any plans to add a PyMVPA interface to Nipype?) The first step of this pipeline finds the behavioral data from the scanning sessions and records the timing of the events to use as labels for the scans during classification. This is a relatively simple function that just receiv</div></div><span class="ago" title="published:Fri, 19 Dec 2014 16:16:07 GMT -- Received: Fri, 19 Dec 2014 16:52:13 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a62d1ae55:9e7e0ec:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2786/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a62d1ae55:9e7e0ec:8340a896" data-navigation="inline" data-title="bips preprocessing_negative values in the brain" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a62d1ae55:9e7e0ec:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7587 --></div><!-- react-empty: 7588 --><div class="content"><a href="http://neurostars.org/p/2786/" target="_blank" rel="noopener" class="title">bips preprocessing_negative values in the brain</a><div class="summary"> I couldn´t find in the bips -77 workflow when some operations apparently demean (or z-score) the time series. As a result some values (probably those that were below the mean before) became negative. Please see the .png in the link, it is the final output (with the cursor on one negative voxel) and the relative timeseries (it looks wired to me). Why is it necessary? How can I get an output with o</div></div><span class="ago" title="published:Fri, 19 Dec 2014 12:27:28 GMT -- Received: Fri, 19 Dec 2014 13:51:56 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a5daa5415:887a77f:b669a9fb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2779/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a5daa5415:887a77f:b669a9fb" data-navigation="inline" data-title="Connection between nodes with different output/input datatypes" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a5daa5415:887a77f:b669a9fb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7603 --></div><!-- react-empty: 7604 --><div class="content"><a href="http://neurostars.org/p/2779/" target="_blank" rel="noopener" class="title">Connection between nodes with different output/input datatypes</a><div class="summary"> Hey community, is there a possibility to connect two nodes, which haven't a matching output to input datatype. For example: node1 has an output which is type of Directory trait node2 needs as input a commaseperated string of absolute paths to files The execution of node2 depends on the result of node1. For now I didn't find a solution and would be delighted to hearing from you. Best regards Jan </div></div><span class="ago" title="published:Thu, 18 Dec 2014 13:31:40 GMT -- Received: Thu, 18 Dec 2014 13:50:51 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a58825822:6d905fd:b669a9fb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2771/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a58825822:6d905fd:b669a9fb" data-navigation="inline" data-title="[nipype] Questions concerning example workflow for rsfMRI preprocessing" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a58825822:6d905fd:b669a9fb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7619 --></div><!-- react-empty: 7620 --><div class="content"><a href="http://neurostars.org/p/2771/" target="_blank" rel="noopener" class="title">[nipype] Questions concerning example workflow for rsfMRI preprocessing</a><div class="summary">Hi, going through this nice example workflow for rsfMRI preprocessing (https://github.com/nipy/nipype/blob/master/examples/rsfmri_vol_surface_preprocessing_nipy.py#L261) I am wondering about some of the steps: 1. The coregistration is computed for an average median volume across runs. Isn't this problematic as the different runs might not be perfectly aligned to each other? 2. The time series are </div></div><span class="ago" title="published:Wed, 17 Dec 2014 10:49:57 GMT -- Received: Wed, 17 Dec 2014 13:49:04 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a57389419:66b1f2c:b669a9fb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2768/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a57389419:66b1f2c:b669a9fb" data-navigation="inline" data-title="Bug in case to execute &quot;Normalize&quot; with modulation." data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a57389419:66b1f2c:b669a9fb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7635 --></div><!-- react-empty: 7636 --><div class="content"><a href="http://neurostars.org/p/2768/" target="_blank" rel="noopener" class="title">Bug in case to execute "Normalize" with modulation.</a><div class="summary">Hello, everyone. I executed SPM Normalize with modulation via pipeline I implemented. import nipype.interfaces.spm as spm import nipype.pipeline.engine as pe ... norm = pe.Node(interface=spm.Normalize(), name="Norm") norm.inputs.source = 'inputs.nii' norm.inputs.write_preserve = True ... However, it returned FileNotFoundError as follows: Traceback: Traceback (most recent call last): File "/usr/lib</div></div><span class="ago" title="published:Wed, 17 Dec 2014 06:16:01 GMT -- Received: Wed, 17 Dec 2014 07:48:53 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a4e33f889:36966dc:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2765/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a4e33f889:36966dc:8340a896" data-navigation="inline" data-title="Interface WarpImageMultiTransform failed to run." data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a4e33f889:36966dc:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7651 --></div><!-- react-empty: 7652 --><div class="content"><a href="http://neurostars.org/p/2765/" target="_blank" rel="noopener" class="title">Interface WarpImageMultiTransform failed to run.</a><div class="summary">Hello, is there anyone, that could help to make sense of this error message (from the crash file)? I am not sure what went wrong here. That would be super nice . Thank you. Leonie nipype_display_crash /home/raid1/lampe/crash-20141215-125915-lampe-21_t1_warped_2_fa.a0.b0.pklz False False File: /home/raid1/lampe/crash-20141215-125915-lampe-21_t1_warped_2_fa.a0.b0.pklz Node: my_workflow.21_t1_warped_</div></div><span class="ago" title="published:Mon, 15 Dec 2014 12:32:25 GMT -- Received: Mon, 15 Dec 2014 13:47:16 GMT">23mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a2c0ee382:7e58f:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2762/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a2c0ee382:7e58f:d38cab4a" data-navigation="inline" data-title="XNATSource and PyXNAT both returning NIFTI files in strange format" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a2c0ee382:7e58f:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7667 --></div><!-- react-empty: 7668 --><div class="content"><a href="http://neurostars.org/p/2762/" target="_blank" rel="noopener" class="title">XNATSource and PyXNAT both returning NIFTI files in strange format</a><div class="summary"> Hello, Can anyone tell me why my NII files from XNAT are downloaded in some format (with a header file) which is not NIFTI? I have tried it both with XNATSource and PyXNAT. With PyXNAT, I can see the names of the files, but the downloaded files are not NII format. This creates a problem because I am unable to send these files into a workflow/node such as BET. The error message I receive is as fol</div></div><span class="ago" title="published:Mon, 08 Dec 2014 21:28:18 GMT -- Received: Mon, 08 Dec 2014 22:39:40 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a22656c8c:275aeb7:c4302584_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2759/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a22656c8c:275aeb7:c4302584" data-navigation="inline" data-title="Tool: XNATSource: Template ... returns no files. XNATSource failed to run" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a22656c8c:275aeb7:c4302584" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7683 --></div><!-- react-empty: 7684 --><div class="content"><a href="http://neurostars.org/p/2759/" target="_blank" rel="noopener" class="title">Tool: XNATSource: Template ... returns no files. XNATSource failed to run</a><div class="summary"> Hello, I am trying to pull DICOM files stored on XNAT to be converted to Nifti via Dicom2Nifti tool. The first part of my code is as follows: dg = pe.Node(XNATSource(), name='xnatsource') dg.inputs.query_template = ('/data/projects/myProjectName/subjects/CENTRAL_S04099/experiments/CENTRAL_E07218/scans/1/resources/DICOM/files') dg.inputs.user = 'userName' dg.inputs.pwd = 'myPassword' dg.inputs.ser</div></div><span class="ago" title="published:Sun, 07 Dec 2014 01:07:14 GMT -- Received: Sun, 07 Dec 2014 01:37:59 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a07fafbfe:14ceac21:113fbbc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2755/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a07fafbfe:14ceac21:113fbbc6" data-navigation="inline" data-title="error while loading shared libraries: libnewimage.so: cannot open shared object file" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14a07fafbfe:14ceac21:113fbbc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7699 --></div><!-- react-empty: 7700 --><div class="content"><a href="http://neurostars.org/p/2755/" target="_blank" rel="noopener" class="title">error while loading shared libraries: libnewimage.so: cannot open shared object file</a><div class="summary">Hi I am new to pycharm and fsl and am trying to get the two to play nicely together.The problem is that they are not. First off, I am running the Neurodebian x64 virtualbox (6.999 +). To start with I was attempting to use Nipype but have moved back to running a simple subprocess command in order to track where the problem I'm getting occurs. If, from PyCharm, I run a simple subprocess.call('fslsta</div></div><span class="ago" title="published:Mon, 01 Dec 2014 21:51:08 GMT -- Received: Mon, 01 Dec 2014 22:31:36 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149f73aa166:100184cc:113fbbc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2754/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149f73aa166:100184cc:113fbbc6" data-navigation="inline" data-title="steps to check rfMRI data preprocessing in BIPS" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149f73aa166:100184cc:113fbbc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7715 --></div><!-- react-empty: 7716 --><div class="content"><a href="http://neurostars.org/p/2754/" target="_blank" rel="noopener" class="title">steps to check rfMRI data preprocessing in BIPS</a><div class="summary"> Hi, can anybody reccommend some sanity check steps to ensure that the preprocessing workflow of restingfMRI data in BIPs run correctly? Thanks </div></div><span class="ago" title="published:Fri, 28 Nov 2014 14:13:10 GMT -- Received: Fri, 28 Nov 2014 16:27:35 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149ecec566e:c9ba16f:113fbbc6_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2742/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149ecec566e:c9ba16f:113fbbc6" data-navigation="inline" data-title="Connecting Dcm2nii interface to Datagrabber (or similar)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149ecec566e:c9ba16f:113fbbc6" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7731 --></div><!-- react-empty: 7732 --><div class="content"><a href="http://neurostars.org/p/2742/" target="_blank" rel="noopener" class="title">Connecting Dcm2nii interface to Datagrabber (or similar)</a><div class="summary"> Hi, I'm trying to set up my first workflow with NiPype and I'm running into some problems with connecting dcm2nii to the rest of the pipeline. I'm using dcm2nii to convert my dicom files to Nifti files. Then I want to use datagrabber to select some of those nifti files to follow up on the processing. I can't figure out how to do this. Please also let me know if there is a better approach to this </div></div><span class="ago" title="published:Wed, 26 Nov 2014 15:58:14 GMT -- Received: Wed, 26 Nov 2014 16:25:51 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149c4565128:7cef5df:8b965bbf_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2729/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149c4565128:7cef5df:8b965bbf" data-navigation="inline" data-title="bips_restingfMRI_multiband sequences_slice time or slice number?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149c4565128:7cef5df:8b965bbf" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7747 --></div><!-- react-empty: 7748 --><div class="content"><a href="http://neurostars.org/p/2729/" target="_blank" rel="noopener" class="title">bips_restingfMRI_multiband sequences_slice time or slice number?</a><div class="summary"> Why in multi band sequences is it required as input the slice time acquisition rather than the slice number? Does it matter if the slice time acquisition slightly differs across participants? Should it be input in ms or sec? Which smoothing is recommended for a voxel size of 2.3x2.3x2.3 mm? Thank you! </div></div><span class="ago" title="published:Tue, 18 Nov 2014 18:17:28 GMT -- Received: Tue, 18 Nov 2014 19:17:11 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149c4565128:7cef5de:8b965bbf_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2728/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149c4565128:7cef5de:8b965bbf" data-navigation="inline" data-title="resting state data with multi band sequences in SPM" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149c4565128:7cef5de:8b965bbf" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7763 --></div><!-- react-empty: 7764 --><div class="content"><a href="http://neurostars.org/p/2728/" target="_blank" rel="noopener" class="title">resting state data with multi band sequences in SPM</a><div class="summary"> Is it true that SPM cannot handle multiband sequences (so that the alternative DPARSF toolkit won´t work anyway)? if so, why? and which alternative pipeline can be used? thanks! </div></div><span class="ago" title="published:Tue, 18 Nov 2014 18:09:05 GMT -- Received: Tue, 18 Nov 2014 19:17:11 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149bf2f3a54:5ba911f:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2726/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149bf2f3a54:5ba911f:8340a896" data-navigation="inline" data-title="Is there any FOSS for tracking animal (mice, tadpoles, moscitos) behavior?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149bf2f3a54:5ba911f:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7779 --></div><!-- react-empty: 7780 --><div class="content"><a href="http://neurostars.org/p/2726/" target="_blank" rel="noopener" class="title">Is there any FOSS for tracking animal (mice, tadpoles, moscitos) behavior?</a><div class="summary"> One of the major demands apparently is a lack of a FOSS for tracking animal behavior. The available commercial offerings (such as from Noldus) are proprietary, limiting, and expensive. </div></div><span class="ago" title="published:Mon, 17 Nov 2014 17:27:00 GMT -- Received: Mon, 17 Nov 2014 19:16:23 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149bde57902:574741f:8b965bbf_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2722/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149bde57902:574741f:8b965bbf" data-navigation="inline" data-title="fMRI_QA Pipeline (BIPS: 5dd866fe8af611e1b9d5001e4fb1404c Task/Resting fMRI Quality Assurance workflow)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149bde57902:574741f:8b965bbf" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7795 --></div><!-- react-empty: 7796 --><div class="content"><a href="http://neurostars.org/p/2722/" target="_blank" rel="noopener" class="title">fMRI_QA Pipeline (BIPS: 5dd866fe8af611e1b9d5001e4fb1404c Task/Resting fMRI Quality Assurance workflow)</a><div class="summary">Hello, I want to use this pipeline in BIPS to create a pdf for the preprocessing with the fMRI-preprocessing script. It executed without errors and all the files are in the right directories. I get the following error message: File: crash-20141117-113303-fbeyer-segstats.a0.pklz Node: QA.tsnr_stddev_roi.getstats.segstats.a0 Working directory: /SCR2/data_fbeyer/Results/BIPS/WDR/QA/tsnr_stddev_roi/ge</div></div><span class="ago" title="published:Mon, 17 Nov 2014 10:55:36 GMT -- Received: Mon, 17 Nov 2014 13:16:12 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149b8197396:391ea10:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2721/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149b8197396:391ea10:8340a896" data-navigation="inline" data-title="ProbtrackX question: How to obtain tracts to visualize" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149b8197396:391ea10:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7811 --></div><!-- react-empty: 7812 --><div class="content"><a href="http://neurostars.org/p/2721/" target="_blank" rel="noopener" class="title">ProbtrackX question: How to obtain tracts to visualize</a><div class="summary">I am looking to visualize tracts from ProbtrackX using Chris Gorgolewski's BedpostX particle reader. (I am looking to generate a connectivity matrix between atlas regions, and this is is invaluable to me in visualizing the tracks. ) For this, I need to output particle files from ProbtrackX using the verbose==2 option, and then feed this to the BedpostX particle reader to generate .trk files. Howev</div></div><span class="ago" title="published:Sun, 16 Nov 2014 07:19:36 GMT -- Received: Sun, 16 Nov 2014 10:15:16 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149afb9b194:154d11f:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2717/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149afb9b194:154d11f:8340a896" data-navigation="inline" data-title="NiPype Pipeline Features/Advantages towards XNAT" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149afb9b194:154d11f:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7827 --></div><!-- react-empty: 7828 --><div class="content"><a href="http://neurostars.org/p/2717/" target="_blank" rel="noopener" class="title">NiPype Pipeline Features/Advantages towards XNAT</a><div class="summary"> Hey everyone, I currently evaluate NiPype against XNAT Server with its pipeline engine. I tried to figure out the advantages / features NiPype offers, which aren't available / or can't be implemented with XNAT. Probably I'm not long enough familiar with the matter to see them. It would be very nice if someone of you could name me some arguments why I should use NiPype. Best regards, Jan </div></div><span class="ago" title="published:Fri, 14 Nov 2014 18:57:10 GMT -- Received: Fri, 14 Nov 2014 19:13:42 GMT">24mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149aa548a0c:4c53798:b669a9fb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2712/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149aa548a0c:4c53798:b669a9fb" data-navigation="inline" data-title="What are the most &quot;open&quot; data use agreements or data licenses?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149aa548a0c:4c53798:b669a9fb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7843 --></div><!-- react-empty: 7844 --><div class="content"><a href="http://neurostars.org/p/2712/" target="_blank" rel="noopener" class="title">What are the most "open" data use agreements or data licenses?</a><div class="summary"> I'd like to get a collection of Data Use Agreements and Data License examples for future reference. Does anyone have examples that they like or support? What are the costs/benefits of choosing one over the other? For example, here are a few that I've found: Datasets PING: http://pingstudy.ucsd.edu/Data.php HCP: http://humanconnectome.org/data/data-use-terms/index.html ADNI: http://adni.loni.usc.e</div></div><span class="ago" title="published:Thu, 13 Nov 2014 15:19:24 GMT -- Received: Thu, 13 Nov 2014 18:05:07 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149a865e8f4:412da82:b669a9fb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2709/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149a865e8f4:412da82:b669a9fb" data-navigation="inline" data-title="Aquisition time of each DICOM image" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149a865e8f4:412da82:b669a9fb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7859 --></div><!-- react-empty: 7860 --><div class="content"><a href="http://neurostars.org/p/2709/" target="_blank" rel="noopener" class="title">Aquisition time of each DICOM image</a><div class="summary"> Dear Experts i have acquired a fmri data without synchronization cable ! my TR was 2 seconds so the proper way for the images was that each image took 2 seconds to be acquired but the problem is that i have noticed my first image took up to 7 seconds and the next image 3 seconds and after that the images was taken each 2 seconds. i want to know about the acquisition time of my first 2 or 3 images</div></div><span class="ago" title="published:Thu, 13 Nov 2014 06:29:56 GMT -- Received: Thu, 13 Nov 2014 09:04:51 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1499d72a898:4dc3a5:b669a9fb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2707/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1499d72a898:4dc3a5:b669a9fb" data-navigation="inline" data-title="How to control scl_slope (scl_inter) in generating new NIfTI file." data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1499d72a898:4dc3a5:b669a9fb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7875 --></div><!-- react-empty: 7876 --><div class="content"><a href="http://neurostars.org/p/2707/" target="_blank" rel="noopener" class="title">How to control scl_slope (scl_inter) in generating new NIfTI file.</a><div class="summary"> Hi, everyone. I'd like to set 'scl_slope' and 'scl_inter' in NIfTI header when I generate new NIfTI image file. I tried codes as follows: import nibabel as nib import numpy as np hdr = Nifti1Header() hdr.set_data_dtype(np.int16) hdr.set_slope_inter(1, inter=0) # Set scl_slope to 1 img = nib.Nifti1Image( &gt;, , header=hdr) img.to_filename( ) However, setting for slope didn't work in these code. And,</div></div><span class="ago" title="published:Tue, 11 Nov 2014 03:07:36 GMT -- Received: Tue, 11 Nov 2014 06:02:57 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149965ccbba:8309d32:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2698/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149965ccbba:8309d32:8340a896" data-navigation="inline" data-title="How do I 'follow' a tag on neurostars?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149965ccbba:8309d32:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7891 --></div><!-- react-empty: 7892 --><div class="content"><a href="http://neurostars.org/p/2698/" target="_blank" rel="noopener" class="title">How do I 'follow' a tag on neurostars?</a><div class="summary"> I would like to get email alerts for tags of interest. Am I missing something obvious? </div></div><span class="ago" title="published:Sun, 09 Nov 2014 19:22:46 GMT -- Received: Sun, 09 Nov 2014 21:01:44 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14981bfe277:1b55d2e:5c7036b5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2689/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14981bfe277:1b55d2e:5c7036b5" data-navigation="inline" data-title="Using Interface to Run Nested Workflows" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14981bfe277:1b55d2e:5c7036b5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7907 --></div><!-- react-empty: 7908 --><div class="content"><a href="http://neurostars.org/p/2689/" target="_blank" rel="noopener" class="title">Using Interface to Run Nested Workflows</a><div class="summary"> I have created a number of workflows for our lab to use for automated processing. Some are run on the subject level and some are run on a scan by scan level. I am looking to create a overarching pipeline that runs on the subject level to connect these individual workflows and submit tasks to the cluster when previous workflows have completed. To do this I am creating a pipeline workflow and then </div></div><span class="ago" title="published:Wed, 05 Nov 2014 20:50:17 GMT -- Received: Wed, 05 Nov 2014 20:57:56 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497de2c3e2:4a13f8:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2687/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497de2c3e2:4a13f8:8340a896" data-navigation="inline" data-title="OpenfMRI data layout modification" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497de2c3e2:4a13f8:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7923 --></div><!-- react-empty: 7924 --><div class="content"><a href="http://neurostars.org/p/2687/" target="_blank" rel="noopener" class="title">OpenfMRI data layout modification</a><div class="summary"> 1. under anatomy: - change highres to be more specific t1_001, t1w_001, t2_001, swi_001, swi_001_phase, etc.,. 2. add a dicom information file associated with each converted nifti file, when available. we currently generate a json formatted file that includes a lot of the dicom parameters. we would add a corresponding file for each converted file. 3. add a dataset level provenance folder while we</div></div><span class="ago" title="published:Wed, 05 Nov 2014 00:48:27 GMT -- Received: Wed, 05 Nov 2014 02:57:33 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497bf44b71:771eb87:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2685/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497bf44b71:771eb87:6b730af7" data-navigation="inline" data-title="Why does lowering the impedance of a recording site improve spike yields?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497bf44b71:771eb87:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7939 --></div><!-- react-empty: 7940 --><div class="content"><a href="http://neurostars.org/p/2685/" target="_blank" rel="noopener" class="title">Why does lowering the impedance of a recording site improve spike yields?</a><div class="summary"> I use silicon microprobes and I've been hearing rumors that I could get better spike yields if I treat my recording sites to lower their impedance. Neuronexus offers Iridium Oxidation for ~ $180 per probe. They also have a protocol for using their NiPOD to do the same. I've also heard that electrodeposition of PEDOT on recording sites will drop the impedance from ~1Mohm to ~100kOhm and drasticall</div></div><span class="ago" title="published:Tue, 04 Nov 2014 17:54:31 GMT -- Received: Tue, 04 Nov 2014 17:57:28 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497aaa85c4:6ff6cf8:b669a9fb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2684/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497aaa85c4:6ff6cf8:b669a9fb" data-navigation="inline" data-title="[nitime] Temporal filtering of fMRI data - choosing the right filter" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1497aaa85c4:6ff6cf8:b669a9fb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7955 --></div><!-- react-empty: 7956 --><div class="content"><a href="http://neurostars.org/p/2684/" target="_blank" rel="noopener" class="title">[nitime] Temporal filtering of fMRI data - choosing the right filter</a><div class="summary"> Hi, I would like to use nitime for bandpass filtering of resting state fMRI data. The example on the nitime homepage explains well how to implement this (http://nipy.org/nitime/examples/filtering_fmri.html) But I am wondering which filter method to use (boxcar, IIR, FIR, fourier). Can anyone provide guidance how to make that decision? (So far I am using fslmaths -bptf performing nonlinear highpas</div></div><span class="ago" title="published:Tue, 04 Nov 2014 09:40:06 GMT -- Received: Tue, 04 Nov 2014 11:57:16 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14958778359:7c1f671:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2680/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14958778359:7c1f671:d38cab4a" data-navigation="inline" data-title="How many surface labels should Mindboggle use?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14958778359:7c1f671:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7971 --></div><!-- react-empty: 7972 --><div class="content"><a href="http://neurostars.org/p/2680/" target="_blank" rel="noopener" class="title">How many surface labels should Mindboggle use?</a><div class="summary">I am preparing to release a new version of the Mindboggle software ( http://mindboggle.info ) and need to make a decision about output labels and am requesting feedback from the community. As is described in Mindboggle's labeling FAQ , the volume outputs (labels, volumes, thicknesses) use the 31 cortical labels per hemisphere in the DKT31 protocol + MGH (wmparc.mgz) noncortical labels. Currently, </div></div><span class="ago" title="published:Tue, 28 Oct 2014 19:57:20 GMT -- Received: Tue, 28 Oct 2014 20:34:27 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14958778359:7c1f670:d38cab4a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2678/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14958778359:7c1f670:d38cab4a" data-navigation="inline" data-title="Tutorial: How to use Neurostars" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14958778359:7c1f670:d38cab4a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 7987 --></div><!-- react-empty: 7988 --><div class="content"><a href="http://neurostars.org/p/2678/" target="_blank" rel="noopener" class="title">Tutorial: How to use Neurostars</a><div class="summary"> Neurostars can be used to post questions: Step 1: Login Step 2: Click on new post </div></div><span class="ago" title="published:Tue, 28 Oct 2014 18:21:37 GMT -- Received: Tue, 28 Oct 2014 20:34:27 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1495499e451:65a27eb:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2677/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1495499e451:65a27eb:6b730af7" data-navigation="inline" data-title="Python nifti viewer" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1495499e451:65a27eb:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8003 --></div><!-- react-empty: 8004 --><div class="content"><a href="http://neurostars.org/p/2677/" target="_blank" rel="noopener" class="title">Python nifti viewer</a><div class="summary"> Anyone using a python nifti viewer they like? Looking to not reinvent the wheel if we don't have to. </div></div><span class="ago" title="published:Mon, 27 Oct 2014 23:40:36 GMT -- Received: Tue, 28 Oct 2014 02:33:31 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76c:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/165/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76c:6b730af7" data-navigation="inline" data-title="custom location for temporary files" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76c:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8019 --></div><!-- react-empty: 8020 --><div class="content"><a href="http://neurostars.org/p/165/" target="_blank" rel="noopener" class="title">custom location for temporary files</a><div class="summary"> I'm trying to run nipype over the IPython cluster. We have a shared file system, however tmpfs is not shared. I believe this causes the crashes. Is there a way to specify custom tmp folder for a nipype workflow? Cheers -- You received this message because you are subscribed to the Google Groups "NiPy Users" group. To unsubscribe from this group and stop receiving emails from it, send an email to </div></div><span class="ago" title="published:Fri, 24 Oct 2014 07:51:39 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76b:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/1367/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76b:6b730af7" data-navigation="inline" data-title="Nipype 0.10.0 release" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76b:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8035 --></div><!-- react-empty: 8036 --><div class="content"><a href="http://neurostars.org/p/1367/" target="_blank" rel="noopener" class="title">Nipype 0.10.0 release</a><div class="summary">Dear all, I'm delighted to announce a brand new release of Nipype - a pipelining and interfacing framework bringing together a variety of neuroimaging software and making your workflows easier to design and more efficient to compute. Nipype is also an important component of many other neuroimaging tools (see: http://nipy.org/nipype/devel/software_using_nipype.html). You can download Nipype from ht</div></div><span class="ago" title="published:Thu, 16 Oct 2014 19:48:44 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76a:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2415/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76a:6b730af7" data-navigation="inline" data-title="Software using or supporting Nipype" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e76a:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8051 --></div><!-- react-empty: 8052 --><div class="content"><a href="http://neurostars.org/p/2415/" target="_blank" rel="noopener" class="title">Software using or supporting Nipype</a><div class="summary"> Hi, I'm trying to compile a page listing all software that uses or supports Nipype. Here is the initial attempt: https://github.com/nipy/nipype/pull/960/files Please let me know if I have missed something or if I should alter some of the descriptions. Best, Chris </div></div><span class="ago" title="published:Wed, 15 Oct 2014 18:51:51 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e769:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2400/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e769:6b730af7" data-navigation="inline" data-title="nipype - sometimes hangs on MatlabCommands" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e769:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8067 --></div><!-- react-empty: 8068 --><div class="content"><a href="http://neurostars.org/p/2400/" target="_blank" rel="noopener" class="title">nipype - sometimes hangs on MatlabCommands</a><div class="summary">Hi all, I use a few SPM scripts wrapped as in example 1 here: http://nipy.sourceforge.net/nipype/devel/matlab_interface_devel.html When running them as part of a workflow (either serially or in parallel), they sometimes "hang". CPU drops to baseline, no further jobs are executed or finished. When I ctrl-c the script, it exists at sleep(2) on line 260, base.py. Further investigation revealed that t</div></div><span class="ago" title="published:Mon, 13 Oct 2014 23:36:33 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e768:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2394/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e768:6b730af7" data-navigation="inline" data-title="Method development for coupled EEG-fMRI (1x Full EPSRC studentship)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e768:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8083 --></div><!-- react-empty: 8084 --><div class="content"><a href="http://neurostars.org/p/2394/" target="_blank" rel="noopener" class="title">Method development for coupled EEG-fMRI (1x Full EPSRC studentship)</a><div class="summary">Method development for coupled EEG-fMRI (1x Full EPSRC studentship) The goal of the project is to develop novel methods for the joint analysis of signals from electroencephalography (EEG) and functional magnetic resonance imaging (fMRI), when they are obtained concurrently. Technological advances of the last 10 years make it possible for scientists to record both modalities concurrently, in what i</div></div><span class="ago" title="published:Fri, 10 Oct 2014 18:29:39 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e767:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/1890/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e767:6b730af7" data-navigation="inline" data-title="Machine learning capabilities and the Function Interface" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e767:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8099 --></div><!-- react-empty: 8100 --><div class="content"><a href="http://neurostars.org/p/1890/" target="_blank" rel="noopener" class="title">Machine learning capabilities and the Function Interface</a><div class="summary"> Hi all, I wanted to use the distributed capabilities of nipype for multivariate pattern analysis (i.e. machine learning) of some fMRI data. However, after playing around with the Function Interface, I got the feeling that due to it's heavy dependency on files and impossibility to pass arbitrary python objects around, nipype is probably not the most suitable engine for pure machine learning tasks.</div></div><span class="ago" title="published:Tue, 07 Oct 2014 06:28:15 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e766:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2389/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e766:6b730af7" data-navigation="inline" data-title="FSL FLAMEO tstats" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e766:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8115 --></div><!-- react-empty: 8116 --><div class="content"><a href="http://neurostars.org/p/2389/" target="_blank" rel="noopener" class="title">FSL FLAMEO tstats</a><div class="summary"> One of the outputs of FLAMEO is 'tstats' what do the values in this image mean? The appear to be the inverse of p-values, it that correct? This might be the wrong place to be asking this, but all of the FLAMEO documentation but out by the FSL team is directed at FEAT users. Thank you, Dalton Combs -- You received this message because you are subscribed to the Google Groups "NiPy Users" group. To </div></div><span class="ago" title="published:Mon, 06 Oct 2014 21:47:11 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e765:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2388/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e765:6b730af7" data-navigation="inline" data-title="FSL FLAMEO tstats" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e765:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8131 --></div><!-- react-empty: 8132 --><div class="content"><a href="http://neurostars.org/p/2388/" target="_blank" rel="noopener" class="title">FSL FLAMEO tstats</a><div class="summary"> One of the outputs of FLAMEO is 'tstats' what do the values in this image mean? The appear to be the inverse of p-values, it that correct? Is there some good documentation of the FLAMEO bin somewhere (not the nipype docs, I'm looking for something from the FSL team)? I can't seem to find any solid documentation put out by the FSL team on how to use their bins from the terminal. Dalton </div></div><span class="ago" title="published:Mon, 06 Oct 2014 21:33:53 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e764:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2395/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e764:6b730af7" data-navigation="inline" data-title="FSL &amp; Nipype IOError" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e764:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8147 --></div><!-- react-empty: 8148 --><div class="content"><a href="http://neurostars.org/p/2395/" target="_blank" rel="noopener" class="title">FSL &amp; Nipype IOError</a><div class="summary">Hello, I recently installed nipype in a neurodebian virtual box. I believe I have correctly altered the necessary environmental variables in .bashrc for FSL to include- FSLDIR=/usr/share/fsl/5.0 PATH=${FSLDIR}/bin:${PATH} export FSLDIR PATH . ${FSLDIR}/etc/fslconf/fsl.sh I am able to run BET, SUSAN ect through terminal, however when I try to run the same commands through nipype in ipython or spyde</div></div><span class="ago" title="published:Sun, 05 Oct 2014 18:25:14 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e763:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2378/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e763:6b730af7" data-navigation="inline" data-title="A 3dSkullStrip issue on Nipype 0.9.2" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e763:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8163 --></div><!-- react-empty: 8164 --><div class="content"><a href="http://neurostars.org/p/2378/" target="_blank" rel="noopener" class="title">A 3dSkullStrip issue on Nipype 0.9.2</a><div class="summary">Hi, Nipype developers, I've met a 3dSkullStrip issue with Nipype, using the code (with my costumed T1) form the Nipype document page. from nipype.interfaces import afni as afni skullstrip = afni.SkullStrip() skullstrip.inputs.in_file = '/media/TOSHIBA_EXT/newdata2/cpac/work/resting_preproc_dp37_2/anat_pre proc_0/anat_skullstrip/t1_resample.nii.gz' 1) skullstrip.inputs.args = '-o_ply' res = skullst</div></div><span class="ago" title="published:Thu, 25 Sep 2014 12:52:38 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e762:6b730af7_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/2368/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e762:6b730af7" data-navigation="inline" data-title="Job: Hiring a NeuroImaging Engineer mine large functional-connectivity databases" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1494b952cda:390e762:6b730af7" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8179 --></div><!-- react-empty: 8180 --><div class="content"><a href="http://neurostars.org/p/2368/" target="_blank" rel="noopener" class="title">Job: Hiring a NeuroImaging Engineer mine large functional-connectivity databases</a><div class="summary">Dear colleagues and friend, Please find the following job posting: ______________________________________________________________________ _________ Job offer: https://team.inria.fr/parietal/job-offers/ Motivational blog post: http://gael-varoquaux.info/blog/?p=175 ______________________________________________________________________ _________ We are looking for a research engineer to assist us in</div></div><span class="ago" title="published:Mon, 22 Sep 2014 18:37:20 GMT -- Received: Sun, 26 Oct 2014 08:31:47 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14938e56a56:146f8f5:b669a9fb_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/159/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14938e56a56:146f8f5:b669a9fb" data-navigation="inline" data-title="CLI wrapper automatization" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14938e56a56:146f8f5:b669a9fb" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8236 --></div><!-- react-empty: 8237 --><div class="content"><a href="http://neurostars.org/p/159/" target="_blank" rel="noopener" class="title">CLI wrapper automatization</a><div class="summary">Hi Community, I just started with my bachelor thesis where I have to set up a pipeline concept using NiPype or Xnat. In the software MITK of the DKFZ in Germany, there are some command line tools which should be integrated into a pipeline concept. These tools are developed with CTK (common toolkit) and each of them deliver XML Information of the usage. For example: ./MitkDiffusionMiniApps - -xml d</div></div><span class="ago" title="published:Wed, 22 Oct 2014 17:04:19 GMT -- Received: Wed, 22 Oct 2014 17:26:39 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1492ca8783e:4de639f:5c7036b5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/156/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1492ca8783e:4de639f:5c7036b5" data-navigation="inline" data-title="Python code to read Gifti files?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1492ca8783e:4de639f:5c7036b5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8252 --></div><!-- react-empty: 8253 --><div class="content"><a href="http://neurostars.org/p/156/" target="_blank" rel="noopener" class="title">Python code to read Gifti files?</a><div class="summary"> Hello, Anyone knows/has python code to read gifti mesh files? thank you! </div></div><span class="ago" title="published:Mon, 20 Oct 2014 07:17:50 GMT -- Received: Mon, 20 Oct 2014 08:24:38 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149296fafe8:4012002:8340a896_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/153/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149296fafe8:4012002:8340a896" data-navigation="inline" data-title="Nipype workflows for small animal fMRI" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_149296fafe8:4012002:8340a896" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8268 --></div><!-- react-empty: 8269 --><div class="content"><a href="http://neurostars.org/p/153/" target="_blank" rel="noopener" class="title">Nipype workflows for small animal fMRI</a><div class="summary"> I work in animal models (rodents) and I am new to fMRI and Nipype. There are numerous tutorials and resources for human fMRI, but I haven't found anything for animal work. I am constructing a workflow based on published methods for AFNI, but it would be great to have something to compare against. Does anyone have an example workflow for a rat? Is there an Animal fMRI community out there? Thanks! </div></div><span class="ago" title="published:Sun, 19 Oct 2014 15:00:01 GMT -- Received: Sun, 19 Oct 2014 17:23:45 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1492825ddc9:3a3ca54:5c7036b5_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/151/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1492825ddc9:3a3ca54:5c7036b5" data-navigation="inline" data-title="How to read the pconn.nii files from Human Connectome Project?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1492825ddc9:3a3ca54:5c7036b5" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8284 --></div><!-- react-empty: 8285 --><div class="content"><a href="http://neurostars.org/p/151/" target="_blank" rel="noopener" class="title">How to read the pconn.nii files from Human Connectome Project?</a><div class="summary"> Hello, I downloaded the Parcelated Connectivity data from the Human Connectome Project (468 Subjects (Many Related), full correlation and 468 Subjects (Many Related), mean-gray-timecourse-regressed). In the HCP500_Parcellation_Timeseries_Netmats directory, netmats subdirectory, there are directories with connectivity matrices for 461 subjects using different numbers of ICA components. The files h</div></div><span class="ago" title="published:Sun, 19 Oct 2014 08:48:20 GMT -- Received: Sun, 19 Oct 2014 11:23:30 GMT">25mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1490fa547bf:a04a0f2:9c034b5e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/145/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1490fa547bf:a04a0f2:9c034b5e" data-navigation="inline" data-title="Using fsl.ImageStats in a nypipe workflow (no output)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1490fa547bf:a04a0f2:9c034b5e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8300 --></div><!-- react-empty: 8301 --><div class="content"><a href="http://neurostars.org/p/145/" target="_blank" rel="noopener" class="title">Using fsl.ImageStats in a nypipe workflow (no output)</a><div class="summary"> Hello, I am trying to program a nypipe workflow and one of the steps is using fsl.ImageStats (a wrapper for fslstats) on an image. The problem is that fslstats just provides the result as a number to the standard output, with no option of saving that number to a file. The current wrapper then, has no way of providing that output for piping it into another node, it just gets lost. I think the solu</div></div><span class="ago" title="published:Tue, 14 Oct 2014 14:27:28 GMT -- Received: Tue, 14 Oct 2014 17:12:10 GMT">26mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148ff87cc64:577e165:9c034b5e_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/144/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148ff87cc64:577e165:9c034b5e" data-navigation="inline" data-title="Review papers and pipelines for MEG resting-state analysis?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148ff87cc64:577e165:9c034b5e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8316 --></div><!-- react-empty: 8317 --><div class="content"><a href="http://neurostars.org/p/144/" target="_blank" rel="noopener" class="title">Review papers and pipelines for MEG resting-state analysis?</a><div class="summary"> I will be analyzing my own resting-state MEG dataset and I was wondering if MNE is the most stablished pipeline to get the most out of such a dataset: http://journal.frontiersin.org/Journal/10.3389/fnins.2013.00267/full http://martinos.org/mne/stable/manual/cookbook.html Good and/or seminal papers on this subject are highly welcome too! </div></div><span class="ago" title="published:Sat, 11 Oct 2014 12:03:31 GMT -- Received: Sat, 11 Oct 2014 14:06:02 GMT">26mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c2f4fd78:56d208c:7762fa0c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/132/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c2f4fd78:56d208c:7762fa0c" data-navigation="inline" data-title="Nipy fails to install on Anaconda" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c2f4fd78:56d208c:7762fa0c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8332 --></div><!-- react-empty: 8333 --><div class="content"><a href="http://neurostars.org/p/132/" target="_blank" rel="noopener" class="title">Nipy fails to install on Anaconda</a><div class="summary">I tried to install nipy on Anaconda via pip but I'm getting linking errors: gcc -bundle -undefined dynamic_lookup -L/Users/filo/anaconda/lib -arch x86_64 -arch x86_64 build/temp.macosx-10.5-x86_64-2.7/build/src.macosx-10.5-x86_64-2.7/nipy/labs/bindings/linalg.o -L/Users/filo/anaconda/lib -L/Users/filo/anaconda/lib -L/Users/filo/anaconda/lib -Lbuild/temp.macosx-10.5-x86_64-2.7 -lcstat -lmkl_lapack9</div></div><span class="ago" title="published:Mon, 29 Sep 2014 18:04:53 GMT -- Received: Mon, 29 Sep 2014 19:48:28 GMT">26mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c10600f4:4c099a8:7762fa0c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/130/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c10600f4:4c099a8:7762fa0c" data-navigation="inline" data-title="Forum: New NeuroStars twitter feed: @neuroquestion" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c10600f4:4c099a8:7762fa0c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><span class="engagement no-icon" data-dot="engagement-count" title="metric showing how popular this story is">1</span></div><!-- react-empty: 8349 --><div class="content"><a href="http://neurostars.org/p/130/" target="_blank" rel="noopener" class="title">Forum: New NeuroStars twitter feed: @neuroquestion</a><div class="summary"> I've just enabled a twitter account, @neuroquestion, that directly connects to this site's RSS feed via IFTTT. Feel free to use it to as new questions notification source and/or interact with this site's administrators and users. Perhaps in the future we'll allow bi-directional communication (from twitter to neurostars) if there's enough popular demand, enjoy! </div></div><span class="ago" title="published:Mon, 29 Sep 2014 08:32:11 GMT -- Received: Mon, 29 Sep 2014 10:47:49 GMT">26mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c0613100:48c2e32:7762fa0c_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/128/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c0613100:48c2e32:7762fa0c" data-navigation="inline" data-title="Nipype Error in import" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148c0613100:48c2e32:7762fa0c" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8364 --></div><!-- react-empty: 8365 --><div class="content"><a href="http://neurostars.org/p/128/" target="_blank" rel="noopener" class="title">Nipype Error in import</a><div class="summary"> Dear all, I am on Windows 7 and I have installed the Pythonxy and the packages nibabel and nipype. When I am going to import nipype I get the following error: &gt;&gt;&gt; import nipype Traceback (most recent call last): File " ", line 1, in File "C:\Python27\Lib\site-packages\nipype\__init__.py", line 9, in from utils.config import NipypeConfig File "C:\Python27\Lib\site-packages\nipype\utils\config.py",</div></div><span class="ago" title="published:Mon, 29 Sep 2014 06:01:55 GMT -- Received: Mon, 29 Sep 2014 07:47:48 GMT">26mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148af6a47ed:1b4dbdb:40a25680_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/127/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148af6a47ed:1b4dbdb:40a25680" data-navigation="inline" data-title="Alternatives to XNAT?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_148af6a47ed:1b4dbdb:40a25680" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8380 --></div><!-- react-empty: 8381 --><div class="content"><a href="http://neurostars.org/p/127/" target="_blank" rel="noopener" class="title">Alternatives to XNAT?</a><div class="summary"> While writing an Ansible automation playbook for XNAT: https://github.com/brainstorm/ansible-xnat And seeing multiple issues deploying and testing it on their official mailing list: https://groups.google.com/forum/#!forum/xnat_discussion I wondered if there are any popular alternatives (open source), being used in hospitals and research centers... Is XNAT the absolute leader in its category? Whic</div></div><span class="ago" title="published:Fri, 26 Sep 2014 00:44:11 GMT -- Received: Fri, 26 Sep 2014 00:44:11 GMT">26mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146feaadc8a:197894f:d89185ed_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/112/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146feaadc8a:197894f:d89185ed" data-navigation="inline" data-title="What is the best way to &quot;deface&quot; an anatomical T1 MRI?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146feaadc8a:197894f:d89185ed" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8396 --></div><!-- react-empty: 8397 --><div class="content"><a href="http://neurostars.org/p/112/" target="_blank" rel="noopener" class="title">What is the best way to "deface" an anatomical T1 MRI?</a><div class="summary"> I'm curious about the experience people have had with defacing T1 MRI scans and what the best practices are. I've come across the following tools/articles: FreeSurfer's mri_deface (http://www.na-mic.org/Wiki/index.php/Mbirn:_Defacer_for_structural_MRI) QuickShear (http://www.ibcb.utulsa.edu/wp-content/uploads/2011/08/SchimkeHealthSec11.pdf) SurfaceMask (http://www.ncbi.nlm.nih.gov/pmc/articles/PM</div></div><span class="ago" title="published:Thu, 03 Jul 2014 23:59:08 GMT -- Received: Thu, 03 Jul 2014 23:59:08 GMT">29mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146fb72028a:938a47:70bfb07f_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/109/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146fb72028a:938a47:70bfb07f" data-navigation="inline" data-title="CT T1 Registration" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146fb72028a:938a47:70bfb07f" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8412 --></div><!-- react-empty: 8413 --><div class="content"><a href="http://neurostars.org/p/109/" target="_blank" rel="noopener" class="title">CT T1 Registration</a><div class="summary"> I'd like to register a T1 with a CT scan. My impression is that most registration tools are for images from the same modality. Is there a straightforward approach to this problem with NiPy and friends, or is it necessary to employ other software like BrainVisa, 3D Slicer, etc? </div></div><span class="ago" title="published:Thu, 03 Jul 2014 08:58:11 GMT -- Received: Thu, 03 Jul 2014 08:58:11 GMT">29mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146f1c7b6c1:3012a14:c80f3eba_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/102/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146f1c7b6c1:3012a14:c80f3eba" data-navigation="inline" data-title="DWIConvert building errors" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146f1c7b6c1:3012a14:c80f3eba" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8428 --></div><!-- react-empty: 8429 --><div class="content"><a href="http://neurostars.org/p/102/" target="_blank" rel="noopener" class="title">DWIConvert building errors</a><div class="summary"> I am try to build DWIConverter but some errors occur when build it. This error is : Linking CXX executable DWIConvertTest /home/vavlab/Desktop/DWIconverterBuild/Prereqs/lib/liboflog.a(oflog.cc.o): In function `OFLog::configureFromCommandLine(OFCommandLine&amp;, OFConsoleApplication&amp;, OFLogger::LogLevel)': oflog.cc:(.text+0xa44): undefined reference to `OFStandard::fileExists(OFString const&amp;)' oflog.c</div></div><span class="ago" title="published:Tue, 01 Jul 2014 11:55:36 GMT -- Received: Tue, 01 Jul 2014 11:55:36 GMT">29mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146c3660265:49a9779:1fb9b1e9_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/100/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146c3660265:49a9779:1fb9b1e9" data-navigation="inline" data-title="Combining ANTs/itk and FSL style deformation fields" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_146c3660265:49a9779:1fb9b1e9" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8444 --></div><!-- react-empty: 8445 --><div class="content"><a href="http://neurostars.org/p/100/" target="_blank" rel="noopener" class="title">Combining ANTs/itk and FSL style deformation fields</a><div class="summary"> Hi, does anyone know of a way to apply deformation fields derived from FSL (warpfield from convertwarp or alternatively shiftmap from FUGUE) and those derived from antsRegsitration in a single transformation? For affine transforms there is the c3d affine tool to get FSL style matrices into itk format which can be used in antsApplyTransforms. Does something similar exist for deformation fields? Or</div></div><span class="ago" title="published:Sun, 22 Jun 2014 11:46:20 GMT -- Received: Sun, 22 Jun 2014 11:46:20 GMT">29mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae7:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/94/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae7:6e323717" data-navigation="inline" data-title="Nipype Custom Interface: Enforce A &quot;Same Length&quot; Constraint On Two Inputs" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae7:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8460 --></div><!-- react-empty: 8461 --><div class="content"><a href="http://neurostars.org/p/94/" target="_blank" rel="noopener" class="title">Nipype Custom Interface: Enforce A "Same Length" Constraint On Two Inputs</a><div class="summary"> Hello, I'd like to ensure that two List inputs in my interface's input spec class have the same length. If field1 and field2 are both List inputs, I'd like to throw an exception e.g. after assigning a value to field2 if field1 has already been assigned and its length is different. What's the most straightforward way of doing this? </div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae6:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/82/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae6:6e323717" data-navigation="inline" data-title="Integrating Multiple Dicom Scans For Single Nifti Image" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae6:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8476 --></div><!-- react-empty: 8477 --><div class="content"><a href="http://neurostars.org/p/82/" target="_blank" rel="noopener" class="title">Integrating Multiple Dicom Scans For Single Nifti Image</a><div class="summary"> (First a disclaimer: I'm taking baby steps with MRI treatment here, sorry to bore you) I've been using pydicom to load sequences of slices. The structural MRIs (at least the ones I have) usually have multiple sequences, along different axes. For a given sequence, the plane tangential to the sequence axis has good resolution, while the along the axis does not. My impression is that these sequences</div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae5:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/73/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae5:6e323717" data-navigation="inline" data-title="Forum: What Are The Current Challenges That Users Face When Using Nipype?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae5:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><span class="engagement no-icon" data-dot="engagement-count" title="metric showing how popular this story is">5</span></div><!-- react-empty: 8493 --><div class="content"><a href="http://neurostars.org/p/73/" target="_blank" rel="noopener" class="title">Forum: What Are The Current Challenges That Users Face When Using Nipype?</a><div class="summary"> There are many things that Nipype can do well and provides great flexibility. But with that flexibility comes complexity and steep learning curves. There are many hurdles that limit adoption. The intent of this post is to request feedback to improve Nipype. We are in the process of re-architecting components of Nipype and feedback would help a lot. Please describe the challenges that you faced wh</div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae4:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/72/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae4:6e323717" data-navigation="inline" data-title="Nipype'S Fsl.First Workflow Outputs Are Being Deleted" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae4:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8508 --></div><!-- react-empty: 8509 --><div class="content"><a href="http://neurostars.org/p/72/" target="_blank" rel="noopener" class="title">Nipype'S Fsl.First Workflow Outputs Are Being Deleted</a><div class="summary">I am trying to implement a NiPype workflow for FSL's run_first_all command via nipype.interfaces.fsl.FIRST(). My worfklow class is as follows: def create_subcort_seg(wf_name = 'subcort_seg'): """ Create FIRST segmentation/registration tool workflow """ # Create the workflow subcort_wf = pe.Workflow(name=wf_name) # Input node - load in the reoriented brain (original space, non-registered) inputNode</div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae3:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/68/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae3:6e323717" data-navigation="inline" data-title="Problems Registering T1 Scan To Mni Space" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae3:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8524 --></div><!-- react-empty: 8525 --><div class="content"><a href="http://neurostars.org/p/68/" target="_blank" rel="noopener" class="title">Problems Registering T1 Scan To Mni Space</a><div class="summary"> Hello, I have created some code here: that is a mix of the rsfmri preproc code here: http://nipy.org/nipype/users/examples/rsfmri_fsl_compcorr.html and the SPM preprocessing stage here: http://nipy.org/nipype/users/examples/fmri_spm_auditory.html I can't seem to get this to run however. The error message doesn't really give me any clues either: It fails on the attempting to normalise the T1 scan.</div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae2:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/66/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae2:6e323717" data-navigation="inline" data-title="Benchmaking Papers On The Spinnaker Neuromorphic Hardware Platform?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae2:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8540 --></div><!-- react-empty: 8541 --><div class="content"><a href="http://neurostars.org/p/66/" target="_blank" rel="noopener" class="title">Benchmaking Papers On The Spinnaker Neuromorphic Hardware Platform?</a><div class="summary"> After reading about the SpiNNaker neuromorphic architecture: http://www.artificialbrains.com/spinnaker I wonder if there are any papers benchmarking several aspects of it against different models, namely: power consumption, wall clock speedups, accuracy of the simulations compared with other platforms, etc... Any insights? </div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae1:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/52/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae1:6e323717" data-navigation="inline" data-title="Forum: Is Email Notification Working?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae1:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><span class="engagement no-icon" data-dot="engagement-count" title="metric showing how popular this story is">8</span></div><!-- react-empty: 8557 --><div class="content"><a href="http://neurostars.org/p/52/" target="_blank" rel="noopener" class="title">Forum: Is Email Notification Working?</a><div class="summary"> Is this working? Try and bookmark me. </div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae0:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/39/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae0:6e323717" data-navigation="inline" data-title="Porting Spm To Python?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ae0:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8572 --></div><!-- react-empty: 8573 --><div class="content"><a href="http://neurostars.org/p/39/" target="_blank" rel="noopener" class="title">Porting Spm To Python?</a><div class="summary"> Given that Python is predating the e-science landscape these days (good thing IMHO :P): http://www.r-bloggers.com/the-homogenization-of-scientific-computing-or-why-python-is-steadily-eating-other-languages-lunch/ I thought about starting a port for neuroimaging SPM ( http://www.fil.ion.ucl.ac.uk/spm/doc/ ), ( PySPM ?) as an INCF GSoC project, along the lines of the previous GSoC 2013 edition: htt</div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adf:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/29/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adf:6e323717" data-navigation="inline" data-title="How To Unload Image From Nibabel When I'M Finished Using It?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adf:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8588 --></div><!-- react-empty: 8589 --><div class="content"><a href="http://neurostars.org/p/29/" target="_blank" rel="noopener" class="title">How To Unload Image From Nibabel When I'M Finished Using It?</a><div class="summary"> I can load an MR image using from the Nibabel Getting Started help page: &gt;&gt;&gt; import nibabel as nib &gt;&gt;&gt; img = nib.load(os.path.join(data_path, 'example4d.nii.gz')) I can then find some summary measures (eg: mean) and store that in another numpy array. Can I "unload" the image data (and only keep the summary array) so that I don't take up too much memory? Do I even need to do this? My gut instinct </div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ade:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/28/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ade:6e323717" data-navigation="inline" data-title="Forum: How Can I Set Up Email Alarms For Certain Questions And Tags?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ade:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8604 --></div><!-- react-empty: 8605 --><div class="content"><a href="http://neurostars.org/p/28/" target="_blank" rel="noopener" class="title">Forum: How Can I Set Up Email Alarms For Certain Questions And Tags?</a><div class="summary"> I would like to be notified by email whenever someone answers a question I posted or replies to my comment. This would also be useful for certain tags (in my case "nipype"- this way we could move part of the support over here). </div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846add:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/20/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846add:6e323717" data-navigation="inline" data-title="Is There A 7T Test-Retest Epi Data Set Available?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846add:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8620 --></div><!-- react-empty: 8621 --><div class="content"><a href="http://neurostars.org/p/20/" target="_blank" rel="noopener" class="title">Is There A 7T Test-Retest Epi Data Set Available?</a><div class="summary"> Does anyone know of a 7T test-retest EPI dataset that I could use? </div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adc:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/19/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adc:6e323717" data-navigation="inline" data-title="What Are The Disadvantages Of Using Non-Linear Epi To T1 Coregistration For Fmri Analysis?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adc:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8636 --></div><!-- react-empty: 8637 --><div class="content"><a href="http://neurostars.org/p/19/" target="_blank" rel="noopener" class="title">What Are The Disadvantages Of Using Non-Linear Epi To T1 Coregistration For Fmri Analysis?</a><div class="summary"> Traditionally to bring results from individual subject to common space one would coregister subject specific mean EPI image to subject specific T1 image coregister subject specific T1 image to a template Step 1 has been usually done with rigid body, or affine transformation sometimes with the use of boundaries extracted from the T1 image. However, EPI images often suffer from geometric distortion</div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adb:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/13/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adb:6e323717" data-navigation="inline" data-title="Is Neurostars Ready For Public Attention (I.E. Should Be Announced In Social Networks)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846adb:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8652 --></div><!-- react-empty: 8653 --><div class="content"><a href="http://neurostars.org/p/13/" target="_blank" rel="noopener" class="title">Is Neurostars Ready For Public Attention (I.E. Should Be Announced In Social Networks)</a><div class="summary"> It is in beta, so I was not sure on how much publicity you want to have at this point. </div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ada:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/6/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ada:6e323717" data-navigation="inline" data-title="Forum: Howto/Where Report A Technical Issue In Neurostarts &quot;Engine&quot;?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ada:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8668 --></div><!-- react-empty: 8669 --><div class="content"><a href="http://neurostars.org/p/6/" target="_blank" rel="noopener" class="title">Forum: Howto/Where Report A Technical Issue In Neurostarts "Engine"?</a><div class="summary"> Since without being logged in, clicking on "log in to comment" in spits me out "404 Error. Cannot find path: /login/" </div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ad9:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/5/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ad9:6e323717" data-navigation="inline" data-title="What Is The Best Opensource Pipeline To Get The Most Out Of A Generic Fmri Dataset?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ad9:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8684 --></div><!-- react-empty: 8685 --><div class="content"><a href="http://neurostars.org/p/5/" target="_blank" rel="noopener" class="title">What Is The Best Opensource Pipeline To Get The Most Out Of A Generic Fmri Dataset?</a><div class="summary"> On the same line as the rather popular BioStars question: 'What is the best pipeline for human whole exome sequencing?' I wonder what are the "canonical" ways to process a dataset from, for instance, OpenFMRI repository. Are NiPype and/or XNAT the first steps to acquire valuable insight about a specific dataset? How much information can one get from those? How automated is that process in the cur</div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ad8:6e323717_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/1/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ad8:6e323717" data-navigation="inline" data-title="Which Neuroinformatics Programming Tasks Are Doable Within A 2-3 Day Coding Hackathon?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1469a274193:846ad8:6e323717" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8700 --></div><!-- react-empty: 8701 --><div class="content"><a href="http://neurostars.org/p/1/" target="_blank" rel="noopener" class="title">Which Neuroinformatics Programming Tasks Are Doable Within A 2-3 Day Coding Hackathon?</a><div class="summary"> First question ever in NeuroStars! :D I would like to know which tasks within neuroinformatics are worth implementing in a hackathon . For instance, there is an upcoming CodeJam in Germany : For instance, I thought about extending http://cloudbiolinux.org/ with some http://neuro.debian.net/ goodness. Another idea would be to work through the NiPype GitHub issues but the dilemma is how to prioriti</div></div><span class="ago" title="published:Sat, 14 Jun 2014 11:33:22 GMT -- Received: Sat, 14 Jun 2014 11:33:22 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1464a210fd3:6ce3ea:41e9a65f_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/97/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1464a210fd3:6ce3ea:41e9a65f" data-navigation="inline" data-title="Passing complex outputs to the next node" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1464a210fd3:6ce3ea:41e9a65f" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><span class="engagement no-icon" data-dot="engagement-count" title="metric showing how popular this story is">18</span></div><!-- react-empty: 8717 --><div class="content"><a href="http://neurostars.org/p/97/" target="_blank" rel="noopener" class="title">Passing complex outputs to the next node</a><div class="summary"> This question is already asked on Nipy users, but since I did not have any responses I thought to put it here as well. I'm wondering when the output of a node is not a simple output, how I can go about connecting the node to the next. So as an example in ANTs Atropos there are 3 outputs for posterior images, but only one output has been defined in Nipype: segment = pe.Node(interface = ants.Atropo</div></div><span class="ago" title="published:Thu, 29 May 2014 22:36:59 GMT -- Received: Thu, 29 May 2014 22:36:59 GMT">30mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14529a4f9b9:316323d:1642a341_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/94/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14529a4f9b9:316323d:1642a341" data-navigation="inline" data-title="Nipype custom interface: Enforce a &quot;same length&quot; constraint on two inputs" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14529a4f9b9:316323d:1642a341" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8732 --></div><!-- react-empty: 8733 --><div class="content"><a href="http://neurostars.org/p/94/" target="_blank" rel="noopener" class="title">Nipype custom interface: Enforce a "same length" constraint on two inputs</a><div class="summary"> Hello, I'd like to ensure that two List inputs in my interface's input spec class have the same length. If field1 and field2 are both List inputs, I'd like to throw an exception e.g. after assigning a value to field2 if field1 has already been assigned and its length is different. What's the most straightforward way of doing this? </div></div><span class="ago" title="published:Thu, 03 Apr 2014 22:10:48 GMT -- Received: Thu, 03 Apr 2014 22:10:48 GMT">32mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_145081f9e66:28cec77:fc6b336a_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/82/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_145081f9e66:28cec77:fc6b336a" data-navigation="inline" data-title="Integrating multiple DICOM scans for single Nifti image" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_145081f9e66:28cec77:fc6b336a" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8748 --></div><!-- react-empty: 8749 --><div class="content"><a href="http://neurostars.org/p/82/" target="_blank" rel="noopener" class="title">Integrating multiple DICOM scans for single Nifti image</a><div class="summary"> (First a disclaimer: I'm taking baby steps with MRI treatment here, sorry to bore you) I've been using pydicom to load sequences of slices. The structural MRIs (at least the ones I have) usually have multiple sequences, along different axes. For a given sequence, the plane tangential to the sequence axis has good resolution, while the along the axis does not. My impression is that these sequences</div></div><span class="ago" title="published:Fri, 28 Mar 2014 09:57:41 GMT -- Received: Fri, 28 Mar 2014 09:57:41 GMT">32mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144f9eb3c5e:37f3db0:85969355_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/73/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144f9eb3c5e:37f3db0:85969355" data-navigation="inline" data-title="Forum: What are the current challenges that users face when using Nipype?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144f9eb3c5e:37f3db0:85969355" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><span class="engagement no-icon" data-dot="engagement-count" title="metric showing how popular this story is">5</span></div><!-- react-empty: 8765 --><div class="content"><a href="http://neurostars.org/p/73/" target="_blank" rel="noopener" class="title">Forum: What are the current challenges that users face when using Nipype?</a><div class="summary"> There are many things that Nipype can do well and provides great flexibility. But with that flexibility comes complexity and steep learning curves. There are many hurdles that limit adoption. The intent of this post is to request feedback to improve Nipype. We are in the process of re-architecting components of Nipype and feedback would help a lot. Please describe the challenges that you faced wh</div></div><span class="ago" title="published:Tue, 25 Mar 2014 15:45:47 GMT -- Received: Tue, 25 Mar 2014 15:45:47 GMT">32mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144f5691d94:2266809:85969355_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/72/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144f5691d94:2266809:85969355" data-navigation="inline" data-title="NiPype's fsl.FIRST workflow outputs are being deleted" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144f5691d94:2266809:85969355" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8780 --></div><!-- react-empty: 8781 --><div class="content"><a href="http://neurostars.org/p/72/" target="_blank" rel="noopener" class="title">NiPype's fsl.FIRST workflow outputs are being deleted</a><div class="summary"> I am trying to implement a NiPype workflow for FSL's run_first_all command via nipype.interfaces.fsl.FIRST(). My worfklow class is as follows: def create_subcort_seg(wf_name = 'subcort_seg'): """ Create FIRST segmentation/registration tool workflow """ # Create the workflow subcort_wf = pe.Workflow(name=wf_name) # Input node - load in the reoriented brain (original space, non-registered) inputNod</div></div><span class="ago" title="published:Mon, 24 Mar 2014 18:45:10 GMT -- Received: Mon, 24 Mar 2014 18:45:10 GMT">32mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144e2bc3c8b:4a335b6:c0cce794_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/68/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144e2bc3c8b:4a335b6:c0cce794" data-navigation="inline" data-title="Problems Registering T1 scan to mni space" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_144e2bc3c8b:4a335b6:c0cce794" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8796 --></div><!-- react-empty: 8797 --><div class="content"><a href="http://neurostars.org/p/68/" target="_blank" rel="noopener" class="title">Problems Registering T1 scan to mni space</a><div class="summary"> Hello, I have created some code here: that is a mix of the rsfmri preproc code here: http://nipy.org/nipype/users/examples/rsfmri_fsl_compcorr.html and the SPM preprocessing stage here: http://nipy.org/nipype/users/examples/fmri_spm_auditory.html I can't seem to get this to run however. The error message doesn't really give me any clues either: It fails on the attempting to normalise the T1 scan.</div></div><span class="ago" title="published:Fri, 21 Mar 2014 03:43:10 GMT -- Received: Fri, 21 Mar 2014 03:43:10 GMT">32mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1442419aa5a:393f17:5e7efe62_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/66/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1442419aa5a:393f17:5e7efe62" data-navigation="inline" data-title="Benchmaking papers on the SpiNNaker neuromorphic hardware platform?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1442419aa5a:393f17:5e7efe62" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8812 --></div><!-- react-empty: 8813 --><div class="content"><a href="http://neurostars.org/p/66/" target="_blank" rel="noopener" class="title">Benchmaking papers on the SpiNNaker neuromorphic hardware platform?</a><div class="summary"> After reading about the SpiNNaker neuromorphic architecture: http://www.artificialbrains.com/spinnaker I wonder if there are any papers benchmarking several aspects of it against different models, namely: power consumption, wall clock speedups, accuracy of the simulations compared with other platforms, etc... Any insights? </div></div><span class="ago" title="published:Wed, 12 Feb 2014 03:17:45 GMT -- Received: Wed, 12 Feb 2014 03:17:45 GMT">34mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143de7bc6b1:b90a75:5e7efe62_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/52/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143de7bc6b1:b90a75:5e7efe62" data-navigation="inline" data-title="Forum: Is email notification working?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143de7bc6b1:b90a75:5e7efe62" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><span class="engagement no-icon" data-dot="engagement-count" title="metric showing how popular this story is">8</span></div><!-- react-empty: 8829 --><div class="content"><a href="http://neurostars.org/p/52/" target="_blank" rel="noopener" class="title">Forum: Is email notification working?</a><div class="summary"> Is this working? Try and bookmark me. </div></div><span class="ago" title="published:Wed, 29 Jan 2014 14:51:30 GMT -- Received: Wed, 29 Jan 2014 14:51:30 GMT">34mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082ce:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/39/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082ce:7ed032" data-navigation="inline" data-title="Porting SPM to Python?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082ce:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8844 --></div><!-- react-empty: 8845 --><div class="content"><a href="http://neurostars.org/p/39/" target="_blank" rel="noopener" class="title">Porting SPM to Python?</a><div class="summary"> Given that Python is predating the e-science landscape these days (good thing IMHO :P): http://www.r-bloggers.com/the-homogenization-of-scientific-computing-or-why-python-is-steadily-eating-other-languages-lunch/ I thought about starting a port for neuroimaging SPM ( http://www.fil.ion.ucl.ac.uk/spm/doc/ ), ( PySPM ?) as an INCF GSoC project, along the lines of the previous GSoC 2013 edition: htt</div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cd:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/29/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cd:7ed032" data-navigation="inline" data-title="How to unload image from nibabel when I'm finished using it?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cd:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8860 --></div><!-- react-empty: 8861 --><div class="content"><a href="http://neurostars.org/p/29/" target="_blank" rel="noopener" class="title">How to unload image from nibabel when I'm finished using it?</a><div class="summary"> I can load an MR image using from the Nibabel Getting Started help page: &gt;&gt;&gt; import nibabel as nib &gt;&gt;&gt; img = nib.load(os.path.join(data_path, 'example4d.nii.gz')) I can then find some summary measures (eg: mean) and store that in another numpy array. Can I "unload" the image data (and only keep the summary array) so that I don't take up too much memory? Do I even need to do this? My gut instinct </div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cc:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/28/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cc:7ed032" data-navigation="inline" data-title="Forum: How can I set up email alarms for certain questions and tags?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cc:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8894 --></div><!-- react-empty: 8895 --><div class="content"><a href="http://neurostars.org/p/28/" target="_blank" rel="noopener" class="title">Forum: How can I set up email alarms for certain questions and tags?</a><div class="summary"> I would like to be notified by email whenever someone answers a question I posted or replies to my comment. This would also be useful for certain tags (in my case "nipype"- this way we could move part of the support over here). </div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cb:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/20/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cb:7ed032" data-navigation="inline" data-title="Is there a 7T test-retest EPI data set available?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082cb:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8910 --></div><!-- react-empty: 8911 --><div class="content"><a href="http://neurostars.org/p/20/" target="_blank" rel="noopener" class="title">Is there a 7T test-retest EPI data set available?</a><div class="summary"> Does anyone know of a 7T test-retest EPI dataset that I could use? </div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082ca:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/19/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082ca:7ed032" data-navigation="inline" data-title="What are the disadvantages of using non-linear EPI to T1 coregistration for fMRI analysis?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082ca:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8926 --></div><!-- react-empty: 8927 --><div class="content"><a href="http://neurostars.org/p/19/" target="_blank" rel="noopener" class="title">What are the disadvantages of using non-linear EPI to T1 coregistration for fMRI analysis?</a><div class="summary"> Traditionally to bring results from individual subject to common space one would coregister subject specific mean EPI image to subject specific T1 image coregister subject specific T1 image to a template Step 1 has been usually done with rigid body, or affine transformation sometimes with the use of boundaries extracted from the T1 image. However, EPI images often suffer from geometric distortion</div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c9:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/13/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c9:7ed032" data-navigation="inline" data-title="Is NeuroStars ready for public attention (i.e. should be announced in social networks)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c9:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8942 --></div><!-- react-empty: 8943 --><div class="content"><a href="http://neurostars.org/p/13/" target="_blank" rel="noopener" class="title">Is NeuroStars ready for public attention (i.e. should be announced in social networks)</a><div class="summary"> It is in beta, so I was not sure on how much publicity you want to have at this point. </div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c8:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/6/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c8:7ed032" data-navigation="inline" data-title="Forum: Howto/where report a technical issue in NeuroStarts &quot;engine&quot;?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c8:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8958 --></div><!-- react-empty: 8959 --><div class="content"><a href="http://neurostars.org/p/6/" target="_blank" rel="noopener" class="title">Forum: Howto/where report a technical issue in NeuroStarts "engine"?</a><div class="summary"> Since without being logged in, clicking on "log in to comment" in spits me out "404 Error. Cannot find path: /login/" </div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c7:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/5/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c7:7ed032" data-navigation="inline" data-title="What is the best opensource pipeline to get the most out of a generic fMRI dataset?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c7:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8974 --></div><!-- react-empty: 8975 --><div class="content"><a href="http://neurostars.org/p/5/" target="_blank" rel="noopener" class="title">What is the best opensource pipeline to get the most out of a generic fMRI dataset?</a><div class="summary"> On the same line as the rather popular BioStars question: 'What is the best pipeline for human whole exome sequencing?' I wonder what are the "canonical" ways to process a dataset from, for instance, OpenFMRI repository. Are NiPype and/or XNAT the first steps to acquire valuable insight about a specific dataset? How much information can one get from those? How automated is that process in the cur</div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c6:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://neurostars.org/p/1/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c6:7ed032" data-navigation="inline" data-title="Which neuroinformatics programming tasks are doable within a 2-3 day coding hackathon?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1438ab8e57a:50082c6:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 8990 --></div><!-- react-empty: 8991 --><div class="content"><a href="http://neurostars.org/p/1/" target="_blank" rel="noopener" class="title">Which neuroinformatics programming tasks are doable within a 2-3 day coding hackathon?</a><div class="summary"> First question ever in NeuroStars! :D I would like to know which tasks within neuroinformatics are worth implementing in a hackathon . For instance, there is an upcoming CodeJam in Germany : For instance, I thought about extending http://cloudbiolinux.org/ with some http://neuro.debian.net/ goodness. Another idea would be to work through the NiPype GitHub issues but the dilemma is how to prioriti</div></div><span class="ago" title="published:Mon, 13 Jan 2014 08:30:09 GMT -- Received: Mon, 13 Jan 2014 08:30:09 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437c8ddb77:20c0aeb:7ed032_main" class="entry read u0 density-29" data-alternate-link="http://example.com/p/39/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437c8ddb77:20c0aeb:7ed032" data-navigation="inline" data-title="Porting SPM to Python?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437c8ddb77:20c0aeb:7ed032" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 9006 --></div><!-- react-empty: 9007 --><div class="content"><a href="http://example.com/p/39/" target="_blank" rel="noopener" class="title">Porting SPM to Python?</a><div class="summary"> Given that Python is predating the e-science landscape these days (good thing IMHO :P): http://www.r-bloggers.com/the-homogenization-of-scientific-computing-or-why-python-is-steadily-eating-other-languages-lunch/ I thought about starting a port for neuroimaging SPM ( http://www.fil.ion.ucl.ac.uk/spm/doc/ ), ( PySPM ?) as an INCF GSoC project, along the lines of the previous GSoC 2013 edition: htt</div></div><span class="ago" title="published:Fri, 10 Jan 2014 14:28:28 GMT -- Received: Fri, 10 Jan 2014 14:28:28 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437950d995:17c3be5:1336b31e_main" class="entry read u0 density-29" data-alternate-link="http://example.com/p/29/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437950d995:17c3be5:1336b31e" data-navigation="inline" data-title="How to unload image from nibabel when I'm finished using it?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437950d995:17c3be5:1336b31e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 9022 --></div><!-- react-empty: 9023 --><div class="content"><a href="http://example.com/p/29/" target="_blank" rel="noopener" class="title">How to unload image from nibabel when I'm finished using it?</a><div class="summary"> I can load an MR image using from the Nibabel Getting Started help page: &gt;&gt;&gt; import nibabel as nib &gt;&gt;&gt; img = nib.load(os.path.join(data_path, 'example4d.nii.gz')) I can then find some summary measures (eg: mean) and store that in another numpy array. Can I "unload" the image data (and only keep the summary array) so that I don't take up too much memory? Do I even need to do this? My gut instinct </div></div><span class="ago" title="published:Thu, 09 Jan 2014 23:22:58 GMT -- Received: Thu, 09 Jan 2014 23:22:58 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794f04d5:17c3be4:1336b31e_main" class="entry read u0 density-29" data-alternate-link="http://example.com/p/28/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794f04d5:17c3be4:1336b31e" data-navigation="inline" data-title="Forum: How can I set up email alarms for certain questions and tags?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794f04d5:17c3be4:1336b31e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 9038 --></div><!-- react-empty: 9039 --><div class="content"><a href="http://example.com/p/28/" target="_blank" rel="noopener" class="title">Forum: How can I set up email alarms for certain questions and tags?</a><div class="summary"> I would like to be notified by email whenever someone answers a question I posted or replies to my comment. This would also be useful for certain tags (in my case "nipype"- this way we could move part of the support over here). </div></div><span class="ago" title="published:Thu, 09 Jan 2014 23:20:58 GMT -- Received: Thu, 09 Jan 2014 23:20:58 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794d3015:17c3be3:1336b31e_main" class="entry read u0 density-29" data-alternate-link="http://example.com/p/20/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794d3015:17c3be3:1336b31e" data-navigation="inline" data-title="Is there a 7T test-retest EPI data set available?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794d3015:17c3be3:1336b31e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 9054 --></div><!-- react-empty: 9055 --><div class="content"><a href="http://example.com/p/20/" target="_blank" rel="noopener" class="title">Is there a 7T test-retest EPI data set available?</a><div class="summary"> Does anyone know of a 7T test-retest EPI dataset that I could use? </div></div><span class="ago" title="published:Thu, 09 Jan 2014 23:18:58 GMT -- Received: Thu, 09 Jan 2014 23:18:58 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794b5b55:17c3be2:1336b31e_main" class="entry read u0 density-29" data-alternate-link="http://example.com/p/19/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794b5b55:17c3be2:1336b31e" data-navigation="inline" data-title="What are the disadvantages of using non-linear EPI to T1 coregistration for fMRI analysis?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_143794b5b55:17c3be2:1336b31e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 9070 --></div><!-- react-empty: 9071 --><div class="content"><a href="http://example.com/p/19/" target="_blank" rel="noopener" class="title">What are the disadvantages of using non-linear EPI to T1 coregistration for fMRI analysis?</a><div class="summary"> Traditionally to bring results from individual subject to common space one would coregister subject specific mean EPI image to subject specific T1 image coregister subject specific T1 image to a template Step 1 has been usually done with rigid body, or affine transformation sometimes with the use of boundaries extracted from the T1 image. However, EPI images often suffer from geometric distortion</div></div><span class="ago" title="published:Thu, 09 Jan 2014 23:16:58 GMT -- Received: Thu, 09 Jan 2014 23:16:58 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379498695:17c3be1:1336b31e_main" class="entry read u0 density-29" data-alternate-link="http://example.com/p/13/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379498695:17c3be1:1336b31e" data-navigation="inline" data-title="Is NeuroStars ready for public attention (i.e. should be announced in social networks)" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379498695:17c3be1:1336b31e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 9086 --></div><!-- react-empty: 9087 --><div class="content"><a href="http://example.com/p/13/" target="_blank" rel="noopener" class="title">Is NeuroStars ready for public attention (i.e. should be announced in social networks)</a><div class="summary"> It is in beta, so I was not sure on how much publicity you want to have at this point. </div></div><span class="ago" title="published:Thu, 09 Jan 2014 23:14:58 GMT -- Received: Thu, 09 Jan 2014 23:14:58 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437947b1d5:17c3be0:1336b31e_main" class="entry read u0 density-29" data-alternate-link="http://example.com/p/6/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437947b1d5:17c3be0:1336b31e" data-navigation="inline" data-title="Forum: Howto/where report a technical issue in NeuroStarts &quot;engine&quot;?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437947b1d5:17c3be0:1336b31e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 9102 --></div><!-- react-empty: 9103 --><div class="content"><a href="http://example.com/p/6/" target="_blank" rel="noopener" class="title">Forum: Howto/where report a technical issue in NeuroStarts "engine"?</a><div class="summary"> Since without being logged in, clicking on "log in to comment" in spits me out "404 Error. Cannot find path: /login/" </div></div><span class="ago" title="published:Thu, 09 Jan 2014 23:12:58 GMT -- Received: Thu, 09 Jan 2014 23:12:58 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437945dd15:17c3bdf:1336b31e_main" class="entry read u0 density-29" data-alternate-link="http://example.com/p/5/" data-dot="first-story" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437945dd15:17c3bdf:1336b31e" data-navigation="inline" data-title="What is the best opensource pipeline to get the most out of a generic fMRI dataset?" data-inlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_1437945dd15:17c3bdf:1336b31e" data-u="0"><button class="save-for-later" title="Save for later" type="button"></button><div class="engagement-container"><!-- react-empty: 9118 --></div><!-- react-empty: 9119 --><div class="content"><a href="http://example.com/p/5/" target="_blank" rel="noopener" class="title">What is the best opensource pipeline to get the most out of a generic fMRI dataset?</a><div class="summary"> On the same line as the rather popular BioStars question: 'What is the best pipeline for human whole exome sequencing?' I wonder what are the "canonical" ways to process a dataset from, for instance, OpenFMRI repository. Are NiPype and/or XNAT the first steps to acquire valuable insight about a specific dataset? How much information can one get from those? How automated is that process in the cur</div></div><span class="ago" title="published:Thu, 09 Jan 2014 23:10:58 GMT -- Received: Thu, 09 Jan 2014 23:10:58 GMT">35mo</span><div class="condensed-tools"><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button></div></div><button class="icon-fx-twitter-ios-md-black icon condensed-toolbar-icon" title="Twitter" type="button"></button><button class="icon-fx-email-ios-md-black icon condensed-toolbar-icon" title="Email" type="button"></button><button class="icon-fx-google-plus-ios-md-black icon condensed-toolbar-icon" title="Google+" type="button"></button><button class="icon condensed-toolbar-icon icon-fx-cross-ios-sm-black" title="Mark as read and hide" type="button"></button></div></div><div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_main" class="inlineFrame read selected u0" data-uninlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e"><i class="close-button icon-fx-cross-ios-sm-black" data-uninlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e"></i><div _controllerid="rot150" style="opacity: 1;"> 
<div id="u100Header" class="u100Header floatingBar"> 
<div class="bar-icon close" data-page-action="unInlineEntry"></div> 
<div class="bar-icon menu" data-page-action="toggleInlineHeader"></div> 
 <h1>Which neuroinformatics programming tasks are doable within a 2-3 day coding hackathon?</h1> 
<div style="width:578px" id="floatingBarWikiWidget"> 
<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_topWikiWidget" class="topWikiWidget wikiWidget" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e" data-ref="forms_top"> 
</div> 
</div> 
</div> 

<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_main"> 
<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_entryHolder" class="entryholder"> 
<div class="frameActionsTop" data-uninlineentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e" style="cursor:pointer" title="close this article and return to the list"> 
</div> 
<div class="u100Entry  " data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e" data-alternate-link="http://example.com/p/1/" data-title="Which neuroinformatics programming tasks are doable within a 2-3 day coding hackathon?" data-selectentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e"> 
 <div id="feedlyPromotedHeader" style="display:none"></div> 
<div class="entryHeader"> 
<a href="http://example.com/p/1/" id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_entry_title" class="entryTitle title read" target="_blank" rel="noopener" cdf_container="10">Which&nbsp;neuroinformatics programming tasks are doable within a 2-3 day coding hackathon?</a> 

<div class="metadata"> 
<span id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_meta" class="meta" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e"> 
 
<span class="noNoiseMarker" id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_nbrRecommendations" data-alternatelink="http://example.com/p/1/" data-age="25399.793901666668" data-engagement="0" data-enriched="no"> 

</span> 
<span id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_noise" class="noiseMarker"> 
Noise 
</span> 

</span> 
 <a class="sourceTitle" href="//feedly.com//i/subscription/feed/http://neurostars.org/feeds/latest/" data-feedid="feed/http://neurostars.org/feeds/latest/" data-uri="subscription/feed/http://neurostars.org/feeds/latest/">Neurostars latest!</a> 
 
<span title="published:Thu, 09 Jan 2014 23:08:58 GMT -- received: Thu, 09 Jan 2014 23:08:58 GMT"> 
 1,058d 

</span> 
 &nbsp;//&nbsp; <span class="action" data-page-entry-action="keepEntryAsUnread" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e">keep unread</span> 
 &nbsp;//&nbsp; <span data-buryentryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e" class="action" title="mark as read and remove from list">hide</span> 
 &nbsp;//&nbsp; <span class="action" data-page-entry-action="previewEntry" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e">preview</span> 
</div> 

<div class="shareBarHolder"><div data-reactroot="" class="headerInfo headerInfo-article" style="position: relative;"><div class="headerInfo-links left"><i class="headerInfo-expanded-img icon-fx-save-ios-md-black" title="Save for later"></i><div class="fx" title="Save to board" style="float: left;"><div class="button-dropdown"><button class="headerInfo-expanded-img tertiary button-icon-only-micro icon icon-fx-save-to-knowledge-board-ios-md-black"></button><!-- react-empty: 7 --></div></div><i class="headerInfo-expanded-img icon-fx-link-ios-md-black" title="Copy link to clipboard"></i></div><div class="headerInfo-links right"><i class="headerInfo-expanded-img icon-fx-email-ios-md-black" title="Share via email"></i><i class="headerInfo-expanded-img icon-fx-google-plus-ios-md-black" title="Share to Google Plus"></i><i class="headerInfo-expanded-img icon-fx-twitter-ios-md-black" title="Share to Twitter"></i><i class="headerInfo-expanded-img icon-fx-wordpress-ios-md-black" title="Share to WordPress"></i><i class="headerInfo-expanded-img icon-fx-more-ios-md-black"></i></div><span style="display: none;"></span></div></div> 
 </div> 
<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_debugger" class="debugger"></div> 
<div class="tagsHolder decoration-holder"><!-- react-empty: 1 --></div> 
<div class="fx annotationsHolder decoration-holder"><!-- react-empty: 1 --></div> 
<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_status" class="toolsStatus" style="display:none"> 
 Also <a href="" target="OneNote">saved in OneNote</a> 
 </div> 
<div class="noiseMetadata" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e"> 

</div> 
<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_forms_top" style="display:none"></div> 
<div class="entryBody" id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_entryBody" cdf_m_top="1" data-dot="add-highlight" title="" style="position: relative;"> 
 
<!-- Summary container all summary markers are relative to this container: do not change the id (3) --> 
<div class="content" itemprop="description" id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_entryContent" cdf_container="3"> 
<p>First question ever in NeuroStars! :D</p>
<p>I would like to know which tasks within neuroinformatics are worth implementing in a <a href="http://en.wikipedia.org/wiki/Hackathon" target="_blank">hackathon</a>. For instance, there is an upcoming <a href="http://www.fz-juelich.de/ias/jsc/EN/Expertise/Workshops/Conferences/CodeJam2014/Program/_node.html" target="_blank">CodeJam in Germany</a>:</p>
<p>For instance, I thought about extending <a href="http://cloudbiolinux.org/" target="_blank">http://cloudbiolinux.org/</a> with some <a href="http://neuro.debian.net/" target="_blank">http://neuro.debian.net/</a> goodness.</p>
<p>Another idea would be to work through the <a href="http://nipy.sourceforge.net/nipype/" target="_blank">NiPype</a> <a href="https://github.com/nipy/nipype/issues" target="_blank">GitHub issues</a> but the dilemma is how to prioritize within such an interesting field :)</p>
<p>Opinions? Thoughts? More ideas?</p> 
</div> 
 <div class="fx"><!-- react-empty: 1 --></div></div> 
<div class="tagsHolder decoration-holder"><!-- react-empty: 1 --></div> 
<div style="clear:both"></div> 
<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_wallHolder"></div> 
 <a href="http://example.com/p/1/" class="fx-button secondary full-width visitWebsiteButton" style="margin-top:24px" target="_blank" rel="noopener"> 
Visit Website 
</a> 
<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_bottomWikiWidget" style="display:none" class="bottomWikiWidget wikiWidget" data-entryid="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e"> 
<div class="shareBarHolder"></div> 
</div> 
<div id="asvvdx865ekZS29zuuNNEfOY/yfHzvvxJ7EvaV1yXyU=_14379440855:17c3bde:1336b31e_forms_bottom" style="display:none"></div> 
 
</div> 
</div> 
</div> 
</div></div><h4>End of feed</h4></div>
